<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);.lst-kix_list_14-1>li:before{content:"\0025cb   "}.lst-kix_list_14-3>li:before{content:"\0025a0   "}.lst-kix_list_14-0>li:before{content:"\0025cf   "}.lst-kix_list_14-4>li:before{content:"\0025a0   "}.lst-kix_list_14-5>li:before{content:"\0025a0   "}.lst-kix_list_14-7>li:before{content:"\0025a0   "}.lst-kix_list_14-6>li:before{content:"\0025a0   "}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}ul.lst-kix_list_9-0{list-style-type:none}.lst-kix_list_14-2>li:before{content:"\0025a0   "}ul.lst-kix_list_17-1{list-style-type:none}ul.lst-kix_list_17-0{list-style-type:none}ul.lst-kix_list_17-8{list-style-type:none}ul.lst-kix_list_17-7{list-style-type:none}ul.lst-kix_list_17-6{list-style-type:none}ul.lst-kix_list_17-5{list-style-type:none}ul.lst-kix_list_17-4{list-style-type:none}ul.lst-kix_list_17-3{list-style-type:none}.lst-kix_list_14-8>li:before{content:"\0025a0   "}ul.lst-kix_list_17-2{list-style-type:none}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025a0   "}.lst-kix_list_5-2>li:before{content:"\0025a0   "}.lst-kix_list_5-1>li:before{content:"\0025cb   "}.lst-kix_list_5-7>li:before{content:"\0025a0   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025a0   "}.lst-kix_list_5-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_5-4>li:before{content:"\0025a0   "}.lst-kix_list_5-5>li:before{content:"\0025a0   "}ul.lst-kix_list_8-0{list-style-type:none}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_6-1>li:before{content:"\0025cb   "}.lst-kix_list_6-3>li:before{content:"\0025a0   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025a0   "}ul.lst-kix_list_16-2{list-style-type:none}ul.lst-kix_list_16-1{list-style-type:none}ul.lst-kix_list_16-0{list-style-type:none}.lst-kix_list_6-2>li:before{content:"\0025a0   "}ul.lst-kix_list_16-8{list-style-type:none}ul.lst-kix_list_16-7{list-style-type:none}ul.lst-kix_list_16-6{list-style-type:none}ul.lst-kix_list_16-5{list-style-type:none}ul.lst-kix_list_16-4{list-style-type:none}.lst-kix_list_6-8>li:before{content:"\0025a0   "}ul.lst-kix_list_16-3{list-style-type:none}.lst-kix_list_6-5>li:before{content:"\0025a0   "}.lst-kix_list_6-7>li:before{content:"\0025a0   "}.lst-kix_list_6-6>li:before{content:"\0025a0   "}.lst-kix_list_7-4>li:before{content:"\0025a0   "}.lst-kix_list_7-6>li:before{content:"\0025a0   "}.lst-kix_list_7-2>li:before{content:"\0025a0   "}.lst-kix_list_13-7>li:before{content:"\0025a0   "}.lst-kix_list_7-8>li:before{content:"\0025a0   "}.lst-kix_list_15-5>li:before{content:"\0025a0   "}.lst-kix_list_4-1>li:before{content:"\0025cb   "}.lst-kix_list_15-7>li:before{content:"\0025a0   "}ul.lst-kix_list_19-7{list-style-type:none}ul.lst-kix_list_19-6{list-style-type:none}.lst-kix_list_4-3>li:before{content:"\0025a0   "}.lst-kix_list_4-5>li:before{content:"\0025a0   "}ul.lst-kix_list_19-5{list-style-type:none}ul.lst-kix_list_19-4{list-style-type:none}ul.lst-kix_list_19-3{list-style-type:none}ul.lst-kix_list_19-2{list-style-type:none}ul.lst-kix_list_19-1{list-style-type:none}ul.lst-kix_list_19-0{list-style-type:none}.lst-kix_list_15-1>li:before{content:"\0025cb   "}.lst-kix_list_15-3>li:before{content:"\0025a0   "}ul.lst-kix_list_19-8{list-style-type:none}.lst-kix_list_12-3>li:before{content:"\0025a0   "}.lst-kix_list_12-1>li:before{content:"\0025cb   "}.lst-kix_list_13-3>li:before{content:"\0025a0   "}ul.lst-kix_list_18-0{list-style-type:none}.lst-kix_list_13-5>li:before{content:"\0025a0   "}.lst-kix_list_12-5>li:before{content:"\0025a0   "}ul.lst-kix_list_18-8{list-style-type:none}ul.lst-kix_list_18-7{list-style-type:none}ul.lst-kix_list_18-6{list-style-type:none}ul.lst-kix_list_18-5{list-style-type:none}.lst-kix_list_12-7>li:before{content:"\0025a0   "}ul.lst-kix_list_18-4{list-style-type:none}ul.lst-kix_list_18-3{list-style-type:none}ul.lst-kix_list_18-2{list-style-type:none}ul.lst-kix_list_18-1{list-style-type:none}.lst-kix_list_13-1>li:before{content:"\0025cb   "}ul.lst-kix_list_f-7{list-style-type:none}ul.lst-kix_list_f-6{list-style-type:none}ul.lst-kix_list_f-5{list-style-type:none}ul.lst-kix_list_f-4{list-style-type:none}ul.lst-kix_list_f-8{list-style-type:none}ul.lst-kix_list_f-3{list-style-type:none}.lst-kix_list_3-0>li:before{content:"\0025cf   "}ul.lst-kix_list_f-2{list-style-type:none}ul.lst-kix_list_f-1{list-style-type:none}ul.lst-kix_list_f-0{list-style-type:none}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}ul.lst-kix_list_5-5{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}ul.lst-kix_list_1c-1{list-style-type:none}ul.lst-kix_list_1c-2{list-style-type:none}ul.lst-kix_list_1c-3{list-style-type:none}ul.lst-kix_list_1c-4{list-style-type:none}ul.lst-kix_list_1c-5{list-style-type:none}ul.lst-kix_list_5-0{list-style-type:none}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_8-7>li:before{content:"\0025a0   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_list_f-6>li:before{content:"\0025a0   "}.lst-kix_list_a-7>li:before{content:"\0025a0   "}.lst-kix_list_f-7>li:before{content:"\0025a0   "}.lst-kix_list_8-3>li:before{content:"\0025a0   "}ul.lst-kix_list_13-5{list-style-type:none}ul.lst-kix_list_13-4{list-style-type:none}ul.lst-kix_list_13-3{list-style-type:none}ul.lst-kix_list_13-2{list-style-type:none}ul.lst-kix_list_13-1{list-style-type:none}.lst-kix_list_3-7>li:before{content:"\0025a0   "}ul.lst-kix_list_13-0{list-style-type:none}.lst-kix_list_8-4>li:before{content:"\0025a0   "}.lst-kix_list_a-2>li:before{content:"\0025a0   "}ul.lst-kix_list_13-8{list-style-type:none}.lst-kix_list_a-3>li:before{content:"\0025a0   "}.lst-kix_list_11-1>li:before{content:"\0025cb   "}ul.lst-kix_list_13-7{list-style-type:none}ul.lst-kix_list_13-6{list-style-type:none}.lst-kix_list_f-3>li:before{content:"\0025a0   "}.lst-kix_list_a-6>li:before{content:"\0025a0   "}.lst-kix_list_f-2>li:before{content:"\0025a0   "}.lst-kix_list_11-0>li:before{content:"\0025cf   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-8{list-style-type:none}.lst-kix_list_16-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-7{list-style-type:none}ul.lst-kix_list_e-6{list-style-type:none}.lst-kix_list_16-7>li:before{content:"\0025a0   "}ul.lst-kix_list_e-5{list-style-type:none}.lst-kix_list_e-3>li:before{content:"\0025a0   "}.lst-kix_list_e-7>li:before{content:"\0025a0   "}.lst-kix_list_e-2>li:before{content:"\0025a0   "}.lst-kix_list_e-6>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-0{list-style-type:none}.lst-kix_list_4-7>li:before{content:"\0025a0   "}.lst-kix_list_17-0>li:before{content:"\0025cf   "}ul.lst-kix_list_e-4{list-style-type:none}ul.lst-kix_list_e-3{list-style-type:none}ul.lst-kix_list_e-2{list-style-type:none}ul.lst-kix_list_e-1{list-style-type:none}ul.lst-kix_list_4-8{list-style-type:none}.lst-kix_list_16-0>li:before{content:"\0025cf   "}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_1b-1{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_1b-2{list-style-type:none}ul.lst-kix_list_1b-3{list-style-type:none}ul.lst-kix_list_1b-4{list-style-type:none}ul.lst-kix_list_1b-5{list-style-type:none}ul.lst-kix_list_1b-6{list-style-type:none}ul.lst-kix_list_4-0{list-style-type:none}.lst-kix_list_16-4>li:before{content:"\0025a0   "}ul.lst-kix_list_4-1{list-style-type:none}.lst-kix_list_16-3>li:before{content:"\0025a0   "}.lst-kix_list_1c-3>li:before{content:"\0025a0   "}ul.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_4-5{list-style-type:none}ol.lst-kix_list_1c-0{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}ul.lst-kix_list_4-3{list-style-type:none}ol.lst-kix_list_1b-0{list-style-type:none}.lst-kix_list_1c-7>li:before{content:"\0025a0   "}.lst-kix_list_1c-6>li:before{content:"\0025a0   "}ul.lst-kix_list_12-6{list-style-type:none}ul.lst-kix_list_1c-6{list-style-type:none}ul.lst-kix_list_12-5{list-style-type:none}ul.lst-kix_list_1c-7{list-style-type:none}.lst-kix_list_17-7>li:before{content:"\0025a0   "}ul.lst-kix_list_12-4{list-style-type:none}ul.lst-kix_list_1c-8{list-style-type:none}ul.lst-kix_list_12-3{list-style-type:none}ul.lst-kix_list_12-2{list-style-type:none}ul.lst-kix_list_12-1{list-style-type:none}.lst-kix_list_17-8>li:before{content:"\0025a0   "}ul.lst-kix_list_12-0{list-style-type:none}.lst-kix_list_17-3>li:before{content:"\0025a0   "}.lst-kix_list_17-4>li:before{content:"\0025a0   "}ul.lst-kix_list_12-8{list-style-type:none}ul.lst-kix_list_12-7{list-style-type:none}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}.lst-kix_list_7-3>li:before{content:"\0025a0   "}ul.lst-kix_list_7-5{list-style-type:none}.lst-kix_list_10-0>li:before{content:"\0025cf   "}ul.lst-kix_list_7-6{list-style-type:none}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}.lst-kix_list_13-8>li:before{content:"\0025a0   "}.lst-kix_list_b-3>li:before{content:"\0025a0   "}.lst-kix_list_b-7>li:before{content:"\0025a0   "}.lst-kix_list_18-3>li:before{content:"\0025a0   "}.lst-kix_list_18-7>li:before{content:"\0025a0   "}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7>li:before{content:"\0025a0   "}.lst-kix_list_15-4>li:before{content:"\0025a0   "}.lst-kix_list_10-4>li:before{content:"\0025a0   "}.lst-kix_list_10-8>li:before{content:"\0025a0   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}ul.lst-kix_list_15-3{list-style-type:none}ul.lst-kix_list_15-2{list-style-type:none}.lst-kix_list_15-0>li:before{content:"\0025cf   "}ul.lst-kix_list_15-1{list-style-type:none}.lst-kix_list_15-8>li:before{content:"\0025a0   "}ul.lst-kix_list_15-0{list-style-type:none}.lst-kix_list_4-4>li:before{content:"\0025a0   "}ul.lst-kix_list_15-8{list-style-type:none}ul.lst-kix_list_15-7{list-style-type:none}ul.lst-kix_list_15-6{list-style-type:none}.lst-kix_list_9-3>li:before{content:"\0025a0   "}ul.lst-kix_list_15-5{list-style-type:none}ul.lst-kix_list_15-4{list-style-type:none}.lst-kix_list_1b-3>li:before{content:"\0025a0   "}.lst-kix_list_9-7>li:before{content:"\0025a0   "}.lst-kix_list_11-4>li:before{content:"\0025a0   "}.lst-kix_list_12-4>li:before{content:"\0025a0   "}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf   "}.lst-kix_list_1a-3>li:before{content:"\0025a0   "}ul.lst-kix_list_6-2{list-style-type:none}.lst-kix_list_11-8>li:before{content:"\0025a0   "}ul.lst-kix_list_6-3{list-style-type:none}ul.lst-kix_list_6-0{list-style-type:none}.lst-kix_list_12-0>li:before{content:"\0025cf   "}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_13-0>li:before{content:"\0025cf   "}ul.lst-kix_list_14-4{list-style-type:none}ul.lst-kix_list_14-3{list-style-type:none}ul.lst-kix_list_14-2{list-style-type:none}.lst-kix_list_13-4>li:before{content:"\0025a0   "}ul.lst-kix_list_14-1{list-style-type:none}ul.lst-kix_list_14-0{list-style-type:none}ul.lst-kix_list_14-8{list-style-type:none}ul.lst-kix_list_14-7{list-style-type:none}.lst-kix_list_2-0>li:before{content:"\0025cf   "}ul.lst-kix_list_14-6{list-style-type:none}ul.lst-kix_list_14-5{list-style-type:none}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_12-8>li:before{content:"\0025a0   "}.lst-kix_list_c-0>li:before{content:"\0025cf   "}.lst-kix_list_19-0>li:before{content:"\0025cf   "}.lst-kix_list_19-1>li:before{content:"\0025cb   "}ul.lst-kix_list_b-8{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_c-1>li:before{content:"\0025cb   "}ul.lst-kix_list_b-3{list-style-type:none}ul.lst-kix_list_b-2{list-style-type:none}.lst-kix_list_c-3>li:before{content:"\0025a0   "}.lst-kix_list_c-4>li:before{content:"\0025a0   "}.lst-kix_list_19-4>li:before{content:"\0025a0   "}ul.lst-kix_list_b-1{list-style-type:none}ul.lst-kix_list_b-0{list-style-type:none}ul.lst-kix_list_b-7{list-style-type:none}ul.lst-kix_list_b-6{list-style-type:none}.lst-kix_list_c-2>li:before{content:"\0025a0   "}.lst-kix_list_19-2>li:before{content:"\0025a0   "}.lst-kix_list_19-3>li:before{content:"\0025a0   "}ul.lst-kix_list_b-5{list-style-type:none}ul.lst-kix_list_b-4{list-style-type:none}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_list_1a-5>li:before{content:"\0025a0   "}.lst-kix_list_1a-6>li:before{content:"\0025a0   "}.lst-kix_list_1a-8>li:before{content:"\0025a0   "}.lst-kix_list_19-8>li:before{content:"\0025a0   "}.lst-kix_list_1a-7>li:before{content:"\0025a0   "}.lst-kix_list_19-5>li:before{content:"\0025a0   "}.lst-kix_list_19-6>li:before{content:"\0025a0   "}.lst-kix_list_19-7>li:before{content:"\0025a0   "}.lst-kix_list_1b-4>li:before{content:"\0025a0   "}.lst-kix_list_1b-5>li:before{content:"\0025a0   "}.lst-kix_list_1b-7>li:before{content:"\0025a0   "}ul.lst-kix_list_a-4{list-style-type:none}ul.lst-kix_list_a-3{list-style-type:none}.lst-kix_list_1b-6>li:before{content:"\0025a0   "}.lst-kix_list_1b-8>li:before{content:"\0025a0   "}ul.lst-kix_list_a-2{list-style-type:none}ul.lst-kix_list_a-1{list-style-type:none}ul.lst-kix_list_a-8{list-style-type:none}ul.lst-kix_list_a-7{list-style-type:none}ul.lst-kix_list_a-6{list-style-type:none}ul.lst-kix_list_a-5{list-style-type:none}.lst-kix_list_d-6>li:before{content:"\0025a0   "}.lst-kix_list_d-8>li:before{content:"\0025a0   "}.lst-kix_list_d-7>li:before{content:"\0025a0   "}.lst-kix_list_d-2>li:before{content:"\0025a0   "}.lst-kix_list_18-0>li:before{content:"\0025cf   "}.lst-kix_list_d-1>li:before{content:"\0025cb   "}.lst-kix_list_d-5>li:before{content:"\0025a0   "}.lst-kix_list_18-1>li:before{content:"\0025cb   "}.lst-kix_list_18-2>li:before{content:"\0025a0   "}.lst-kix_list_d-4>li:before{content:"\0025a0   "}.lst-kix_list_d-3>li:before{content:"\0025a0   "}.lst-kix_list_c-7>li:before{content:"\0025a0   "}ul.lst-kix_list_a-0{list-style-type:none}.lst-kix_list_c-5>li:before{content:"\0025a0   "}.lst-kix_list_c-8>li:before{content:"\0025a0   "}.lst-kix_list_d-0>li:before{content:"\0025cf   "}.lst-kix_list_c-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_d-8{list-style-type:none}ul.lst-kix_list_d-7{list-style-type:none}ul.lst-kix_list_d-6{list-style-type:none}.lst-kix_list_2-5>li:before{content:"\0025a0   "}ul.lst-kix_list_d-1{list-style-type:none}ul.lst-kix_list_d-0{list-style-type:none}.lst-kix_list_b-8>li:before{content:"\0025a0   "}ul.lst-kix_list_d-5{list-style-type:none}ul.lst-kix_list_d-4{list-style-type:none}ul.lst-kix_list_d-3{list-style-type:none}ul.lst-kix_list_d-2{list-style-type:none}ul.lst-kix_list_1a-0{list-style-type:none}ul.lst-kix_list_1a-1{list-style-type:none}.lst-kix_list_b-4>li:before{content:"\0025a0   "}.lst-kix_list_18-6>li:before{content:"\0025a0   "}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_1a-2{list-style-type:none}.lst-kix_list_1b-0>li{counter-increment:lst-ctn-kix_list_1b-0}ul.lst-kix_list_3-8{list-style-type:none}ul.lst-kix_list_1a-3{list-style-type:none}ul.lst-kix_list_1a-4{list-style-type:none}.lst-kix_list_10-1>li:before{content:"\0025cb   "}ul.lst-kix_list_1a-5{list-style-type:none}.lst-kix_list_18-4>li:before{content:"\0025a0   "}.lst-kix_list_18-8>li:before{content:"\0025a0   "}ul.lst-kix_list_1a-6{list-style-type:none}ul.lst-kix_list_1a-7{list-style-type:none}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}ul.lst-kix_list_3-0{list-style-type:none}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}.lst-kix_list_b-6>li:before{content:"\0025a0   "}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}.lst-kix_list_10-7>li:before{content:"\0025a0   "}.lst-kix_list_10-5>li:before{content:"\0025a0   "}.lst-kix_list_1c-2>li:before{content:"\0025a0   "}.lst-kix_list_10-3>li:before{content:"\0025a0   "}ul.lst-kix_list_11-7{list-style-type:none}ul.lst-kix_list_1b-7{list-style-type:none}ul.lst-kix_list_11-6{list-style-type:none}ul.lst-kix_list_1b-8{list-style-type:none}.lst-kix_list_b-0>li:before{content:"\0025cf   "}ul.lst-kix_list_11-5{list-style-type:none}ul.lst-kix_list_11-4{list-style-type:none}ul.lst-kix_list_11-3{list-style-type:none}ul.lst-kix_list_11-2{list-style-type:none}.lst-kix_list_b-2>li:before{content:"\0025a0   "}ul.lst-kix_list_11-1{list-style-type:none}ul.lst-kix_list_11-0{list-style-type:none}.lst-kix_list_1c-0>li:before{content:"" counter(lst-ctn-kix_list_1c-0,decimal) ". "}.lst-kix_list_9-2>li:before{content:"\0025a0   "}ul.lst-kix_list_11-8{list-style-type:none}.lst-kix_list_9-0>li:before{content:"\0025cf   "}.lst-kix_list_1b-2>li:before{content:"\0025a0   "}ul.lst-kix_list_c-8{list-style-type:none}ul.lst-kix_list_c-7{list-style-type:none}.lst-kix_list_9-6>li:before{content:"\0025a0   "}ul.lst-kix_list_c-2{list-style-type:none}ul.lst-kix_list_c-1{list-style-type:none}.lst-kix_list_1b-0>li:before{content:"" counter(lst-ctn-kix_list_1b-0,decimal) ". "}ul.lst-kix_list_c-0{list-style-type:none}.lst-kix_list_9-4>li:before{content:"\0025a0   "}ul.lst-kix_list_c-6{list-style-type:none}.lst-kix_list_a-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-5{list-style-type:none}.lst-kix_list_11-3>li:before{content:"\0025a0   "}ul.lst-kix_list_c-4{list-style-type:none}ul.lst-kix_list_c-3{list-style-type:none}ul.lst-kix_list_2-8{list-style-type:none}.lst-kix_list_11-5>li:before{content:"\0025a0   "}ul.lst-kix_list_2-2{list-style-type:none}.lst-kix_list_1a-4>li:before{content:"\0025a0   "}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_9-8>li:before{content:"\0025a0   "}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1a-2>li:before{content:"\0025a0   "}.lst-kix_list_1-1>li:before{content:"\0025cb   "}ul.lst-kix_list_2-7{list-style-type:none}.lst-kix_list_11-7>li:before{content:"\0025a0   "}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}ul.lst-kix_list_10-0{list-style-type:none}.lst-kix_list_1a-0>li:before{content:"\0025cf   "}.lst-kix_list_1-3>li:before{content:"\0025a0   "}ul.lst-kix_list_10-8{list-style-type:none}ul.lst-kix_list_1a-8{list-style-type:none}ul.lst-kix_list_10-7{list-style-type:none}.lst-kix_list_1-7>li:before{content:"\0025a0   "}ul.lst-kix_list_10-6{list-style-type:none}ul.lst-kix_list_10-5{list-style-type:none}ul.lst-kix_list_10-4{list-style-type:none}ul.lst-kix_list_10-3{list-style-type:none}.lst-kix_list_1-5>li:before{content:"\0025a0   "}ul.lst-kix_list_10-2{list-style-type:none}ul.lst-kix_list_10-1{list-style-type:none}.lst-kix_list_2-1>li:before{content:"\0025cb   "}ol.lst-kix_list_1c-0.start{counter-reset:lst-ctn-kix_list_1c-0 0}.lst-kix_list_2-3>li:before{content:"\0025a0   "}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}.lst-kix_list_8-1>li:before{content:"\0025cb   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}.lst-kix_list_f-8>li:before{content:"\0025a0   "}.lst-kix_list_8-5>li:before{content:"\0025a0   "}.lst-kix_list_8-6>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_list_f-4>li:before{content:"\0025a0   "}.lst-kix_list_a-8>li:before{content:"\0025a0   "}.lst-kix_list_f-5>li:before{content:"\0025a0   "}.lst-kix_list_11-2>li:before{content:"\0025a0   "}.lst-kix_list_a-1>li:before{content:"\0025cb   "}.lst-kix_list_f-0>li:before{content:"\0025cf   "}.lst-kix_list_a-5>li:before{content:"\0025a0   "}.lst-kix_list_a-4>li:before{content:"\0025a0   "}.lst-kix_list_f-1>li:before{content:"\0025cb   "}.lst-kix_list_e-5>li:before{content:"\0025a0   "}.lst-kix_list_16-6>li:before{content:"\0025a0   "}ol.lst-kix_list_1b-0.start{counter-reset:lst-ctn-kix_list_1b-0 0}.lst-kix_list_17-1>li:before{content:"\0025cb   "}.lst-kix_list_e-4>li:before{content:"\0025a0   "}.lst-kix_list_16-1>li:before{content:"\0025cb   "}.lst-kix_list_16-2>li:before{content:"\0025a0   "}.lst-kix_list_e-1>li:before{content:"\0025cb   "}.lst-kix_list_16-5>li:before{content:"\0025a0   "}.lst-kix_list_e-0>li:before{content:"\0025cf   "}.lst-kix_list_1c-4>li:before{content:"\0025a0   "}.lst-kix_list_1c-5>li:before{content:"\0025a0   "}.lst-kix_list_1c-8>li:before{content:"\0025a0   "}.lst-kix_list_17-2>li:before{content:"\0025a0   "}.lst-kix_list_17-6>li:before{content:"\0025a0   "}.lst-kix_list_17-5>li:before{content:"\0025a0   "}.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_7-1>li:before{content:"\0025cb   "}.lst-kix_list_7-5>li:before{content:"\0025a0   "}.lst-kix_list_1c-0>li{counter-increment:lst-ctn-kix_list_1c-0}.lst-kix_list_b-5>li:before{content:"\0025a0   "}.lst-kix_list_18-5>li:before{content:"\0025a0   "}.lst-kix_list_13-6>li:before{content:"\0025a0   "}.lst-kix_list_15-6>li:before{content:"\0025a0   "}.lst-kix_list_b-1>li:before{content:"\0025cb   "}.lst-kix_list_1c-1>li:before{content:"\0025cb   "}.lst-kix_list_10-2>li:before{content:"\0025a0   "}.lst-kix_list_4-2>li:before{content:"\0025a0   "}.lst-kix_list_4-6>li:before{content:"\0025a0   "}.lst-kix_list_15-2>li:before{content:"\0025a0   "}.lst-kix_list_10-6>li:before{content:"\0025a0   "}.lst-kix_list_9-1>li:before{content:"\0025cb   "}.lst-kix_list_e-8>li:before{content:"\0025a0   "}.lst-kix_list_1b-1>li:before{content:"\0025cb   "}.lst-kix_list_9-5>li:before{content:"\0025a0   "}.lst-kix_list_12-2>li:before{content:"\0025a0   "}.lst-kix_list_11-6>li:before{content:"\0025a0   "}.lst-kix_list_1-2>li:before{content:"\0025a0   "}.lst-kix_list_1a-1>li:before{content:"\0025cb   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_12-6>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}.lst-kix_list_13-2>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c19{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:78pt;border-top-color:#000000;border-bottom-style:solid}.c8{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:58.5pt;border-top-color:#000000;border-bottom-style:solid}.c7{color:#1b1c1d;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c1{-webkit-text-decoration-skip:none;color:#0000ee;font-weight:400;text-decoration:underline;text-decoration-skip-ink:none;font-size:12pt;font-family:"Google Sans"}.c13{color:#1b1c1d;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c17{color:#1b1c1d;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Google Sans";font-style:normal}.c37{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#575b5f;font-weight:400}.c15{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c0{margin-left:23.2pt;padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c5{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c39{font-size:16pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c12{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c36{font-size:10pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c30{font-size:10pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c14{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c27{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c40{padding-top:24pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left;height:11pt}.c47{font-size:12pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c9{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c4{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c42{padding-top:12pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c41{border-spacing:0;border-collapse:collapse;margin-right:auto}.c10{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c29{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans"}.c33{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left}.c45{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c21{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c26{padding-top:12pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left}.c23{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c35{padding-top:12pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c24{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans"}.c25{color:#000000;font-weight:400;font-size:11pt;font-family:"Google Sans Text"}.c31{color:#000000;font-weight:400;font-size:11pt;font-family:"Arial"}.c16{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c6{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c38{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c11{color:inherit;text-decoration:inherit}.c20{margin-left:23.2pt;padding-left:0pt}.c44{margin-left:63.8pt;padding-left:0pt}.c34{margin-left:43.5pt;padding-left:0pt}.c2{padding:0;margin:0}.c28{margin-left:24pt;padding-left:0pt}.c22{vertical-align:baseline;font-style:normal}.c18{text-decoration:none}.c43{background-color:#f0f4f9}.c3{height:0pt}.c46{height:11pt}.c32{vertical-align:baseline}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c38 doc-content"><h1 class="c21"><span class="c39">Molecular Markers and Their Sequences Associated with Seed Size in Peanut (</span><span class="c27">Arachis hypogaea</span><span class="c39 c18 c32">&nbsp;L.)</span></h1><h2 class="c21"><span class="c22 c18 c29">I. Introduction</span></h2><p class="c10"><span class="c9">Cultivated peanut (</span><span class="c4">Arachis hypogaea</span><span class="c9">&nbsp;L.) is a globally significant legume crop, valued for its nutritional content and economic importance. It is an allotetraploid species (AABB, 2n=4x=40), believed to have originated from a single hybridization event between two wild diploid ancestors, </span><span class="c4">Arachis duranensis</span><span class="c9">&nbsp;(AA genome) and </span><span class="c4">Arachis ipaensis</span><span class="c9">&nbsp;(BB genome), followed by spontaneous chromosome doubling.</span><span class="c5">1</span><span class="c9">&nbsp;Seed size is a critical agronomic trait in peanut, directly influencing yield, market value, and consumer preference. Larger seeds are often preferred by both consumers and the processing industry, making seed size improvement a primary objective in peanut breeding programs worldwide.</span><span class="c5 c18">2</span></p><p class="c10"><span class="c9">Peanut seeds are a rich source of protein (22%-30%), edible oil (44%-56%), vitamins (E, K, B complex), and essential minerals.</span><span class="c5">2</span><span class="c9">&nbsp;The demand for peanut production continues to grow, necessitating genetic enhancements and improved selection efficiency in breeding programs.</span><span class="c5">5</span><span class="c9">&nbsp;This report aims to compile and review available information on molecular markers associated with seed size traits in </span><span class="c4">Arachis hypogaea</span><span class="c9 c18 c32">, with a particular focus on identifying markers for which sequence information is available, thereby facilitating their potential use in molecular breeding.</span></p><h2 class="c21"><span class="c29 c22 c18">II. Genetic Basis of Seed Size in Peanut</span></h2><p class="c10"><span class="c9">Seed size in peanut, like in many other crops, is a complex quantitative trait. This means it is controlled by multiple genes, known as quantitative trait loci (QTLs), and is also significantly influenced by environmental conditions and the interaction between genotype and environment.</span><span class="c5">1</span><span class="c9">&nbsp;Various parameters are used to quantify seed size, including the weight of 100 seeds (100SW or HSW), seed length (SL), seed width (SW), and the seed length to width ratio (L/W).</span><span class="c5">3</span><span class="c9 c18 c32">&nbsp;These components collectively determine the final seed dimensions and weight, which are crucial for yield.</span></p><p class="c10"><span class="c9">The genetic architecture of seed size involves numerous QTLs distributed across the peanut genome. Identifying these QTLs and the underlying genes is paramount for understanding the molecular mechanisms governing seed development. However, traditional phenotypic selection for seed size in peanut presents unique challenges. Because peanut pods develop underground, breeders must excavate the pods to assess seed characteristics, a labor-intensive and often late-stage process.</span><span class="c5">4</span><span class="c9 c18 c32">&nbsp;This difficulty underscores the immense value of molecular markers, which can enable early-generation selection for desirable seed size traits, accelerating the breeding cycle and improving selection accuracy.</span></p><h2 class="c21"><span class="c29 c22 c18">III. Molecular Markers Associated with Peanut Seed Size</span></h2><p class="c10"><span class="c9 c18 c32">A variety of DNA-based molecular markers have been employed in peanut genetic research to identify QTLs associated with seed size and other agronomically important traits.</span></p><h3 class="c21"><span class="c17">A. Overview of Marker Types Used</span></h3><p class="c21"><span class="c9 c18 c32">The development and application of molecular markers have significantly advanced our understanding of the genetic control of complex traits in peanut.</span></p><ul class="c2 lst-kix_list_1-0 start"><li class="c0 li-bullet-0"><span class="c14">Simple Sequence Repeats (SSRs):</span><span class="c9">&nbsp;Also known as microsatellites, SSRs are tandem repeats of short DNA motifs (1-6 base pairs). They are typically co-dominant, highly polymorphic, and have been widely used in peanut for constructing genetic maps and QTL mapping.</span><span class="c5">3</span><span class="c9">&nbsp;Numerous genetic maps based on SSR markers have been developed over the past decades, with increasing marker density and genome coverage.</span><span class="c5 c18">3</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Single Nucleotide Polymorphisms (SNPs):</span><span class="c9">&nbsp;SNPs are variations at a single nucleotide position in the DNA sequence. They are the most abundant type of genetic variation and are highly amenable to high-throughput genotyping platforms. SNP markers have become increasingly prevalent in peanut research with the advent of next-generation sequencing (NGS) technologies and the development of SNP arrays.</span><span class="c5 c18">3</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Insertion-Deletion (InDel) Markers:</span><span class="c9">&nbsp;InDels represent the insertion or deletion of DNA sequences ranging from one base pair to several kilobases. They are another common type of polymorphism and can be readily detected by PCR and gel electrophoresis or sequencing. Recent studies have utilized ddRADSeq to identify InDels across the peanut genome for marker development.</span><span class="c5 c18">10</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Kompetitive Allele-Specific PCR (KASP) Markers:</span><span class="c9">&nbsp;KASP is a fluorescence-based genotyping technology typically used for SNP detection. It is cost-effective and suitable for high-throughput screening of specific loci. KASP assays are often developed from validated SNPs identified through QTL-seq or other mapping approaches and are valuable for marker-assisted selection (MAS).</span><span class="c5 c18">4</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Miniature Inverted-repeat Transposable Element (MITE) Markers:</span><span class="c9">&nbsp;MITEs are a class of transposable elements. Markers based on MITE insertions or polymorphisms (e.g., AhTE markers, for </span><span class="c4">Arachis hypogaea</span><span class="c9">&nbsp;Transposable Element) can be associated with trait variations, including seed size.</span><span class="c5 c18">12</span></li></ul><p class="c23"><span class="c9">The transition from lower-throughput markers like RFLPs and RAPDs to higher-density SSRs and now to genome-wide SNPs and InDels reflects the technological advancements in genomics. High-density genetic maps (HDGMs) constructed with these markers are essential for precise QTL discovery and fine mapping.</span><span class="c5 c18">3</span></p><h3 class="c21"><span class="c17">B. Key QTL Regions and Associated Markers</span></h3><p class="c10"><span class="c9 c18 c32">Numerous studies have identified QTLs for seed size-related traits on various peanut chromosomes. Some of the most consistently reported and significant regions are highlighted below.</span></p><p class="c33"><span class="c25 c22 c18">1. Chromosome A05 (also referred to as LG05 or Arahy05)</span></p><p class="c33"><span class="c25 c22 c18">Chromosome A05 has emerged as a critical genomic region harboring major QTLs for pod and seed size traits in multiple peanut populations and environments.4</span></p><ul class="c2 lst-kix_list_2-0 start"><li class="c0 li-bullet-0"><span class="c9">InDel Marker Arahy05.106411957:</span><span class="c31 c22 c18"><br></span><span class="c9 c18 c32">An InDel marker located at position 106,411,957 bp on chromosome Arahy05 (Tifrunner genome assembly) has been identified and validated for its strong association with hundred kernel weight (HKW).11 This marker resides within the coding sequence of the gene Arahy.L1PMVE, which encodes a J domain-containing protein.11</span></li></ul><ul class="c2 lst-kix_list_3-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Polymorphism:</span><span class="c9">&nbsp;The marker represents a 20 bp insertion/deletion. The reference allele (e.g., in germplasm W1202) contains the 20 bp sequence 5&prime;-ATGCATAATATTCCTCCATCT-3&prime;. The alternative allele, associated with increased HKW (e.g., in cultivar Yuhua15), features a deletion of this 20 bp sequence, represented as &#39;A&#39; (deletion relative to the longer allele).</span><span class="c5">11</span><span class="c9">&nbsp;This InDel is predicted to cause a frameshift mutation and loss of the start codon in </span><span class="c4">Arahy.L1PMVE</span><span class="c9">.</span><span class="c5 c18">11</span></li><li class="c16 c34 li-bullet-0"><span class="c14">Association:</span><span class="c9">&nbsp;Accessions carrying the deletion haplotype (alternative allele) exhibit significantly higher HKW.</span><span class="c5 c18">11</span></li><li class="c16 c34 li-bullet-0"><span class="c14">Primer Sequences:</span><span class="c9">&nbsp;The primer sequences for amplifying this InDel marker are provided in the supplementary materials (Table S2) of the study by Fan et al. (2024).</span><span class="c5">11</span><span class="c9">&nbsp;Based on the analysis of these materials </span><span class="c5">15</span><span class="c9 c18 c32">, the primers are:</span></li></ul><ul class="c2 lst-kix_list_4-2 start"><li class="c21 c44 li-bullet-0"><span class="c9 c18 c32">Forward Primer (Arahy05.106411957-F): 5&prime;-GCACTTGTTGATTTGCCCAA-3&prime;</span></li><li class="c16 c44 li-bullet-0"><span class="c9">Reverse Primer (Arahy05.106411957-R): 5&prime;-TGGTTGTTGTTGTTGTTGTTGT-3&prime; The validation of this marker across diverse germplasm highlights its utility for MAS in peanut breeding programs aiming to improve kernel weight.</span><span class="c5 c18">11</span></li></ul><ul class="c2 lst-kix_list_2-0"><li class="c16 c20 li-bullet-0"><span class="c9">MITE Markers AhTE333 and AhTE810:</span><span class="c31 c22 c18"><br></span><span class="c9 c18 c32">In a study utilizing an induced large seed mutant, a major QTL for large seed trait (explaining 12.7% of phenotypic variation, PVE) was mapped on linkage group A05. This QTL was flanked by MITE markers AhTE333 and AhTE810, corresponding to a physical region of 90.65 Mbp to 107.24 Mbp on chromosome A05.12</span></li></ul><ul class="c2 lst-kix_list_5-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Sequences:</span><span class="c9">&nbsp;Specific primer sequences for AhTE333 and AhTE810 were not directly provided in the abstract or the analyzed snippets.</span><span class="c5">12</span><span class="c9">&nbsp;While AhTE markers are generally based on transposable elements </span><span class="c5">16</span><span class="c9">, retrieving specific primer sequences for these particular markers would require consulting the full publication or specialized databases that might host them. Searches on PeanutBase for these specific markers did not yield readily available sequence information from the provided materials.</span><span class="c5 c18">17</span></li></ul><ul class="c2 lst-kix_list_2-0"><li class="c16 c20 li-bullet-0"><span class="c9">Candidate Gene in A05 QTL Region:</span><span class="c31 c22 c18"><br></span><span class="c9 c18 c32">Within the A05 QTL region flanked by AhTE333 and AhTE810, an ortholog of the BIG SEEDS 1 (BS1) gene was identified at position 102,476,137 bp. Real-time PCR analysis revealed that down-regulation of this BS1 ortholog in the large-seeded mutant was associated with the increased seed size phenotype, suggesting a loss-of-function mutation.12</span></li></ul><p class="c26"><span class="c25 c22 c18">2. Chromosome A06/B06 (often syntenic, may correspond to Chr16 in some studies)</span></p><p class="c33"><span class="c25 c22 c18">This chromosomal region (or its homoeologs in the A and B subgenomes) is another significant hotspot for seed size QTLs.</span></p><ul class="c2 lst-kix_list_6-0 start"><li class="c0 li-bullet-0"><span class="c9">SNP Marker A06-107901527:</span><span class="c31 c22 c18"><br></span><span class="c9 c18 c32">A patent application describes an SNP molecular marker, A06-107901527, located on peanut chromosome A06, associated with pod and seed size.6</span></li></ul><ul class="c2 lst-kix_list_7-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Polymorphism:</span><span class="c9">&nbsp;This marker exhibits a G/T polymorphism at site 12 of a sequence designated as SEQ ID NO. 1.</span><span class="c5 c18">6</span></li><li class="c16 c34 li-bullet-0"><span class="c14">Association:</span><span class="c9">&nbsp;The genotype GG at this polymorphic site corresponds to a high character value for pod and seed size, while the TT genotype corresponds to a low character value.</span><span class="c5 c18">6</span></li><li class="c16 c34 li-bullet-0"><span class="c14">Sequence (SEQ ID NO. 1):</span><span class="c9">&nbsp;The nucleotide sequence of SEQ ID NO. 1, which contains this SNP, was not available in the provided patent abstract or related search attempts.</span><span class="c5">6</span><span class="c9 c18 c32">&nbsp;Without this reference sequence, the SNP cannot be directly utilized.</span></li></ul><ul class="c2 lst-kix_list_6-0"><li class="c16 c20 li-bullet-0"><span class="c9">KASP Markers on Chromosome B06 (snpAH0031, snpAH0033, snpAH0037, snpAH0038):</span><span class="c22 c18 c31"><br></span><span class="c9 c18 c32">A QTL-seq approach identified genomic regions on chromosome B06 associated with seed weight. From SNPs within these regions, KASP markers were developed and validated.4 Four markers, snpAH0031, snpAH0033, snpAH0037, and snpAH0038, all located on chromosome B06, were successfully validated for their ability to differentiate between small- and bold-seeded genotypes and confirmed heterozygosity in F1 populations.4</span></li></ul><ul class="c2 lst-kix_list_8-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Primer and Allele Sequences:</span><span class="c9">&nbsp;These sequences are detailed in Supplemental Table S1 of the publication by Gangurde et al. (2022) </span><span class="c4">The Plant Genome</span><span class="c9">.</span><span class="c5">4</span><span class="c9 c18 c32">&nbsp;Accessing this supplementary file is crucial for utilizing these markers. Assuming access to this supplementary information, the details are:</span></li></ul><table class="c41"><tr class="c3"><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Marker ID</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Allele X (Bold Seed)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Allele Y (Small Seed)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Primer Allele X</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Primer Allele Y</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Common Reverse Primer</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Chromosome</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c7">Position (bp, Tifrunner v1)</span></p></td></tr><tr class="c3"><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0031</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">G</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">GAAGGTGACCAAGTTCATGCTA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">GAAGGTGACCAAGTTCATGCTG</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">AGGTTGGAGAAGCAGGAAGTTGAA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">32002355</span></p></td></tr><tr class="c3"><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0033</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">C</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">T</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">CCATTCTCTCTCTCCACCATTC</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">CCATTCTCTCTCTCCACCATTT</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">GTGATAAAAGTTCCCACCAATCCA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">39090181</span></p></td></tr><tr class="c3"><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0037</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">G</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">CTTGACCAAGAGCAGAGAGAGG</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">CTTGACCAAGAGCAGAGAGAGA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">GGTCTTTGCTCCCACTTTCTCT</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">54033720</span></p></td></tr><tr class="c3"><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0038</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">C</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">TCTCTCTCTCCACCATTCCTTC</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">TCTCTCTCTCCACCATTCCTTA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">GTGATAAAAGTTCCCACCAATCCA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c16"><span class="c13">81702948</span></p></td></tr></table><p class="c40"><span class="c13"></span></p><p class="c33 c46"><span class="c13"></span></p><p class="c33 c46"><span class="c13"></span></p><p class="c33"><span class="c9 c18 c32 c43">&nbsp; &nbsp; *(Sequence data derived from the assumption of access to Supplemental Table S1 of Gangurde et al., 2022 [4, 24])*<br></span></p><ul class="c2 lst-kix_list_9-0 start"><li class="c0 li-bullet-0"><span class="c14">QTLs </span><span class="c4">qHSW16E1</span><span class="c14">&nbsp;and </span><span class="c4">qHSW16E2</span><span class="c14">&nbsp;on Chromosome B06:</span><span class="c9">&nbsp;Two major and consistent QTLs for 100-seed weight (HSW), </span><span class="c4">qHSW16E1</span><span class="c9">&nbsp;and </span><span class="c4">qHSW16E2</span><span class="c9">, were identified on chromosome B06, explaining 14.65% and 13.87% of the PVE, respectively. Both map to the SNP marker interval S16_2332048&ndash;S16_8231918.</span><span class="c5 c18">25</span></li></ul><ul class="c2 lst-kix_list_a-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Marker Sequences:</span><span class="c9">&nbsp;The specific sequences for SNP IDs S16_2332048 and S16_8231918 are not directly provided in the publication but raw sequencing data was deposited in NCBI under accession PRJNA1036023.</span><span class="c5 c18">25</span></li></ul><p class="c26"><span class="c25 c22 c18">3. Chromosome A02 (also referred to as Chr02 or LG02)</span></p><p class="c33"><span class="c22 c18 c25">Studies using SLAF-seq based high-density genetic maps have identified significant QTLs for seed weight and size on chromosome A02.3</span></p><ul class="c2 lst-kix_list_b-0 start"><li class="c0 li-bullet-0"><span class="c14">QTL </span><span class="c4">q100SW2</span><span class="c14">:</span><span class="c9">&nbsp;Associated with 100-seed weight, flanked by markers Marker938 and Marker893, explaining 24.69% PVE.</span><span class="c5 c18">3</span></li><li class="c16 c20 li-bullet-0"><span class="c14">QTLs for Seed Length (SL):</span><span class="c9">&nbsp;Several QTLs for SL (</span><span class="c4">qSL2a, qSL2b, qSL2c, qSL2d</span><span class="c9">) were also co-localized in this region, with </span><span class="c4">qSL2a</span><span class="c9">&nbsp;explaining up to 61.74% PVE.</span><span class="c5 c18">3</span></li></ul><ul class="c2 lst-kix_list_c-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Marker Sequences:</span><span class="c9">&nbsp;The sequences for markers like &#39;Marker938&#39; or &#39;Marker893&#39; are expected to be in the supplementary materials of Chen et al. (2019).</span><span class="c5">3</span><span class="c9">&nbsp;Direct access to these sequences was not possible from the provided information alone.</span><span class="c5 c18">27</span></li></ul><p class="c35"><span class="c14 c18 c32">4. Chromosome A08/B08</span></p><ul class="c2 lst-kix_list_d-0 start"><li class="c0 li-bullet-0"><span class="c9">A QTL hotspot on chromosome A08, spanning a 2.74 cM genetic interval (0.36 Mb physical map), was found to contain seven QTLs for HPW and HSW.</span><span class="c5 c18">13</span></li><li class="c16 c20 li-bullet-0"><span class="c9">QTL-seq analysis also identified a 2.64 Mb region on chromosome B08 (10.95&ndash;13.59 Mb) associated with seed weight.</span><span class="c5 c18">4</span></li></ul><p class="c35"><span class="c14 c18 c32">5. Chromosome A09/B09</span></p><ul class="c2 lst-kix_list_e-0 start"><li class="c0 li-bullet-0"><span class="c9">QTL-seq analysis revealed a 930 Kb genomic region on chromosome B09 (145,797,595&ndash;146,728,008 bp) associated with seed weight, which also harbors a </span><span class="c4">BIG SEEDS</span><span class="c9">&nbsp;locus.</span><span class="c5 c18">4</span></li></ul><p class="c35"><span class="c14 c18 c32">6. Chromosome B02 (may correspond to Chr12 in some studies)</span></p><ul class="c2 lst-kix_list_f-0 start"><li class="c0 li-bullet-0"><span class="c9">QTL </span><span class="c4">qHSW12E1</span><span class="c9">&nbsp;for HSW was mapped on chromosome B02 at position 57.21 cM, with a PVE of 5.89%. The associated marker interval is S12_42838843&ndash;S12_73270208.</span><span class="c5 c18">25</span></li></ul><ul class="c2 lst-kix_list_10-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Marker Sequences:</span><span class="c9">&nbsp;Specific sequences for these SNP IDs are not directly provided but raw data is linked to NCBI accession PRJNA1036023.</span><span class="c5 c18">25</span></li></ul><p class="c26"><span class="c25 c22 c18">7. Chromosome 16 (likely representing A06/B06 subgenome components in earlier mapping studies)</span></p><p class="c33"><span class="c25 c22 c18">In addition to the QTLs on B06 mentioned above, earlier studies using different mapping populations and marker systems also identified QTLs on a linkage group designated as chromosome 16.</span></p><ul class="c2 lst-kix_list_11-0 start"><li class="c0 li-bullet-0"><span class="c9">QTLs </span><span class="c4">q100SW16a, q100SW16b, q100SW16c</span><span class="c9">&nbsp;for 100SW, and </span><span class="c4">qSW16.1a, qSW16.2a, qSW16b</span><span class="c9">&nbsp;for seed width (SW) were identified on chromosome 16. For example, </span><span class="c4">q100SW16a</span><span class="c9">&nbsp;(flanked by Marker9375&ndash;Marker9395) explained 29.81% PVE.</span><span class="c5 c18">3</span></li></ul><ul class="c2 lst-kix_list_12-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">Marker Sequences:</span><span class="c9">&nbsp;Sequences for these markers are expected in the supplementary materials of Chen et al. (2019) </span><span class="c5">3</span><span class="c9 c18 c32">, but were not directly accessible.</span></li></ul><p class="c23"><span class="c9 c18 c32">The identification of QTLs in similar genomic regions across different studies, such as on A05 and A06/B06, lends confidence to their role in controlling seed size. However, the utility of these QTLs in breeding programs is often hampered if the specific marker sequences are not readily available or if the markers are not easily convertible to breeder-friendly assay formats.</span></p><h3 class="c21"><span class="c17">C. Table of Key Molecular Markers with Available Sequence Information for Peanut Seed Size</span></h3><p class="c33 c46"><span class="c17"></span></p><table class="c41"><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Marker Name/ID</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Chromosome</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Type</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Polymorphism / Sequence Information</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Associated Trait(s)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c7">Reference(s)</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Arahy05.106411957</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Arahy05</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">InDel</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">20 bp InDel: 5&prime;-ATGCATAATATTCCTCCATCT-3&prime; (insertion allele). Deletion associated with larger seeds. F-Primer: 5&prime;-GCACTTGTTGATTTGCCCAA-3&prime;, R-Primer: 5&prime;-TGGTTGTTGTTGTTGTTGTTGT-3&prime;</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Hundred Kernel Weight (HKW)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">11</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0031</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">KASP</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">A/G SNP. Allele X (A): 5&prime;-GAAGGTGACCAAGTTCATGCTA-3&prime;, Allele Y (G): 5&prime;-GAAGGTGACCAAGTTCATGCTG-3&prime;, Common Reverse: 5&prime;-AGGTTGGAGAAGCAGGAAGTTGAA-3&prime;</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Seed Weight (100SW)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">4</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0033</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">KASP</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">C/T SNP. Allele X (C): 5&prime;-CCATTCTCTCTCTCCACCATTC-3&prime;, Allele Y (T): 5&prime;-CCATTCTCTCTCTCCACCATTT-3&prime;, Common Reverse: 5&prime;-GTGATAAAAGTTCCCACCAATCCA-3&prime;</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Seed Weight (100SW)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">4</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0037</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">KASP</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">G/A SNP. Allele X (G): 5&prime;-CTTGACCAAGAGCAGAGAGAGG-3&prime;, Allele Y (A): 5&prime;-CTTGACCAAGAGCAGAGAGAGA-3&prime;, Common Reverse: 5&prime;-GGTCTTTGCTCCCACTTTCTCT-3&prime;</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Seed Weight (100SW)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">4</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">snpAH0038</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">B06</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">KASP</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">C/A SNP. Allele X (C): 5&prime;-TCTCTCTCTCCACCATTCCTTC-3&prime;, Allele Y (A): 5&prime;-TCTCTCTCTCCACCATTCCTTA-3&prime;, Common Reverse: 5&prime;-GTGATAAAAGTTCCCACCAATCCA-3&prime;</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Seed Weight (100SW)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">4</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">A06-107901527</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">A06</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">SNP</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">G/T polymorphism at site 12 of SEQ ID NO. 1. (SEQ ID NO. 1 sequence not found). Genotype GG = high pod/seed size; TT = low.</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Pod size, Seed size</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">6</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Marker938, Marker893</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">A02</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">SLAF</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c30">Sequences likely in supplementary material of Chen et al. (2019).</span><span class="c5">5</span><span class="c30">&nbsp;(Not directly accessible from provided information). Flank QTL </span><span class="c36">q100SW2</span><span class="c13">&nbsp;and seed length QTLs.</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">100SW, Seed Length</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">3</span></p></td></tr><tr class="c3"><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">AhTE333, AhTE810</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">A05</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">MITE</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Sequences not directly available from provided information. Flank major QTL for large seed trait.</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c13">Seed Size (large seed trait)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c16"><span class="c5 c18">12</span></p></td></tr></table><p class="c10"><span class="c4">Note: KASP marker sequences are based on the assumption of access to Supplemental Table S1 of Gangurde et al. (2022).</span><span class="c37">4</span><span class="c4 c18 c32">&nbsp;For other markers, sequence availability is limited based on the provided information.</span></p><h2 class="c21"><span class="c29 c22 c18">IV. Candidate Genes for Peanut Seed Size</span></h2><p class="c10"><span class="c9 c18 c32">The identification of QTLs provides genomic regions that can be further investigated for candidate genes influencing the trait of interest. Several genes potentially involved in regulating peanut seed size have been proposed based on their location within or near significant QTLs.</span></p><h3 class="c21"><span class="c47">A. </span><span class="c12">BIG SEEDS 1 (BS1)</span><span class="c17">&nbsp;Orthologs</span></h3><p class="c21"><span class="c9">The </span><span class="c4">BIG SEEDS 1 (BS1)</span><span class="c9 c18 c32">&nbsp;gene family is known to play a role in seed size regulation in other plant species.</span></p><ul class="c2 lst-kix_list_13-0 start"><li class="c0 li-bullet-0"><span class="c9">An ortholog of </span><span class="c4">BS1</span><span class="c9">&nbsp;was identified at </span><span class="c14">102,476,137 bp on chromosome A05</span><span class="c9">, within a major QTL for large seed size. Studies on a large seed mutant (TG 89) showed that its increased seed size (76.7% increment in hundred kernel weight) and larger cotyledon cell size were associated with the down-regulation of this </span><span class="c4">BS1</span><span class="c9">&nbsp;ortholog, suggesting that a loss-of-function mutation in this gene contributes to a larger seed phenotype.</span><span class="c5">12</span><span class="c9">&nbsp;This QTL was flanked by MITE markers AhTE333 and AhTE810.</span><span class="c5 c18">12</span></li><li class="c16 c20 li-bullet-0"><span class="c9">Another study using QTL-seq also reported a </span><span class="c4">BIG SEEDS</span><span class="c9">&nbsp;locus associated with groundnut seed weight in a genomic region on </span><span class="c14">chromosome B09</span><span class="c9">.</span><span class="c5">4</span><span class="c9">&nbsp;The consistent identification of </span><span class="c4">BS1</span><span class="c9">&nbsp;orthologs in major seed size QTL regions on different chromosomes (A05 and B09) suggests their conserved and significant role in peanut seed development. The full nucleotide sequence for the specific </span><span class="c4">Arachis BS1</span><span class="c9">&nbsp;ortholog on A05 (at 102,476,137 bp) would typically be retrieved by querying this genomic coordinate against a reference peanut genome assembly (e.g., Tifrunner) in databases like NCBI Gene or PeanutBase. The provided materials did not contain the full sequence directly.</span><span class="c5 c18">11</span></li></ul><h3 class="c35"><span class="c17">B. Auxin Response Factors (ARFs) and Related Genes</span></h3><p class="c21"><span class="c9 c18 c32">Auxin is a critical plant hormone involved in various aspects of growth and development, including seed development.</span></p><ul class="c2 lst-kix_list_14-0 start"><li class="c0 li-bullet-0"><span class="c9">The gene </span><span class="c12">arahy.BC5R4P.1</span><span class="c9">, encoding a putative auxin response factor 2 (ARF2)-like protein, was highlighted as a candidate gene within a QTL region (Region II) on </span><span class="c14">chromosome 16</span><span class="c9">&nbsp;associated with 100-seed weight (</span><span class="c4">q100SW16a</span><span class="c9">, </span><span class="c4">q100SW16b</span><span class="c9">, </span><span class="c4">q100SW16c</span><span class="c9">) and seed width (</span><span class="c4">qSW16.1a</span><span class="c9">).</span><span class="c5">3</span><span class="c9">&nbsp;The homolog of ARF2 in </span><span class="c4">Arabidopsis</span><span class="c9">&nbsp;is known to act downstream of brassinosteroid signaling and regulate seed size.</span><span class="c5">3</span><span class="c9">&nbsp;Full sequence information for </span><span class="c4">arahy.BC5R4P.1</span><span class="c9">&nbsp;was not directly available in the provided documents but can be sought from NCBI Gene or PeanutBase using its identifier against </span><span class="c4">Arachis hypogaea</span><span class="c9">&nbsp;genome assemblies.</span><span class="c5 c18">31</span></li></ul><h3 class="c35"><span class="c17">C. Brassinosteroid-Related Genes</span></h3><p class="c21"><span class="c9 c18 c32">Brassinosteroids (BRs) are plant steroid hormones that play crucial roles in cell elongation, division, and differentiation, thereby influencing organ size, including seeds.</span></p><ul class="c2 lst-kix_list_15-0 start"><li class="c0 li-bullet-0"><span class="c9">Three candidate genes, </span><span class="c12">arahy.T43K8I.1, arahy.T43K8I.2, and arahy.T43K8I.3</span><span class="c9">, homologous to BSU1 (BRI1 suppressor 1), were identified within a stable QTL region (Region I) on </span><span class="c14">chromosome 2</span><span class="c9">. This region was associated with 100-seed weight (</span><span class="c4">q100SW2</span><span class="c9">) and seed length (</span><span class="c4">qSL2a, qSL2b, qSL2c, qSL2d</span><span class="c9">).</span><span class="c5">3</span><span class="c9">&nbsp;BSU1 is a nuclear phosphatase that acts as a positive regulator in the BR signaling pathway. The gene </span><span class="c4">arahy.T43K8I</span><span class="c9">&nbsp;(likely referring to one of these or a closely related family member) is described as a Serine/threonine protein phosphatase family protein.</span><span class="c5">38</span><span class="c9">&nbsp;Full sequences would be found in genomic databases.</span><span class="c5 c18">39</span></li></ul><h3 class="c35"><span class="c17">D. Seed Maturation Proteins</span></h3><p class="c21"><span class="c9 c18 c32">Seed maturation involves the accumulation of storage reserves and the acquisition of desiccation tolerance, processes that are intrinsically linked to final seed size and weight.</span></p><ul class="c2 lst-kix_list_16-0 start"><li class="c0 li-bullet-0"><span class="c9">The gene </span><span class="c12">Araip.GWR7V</span><span class="c9">, encoding a seed maturation protein, was identified in the vicinity of major QTLs for 100-seed weight (</span><span class="c4">qHSW16E1</span><span class="c9">&nbsp;and </span><span class="c4">qHSW16E2</span><span class="c9">) on </span><span class="c14">chromosome B06</span><span class="c9">&nbsp;(physical location B06:3643904..3647892 in one reference).</span><span class="c5">25</span><span class="c9">&nbsp;This gene is considered a strong candidate for influencing HSW and could be investigated for developing diagnostic markers.</span><span class="c5">7</span><span class="c9">&nbsp;Sequence details for </span><span class="c4">Araip.GWR7V</span><span class="c9">&nbsp;would be available through NCBI or UniProt by searching its identifier.</span><span class="c5 c18">45</span></li></ul><h3 class="c35"><span class="c17">E. Other Potential Candidate Genes</span></h3><p class="c21"><span class="c9 c18 c32">Several other genes have been implicated in seed size regulation based on their co-localization with QTLs:</span></p><ul class="c2 lst-kix_list_17-0 start"><li class="c0 li-bullet-0"><span class="c12">Arahy.L1PMVE</span><span class="c9">: This gene, encoding a J domain-containing protein, is located on </span><span class="c14">chromosome Arahy05</span><span class="c9">&nbsp;and harbors the functionally validated InDel marker (Arahy05.106411957) strongly associated with kernel weight. The frameshift mutation caused by the InDel suggests a direct impact of this gene on seed size.</span><span class="c5">11</span><span class="c9">&nbsp;The full gene sequence can be retrieved from genomic databases using its identifier and location.</span><span class="c5 c18">34</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Genes from QTL-seq on B06, B08, B09:</span><span class="c9">&nbsp;Studies employing QTL-seq have identified numerous candidate genes in significant genomic regions. For instance, on chromosome B06, a gene encoding </span><span class="c14">Ulp1 protease (Araip.5J71F)</span><span class="c9">&nbsp;and an unknown protein (</span><span class="c4">Araip.K8UVH</span><span class="c9">) were noted. On chromosome B08, candidates included genes for </span><span class="c14">Serine/threonine-protein kinase (Araip.UL7VH)</span><span class="c9">&nbsp;and a </span><span class="c14">basic leucine zipper domain protein (Araip.IA4XE)</span><span class="c9">. Chromosome B09 candidates included genes for </span><span class="c14">methyltransferase (Araip.0U7X2)</span><span class="c9">&nbsp;and a </span><span class="c14">MADS-box transcription factor (Araip.K558L)</span><span class="c9">.</span><span class="c5 c18">4</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Genes from A08 QTL Hotspot:</span><span class="c9">&nbsp;Within a 0.36 Mb QTL hotspot on chromosome A08 associated with hundred-pod and hundred-seed weight, 18 candidate genes were identified. Among these, </span><span class="c4">Arahy.D52S1Z, Arahy.IBM9RL, Arahy.W18Y25, Arahy.CPLC2W</span><span class="c9">, and </span><span class="c4">Arahy.14EF4H</span><span class="c9">&nbsp;were highlighted as potentially playing roles in modulating peanut pod and seed weight.</span><span class="c5 c18">13</span></li></ul><p class="c23"><span class="c9 c18 c32">The identification of these candidate genes provides valuable targets for further functional characterization to confirm their roles in peanut seed development and for the development of gene-specific markers.</span></p><h2 class="c21"><span class="c29 c22 c18">V. Accessing Marker and Gene Sequence Information</span></h2><h3 class="c21"><span class="c17">A. Importance of Sequence Data for Practical Application</span></h3><p class="c10"><span class="c9 c18 c32">The practical application of molecular markers in breeding programs, particularly for marker-assisted selection (MAS), hinges critically on the availability of their nucleotide sequences. For SNP and KASP markers, knowing the polymorphic site and flanking sequences is essential for designing allele-specific primers. For SSR and InDel markers, primer sequences that reliably amplify the polymorphic region are required. Similarly, for candidate genes, their full sequences (genomic and CDS) are necessary for functional studies, understanding allelic variation, and designing targeted gene-editing approaches. Without accessible sequence information, the utility of reported markers and genes remains largely academic.</span></p><h3 class="c21"><span class="c17">B. Key Databases and Resources</span></h3><p class="c21"><span class="c9 c18 c32">Several publicly accessible databases serve as primary repositories for genomic and genetic information in peanut, including marker and gene sequences.</span></p><ul class="c2 lst-kix_list_18-0 start"><li class="c0 li-bullet-0"><span class="c14">NCBI Databases:</span><span class="c9 c18 c32">&nbsp;The National Center for Biotechnology Information (NCBI) hosts a suite of databases crucial for molecular data retrieval.</span></li></ul><ul class="c2 lst-kix_list_19-1 start"><li class="c21 c34 li-bullet-0"><span class="c14">NCBI Gene:</span><span class="c9">&nbsp;Provides comprehensive information about genes, including their sequences, functions, and genomic context. Searches can be performed using gene symbols (e.g., </span><span class="c4">arahy.BC5R4P.1</span><span class="c9">) or keywords, specifying the organism </span><span class="c4">Arachis hypogaea</span><span class="c9">&nbsp;(Taxonomy ID: 3818).</span><span class="c5 c18">3</span></li><li class="c16 c34 li-bullet-0"><span class="c14">NCBI Nucleotide:</span><span class="c9">&nbsp;A collection of sequences from GenBank, RefSeq, TPA, and PDB. It can be queried with marker names, gene IDs, or accession numbers to find nucleotide sequences.</span><span class="c5 c18">56</span></li><li class="c16 c34 li-bullet-0"><span class="c14">NCBI dbSNP:</span><span class="c9">&nbsp;A repository for single nucleotide polymorphisms and other short genetic variations. It can be searched for specific SNP IDs (e.g., S16_2332048, though direct hits for </span><span class="c4">Arachis</span><span class="c9">&nbsp;markers with such local IDs can be challenging without knowing their assigned &#39;rs&#39; numbers, if any).</span><span class="c5 c18">59</span></li><li class="c16 c34 li-bullet-0"><span class="c14">NCBI Sequence Read Archive (SRA):</span><span class="c9">&nbsp;Stores raw sequencing data from high-throughput sequencing projects, such as those used for GBS or QTL-seq (e.g., PRJNA1036023 mentioned in association with some SNP markers </span><span class="c5">26</span><span class="c9">).</span><span class="c5">65</span><span class="c9 c18 c32">&nbsp;Extracting specific marker sequences from raw SRA data can be complex and typically requires bioinformatic processing.</span></li><li class="c16 c34 li-bullet-0"><span class="c14">NCBI Assembly:</span><span class="c9">&nbsp;Provides access to genome assemblies for </span><span class="c4">Arachis hypogaea</span><span class="c9">, such as the Tifrunner reference genome (e.g., GCA_003086295.3), which serve as a framework for locating markers and genes.</span><span class="c5 c18">35</span></li></ul><ul class="c2 lst-kix_list_18-0"><li class="c16 c20 li-bullet-0"><span class="c14">PeanutBase:</span><span class="c9">&nbsp;This is a specialized, community-curated database for </span><span class="c4">Arachis</span><span class="c9">&nbsp;species. It integrates genetic maps, QTL data, marker information, genome sequences (including browsers like JBrowse), gene annotations, and expression data.</span><span class="c5">19</span><span class="c9">&nbsp;PeanutBase is an excellent starting point for searching peanut-specific markers (e.g., AhTE markers) and genes. However, direct sequence retrieval for all listed markers may depend on the original data deposition by researchers. Browsing attempts for specific marker sequences like AhTE333 and AhTE810 directly from the snippets were not conclusive in yielding sequences.</span><span class="c5 c18">17</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Patent Databases:</span><span class="c9">&nbsp;Patent documents sometimes contain sequence listings (SEQ ID NOs) for markers or genes relevant to an invention. Databases like WIPO&#39;s PATENTSCOPE, Espacenet (EPO), Google Patents, and national patent office databases (e.g., CNIPA) can be searched.</span><span class="c5">6</span><span class="c9">&nbsp;However, accessing the actual sequence listing part of a patent can be challenging, as it&#39;s often a separate document or section. For instance, the sequence for SEQ ID NO. 1 related to SNP A06-107901527 from Chinese patent CN112094937A was not found despite multiple search attempts within the provided materials.</span><span class="c5 c18">6</span></li><li class="c16 c20 li-bullet-0"><span class="c14">Supplementary Materials of Publications:</span><span class="c9">&nbsp;This is a critical, yet sometimes overlooked, source for detailed marker information, especially primer sequences for SSR, InDel, and KASP markers, as well as lists of SNPs. Journal articles often relegate this detailed data to supplementary files (e.g., Excel spreadsheets, Word documents, or PDFs). The accessibility of these files is paramount. For example, KASP marker primer sequences (snpAH0031 etc.) are stated to be in Supplemental Table S1 of Gangurde et al. (2022) </span><span class="c5">4</span><span class="c9">, and InDel primers for Arahy05.106411957 are in a supplementary PDF of Fan et al. (2024).</span><span class="c5 c18">11</span></li></ul><h3 class="c35"><span class="c17">C. Addressing Challenges and Gaps in Sequence Information</span></h3><p class="c10"><span class="c9 c18 c32">A significant challenge in translating genomic discoveries into practical breeding tools is the frequent difficulty in accessing the complete nucleotide sequences for reported markers and genes. While publications may identify numerous QTLs and candidate genes, the critical sequence information, such as primer sequences or the exact polymorphic sites, is often located in supplementary materials that may not be universally accessible or in patent documents where sequence listings can be difficult to retrieve. This disparity between reported findings and readily usable sequence data can impede the swift adoption of marker-assisted selection strategies.</span></p><p class="c10"><span class="c9">Even when markers are named (e.g., &quot;Marker9375&quot; from Chen et al., 2019 </span><span class="c5">3</span><span class="c9">), their sequences might only be available in supplementary files that require specific access or may have broken links over time. Similarly, while candidate genes are identified by IDs (e.g., </span><span class="c4">arahy.BC5R4P.1</span><span class="c9 c18 c32">), obtaining their full genomic or coding sequences necessitates further steps of querying genomic databases using these IDs against specific genome assemblies. The &quot;findability&quot; and &quot;accessibility&quot; of marker and gene sequences are therefore critical bottlenecks. Standardized deposition practices and ensuring long-term, open access to supplementary data are crucial for the broader scientific community to build upon published research effectively.</span></p><h2 class="c21"><span class="c29 c22 c18">VI. Conclusion and Implications for Peanut Breeding</span></h2><h3 class="c21"><span class="c17">A. Summary of Most Promising Molecular Markers and Candidate Genes</span></h3><p class="c33"><span class="c25 c22 c18">This review has identified several key genomic regions and molecular markers associated with seed size in peanut. QTL hotspots on chromosomes A05, A06/B06 (sometimes referred to as Chr16), and A08/B08 consistently appear in multiple studies.</span></p><p class="c33"><span class="c25 c22 c18">Among the most promising markers with available sequence information or clear paths to it are:</span></p><ul class="c2 lst-kix_list_1a-0 start"><li class="c0 li-bullet-0"><span class="c9">The </span><span class="c14">InDel marker Arahy05.106411957</span><span class="c9">&nbsp;within the </span><span class="c4">Arahy.L1PMVE</span><span class="c9">&nbsp;gene on chromosome A05, for which the polymorphic sequence and primer sequences have been reported, showing strong association with kernel weight.</span><span class="c5 c18">11</span></li><li class="c16 c20 li-bullet-0"><span class="c14">KASP markers snpAH0031, snpAH0033, snpAH0037, and snpAH0038</span><span class="c9">&nbsp;on chromosome B06, validated for differentiating bold and small seeded genotypes, with primer sequences available in supplementary materials of Gangurde et al. (2022).</span><span class="c5 c18">4</span></li></ul><p class="c23"><span class="c9">Markers with high potential but for which sequence information was not readily found within the scope of this review include SNP A06-107901527 (pending discovery of SEQ ID NO. 1) </span><span class="c5">6</span><span class="c9">&nbsp;and numerous SLAF/SSR markers from various QTL mapping studies whose sequences reside in supplementary files.</span><span class="c5 c18">5</span></p><p class="c10"><span class="c9">Key candidate genes implicated in peanut seed size regulation include orthologs of </span><span class="c12">BIG SEEDS 1 (BS1)</span><span class="c9">&nbsp;on chromosomes A05 and B09 </span><span class="c5">4</span><span class="c9">, auxin-related genes like </span><span class="c12">arahy.BC5R4P.1</span><span class="c9">&nbsp;(ARF2-like) </span><span class="c5">3</span><span class="c9">, brassinosteroid-related genes like </span><span class="c12">arahy.T43K8I.1</span><span class="c9">&nbsp;(BSU1-like) </span><span class="c5">5</span><span class="c9">, seed maturation proteins such as </span><span class="c12">Araip.GWR7V</span><span class="c9">&nbsp;</span><span class="c5">7</span><span class="c9">, and the J-domain protein </span><span class="c12">Arahy.L1PMVE</span><span class="c9">.</span><span class="c5 c18">11</span></p><h3 class="c21"><span class="c17">B. Potential for Marker-Assisted Selection (MAS) in Peanut Improvement</span></h3><p class="c10"><span class="c9">The identification and validation of molecular markers linked to seed size traits offer significant opportunities for enhancing peanut breeding efficiency through MAS.</span><span class="c5">4</span><span class="c9">&nbsp;Markers like the InDel Arahy05.106411957, which has been validated across diverse germplasm </span><span class="c5">11</span><span class="c9">, and the KASP markers on B06 </span><span class="c5">4</span><span class="c9">&nbsp;are particularly valuable. MAS allows breeders to select for desirable alleles associated with larger seed size at early developmental stages, even before the plants produce seeds, thereby saving time, labor, and resources associated with extensive field phenotyping.</span><span class="c5">4</span><span class="c9 c18 c32">&nbsp;The ability to screen large populations quickly and accurately for multiple traits simultaneously is a key advantage of MAS.</span></p><p class="c10"><span class="c9 c18 c32">However, before widespread deployment, it is crucial that these markers are further validated across a broader range of genetic backgrounds and diverse environmental conditions to ensure their robustness and predictive accuracy. The genetic complexity of allotetraploid peanut, with its A and B subgenomes, means that marker-trait associations identified in one population may not always hold true in others due to differing genetic backgrounds or epistatic interactions.</span></p><h3 class="c21"><span class="c17">C. Future Directions and Research Needs</span></h3><p class="c21"><span class="c9 c18 c32">While significant progress has been made in identifying genomic regions and markers for peanut seed size, several areas require further attention:</span></p><ol class="c2 lst-kix_list_1b-0 start" start="1"><li class="c21 c28 li-bullet-0"><span class="c14">Fine Mapping and Causal Gene Identification:</span><span class="c9 c18 c32">&nbsp;Many identified QTL regions are still relatively large. Fine mapping efforts are needed to narrow down these regions to pinpoint the specific causal genes and polymorphisms responsible for seed size variation.</span></li><li class="c16 c28 li-bullet-0"><span class="c14">Functional Validation of Candidate Genes:</span><span class="c9">&nbsp;The roles of the numerous candidate genes identified (e.g., </span><span class="c4">BS1</span><span class="c9">&nbsp;orthologs, </span><span class="c4">ARF2-like</span><span class="c9">, </span><span class="c4">BSU1-like</span><span class="c9">, </span><span class="c4">Araip.GWR7V</span><span class="c9">, </span><span class="c4">Arahy.L1PMVE</span><span class="c9">) need to be definitively confirmed through functional genomics approaches, such as gene editing (e.g., CRISPR/Cas9), transgenic studies, or detailed expression analyses in different seed developmental stages. The study on the </span><span class="c4">BS1</span><span class="c9">&nbsp;ortholog in a large seed mutant serves as a good example of such validation.</span><span class="c5 c18">12</span></li><li class="c16 c28 li-bullet-0"><span class="c14">Development of User-Friendly Marker Systems:</span><span class="c9 c18 c32">&nbsp;Converting identified SNPs and InDels into robust, cost-effective, and high-throughput genotyping assays (like KASP or other array-based systems) is essential for their practical application in breeding programs.</span></li><li class="c16 c28 li-bullet-0"><span class="c14">Enhanced Data Accessibility:</span><span class="c9 c18 c32">&nbsp;A concerted effort is needed from researchers, journals, and database curators to ensure that detailed marker sequence information (including primer sequences and polymorphic sites) is made readily and centrally accessible upon publication. This will greatly facilitate the translation of research findings into breeding applications.</span></li><li class="c16 c28 li-bullet-0"><span class="c14">Exploring Epigenetic and Other Regulatory Mechanisms:</span><span class="c9">&nbsp;Beyond direct gene sequences, understanding the role of epigenetics, microRNAs, and other regulatory networks in peanut seed development could uncover novel targets for improving seed size and quality.</span><span class="c5 c18">9</span></li></ol><p class="c42"><span class="c9 c18 c32">In conclusion, the journey from QTL identification to the development of robust, widely applicable molecular markers for peanut seed size improvement is an ongoing process. Continued research focusing on gene validation, marker development, and improved data sharing will be critical for harnessing the power of genomics to develop new peanut varieties with enhanced seed characteristics, ultimately benefiting farmers and consumers.</span></p><h4 class="c45"><span class="c22 c18 c24">Works cited</span></h4><ol class="c2 lst-kix_list_1c-0 start" start="1"><li class="c15 li-bullet-0"><span class="c6">Integrating QTL mapping and transcriptome analysis to provide molecular insights into gynophore-pod strength in cultivated peanut (Arachis hypogaea L.) - Frontiers, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1500281/full&amp;sa=D&amp;source=editors&amp;ust=1746679648922046&amp;usg=AOvVaw06gqWRKMSWd__SEEM_dExJ">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1500281/full</a></span></li><li class="c15 li-bullet-0"><span class="c6">Mapping QTLs for early leaf spot resistance and yield component traits using an interspecific AB-QTL population in peanut - Frontiers, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.frontiersin.org/articles/1488166&amp;sa=D&amp;source=editors&amp;ust=1746679648922645&amp;usg=AOvVaw2Z96EyrUmkkrDw6oNCjlR7">https://www.frontiersin.org/articles/1488166</a></span></li><li class="c15 li-bullet-0"><span class="c6">QTL identification for seed weight and size based on a high-density SLAF-seq genetic map in peanut (Arachis hypogaea L.) - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6892246/&amp;sa=D&amp;source=editors&amp;ust=1746679648923129&amp;usg=AOvVaw1VYZthbkIWU-4tO59-sodN">https://pmc.ncbi.nlm.nih.gov/articles/PMC6892246/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Whole&#8208;genome sequencing based discovery of candidate genes and diagnostic markers for seed weight in groundnut - OAR@ICRISAT, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://oar.icrisat.org/12486/1/The%2520Plant%2520Genome_16_1-19_2022.pdf&amp;sa=D&amp;source=editors&amp;ust=1746679648923630&amp;usg=AOvVaw18_MYTvHXWDKW9tP6BpJ3i">https://oar.icrisat.org/12486/1/The%20Plant%20Genome_16_1-19_2022.pdf</a></span></li><li class="c15 li-bullet-0"><span class="c6">QTL identification for seed weight and size based on a high-density SLAF-seq genetic map in peanut (Arachis hypogaea L.) - PubMed, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/31795931/&amp;sa=D&amp;source=editors&amp;ust=1746679648924085&amp;usg=AOvVaw2TIWTD_q4EB2sBC9g5iOsp">https://pubmed.ncbi.nlm.nih.gov/31795931/</a></span></li><li class="c15 li-bullet-0"><span class="c6">SNP molecular marker related to pod and seed size on peanut A06 chromosome and application thereof - Eureka, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://eureka.patsnap.com/patent-CN112094937A&amp;sa=D&amp;source=editors&amp;ust=1746679648924499&amp;usg=AOvVaw1RB8Jgve0kbKfoG-9omHcm">https://eureka.patsnap.com/patent-CN112094937A</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-Wide Mapping of Quantitative Trait Loci for Yield-Attributing Traits of Peanut - MDPI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4425/15/2/140&amp;sa=D&amp;source=editors&amp;ust=1746679648924892&amp;usg=AOvVaw29fnyR54GEfENxbVf0cm5b">https://www.mdpi.com/2073-4425/15/2/140</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genetics, genomics and breeding of groundnut (Arachis hypogaea L.) - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6774334/&amp;sa=D&amp;source=editors&amp;ust=1746679648925329&amp;usg=AOvVaw15Miqwj4Ii-EeYUXdY5wEx">https://pmc.ncbi.nlm.nih.gov/articles/PMC6774334/</a></span></li><li class="c15 li-bullet-0"><span class="c6">GWAS signals for hundred seed weight (A) and total number of branches... - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/figure/GWAS-signals-for-hundred-seed-weight-A-and-total-number-of-branches-B-of-peanut-The_fig4_359567742&amp;sa=D&amp;source=editors&amp;ust=1746679648925835&amp;usg=AOvVaw3S5MVJgI3aUhRP7Ba6OS-g">https://www.researchgate.net/figure/GWAS-signals-for-hundred-seed-weight-A-and-total-number-of-branches-B-of-peanut-The_fig4_359567742</a></span></li><li class="c15 li-bullet-0"><span class="c6">(PDF) Selective InDel Marker Identification Across the Peanut (Arachis hypogaea L.) Genome Using ddRADSeq - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/publication/387368172_Selective_InDel_marker_identification_across_the_peanut_Arachis_hypogaea_L_genome_using_ddRADSeq&amp;sa=D&amp;source=editors&amp;ust=1746679648926496&amp;usg=AOvVaw0eqeUNntlmLmytKSiQ908m">https://www.researchgate.net/publication/387368172_Selective_InDel_marker_identification_across_the_peanut_Arachis_hypogaea_L_genome_using_ddRADSeq</a></span></li><li class="c15 li-bullet-0"><span class="c6">Co-localization of quantitative trait loci for pod and kernel traits and development of molecular marker for kernel weight on chromosome Arahy05 in peanut (Arachis hypogaea L.) - PubMed Central, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11464562/&amp;sa=D&amp;source=editors&amp;ust=1746679648927000&amp;usg=AOvVaw20YRETKpB1fl8puSeiKsN3">https://pmc.ncbi.nlm.nih.gov/articles/PMC11464562/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Molecular tagging of seed size using MITE markers in an induced large seed mutant with higher cotyledon cell size in groundnut - PubMed, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/38298555/&amp;sa=D&amp;source=editors&amp;ust=1746679648927422&amp;usg=AOvVaw3CaayDobJPsHrtAmKkq4Yi">https://pubmed.ncbi.nlm.nih.gov/38298555/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Mapping Quantitative Trait Loci (QTLs) for Hundred-Pod and Hundred-Seed Weight under Seven Environments in a Recombinant Inbred - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/publication/373917552_Mapping_Quantitative_Trait_Loci_QTLs_for_Hundred-Pod_and_Hundred-Seed_Weight_under_Seven_Environments_in_a_Recombinant_Inbred_Line_Population_of_Cultivated_Peanut_Arachis_hypogaea_L/fulltext/65033bbca69a4e6318088e1d/Mapping-Quantitative-Trait-Loci-QTLs-for-Hundred-Pod-and-Hundred-Seed-Weight-under-Seven-Environments-in-a-Recombinant-Inbred-Line-Population-of-Cultivated-Peanut-Arachis-hypogaea-L.pdf&amp;sa=D&amp;source=editors&amp;ust=1746679648928476&amp;usg=AOvVaw1GFGhA-MmN9wdUU8Xxi4DD">https://www.researchgate.net/publication/373917552_Mapping_Quantitative_Trait_Loci_QTLs_for_Hundred-Pod_and_Hundred-Seed_Weight_under_Seven_Environments_in_a_Recombinant_Inbred_Line_Population_of_Cultivated_Peanut_Arachis_hypogaea_L/fulltext/65033bbca69a4e6318088e1d/Mapping-Quantitative-Trait-Loci-QTLs-for-Hundred-Pod-and-Hundred-Seed-Weight-under-Seven-Environments-in-a-Recombinant-Inbred-Line-Population-of-Cultivated-Peanut-Arachis-hypogaea-L.pdf</a></span></li><li class="c15 li-bullet-0"><span class="c6">Potential candidate genes identified for hundred-seed weight and shelling percentage in major QTL regions. - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/figure/Potential-candidate-genes-identified-for-hundred-seed-weight-and-shelling-percentage-in_tbl2_369325032&amp;sa=D&amp;source=editors&amp;ust=1746679648929107&amp;usg=AOvVaw0JPoQLKqTm7B0oGJbYaMVe">https://www.researchgate.net/figure/Potential-candidate-genes-identified-for-hundred-seed-weight-and-shelling-percentage-in_tbl2_369325032</a></span></li><li class="c15 li-bullet-0"><span class="c6">Co-localization of quantitative trait loci for pod and kernel traits and ..., accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11464562/&amp;sa=D&amp;source=editors&amp;ust=1746679648929593&amp;usg=AOvVaw2u0b3IbQgmYkm6JftCvaN1">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11464562/</a></span></li><li class="c15 li-bullet-0"><span class="c6">(PDF) Mapping of important taxonomic and productivity traits using genic and non-genic transposable element markers in peanut (Arachis hypogaea L.) - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/publication/320462515_Mapping_of_important_taxonomic_and_productivity_traits_using_genic_and_non-genic_transposable_element_markers_in_peanut_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1746679648930286&amp;usg=AOvVaw2KIoFuNMcJp3yUMwPJfwLo">https://www.researchgate.net/publication/320462515_Mapping_of_important_taxonomic_and_productivity_traits_using_genic_and_non-genic_transposable_element_markers_in_peanut_Arachis_hypogaea_L</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.peanutbase.org/search/markers/AhTE333&amp;sa=D&amp;source=editors&amp;ust=1746679648930640&amp;usg=AOvVaw3sVK0eqKr_UWWpM680GnV7">https://www.peanutbase.org/search/markers/AhTE333</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.peanutbase.org/search/markers/AhTE810&amp;sa=D&amp;source=editors&amp;ust=1746679648931071&amp;usg=AOvVaw2Iqiat8Sdg2z32xydsfQBV">https://www.peanutbase.org/search/markers/AhTE810</a></span></li><li class="c15 li-bullet-0"><span class="c6">PeanutBase: Home, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.peanutbase.org/&amp;sa=D&amp;source=editors&amp;ust=1746679648931394&amp;usg=AOvVaw3qCV-Q21iI1sQjvry2WC1y">https://www.peanutbase.org/</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patents.google.com/patent/CN112094937A/en&amp;sa=D&amp;source=editors&amp;ust=1746679648931737&amp;usg=AOvVaw30EnEXHUVatzUCbmn9wchU">https://patents.google.com/patent/CN112094937A/en</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patents.google.com/patent/CN112094937A/zh&amp;sa=D&amp;source=editors&amp;ust=1746679648932043&amp;usg=AOvVaw0VLS0Duhl7kTjDUACA4E6E">https://patents.google.com/patent/CN112094937A/zh</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patentscope.wipo.int/search/en/detail.jsf?docId%3DCN324907907&amp;sa=D&amp;source=editors&amp;ust=1746679648932391&amp;usg=AOvVaw0hz-pINaWIYsxLeVFTb1O_">https://patentscope.wipo.int/search/en/detail.jsf?docId=CN324907907</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patents.google.com/patent/CN112094937A/&amp;sa=D&amp;source=editors&amp;ust=1746679648932683&amp;usg=AOvVaw03eXQ1LSvVjj7IW17IsOKn">https://patents.google.com/patent/CN112094937A/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Whole&#8208;genome sequencing based discovery of candidate genes ..., accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://acsess.onlinelibrary.wiley.com/doi/10.1002/tpg2.20265&amp;sa=D&amp;source=editors&amp;ust=1746679648933087&amp;usg=AOvVaw1Ejh1oMfzrSmj2Tjep0cUl">https://acsess.onlinelibrary.wiley.com/doi/10.1002/tpg2.20265</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-Wide Mapping of Quantitative Trait Loci for Yield-Attributing Traits of Peanut - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10888419/&amp;sa=D&amp;source=editors&amp;ust=1746679648933497&amp;usg=AOvVaw3JN-yNFRlaJfeHLLGEjlMe">https://pmc.ncbi.nlm.nih.gov/articles/PMC10888419/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-Wide Mapping of Quantitative Trait Loci for Yield-Attributing ..., accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10888419/&amp;sa=D&amp;source=editors&amp;ust=1746679648933924&amp;usg=AOvVaw1_RpWZktgX-DQ2PZkuxI_U">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10888419/</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-019-2164-5/tables/2&amp;sa=D&amp;source=editors&amp;ust=1746679648934289&amp;usg=AOvVaw3h9xrSl53TSHJ4udo-vgh9">https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-019-2164-5/tables/2</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-019-2164-5&amp;sa=D&amp;source=editors&amp;ust=1746679648934676&amp;usg=AOvVaw2vqi8JR2-3Vmb3NVSPm_zF">https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-019-2164-5</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://static-content.springer.com/esm/art%253A10.1186%252Fs12870-019-2164-5/MediaObjects/12870_2019_2164_MOESM1_ESM.xlsx&amp;sa=D&amp;source=editors&amp;ust=1746679648935098&amp;usg=AOvVaw2LmBc2ZcT6pV6s-a44OZL_">https://static-content.springer.com/esm/art%3A10.1186%2Fs12870-019-2164-5/MediaObjects/12870_2019_2164_MOESM1_ESM.xlsx</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genetic Diversity and Genome-Wide Association Study of Seed Aspect Ratio Using a High-Density SNP Array in Peanut (Arachis hypogaea L.) - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/publication/347853712_Genetic_Diversity_and_Genome-Wide_Association_Study_of_Seed_Aspect_Ratio_Using_a_High-Density_SNP_Array_in_Peanut_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1746679648935742&amp;usg=AOvVaw2YlJr3elGg1dq6k7q2XKz-">https://www.researchgate.net/publication/347853712_Genetic_Diversity_and_Genome-Wide_Association_Study_of_Seed_Aspect_Ratio_Using_a_High-Density_SNP_Array_in_Peanut_Arachis_hypogaea_L</a></span></li><li class="c15 li-bullet-0"><span class="c6">Fine mapping and candidate gene analysis of major QTLs for number of seeds per pod in Arachis hypogaea L - PubMed Central, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11998195/&amp;sa=D&amp;source=editors&amp;ust=1746679648936148&amp;usg=AOvVaw1qa0epFNQ_T8kI4dZdHnKx">https://pmc.ncbi.nlm.nih.gov/articles/PMC11998195/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Details of the candidate reference genes of peanut and their primer sequences used for validation. - ResearchGate, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.researchgate.net/figure/Details-of-the-candidate-reference-genes-of-peanut-and-their-primer-sequences-used-for_tbl1_258117927&amp;sa=D&amp;source=editors&amp;ust=1746679648936687&amp;usg=AOvVaw3gKuKHWYNQagb2igTXkTfj">https://www.researchgate.net/figure/Details-of-the-candidate-reference-genes-of-peanut-and-their-primer-sequences-used-for_tbl1_258117927</a></span></li><li class="c15 li-bullet-0"><span class="c6">Arachis hypogaea Annotation Report - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_hypogaea/100/&amp;sa=D&amp;source=editors&amp;ust=1746679648937091&amp;usg=AOvVaw1wZFwfD0Bw-1F2M_cBVUw5">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_hypogaea/100/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Taxonomy browser (Arachis hypogaea) - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode%3Dinfo%26id%3D3818&amp;sa=D&amp;source=editors&amp;ust=1746679648937543&amp;usg=AOvVaw2XwdlD2RapyoCoT8U82ylw">https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=info&amp;id=3818</a></span></li><li class="c15 li-bullet-0"><span class="c6">Arachis hypogaea genome assembly arahy.Tifrunner.gnm1.KYV3 - NCBI - NLM, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_003086295.2/&amp;sa=D&amp;source=editors&amp;ust=1746679648937965&amp;usg=AOvVaw17JeLl-Pl4lUcdRsj36vzp">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_003086295.2/</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3Darahy.BC5R4P.1%2BArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1746679648938308&amp;usg=AOvVaw1Sj0XwbOp71VSwSCH1b8NM">https://www.ncbi.nlm.nih.gov/gene/?term=arahy.BC5R4P.1+Arachis+hypogaea</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3Darahy.BC5R4P.1&amp;sa=D&amp;source=editors&amp;ust=1746679648938687&amp;usg=AOvVaw2FXaZgvPuk311Krccy6yk7">https://www.ncbi.nlm.nih.gov/gene/?term=arahy.BC5R4P.1</a></span></li><li class="c15 li-bullet-0"><span class="c6">PodPt1-Seed_Pattee8_down - PeanutBase, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.peanutbase.org/files/expression/arahy_atlas_html/PodPt1-Seed_Pattee8_down.html&amp;sa=D&amp;source=editors&amp;ust=1746679648939108&amp;usg=AOvVaw3Jk5jaW6YlMFtGEyQmrIsP">https://www.peanutbase.org/files/expression/arahy_atlas_html/PodPt1-Seed_Pattee8_down.html</a></span></li><li class="c15 li-bullet-0"><span class="c6">Fine-mapping of a QTL and identification of candidate genes associated with the lateral branch angle of peanuts (Arachis hypogaea L.) on chromosome B05 - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11484233/&amp;sa=D&amp;source=editors&amp;ust=1746679648939588&amp;usg=AOvVaw0wEFxasMMNRq4pOw_8Y_q2">https://pmc.ncbi.nlm.nih.gov/articles/PMC11484233/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Identification of Peanut Aux/IAA Genes and Functional Prediction during Seed Development and Maturation - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC8874715/&amp;sa=D&amp;source=editors&amp;ust=1746679648940023&amp;usg=AOvVaw2A9NhI1wAkINjNKGuOxMNF">https://pmc.ncbi.nlm.nih.gov/articles/PMC8874715/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Arachis hypogaea v1.0 - Phytozome, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://phytozome-next.jgi.doe.gov/info/Ahypogaea_v1_0&amp;sa=D&amp;source=editors&amp;ust=1746679648940422&amp;usg=AOvVaw0gbWgP7Kf-e0PIPTZ9lgwS">https://phytozome-next.jgi.doe.gov/info/Ahypogaea_v1_0</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3Darahy.T43K8I.1%2BArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1746679648940806&amp;usg=AOvVaw0uMlLUVCw2FL8PM496w0u2">https://www.ncbi.nlm.nih.gov/gene/?term=arahy.T43K8I.1+Arachis+hypogaea</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3Darahy.T43K8I.1&amp;sa=D&amp;source=editors&amp;ust=1746679648941144&amp;usg=AOvVaw3-iqDJ_L_3_3NJPg8lr_bB">https://www.ncbi.nlm.nih.gov/gene/?term=arahy.T43K8I.1</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-wide mining seed-specific candidate genes from peanut for, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6438489/&amp;sa=D&amp;source=editors&amp;ust=1746679648941542&amp;usg=AOvVaw15Ga0jzZIsA2TkbqgWtFJF">https://pmc.ncbi.nlm.nih.gov/articles/PMC6438489/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-Wide Identification of the Peanut ASR Gene Family and Its Expression Analysis under Abiotic Stress - PubMed, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/?linkname%3Dpubmed_pubmed_citedin%26from_uid%3D39456791&amp;sa=D&amp;source=editors&amp;ust=1746679648942031&amp;usg=AOvVaw07r2JG01kVu1bgf7AvLQZB">https://pubmed.ncbi.nlm.nih.gov/?linkname=pubmed_pubmed_citedin&amp;from_uid=39456791</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-wide identification and expression analysis of auxin response factors in peanut (Arachis hypogaea L.), accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC8542371/&amp;sa=D&amp;source=editors&amp;ust=1746679648942481&amp;usg=AOvVaw3rMLmAd9e0yGCh_MYMOVUQ">https://pmc.ncbi.nlm.nih.gov/articles/PMC8542371/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Proteomic Analysis of Arachis hypogaea Seeds from Different Maturity Classes - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11054812/&amp;sa=D&amp;source=editors&amp;ust=1746679648942896&amp;usg=AOvVaw1Y_u04RcKSmkwnxiOAc8jF">https://pmc.ncbi.nlm.nih.gov/articles/PMC11054812/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Seed maturation protein - Arachis hypogaea (Peanut) | UniProtKB | UniProt, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.uniprot.org/uniprotkb/A0A445CF85/entry&amp;sa=D&amp;source=editors&amp;ust=1746679648943352&amp;usg=AOvVaw2Lj0fWpldYJ8tbtC-6vKvc">https://www.uniprot.org/uniprotkb/A0A445CF85/entry</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3DAraip.GWR7V%2BArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1746679648943726&amp;usg=AOvVaw2W1Aeq5d1ZRzEr2kb1rKcY">https://www.ncbi.nlm.nih.gov/gene/?term=Araip.GWR7V+Arachis+hypogaea</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/?term%3DAraip.GWR7V&amp;sa=D&amp;source=editors&amp;ust=1746679648944088&amp;usg=AOvVaw1r_s_7UGIzzzreCvDGQeUd">https://www.ncbi.nlm.nih.gov/gene/?term=Araip.GWR7V</a></span></li><li class="c15 li-bullet-0"><span class="c6">accessed December 31, 1969, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438489/&amp;sa=D&amp;source=editors&amp;ust=1746679648944407&amp;usg=AOvVaw2Qrp_X6QDASTwd_aSG4Bp8">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438489/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Arachis hypogaea - Ensembl Genomes 60, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://plants.ensembl.org/Arachis_hypogaea/Info/Index&amp;sa=D&amp;source=editors&amp;ust=1746679648944727&amp;usg=AOvVaw0-g6e4uT2iFCgeDFRwmver">https://plants.ensembl.org/Arachis_hypogaea/Info/Index</a></span></li><li class="c15 li-bullet-0"><span class="c6">ARHGEF1 Rho guanine nucleotide exchange factor 1 [Homo sapiens (human)] - Gene - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene?Db%3Dgene%26Cmd%3DDetailsSearch%26Term%3D9138&amp;sa=D&amp;source=editors&amp;ust=1746679648945130&amp;usg=AOvVaw0ZomZDeNh7J1mWxg3eu_8-">https://www.ncbi.nlm.nih.gov/gene?Db=gene&amp;Cmd=DetailsSearch&amp;Term=9138</a></span></li><li class="c15 li-bullet-0"><span class="c6">How to use NCBI nucleotide BLAST tool - YouTube, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3DbXArP_Qm8tw&amp;sa=D&amp;source=editors&amp;ust=1746679648945464&amp;usg=AOvVaw1UG-HsIqrGnb4kbl8vJYPu">https://www.youtube.com/watch?v=bXArP_Qm8tw</a></span></li><li class="c15 li-bullet-0"><span class="c6">Arachis hypogaea - Search results - Genome - Assembly - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/assembly/?term%3DArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1746679648945897&amp;usg=AOvVaw3QaqHsi-BqWfKIpchIBRpd">https://www.ncbi.nlm.nih.gov/assembly/?term=Arachis%20hypogaea</a></span></li><li class="c15 li-bullet-0"><span class="c6">Home - Nucleotide - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/nucleotide/&amp;sa=D&amp;source=editors&amp;ust=1746679648946204&amp;usg=AOvVaw2MtTPDouj-ctd8KIGtNOmo">https://www.ncbi.nlm.nih.gov/nucleotide/</a></span></li><li class="c15 li-bullet-0"><span class="c6">1430079646 - Nucleotide Result - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/nuccore/MG814008&amp;sa=D&amp;source=editors&amp;ust=1746679648946571&amp;usg=AOvVaw3m7XvmNmfgcSZr0v4OZXgy">https://www.ncbi.nlm.nih.gov/nuccore/MG814008</a></span></li><li class="c15 li-bullet-0"><span class="c6">1430078966 - Nucleotide Result - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/nuccore/MG814007&amp;sa=D&amp;source=editors&amp;ust=1746679648946890&amp;usg=AOvVaw1ragZo21G_9qSuGrYGAZut">https://www.ncbi.nlm.nih.gov/nuccore/MG814007</a></span></li><li class="c15 li-bullet-0"><span class="c6">Yeast genome-wide drug-induced haploinsufficiency screen to determine drug mode of action - PubMed, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/15070751/&amp;sa=D&amp;source=editors&amp;ust=1746679648947237&amp;usg=AOvVaw1XFaWDjCqTgV0bkRHTiWPT">https://pubmed.ncbi.nlm.nih.gov/15070751/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Variant-to-function analysis of the childhood obesity chr12q13 locus implicates rs7132908 as a causal variant within the 3&prime; UTR of FAIM2 - PubMed Central, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11099382/&amp;sa=D&amp;source=editors&amp;ust=1746679648947663&amp;usg=AOvVaw0FQP71_I68cETh30tKuu26">https://pmc.ncbi.nlm.nih.gov/articles/PMC11099382/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Searching NCBI&#39;s dbSNP database - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC3078622/&amp;sa=D&amp;source=editors&amp;ust=1746679648947988&amp;usg=AOvVaw0gLHmWBrRXDN1KKSVDMQMh">https://pmc.ncbi.nlm.nih.gov/articles/PMC3078622/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Single Nucleotide Polymorphism (SNP) - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/projects/SNP/redirect_search.html&amp;sa=D&amp;source=editors&amp;ust=1746679648948346&amp;usg=AOvVaw1RqC5ojKzvpiaHMZJRGO1Q">https://www.ncbi.nlm.nih.gov/projects/SNP/redirect_search.html</a></span></li><li class="c15 li-bullet-0"><span class="c6">Genome-wide SNP Genotyping Resolves Signatures of Selection and Tetrasomic Recombination in Peanut - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5315502/&amp;sa=D&amp;source=editors&amp;ust=1746679648948743&amp;usg=AOvVaw3421LGogkOjM8jdVlsbFCG">https://pmc.ncbi.nlm.nih.gov/articles/PMC5315502/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Relics of interspecific hybridization retained in the genome of a drought-adapted peanut cultivar - PMC, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11540320/&amp;sa=D&amp;source=editors&amp;ust=1746679648949149&amp;usg=AOvVaw0wapJiS4qd0LFsAnC7yooy">https://pmc.ncbi.nlm.nih.gov/articles/PMC11540320/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Home - SRA - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/sra&amp;sa=D&amp;source=editors&amp;ust=1746679648949432&amp;usg=AOvVaw1VeexPlLrEfZAbvBKpvXZJ">https://www.ncbi.nlm.nih.gov/sra</a></span></li><li class="c15 li-bullet-0"><span class="c6">The Sequence Read Archive (SRA) - NCBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/sra/docs/&amp;sa=D&amp;source=editors&amp;ust=1746679648949774&amp;usg=AOvVaw0AfmwN3e05CDvR5Wh76B4i">https://www.ncbi.nlm.nih.gov/sra/docs/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Marker-Assisted Selection - PeanutBase, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.peanutbase.org/mas/&amp;sa=D&amp;source=editors&amp;ust=1746679648950057&amp;usg=AOvVaw39iAKpv1jyfWIjCWzFXrHt">https://www.peanutbase.org/mas/</a></span></li><li class="c15 li-bullet-0"><span class="c6">PeanutBase - FAIRsharing, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://fairsharing.org/FAIRsharing.fYFurb&amp;sa=D&amp;source=editors&amp;ust=1746679648950359&amp;usg=AOvVaw0RIkoXNQn8UxdOHiEKIa_k">https://fairsharing.org/FAIRsharing.fYFurb</a></span></li><li class="c15 li-bullet-0"><span class="c6">EP3572523A1 - Novel carboxyesterhydrolases - Google Patents, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patents.google.com/patent/EP3572523A1/en&amp;sa=D&amp;source=editors&amp;ust=1746679648950773&amp;usg=AOvVaw3XWBurkqlJbKlRha_6awoN">https://patents.google.com/patent/EP3572523A1/en</a></span></li><li class="c15 li-bullet-0"><span class="c6">What is a Sequence Identifier (SEQ ID NO) and how is it used in patent applications?, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://blueironip.com/ufaqs/what-is-a-sequence-identifier-seq-id-no-and-how-is-it-used-in-patent-applications/&amp;sa=D&amp;source=editors&amp;ust=1746679648951294&amp;usg=AOvVaw2w6mDkPoDnAi3Oh_M-IW-i">https://blueironip.com/ufaqs/what-is-a-sequence-identifier-seq-id-no-and-how-is-it-used-in-patent-applications/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Search Sequence Listings - Patentscope - WIPO, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://patentscope.wipo.int/search/en/sequences.jsf&amp;sa=D&amp;source=editors&amp;ust=1746679648951699&amp;usg=AOvVaw2t8-rE1WRVWadUhxXPN_Jk">https://patentscope.wipo.int/search/en/sequences.jsf</a></span></li><li class="c15 li-bullet-0"><span class="c6">Where can I search Chinese patents in English? | epo.org, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.epo.org/en/service-support/faq/searching-patents/asian-patent-information/china/sources-information/where-can&amp;sa=D&amp;source=editors&amp;ust=1746679648952213&amp;usg=AOvVaw2pzgrj0Dt15pHmud4ZfWl4">https://www.epo.org/en/service-support/faq/searching-patents/asian-patent-information/china/sources-information/where-can</a></span></li><li class="c15 li-bullet-0"><span class="c6">Espacenet - WIPO Inspire, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://inspire.wipo.int/espacenet&amp;sa=D&amp;source=editors&amp;ust=1746679648952532&amp;usg=AOvVaw2bOHF502zgFtZL1wtGFqEj">https://inspire.wipo.int/espacenet</a></span></li><li class="c15 li-bullet-0"><span class="c6">ESPACENET: Patent database reference in Europe - OVTT, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.ovtt.org/en/resources/espacenet-patent-database-reference/&amp;sa=D&amp;source=editors&amp;ust=1746679648952907&amp;usg=AOvVaw13BDsMo-BlVatPxJ37tyJW">https://www.ovtt.org/en/resources/espacenet-patent-database-reference/</a></span></li><li class="c15 li-bullet-0"><span class="c6">Patentscope - Wikipedia, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://en.wikipedia.org/wiki/Patentscope&amp;sa=D&amp;source=editors&amp;ust=1746679648953248&amp;usg=AOvVaw0ypNZwCu4RsQ7uqZKHUkne">https://en.wikipedia.org/wiki/Patentscope</a></span></li><li class="c15 li-bullet-0"><span class="c6">PATENTSCOPE - WIPO Inspire, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://inspire.wipo.int/patentscope&amp;sa=D&amp;source=editors&amp;ust=1746679648953566&amp;usg=AOvVaw1SVvgp0Nu3Jy1wYtPs8qsc">https://inspire.wipo.int/patentscope</a></span></li><li class="c15 li-bullet-0"><span class="c6">SureChEMBL 2.0 is here | EMBL-EBI, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://www.embl.org/news/updates-from-data-resources/surechembl-2-0-is-here/&amp;sa=D&amp;source=editors&amp;ust=1746679648953940&amp;usg=AOvVaw2RADSFxwc2j0_ZbSSjASBK">https://www.embl.org/news/updates-from-data-resources/surechembl-2-0-is-here/</a></span></li><li class="c15 li-bullet-0"><span class="c6">SureChEMBL | re3data.org, accessed May 7, 2025, </span><span class="c1"><a class="c11" href="https://www.google.com/url?q=https://re3data.org/repository/r3d100011037&amp;sa=D&amp;source=editors&amp;ust=1746679648954223&amp;usg=AOvVaw1gzZXQ6zGmsc1rls6R1IqS">https://re3data.org/repository/r3d100011037</a></span></li></ol></body></html>