<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);.lst-kix_list_c-0>li:before{content:"\0025cf   "}ul.lst-kix_list_b-8{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_c-1>li:before{content:"\0025cb   "}ul.lst-kix_list_b-3{list-style-type:none}ul.lst-kix_list_b-2{list-style-type:none}.lst-kix_list_c-3>li:before{content:"\0025a0   "}.lst-kix_list_c-4>li:before{content:"\0025a0   "}ul.lst-kix_list_b-1{list-style-type:none}ul.lst-kix_list_b-0{list-style-type:none}ul.lst-kix_list_b-7{list-style-type:none}ul.lst-kix_list_b-6{list-style-type:none}.lst-kix_list_c-2>li:before{content:"\0025a0   "}ul.lst-kix_list_b-5{list-style-type:none}ul.lst-kix_list_b-4{list-style-type:none}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}.lst-kix_list_11-0>li{counter-increment:lst-ctn-kix_list_11-0}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_9-0{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025a0   "}ul.lst-kix_list_a-4{list-style-type:none}ul.lst-kix_list_a-3{list-style-type:none}.lst-kix_list_5-2>li:before{content:"\0025a0   "}ul.lst-kix_list_a-2{list-style-type:none}ul.lst-kix_list_a-1{list-style-type:none}.lst-kix_list_5-1>li:before{content:"\0025cb   "}ul.lst-kix_list_a-8{list-style-type:none}ul.lst-kix_list_a-7{list-style-type:none}ul.lst-kix_list_a-6{list-style-type:none}ul.lst-kix_list_a-5{list-style-type:none}.lst-kix_list_5-7>li:before{content:"\0025a0   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025a0   "}.lst-kix_list_5-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_d-6>li:before{content:"\0025a0   "}.lst-kix_list_5-4>li:before{content:"\0025a0   "}.lst-kix_list_5-5>li:before{content:"\0025a0   "}ul.lst-kix_list_8-0{list-style-type:none}.lst-kix_list_d-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_d-7>li:before{content:"\0025a0   "}.lst-kix_list_d-2>li:before{content:"\0025a0   "}.lst-kix_list_6-1>li:before{content:"\0025cb   "}.lst-kix_list_6-3>li:before{content:"\0025a0   "}.lst-kix_list_d-1>li:before{content:"\0025cb   "}.lst-kix_list_d-5>li:before{content:"\0025a0   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025a0   "}.lst-kix_list_d-4>li:before{content:"\0025a0   "}.lst-kix_list_d-3>li:before{content:"\0025a0   "}.lst-kix_list_6-2>li:before{content:"\0025a0   "}.lst-kix_list_c-7>li:before{content:"\0025a0   "}ul.lst-kix_list_a-0{list-style-type:none}.lst-kix_list_c-5>li:before{content:"\0025a0   "}.lst-kix_list_c-8>li:before{content:"\0025a0   "}.lst-kix_list_6-8>li:before{content:"\0025a0   "}.lst-kix_list_6-5>li:before{content:"\0025a0   "}.lst-kix_list_6-7>li:before{content:"\0025a0   "}.lst-kix_list_d-0>li:before{content:"\0025cf   "}.lst-kix_list_c-6>li:before{content:"\0025a0   "}.lst-kix_list_6-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_d-8{list-style-type:none}ul.lst-kix_list_d-7{list-style-type:none}.lst-kix_list_7-4>li:before{content:"\0025a0   "}.lst-kix_list_7-6>li:before{content:"\0025a0   "}ul.lst-kix_list_d-6{list-style-type:none}.lst-kix_list_2-5>li:before{content:"\0025a0   "}ul.lst-kix_list_d-1{list-style-type:none}ul.lst-kix_list_d-0{list-style-type:none}.lst-kix_list_b-8>li:before{content:"\0025a0   "}.lst-kix_list_7-2>li:before{content:"\0025a0   "}ul.lst-kix_list_d-5{list-style-type:none}ul.lst-kix_list_d-4{list-style-type:none}ul.lst-kix_list_d-3{list-style-type:none}ul.lst-kix_list_d-2{list-style-type:none}.lst-kix_list_b-4>li:before{content:"\0025a0   "}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}.lst-kix_list_10-1>li:before{content:"\0025cb   "}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}.lst-kix_list_7-8>li:before{content:"\0025a0   "}ul.lst-kix_list_3-0{list-style-type:none}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}.lst-kix_list_b-6>li:before{content:"\0025a0   "}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}.lst-kix_list_10-7>li:before{content:"\0025a0   "}.lst-kix_list_10-5>li:before{content:"\0025a0   "}.lst-kix_list_10-3>li:before{content:"\0025a0   "}ul.lst-kix_list_11-7{list-style-type:none}ul.lst-kix_list_11-6{list-style-type:none}.lst-kix_list_b-0>li:before{content:"\0025cf   "}.lst-kix_list_4-1>li:before{content:"\0025cb   "}ul.lst-kix_list_11-5{list-style-type:none}ul.lst-kix_list_11-4{list-style-type:none}ul.lst-kix_list_11-3{list-style-type:none}ul.lst-kix_list_11-2{list-style-type:none}.lst-kix_list_b-2>li:before{content:"\0025a0   "}ul.lst-kix_list_11-1{list-style-type:none}.lst-kix_list_9-2>li:before{content:"\0025a0   "}.lst-kix_list_4-3>li:before{content:"\0025a0   "}.lst-kix_list_4-5>li:before{content:"\0025a0   "}ul.lst-kix_list_11-8{list-style-type:none}.lst-kix_list_9-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-8{list-style-type:none}ul.lst-kix_list_c-7{list-style-type:none}.lst-kix_list_9-6>li:before{content:"\0025a0   "}ul.lst-kix_list_c-2{list-style-type:none}ul.lst-kix_list_c-1{list-style-type:none}ul.lst-kix_list_c-0{list-style-type:none}.lst-kix_list_9-4>li:before{content:"\0025a0   "}ul.lst-kix_list_c-6{list-style-type:none}.lst-kix_list_a-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-5{list-style-type:none}.lst-kix_list_11-3>li:before{content:"\0025a0   "}ul.lst-kix_list_c-4{list-style-type:none}ul.lst-kix_list_c-3{list-style-type:none}ul.lst-kix_list_2-8{list-style-type:none}.lst-kix_list_11-5>li:before{content:"\0025a0   "}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_9-8>li:before{content:"\0025a0   "}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}ul.lst-kix_list_2-7{list-style-type:none}.lst-kix_list_11-7>li:before{content:"\0025a0   "}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}ul.lst-kix_list_10-0{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}ul.lst-kix_list_10-8{list-style-type:none}ul.lst-kix_list_10-7{list-style-type:none}.lst-kix_list_1-7>li:before{content:"\0025a0   "}ul.lst-kix_list_10-6{list-style-type:none}ul.lst-kix_list_10-5{list-style-type:none}ul.lst-kix_list_10-4{list-style-type:none}ul.lst-kix_list_10-3{list-style-type:none}.lst-kix_list_1-5>li:before{content:"\0025a0   "}ul.lst-kix_list_10-2{list-style-type:none}ul.lst-kix_list_10-1{list-style-type:none}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_list_2-3>li:before{content:"\0025a0   "}ul.lst-kix_list_f-7{list-style-type:none}ul.lst-kix_list_f-6{list-style-type:none}ul.lst-kix_list_f-5{list-style-type:none}ul.lst-kix_list_f-4{list-style-type:none}ul.lst-kix_list_f-8{list-style-type:none}ul.lst-kix_list_f-3{list-style-type:none}.lst-kix_list_3-0>li:before{content:"\0025cf   "}ul.lst-kix_list_f-2{list-style-type:none}ul.lst-kix_list_f-1{list-style-type:none}ul.lst-kix_list_f-0{list-style-type:none}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}ul.lst-kix_list_5-5{list-style-type:none}ol.lst-kix_list_11-0{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}.lst-kix_list_8-1>li:before{content:"\0025cb   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}ul.lst-kix_list_5-0{list-style-type:none}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_8-7>li:before{content:"\0025a0   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_list_f-6>li:before{content:"\0025a0   "}.lst-kix_list_f-8>li:before{content:"\0025a0   "}.lst-kix_list_8-5>li:before{content:"\0025a0   "}.lst-kix_list_a-7>li:before{content:"\0025a0   "}.lst-kix_list_8-6>li:before{content:"\0025a0   "}.lst-kix_list_f-7>li:before{content:"\0025a0   "}.lst-kix_list_8-3>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_list_f-4>li:before{content:"\0025a0   "}.lst-kix_list_3-7>li:before{content:"\0025a0   "}.lst-kix_list_a-8>li:before{content:"\0025a0   "}.lst-kix_list_8-4>li:before{content:"\0025a0   "}.lst-kix_list_f-5>li:before{content:"\0025a0   "}.lst-kix_list_11-2>li:before{content:"\0025a0   "}.lst-kix_list_a-1>li:before{content:"\0025cb   "}.lst-kix_list_a-2>li:before{content:"\0025a0   "}.lst-kix_list_f-0>li:before{content:"\0025cf   "}.lst-kix_list_a-3>li:before{content:"\0025a0   "}.lst-kix_list_11-1>li:before{content:"\0025cb   "}.lst-kix_list_f-3>li:before{content:"\0025a0   "}.lst-kix_list_a-5>li:before{content:"\0025a0   "}.lst-kix_list_a-6>li:before{content:"\0025a0   "}.lst-kix_list_f-2>li:before{content:"\0025a0   "}.lst-kix_list_11-0>li:before{content:"" counter(lst-ctn-kix_list_11-0,decimal) ". "}.lst-kix_list_a-4>li:before{content:"\0025a0   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}.lst-kix_list_f-1>li:before{content:"\0025cb   "}.lst-kix_list_e-5>li:before{content:"\0025a0   "}ul.lst-kix_list_e-8{list-style-type:none}ul.lst-kix_list_e-7{list-style-type:none}ul.lst-kix_list_e-6{list-style-type:none}ul.lst-kix_list_e-5{list-style-type:none}.lst-kix_list_e-3>li:before{content:"\0025a0   "}.lst-kix_list_e-7>li:before{content:"\0025a0   "}.lst-kix_list_e-2>li:before{content:"\0025a0   "}.lst-kix_list_e-6>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-0{list-style-type:none}.lst-kix_list_4-7>li:before{content:"\0025a0   "}ul.lst-kix_list_e-4{list-style-type:none}ul.lst-kix_list_e-3{list-style-type:none}ul.lst-kix_list_e-2{list-style-type:none}.lst-kix_list_e-4>li:before{content:"\0025a0   "}ul.lst-kix_list_e-1{list-style-type:none}ul.lst-kix_list_4-8{list-style-type:none}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_4-0{list-style-type:none}.lst-kix_list_e-1>li:before{content:"\0025cb   "}ul.lst-kix_list_4-1{list-style-type:none}ul.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}.lst-kix_list_e-0>li:before{content:"\0025cf   "}ul.lst-kix_list_4-3{list-style-type:none}ol.lst-kix_list_11-0.start{counter-reset:lst-ctn-kix_list_11-0 0}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}.lst-kix_list_7-1>li:before{content:"\0025cb   "}.lst-kix_list_7-5>li:before{content:"\0025a0   "}.lst-kix_list_7-3>li:before{content:"\0025a0   "}ul.lst-kix_list_7-5{list-style-type:none}.lst-kix_list_10-0>li:before{content:"\0025cf   "}ul.lst-kix_list_7-6{list-style-type:none}.lst-kix_list_b-5>li:before{content:"\0025a0   "}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}.lst-kix_list_b-3>li:before{content:"\0025a0   "}.lst-kix_list_b-7>li:before{content:"\0025a0   "}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7>li:before{content:"\0025a0   "}.lst-kix_list_10-4>li:before{content:"\0025a0   "}.lst-kix_list_10-8>li:before{content:"\0025a0   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}.lst-kix_list_b-1>li:before{content:"\0025cb   "}.lst-kix_list_10-2>li:before{content:"\0025a0   "}.lst-kix_list_4-4>li:before{content:"\0025a0   "}.lst-kix_list_4-2>li:before{content:"\0025a0   "}.lst-kix_list_4-6>li:before{content:"\0025a0   "}.lst-kix_list_9-3>li:before{content:"\0025a0   "}.lst-kix_list_10-6>li:before{content:"\0025a0   "}.lst-kix_list_9-1>li:before{content:"\0025cb   "}.lst-kix_list_e-8>li:before{content:"\0025a0   "}.lst-kix_list_9-7>li:before{content:"\0025a0   "}.lst-kix_list_11-4>li:before{content:"\0025a0   "}.lst-kix_list_9-5>li:before{content:"\0025a0   "}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_11-6>li:before{content:"\0025a0   "}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_6-2{list-style-type:none}.lst-kix_list_11-8>li:before{content:"\0025a0   "}ul.lst-kix_list_6-3{list-style-type:none}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_6-0{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"\0025cf   "}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c6{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:78pt;border-top-color:#000000;border-bottom-style:solid}.c16{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:66.9pt;border-top-color:#000000;border-bottom-style:solid}.c4{color:#1b1c1d;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:15pt;font-family:"Google Sans";font-style:normal}.c8{color:#575b5f;font-weight:400;text-decoration:none;vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal}.c2{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c12{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c7{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;vertical-align:baseline;text-decoration-skip-ink:none;font-style:normal}.c24{color:#1b1c1d;font-weight:700;font-size:16pt;font-family:"Google Sans";font-style:normal}.c27{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c30{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c32{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c10{color:#1b1c1d;font-weight:400;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c29{color:#1b1c1d;font-weight:700;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c1{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c0{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c17{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c23{color:#000000;font-weight:400;font-size:11pt;font-family:"Google Sans Text";font-style:normal}.c31{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c11{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c25{border-spacing:0;border-collapse:collapse;margin-right:auto}.c15{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c21{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c34{padding-top:0pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c14{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c33{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c18{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c13{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left}.c5{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c28{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c3{color:inherit;text-decoration:inherit}.c20{margin-left:23.2pt;padding-left:0pt}.c9{text-decoration:none;vertical-align:baseline}.c19{padding:0;margin:0}.c26{height:11pt}.c22{height:0pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c28 doc-content"><h1 class="c15"><span class="c24">Genetic Determinants of Seed Size in Cultivated Peanut (</span><span class="c32">Arachis hypogaea</span><span class="c9 c24">&nbsp;L.): A Synthesis of QTL Mapping and Candidate Gene Analyses</span></h1><h2 class="c15"><span class="c4">I. Introduction</span></h2><h3 class="c15"><span class="c27">A. Significance of Peanut (</span><span class="c31">Arachis hypogaea</span><span class="c27 c9">) and Seed Size</span></h3><p class="c18"><span class="c0">Cultivated peanut (</span><span class="c1">Arachis hypogaea</span><span class="c0">&nbsp;L.) stands as a crop of global agricultural and economic significance. It serves as a vital source of edible oil and high-quality plant protein, contributing substantially to human nutrition and food security, particularly across Asia and Sub-Saharan Africa where it is a source of livelihood for millions of smallholder farmers.</span><span class="c12">1</span><span class="c0">&nbsp;The seeds are rich in oil (typically 48&ndash;50%), protein (25&ndash;28%), along with essential vitamins, minerals, and bioactive compounds, making peanut a valuable component of diets worldwide.</span><span class="c8">2</span></p><p class="c18"><span class="c0">Seed size, encompassing traits such as seed weight (often measured as hundred-seed weight, HSW, or hundred-kernel weight, HKW), seed length (SL), and seed width (SW), is a paramount characteristic in peanut production and breeding.</span><span class="c12">10</span><span class="c0">&nbsp;It directly impacts overall grain yield, influences harvesting efficiency, and dictates the suitability of cultivars for specific market classes and end uses.</span><span class="c12">14</span><span class="c0">&nbsp;Modern peanut cultivars are categorized into market types like Runner, Virginia, Spanish, and Valencia, which are distinguished primarily by seed characteristics, including size.</span><span class="c12">27</span><span class="c0">&nbsp;For instance, large-seeded Virginia types are preferred for in-shell consumption and gourmet snacks, whereas smaller-seeded Spanish and Valencia types are often utilized in confectionery, peanut butter production, or oil extraction.</span><span class="c12">23</span><span class="c0">&nbsp;This distinction underscores the complex breeding objectives in peanut improvement; optimizing seed size is not merely about increasing bulk yield but also about tailoring seed dimensions to meet specific quality standards and consumer preferences.</span><span class="c12">27</span><span class="c0">&nbsp;Historical evidence suggests that larger pods and seeds were likely targets during the early domestication of peanut, as they offered advantages in yield, harvesting ease, and potentially enhanced adaptability by providing more space and nutrients for the developing seed.</span><span class="c12">12</span><span class="c0">&nbsp;Consequently, understanding and manipulating the genetic factors controlling seed size remains a central goal in contemporary peanut breeding programs aiming to enhance yield potential and meet diverse market demands.</span><span class="c8">10</span></p><h3 class="c15"><span class="c27 c9">B. Genetic Complexity of Peanut Seed Size</span></h3><p class="c18"><span class="c0">The genetic dissection of seed size in peanut is inherently complex, owing to the unique evolutionary history and genomic architecture of the species. Cultivated peanut is a relatively recent allotetraploid (2n = 4x = 40), possessing A and B subgenomes (AABB) that originated from a single hybridization event between two wild diploid ancestors, </span><span class="c1">Arachis duranensis</span><span class="c0">&nbsp;(AA, 2n = 20) and </span><span class="c1">Arachis ipaensis</span><span class="c0">&nbsp;(BB, 2n = 20).</span><span class="c12">1</span><span class="c0">&nbsp;This polyploidization event, coupled with subsequent domestication, created a significant genetic bottleneck, resulting in markedly narrow genetic diversity within the cultivated gene pool compared to its wild relatives.</span><span class="c8">1</span></p><p class="c18"><span class="c0">Seed size itself is recognized as a classic quantitative trait, meaning it is controlled by the cumulative action of multiple genes (polygenic inheritance), each potentially having a small effect, and is significantly modulated by environmental conditions.</span><span class="c12">21</span><span class="c0">&nbsp;This genotype-by-environment (GxE) interaction necessitates evaluating traits across multiple locations and years to obtain reliable genetic estimates.</span><span class="c12">4</span><span class="c0">&nbsp;Early genetic studies on pod and seed size inheritance produced conflicting results, with reports suggesting dominance of large seeds, dominance of small seeds, or control by varying numbers of genes (from one to five).</span><span class="c12">25</span><span class="c0">&nbsp;These discrepancies likely arose from the use of different parental lines, insufficient population sizes to resolve complex inheritance patterns, and the confounding influences of polyploidy (gene redundancy and interaction between homoeologous genes), as well as potential cytoplasmic or maternal effects on seed development.</span><span class="c12">25</span><span class="c0">&nbsp;Despite these historical ambiguities, more recent studies employing robust quantitative genetic analyses consistently indicate that seed size traits exhibit quantitative inheritance, with additive gene action playing a significant role, suggesting that selection for desired seed sizes can be effective, albeit challenging.</span><span class="c8">21</span></p><p class="c18"><span class="c0">Further complicating genetic analysis is the large size of the peanut genome (approximately 2.7 to 2.8 Gb, comparable to the human genome) and the high degree of sequence similarity between the A and B subgenomes, stemming from their recent shared ancestry.</span><span class="c12">2</span><span class="c0">&nbsp;This similarity makes it difficult to develop subgenome-specific markers and accurately map genes to their respective homoeologous chromosomes. These combined factors&mdash;allotetraploidy, narrow genetic base, quantitative inheritance, GxE interaction, and a large, repetitive genome&mdash;render the identification of specific causal genes for seed size a formidable task. Overcoming these challenges requires sophisticated methodologies, including high-density genetic mapping, genome-wide association studies (GWAS), comparative transcriptomics, and the analysis of induced or natural mutants, often demanding large experimental populations and multi-environment trials to robustly detect and validate genetic loci.</span><span class="c8">2</span></p><h3 class="c15"><span class="c27 c9">C. Overview of Methodologies Employed</span></h3><p class="c18"><span class="c0">Researchers have employed a multifaceted array of molecular and genetic techniques to unravel the genetic basis of peanut seed size. Quantitative Trait Locus (QTL) mapping has been extensively utilized as a primary tool.</span><span class="c12">1</span><span class="c0">&nbsp;This approach involves creating segregating populations (such as Recombinant Inbred Lines (RILs), backcross populations, Chromosome Segment Substitution Lines (CSSLs), or diverse germplasm collections) and genotyping them with molecular markers to identify statistical associations between marker loci and phenotypic variation in seed size traits (HSW, SL, SW, and correlated pod traits).</span><span class="c12">2</span><span class="c0">&nbsp;The marker technologies used have evolved from Simple Sequence Repeats (SSRs) to higher-density platforms like Single Nucleotide Polymorphism (SNP) arrays and sequencing-based genotyping methods (e.g., Specific Locus Amplified Fragment sequencing (SLAF-seq), whole-genome re-sequencing), which allow for the construction of more saturated genetic maps and consequently, more precise localization of QTLs.</span><span class="c8">2</span></p><p class="c18"><span class="c0">Complementing QTL mapping, transcriptomics, primarily through RNA sequencing (RNA-Seq), has emerged as a powerful approach to identify genes actively involved in seed development.</span><span class="c12">10</span><span class="c0">&nbsp;By comparing gene expression profiles across different seed developmental stages (e.g., 15, 30, 45, 60 Days Post Flowering/Pegging - DPF/DAP) and between genotypes exhibiting contrasting seed size phenotypes (e.g., large vs. small seeded cultivars, cultivated vs. wild accessions, or wild-type vs. mutants), researchers can identify Differentially Expressed Genes (DEGs) potentially regulating seed growth and maturation.</span><span class="c12">10</span><span class="c0">&nbsp;Advanced analyses like Weighted Gene Co-expression Network Analysis (WGCNA) further help in identifying modules of co-expressed genes and key &#39;hub&#39; genes within these networks that are strongly correlated with the phenotype.</span><span class="c8">11</span></p><p class="c18"><span class="c0">The study of induced mutants has also provided valuable insights. By creating and characterizing mutants with altered seed size (e.g., the </span><span class="c1">ssm1</span><span class="c0">&nbsp;small seed mutant, or </span><span class="c1">hybs</span><span class="c0">&nbsp;&#39;big seed&#39; mutants generated via irradiation), researchers can perform comparative genomic and transcriptomic analyses against the wild-type background to pinpoint specific genes whose disruption leads to the altered phenotype.</span><span class="c8">10</span></p><p class="c18"><span class="c0">Finally, a candidate gene approach is often integrated with these methods. This involves leveraging functional knowledge from well-studied model organisms like </span><span class="c1">Arabidopsis thaliana</span><span class="c0">&nbsp;or rice (</span><span class="c1">Oryza sativa</span><span class="c0">) to investigate the roles of homologous genes in peanut.</span><span class="c12">15</span><span class="c0">&nbsp;Candidate genes are frequently proposed based on their physical location within mapped QTL intervals, especially those narrowed down by fine-mapping, combined with supporting evidence from differential expression data, sequence variations (SNPs, insertions/deletions - Indels) between contrasting genotypes, and functional annotations suggesting a role in relevant biological processes like cell division, hormone signaling, or metabolism.</span><span class="c12">9</span><span class="c0">&nbsp;The availability of reference genome sequences for </span><span class="c1">A. hypogaea</span><span class="c0">&nbsp;and its diploid progenitors (</span><span class="c1">A. duranensis</span><span class="c0">, </span><span class="c1">A. ipaensis</span><span class="c0">), along with curated databases like PeanutBase and NCBI Gene, are indispensable resources underpinning these investigations.</span><span class="c8">8</span></p><h3 class="c15"><span class="c27 c9">D. Report Aim and Scope</span></h3><p class="c18"><span class="c0">This report aims to comprehensively synthesize the current understanding of the genetic control of seed size traits (weight, length, width) in cultivated peanut (</span><span class="c1">Arachis hypogaea</span><span class="c0">), drawing exclusively upon the findings presented in the provided collection of research documentation.</span><span class="c12">1</span><span class="c0 c9">&nbsp;The primary objective is to compile a detailed inventory of identified Quantitative Trait Loci (QTLs) and putative candidate genes implicated in regulating peanut seed size. For each identified genetic factor, the report will critically evaluate the supporting evidence derived from diverse methodological approaches, including QTL mapping, transcriptomic profiling, mutant analyses, and sequence variation studies. The scope is strictly limited to the information contained within the provided materials, presented in a technically rigorous and objective manner suitable for a scientific audience comprising researchers, breeders, and students in plant genetics and related fields.</span></p><h2 class="c15"><span class="c4">II. Major Quantitative Trait Loci (QTLs) Governing Peanut Seed Size</span></h2><h3 class="c15"><span class="c27 c9">A. Overview of QTL Mapping Efforts</span></h3><p class="c18"><span class="c0">Quantitative Trait Locus (QTL) mapping has been a cornerstone methodology in peanut genetics for dissecting complex traits like seed size, despite the inherent challenges posed by the crop&#39;s narrow genetic base and allotetraploid nature.</span><span class="c12">1</span><span class="c0">&nbsp;This approach is instrumental in identifying specific genomic regions that contribute to phenotypic variation, estimating the magnitude of genetic effects, elucidating the mode of gene action (additive, dominant, or epistatic interactions), and uncovering genetic correlations between different traits.</span><span class="c12">1</span><span class="c0">&nbsp;Seed size, encompassing hundred-seed weight (HSW), seed length (SL), and seed width (SW), along with highly correlated pod size and weight traits, has been the focus of numerous QTL studies utilizing diverse genetic materials, including RIL populations, backcross populations, CSSLs derived from wild relatives, and germplasm association panels.</span><span class="c12">2</span><span class="c0">&nbsp;Collectively, these efforts have led to the reporting of over 300 QTLs associated with various yield-related traits, with a substantial portion (~250) linked specifically to seed and/or pod size and weight phenotypes, distributed across the peanut genome.</span><span class="c8">10</span></p><p class="c18"><span class="c0">The resolution and precision of QTL mapping in peanut have significantly advanced with the development and application of progressively higher-density molecular marker systems. Early studies often relied on SSR markers, while more recent investigations leverage SNP arrays (e.g., Axiom_Arachis 58K array) and high-throughput sequencing technologies like SLAF-seq and whole-genome re-sequencing.</span><span class="c12">2</span><span class="c0">&nbsp;These advanced techniques enable the construction of highly saturated genetic maps, facilitating the localization of QTLs to much smaller genomic intervals, sometimes down to kilobase resolution, which is a critical step towards identifying the underlying causal genes.</span><span class="c8">24</span></p><p class="c18"><span class="c0">The large number of reported QTLs associated with seed size underscores its complex, polygenic nature, where many genes likely contribute to the final phenotype.</span><span class="c12">10</span><span class="c0">&nbsp;However, a crucial observation emerging from the collective body of research is the recurrent identification of QTLs within specific chromosomal regions across independent studies, diverse genetic backgrounds, and varying environmental conditions. This convergence, particularly evident with the enhanced resolution offered by modern mapping techniques, strongly suggests that certain genomic loci harbor genes with major and stable effects on peanut seed size. These consistently identified regions, often referred to as QTL hotspots or clusters, represent high-priority targets for further investigation aimed at gene discovery and for application in marker-assisted breeding programs.</span><span class="c8">2</span></p><h3 class="c15"><span class="c27 c9">B. Key Chromosomal Regions and Specific QTLs</span></h3><p class="c18"><span class="c0 c9">Analysis of the available literature reveals several key chromosomal regions consistently associated with major QTLs for peanut seed size traits.</span></p><p class="c13"><span class="c23 c9">1. Chromosome A05: A Major Hub for Seed Size Control</span></p><p class="c13"><span class="c23 c9">Chromosome A05 has emerged as arguably the most significant genomic region influencing seed and pod size in cultivated peanut. Numerous independent studies, employing different populations (RILs, germplasm collections, interspecific crosses) and marker systems, have consistently mapped major QTLs for HSW, SL, SW, and pod dimensions to this chromosome.14 It has been explicitly described as a QTL enrichment region or hotspot.24 Specific mapping studies provide details:</span></p><ul class="c19 lst-kix_list_1-0 start"><li class="c15 c20 li-bullet-0"><span class="c0">One study using advanced generation RILs identified multiple co-localizing QTLs for HSW, SL, and SW within the approximate genetic interval of 19-52 cM on A05. Notably, QTLs like </span><span class="c1">qHSWA05.2</span><span class="c0">&nbsp;(PVE 32.20%) and </span><span class="c1">qSLA05.2</span><span class="c0">&nbsp;(PVE 40.73%) explained a very large proportion of the phenotypic variance.</span><span class="c12">24</span><span class="c0">&nbsp;Subsequent fine-mapping efforts narrowed down major QTLs for HSW (</span><span class="c1">qHSWA05.6</span><span class="c0">) and SW (</span><span class="c1">qSWA05.5</span><span class="c0">) to physical intervals of 600.9 kb and 721.2 kb, respectively, within this broader region.</span><span class="c8">24</span></li><li class="c11 c20 li-bullet-0"><span class="c0">Analysis of the US peanut mini-core collection identified a major locus on linkage group A05, corresponding to the physical region of 93-102 Mbp, which accounted for an exceptionally high phenotypic variation (up to 66%) for multiple pod and seed traits.</span><span class="c8">27</span></li><li class="c11 c20 li-bullet-0"><span class="c0">Comparative studies between cultivated accessions and the wild tetraploid relative </span><span class="c1">A. monticola</span><span class="c0">&nbsp;also pinpointed stable QTLs controlling seed size/weight differences within specific regions of A05 (e.g., 107.44&ndash;114.86 Mb).</span><span class="c8">14</span></li></ul><p class="c21"><span class="c0">The remarkably high PVE values reported for loci on chromosome A05 in some studies </span><span class="c12">24</span><span class="c0">&nbsp;are particularly noteworthy. PVE values exceeding 40% or even 60% for a single QTL are unusual for quantitative traits in complex polyploids. This strongly suggests the presence of one or perhaps a few closely linked genes within this region that exert a disproportionately large influence on seed size. Such large-effect genes are often targets of strong selection during domestication or breeding. The consistent detection of major QTLs in this region across diverse germplasm </span><span class="c12">14</span><span class="c0 c9">&nbsp;reinforces its fundamental importance in peanut seed development and makes it a prime target for both evolutionary studies and practical breeding applications aimed at manipulating seed size.</span></p><p class="c13"><span class="c23 c9">2. Chromosome A07: Pod and Seed Size QTLs (Fine-Mapped)</span></p><p class="c13"><span class="c23 c9">Chromosome A07 is another critical region repeatedly implicated in the control of pod and seed size, often revealed in studies involving interspecific crosses or those focused on fine-mapping QTLs to higher resolution.4 Several research groups have successfully narrowed down major QTL intervals on this chromosome:</span></p><ul class="c19 lst-kix_list_2-0 start"><li class="c15 c20 li-bullet-0"><span class="c0">Early reports cited the fine-mapping of </span><span class="c1">qAHPS07</span><span class="c0">, a major pod size QTL, to a 36.46 kb interval </span><span class="c12">24</span><span class="c0">&nbsp;and another major QTL to a 20.4 kb interval using F6:7 and F6:8 populations.</span><span class="c8">24</span></li><li class="c11 c20 li-bullet-0"><span class="c0">Zhuang et al. reportedly fine-mapped a seed size QTL to a 1 Mb interval on A07 using BSA and RIL analysis.</span><span class="c8">39</span></li><li class="c11 c20 li-bullet-0"><span class="c0">A study utilizing an Advanced Backcross QTL (AB-QTL) population derived from a cross involving the wild diploid </span><span class="c1">A. ipaensis</span><span class="c0">&nbsp;initially mapped a major pod/seed size QTL to a ~5 Mb region on A07. Subsequent fine-mapping using Near-Isogenic Lines (NILs) and a large F2 population successfully narrowed this QTL down to a 168.37 kb segment located between SNP markers Aradu_A07_1148327 and Aradu_A07_1316694 (physical position ~1.1-1.3 Mb).</span><span class="c8">29</span></li></ul><p class="c21"><span class="c0">These successful fine-mapping efforts, reducing large initial QTL intervals (often several megabases) down to the kilobase scale, demonstrate the increasing feasibility of progressing towards positional cloning of causal genes in peanut, despite its genomic complexity. The identification of QTLs in this region that differentiate cultivated peanut from its wild relatives </span><span class="c12">4</span><span class="c0">&nbsp;suggests that genes within this locus were likely subjected to selection pressures during peanut domestication and improvement. The clustering of QTLs for various pod and seed traits in the proximal region of A07 further highlights its importance.</span><span class="c8">4</span></p><p class="c13"><span class="c23 c9">3. Chromosome A08: QTL Hotspot for HSW and HPW</span></p><p class="c13"><span class="c23 c9">Chromosome A08 has been identified as harboring multiple QTLs influencing both hundred-pod weight (HPW) and hundred-seed weight (HSW).24 A particularly significant finding is the identification of a QTL hotspot within a relatively small genomic region on this chromosome.31</span></p><ul class="c19 lst-kix_list_3-0 start"><li class="c15 c20 li-bullet-0"><span class="c0">This hotspot spans a genetic distance of approximately 2.75 cM, flanked by markers AhTE0658 and TC22C01, which corresponds to a physical interval of only 0.36 Mb.</span><span class="c8">31</span></li><li class="c11 c20 li-bullet-0"><span class="c0">Within this compact region, 10 distinct QTLs for HPW and HSW were detected across various environments in one study.</span><span class="c12">31</span><span class="c0">&nbsp;Individual QTLs within this hotspot, such as </span><span class="c1">qHPWA08.3</span><span class="c0">&nbsp;and </span><span class="c1">qHSWA08.6</span><span class="c0">, were found to be stable across multiple environments, each explaining a moderate proportion (3.7% to 8.9%) of the phenotypic variance.</span><span class="c8">31</span></li></ul><p class="c21"><span class="c0">The co-localization of numerous QTLs for highly correlated traits (HPW and HSW) within such a confined physical interval (0.36 Mb), coupled with their stability across environments, strongly points towards the presence of either a single pleiotropic gene influencing both pod and seed weight, or a tightly linked cluster of genes acting coordinately. This makes the 18 candidate genes annotated within this 0.36 Mb interval prime subjects for further investigation to uncover the molecular basis of this QTL hotspot.</span><span class="c8">31</span></p><p class="c13"><span class="c23 c9">4. Chromosome B06: Major QTLs for Pod and Seed Weight (LG B06_1)</span></p><p class="c13"><span class="c23 c9">Linkage group B06 (corresponding to chromosome B06) has also been identified as carrying major QTLs contributing significantly to pod area (PA), pod weight (PW), and seed weight (SdW).2</span></p><ul class="c19 lst-kix_list_4-0 start"><li class="c15 c20 li-bullet-0"><span class="c0">A study using a RIL population derived from a cross involving the cultivar NC 3033 mapped seven major QTLs (defined as PVE &gt; 20%) for PA, PW, and SdW to linkage group B06_1, with the favorable alleles contributed by NC 3033.</span><span class="c8">2</span></li><li class="c11 c20 li-bullet-0"><span class="c0">Six of these major QTLs (</span><span class="c1">qPWB06_1.2</span><span class="c0">, </span><span class="c1">qPWB06_1.3</span><span class="c0">, </span><span class="c1">qSdWB06_1.2</span><span class="c0">, </span><span class="c1">qSdWB06_1.3</span><span class="c0">, </span><span class="c1">qPAB06_1.2</span><span class="c0">, </span><span class="c1">qPAB06_1.3</span><span class="c0">) were found to cluster tightly within a ~3.3 cM interval, delimited by markers AX-147226319_A06 and AX-147226313_A06. These QTLs exhibited substantial PVE values, ranging from 24.4% for pod area to 31.4% for seed weight.</span><span class="c8">2</span></li></ul><p class="c21"><span class="c0 c9">Similar to the findings on chromosome A05, the concentration of multiple, co-localized QTLs with large phenotypic effects on chromosome B06 underscores its status as another critical genomic region governing pod and seed size variation in peanut. The substantial PVE associated with these QTLs makes this region highly relevant for marker-assisted selection (MAS) strategies aimed at improving these traits concurrently.</span></p><p class="c13"><span class="c23 c9">5. Chromosome B07: Consensus QTL for Hundred Kernel Weight (HKW)</span></p><p class="c13"><span class="c23 c9">Chromosome B07 is recognized for harboring QTLs associated with seed size, specifically hundred kernel weight (HKW), as well as related pod characteristics like constriction.2</span></p><ul class="c19 lst-kix_list_5-0 start"><li class="c15 c20 li-bullet-0"><span class="c0">A key finding is the identification of a major consensus QTL, designated </span><span class="c1">qHKW-B07</span><span class="c0">, which was consistently detected across six different environments.</span><span class="c8">44</span></li><li class="c11 c20 li-bullet-0"><span class="c0">This stable QTL is located between SSR markers B07_109 and pPGPseq_2E06 (a 20.6 cM interval) and explained a significant portion of the HKW variance, ranging from 10.5% to 23.9% depending on the environment.</span><span class="c12">44</span><span class="c0">&nbsp;Genomic localization places this region on chromosome </span><span class="c1">Arahy</span><span class="c0">.17 of the cultivated peanut reference genome.</span><span class="c8">44</span></li></ul><p class="c21"><span class="c0">The consistent detection of </span><span class="c1">qHKW-B07</span><span class="c0 c9">&nbsp;across diverse environmental conditions highlights its robustness and reliability as a genetic determinant of seed weight. Its major effect size (PVE up to ~24%) further emphasizes its importance. Consequently, the flanking markers (B07_109 and pPGPseq_2E06) represent valuable tools for breeders employing MAS to select for increased seed weight in peanut improvement programs.</span></p><p class="c13"><span class="c23 c9">6. Chromosomes A02 &amp; A16: Stable QTL Regions for Seed Dimensions</span></p><p class="c13"><span class="c23 c9">Studies employing high-density mapping techniques like SLAF-seq have identified stable QTL regions on chromosomes A02 and A16 that appear to specifically influence seed dimensions.17</span></p><ul class="c19 lst-kix_list_6-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">Chromosome A02:</span><span class="c0">&nbsp;A stable QTL region, physically located between 92.75&ndash;99.81 Mb and genetically flanked by markers Marker938&ndash;Marker893, was strongly associated with seed length (</span><span class="c1">qSL2</span><span class="c0">) and the seed length-to-width ratio (</span><span class="c1">qLW2</span><span class="c0">). This QTL was detected robustly across all four tested environments.</span><span class="c12">17</span><span class="c0">&nbsp;Another study also identified a QTL in this same physical interval.</span><span class="c12">35</span><span class="c0">&nbsp;The reported PVE values for this locus are exceptionally high, reaching up to 61.47% for SL and an astonishing 83.23% for L/W ratio.</span><span class="c8">17</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Chromosome A16:</span><span class="c0">&nbsp;Stable QTLs influencing 100-seed weight (</span><span class="c1">q100SW16</span><span class="c0">) and seed width (</span><span class="c1">qSW16</span><span class="c0">) were mapped to chromosome A16. The </span><span class="c1">q100SW16</span><span class="c0">&nbsp;locus (physical position 8.49&ndash;18.31 Mb) was detected across multiple environments, explaining up to 35.39% PVE. The </span><span class="c1">qSW16</span><span class="c0">&nbsp;locus (physical position spanning 7.33&ndash;53.24 Mb, with specific sub-QTLs in narrower intervals) explained up to 21.58% PVE.</span><span class="c12">17</span><span class="c0">&nbsp;A QTL was also identified in the 7.33-19.54 Mb region in another study.</span><span class="c8">35</span></li></ul><p class="c21"><span class="c0">The extraordinarily high PVE reported for the QTL on chromosome A02, particularly for the length-to-width ratio </span><span class="c12">17</span><span class="c0 c9">, is highly significant. Such a large effect size might suggest the presence of a gene with a near-Mendelian effect on seed shape, potentially controlling a fundamental aspect of cell division or elongation along the longitudinal axis of the seed. This region clearly warrants intensive investigation to identify the underlying gene(s).</span></p><p class="c13"><span class="c23 c9">7. Other Significant QTLs</span></p><p class="c13"><span class="c23 c9">Beyond the major hubs discussed above, QTLs associated with seed size traits have also been reported on several other chromosomes, reinforcing the polygenic nature of this trait:</span></p><ul class="c19 lst-kix_list_7-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">Chromosome B04:</span><span class="c0">&nbsp;Identified as carrying major QTLs for HPW/HSW.</span><span class="c8">24</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Chromosome B08:</span><span class="c0">&nbsp;Also reported to harbor major QTLs for HPW/HSW.</span><span class="c8">24</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Chromosome B10:</span><span class="c0">&nbsp;QTLs for HSW (</span><span class="c1">qHSWB10.1</span><span class="c0">, </span><span class="c1">qHSWB10.2</span><span class="c0">, </span><span class="c1">qHSWB10.3</span><span class="c0">) have been mapped, with </span><span class="c1">qHSWB10.1</span><span class="c0">&nbsp;explaining 12.82% PVE.</span><span class="c12">24</span><span class="c0">&nbsp;Interestingly, a major QTL controlling seed protein content (</span><span class="c1">qSPCB10.1</span><span class="c0">) was also mapped to this chromosome </span><span class="c12">54</span><span class="c0">, suggesting potential genetic linkage or pleiotropy between seed size and composition traits, which could have implications for breeding (e.g., considering the reported negative correlation between oil and protein content </span><span class="c12">54</span><span class="c0 c9">).</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Other Chromosomes:</span><span class="c0">&nbsp;QTLs associated with pod or seed size have also been detected less consistently or with smaller effects on chromosomes A03, A09, B02, and B05.</span><span class="c8">4</span></li></ul><p class="c21"><span class="c0 c9">The distribution of QTLs across numerous chromosomes confirms that seed size is influenced by a complex network of genes acting throughout the genome. The co-localization of QTLs for different traits, such as size and protein content on B10, highlights the need for breeders to consider potential genetic correlations, which could result in trade-offs when selecting for multiple desirable traits simultaneously.</span></p><h3 class="c15"><span class="c27 c9">C. Table 1: Summary of Major Seed Size QTLs in Peanut</span></h3><p class="c13 c26"><span class="c27 c9"></span></p><table class="c25"><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Chromosome</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">QTL Name / Region</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Interval / Flanking Markers</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Trait(s) Affected</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Max PVE (%)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Stability (Environments)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Key Reference(s)</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A02</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">92.75&ndash;99.81 Mb</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Marker938&ndash;Marker893</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">SL, L/W</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">83.23 (L/W)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">4/4</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">17</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">19-52 cM / 93-102 Mb</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Multiple (e.g., Ah900023-Ah900027)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HSW, SL, SW, Pod Size/Weight</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">66.0</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">High (Multiple studies, envs)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">14</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A07</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">168.37 kb (fine-mapped)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Aradu_A07_1148327 &ndash; Aradu_A07_1316694</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Seed Size, Pod Size</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">~26 (Pod size QTL in AB-QTL)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Validated in NILs</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">39</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A07</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">36.46 kb (fine-mapped)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">(Markers not specified in snippet)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pod Size (qAHPS07)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">38.6 (SPW)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Fine-mapped</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">0.36 Mb Hotspot</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">AhTE0658 &ndash; TC22C01</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HPW, HSW</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">8.9 (qHPWA08.3)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">High (Multiple QTLs, envs)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">31</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A16</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">8.49&ndash;18.31 Mb</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Multiple (e.g., Marker9375&ndash;Marker9395)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HSW (q100SW16)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">35.4</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">2-3/4</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">17</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A16</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">7.33&ndash;53.24 Mb</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Multiple (e.g., Marker9360&ndash;Marker9483)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">SW (qSW16)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">21.6</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">1-2/4</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">17</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">B06</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">~3.3 cM (LG B06_1)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">AX-147226319_A06 &ndash; AX-147226313_A06</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">PW, SdW, PA</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">31.4 (SdW)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">2/3</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">2</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">B07</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">20.6 cM</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">B07_109 &ndash; pPGPseq_2E06</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HKW (qHKW-B07)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">23.9</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">6/6</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">44</span></p></td></tr><tr class="c22"><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">B10</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">20.64&ndash;35.72 cM</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">(Markers not specified in snippet)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HSW (qHSWB10.1)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">12.8</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">1 env</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr></table><p class="c18"><span class="c1 c9">Note: PVE = Phenotypic Variance Explained. HSW = Hundred Seed Weight, SL = Seed Length, SW = Seed Width, L/W = Length/Width Ratio, PA = Pod Area, PW = Pod Weight, SdW = Seed Weight, HKW = Hundred Kernel Weight. Stability indicates the number of environments/studies where the QTL was detected.</span></p><h2 class="c15"><span class="c4">III. Candidate Genes Implicated in Peanut Seed Size Regulation</span></h2><h3 class="c15"><span class="c27 c9">A. Overview of Identification Strategies</span></h3><p class="c15"><span class="c0 c9">Moving beyond the identification of genomic regions (QTLs), significant efforts have focused on pinpointing specific candidate genes within these regions that are likely responsible for the observed variation in seed size. Several complementary strategies are employed to nominate and prioritize these candidates:</span></p><ul class="c19 lst-kix_list_8-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">QTL Co-localization:</span><span class="c0">&nbsp;The most direct approach involves identifying genes physically located within the boundaries of validated, stable, and preferably fine-mapped QTL intervals. Genes residing within major QTL hotspots or regions narrowed down to kilobase resolution are considered strong positional candidates.</span><span class="c8">9</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Transcriptomic Evidence:</span><span class="c0">&nbsp;Analyzing gene expression data (RNA-Seq) from developing seeds of genotypes with contrasting sizes allows for the identification of Differentially Expressed Genes (DEGs). Genes showing significant expression differences that correlate with the seed size phenotype, particularly during critical developmental windows (e.g., early cell division phase or later storage accumulation phase), are flagged as potential regulators.</span><span class="c12">9</span><span class="c0 c9">&nbsp;Combining DEG analysis with QTL information (i.e., identifying DEGs located within QTL intervals) significantly strengthens candidacy.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Mutant Analysis:</span><span class="c0">&nbsp;Characterizing induced or spontaneous mutants exhibiting distinct seed size phenotypes (either significantly larger or smaller than wild-type) provides a powerful tool for identifying causal genes. Comparing the genomes and transcriptomes of mutants and wild-type progenitors can reveal specific gene mutations (SNPs, Indels, frameshifts) or expression changes directly linked to the altered phenotype.</span><span class="c8">10</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Sequence Variation Analysis:</span><span class="c0">&nbsp;Identifying polymorphisms (SNPs, Indels) within the coding or regulatory regions of positional candidate genes is crucial. If these variations are predicted to alter protein function (e.g., non-synonymous SNPs causing amino acid changes, frameshifts leading to truncated proteins) and if their allelic distribution correlates with seed size variation across different genotypes or in segregating populations, it provides strong evidence for their functional relevance.</span><span class="c8">9</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Functional Annotation and Homology:</span><span class="c0">&nbsp;Predicting the function of candidate genes based on sequence homology to genes with known roles in other plant species (especially model systems like Arabidopsis or related legumes) is essential. Genes annotated with functions related to key processes like seed development, cell cycle control, cell expansion, hormone biosynthesis or signaling, transcription regulation, or primary/secondary metabolism are prioritized.</span><span class="c8">9</span></li></ul><p class="c21"><span class="c0 c9">It is important to recognize that relying on a single line of evidence can be insufficient. For example, a gene merely located within a large QTL interval may not be the causal gene, and a gene showing differential expression might be downstream of the primary regulator or responding to physiological changes rather than causing them. Therefore, the most compelling candidate genes are those supported by converging evidence from multiple approaches. A gene that resides within a fine-mapped, major QTL interval, shows differential expression correlating with the phenotype, possesses functional sequence variation, and has a relevant predicted biological function represents a high-confidence candidate warranting further functional validation.</span></p><h3 class="c15"><span class="c9 c27">B. Detailed Discussion of Key Candidate Genes and Families</span></h3><p class="c18"><span class="c0 c9">Several gene families and specific genes have emerged as prominent candidates for regulating seed size in peanut, supported by varying degrees of evidence from the strategies outlined above.</span></p><p class="c13"><span class="c23 c9">1. Pentatricopeptide Repeat (PPR) Proteins</span></p><p class="c13"><span class="c23 c9">PPR proteins constitute one of the largest nuclear-encoded protein families in plants.56 They are primarily known for their roles in post-transcriptional processes within mitochondria and chloroplasts, including RNA editing, splicing, stabilization, and translation initiation.55 Given the importance of organellar function (energy production, photosynthesis) for seed development and filling, disruptions in PPR gene function often lead to defects in embryogenesis and seed development in various plant species, including maize and Arabidopsis.9 Several PPR genes have been implicated in peanut seed size control:</span></p><ul class="c19 lst-kix_list_9-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">EVM0025654 (Potentially Arahy.TNFT39 in A05 hotspot):</span><span class="c0">&nbsp;This gene stands out as a particularly strong candidate. It is located within a stable seed size/weight QTL region on chromosome A05 (physical interval 107.44&ndash;114.86 Mb) identified through comparisons between cultivated peanut and its wild relative </span><span class="c1">A. monticola</span><span class="c0">.</span><span class="c12">14</span><span class="c0">&nbsp;Crucially, a non-synonymous SNP (C&gt;G transition at position 111201604) within its coding sequence was identified, resulting in an amino acid change. The &#39;G&#39; allele is present in cultivated accessions (associated with larger seeds), while the &#39;C&#39; allele is found in </span><span class="c1">A. monticola</span><span class="c0">&nbsp;(smaller seeds). This SNP showed a strong association with seed size/weight when tested across a panel of diverse peanut landraces and within a RIL population segregating for this locus.</span><span class="c12">14</span><span class="c0">&nbsp;This convergence of QTL co-localization and functional sequence variation strongly suggests EVM0025654 played a role in the increase in seed size during peanut domestication or improvement. Another PPR gene, </span><span class="c1">Arahy.TNFT39</span><span class="c0">, was identified within the A05 hotspot region defined in a different study </span><span class="c12">24</span><span class="c0 c9">, potentially representing the same locus or a closely related family member.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.W18Y25 &amp; Arahy.CPLC2W:</span><span class="c0">&nbsp;These two PPR-encoding genes (one annotated as PPR-like) were identified among the 18 candidates residing within the 0.36 Mb QTL hotspot on chromosome A08, which is strongly associated with HPW and HSW.</span><span class="c12">31</span><span class="c0 c9">&nbsp;Their candidacy is based on their co-location within this significant QTL interval and the established general role of PPR proteins in plant and seed development.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.JX1V6X:</span><span class="c0">&nbsp;This gene, encoding a PPR superfamily protein, was identified as a candidate within a 0.16 Mb interval on chromosome Arahy07 resulting from fine-mapping of QTLs for kernel weight and size.</span><span class="c12">9</span><span class="c0">&nbsp;Expression analysis indicated it is constitutively expressed in developing seeds.</span><span class="c12">9</span><span class="c0">&nbsp;While its precise function remains unclear from the provided data </span><span class="c12">23</span><span class="c0 c9">, its location in a refined QTL makes it noteworthy.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.7TY6PE &amp; Arahy.X5R2L4:</span><span class="c0">&nbsp;These two additional PPR protein-encoding genes were identified as candidates within the fine-mapped A05 hotspot region (specifically linked to QTLs </span><span class="c1">qPLA05.7</span><span class="c0">&nbsp;and </span><span class="c1">qHSWA05.6</span><span class="c0">).</span><span class="c12">24</span><span class="c0 c9">&nbsp;Their inclusion is based on positional evidence and the known functions of PPRs.</span></li></ul><p class="c21"><span class="c0 c9">The recurrent identification of distinct PPR genes as candidates within major seed size QTLs on different chromosomes (A05, A07, A08) is striking. It suggests that this gene family plays a multifaceted and significant role in determining peanut seed size, potentially influencing various aspects of development through their regulation of organellar gene expression, which could impact critical processes like energy supply or chloroplast function essential for seed filling.</span></p><p class="c13"><span class="c23 c9">2. NAC Domain Transcription Factors</span></p><p class="c13"><span class="c23 c9">NAC (NAM, ATAF1/2, CUC2) transcription factors form a large, plant-specific family involved in regulating a wide array of processes, including plant development, secondary metabolism, senescence, and responses to biotic and abiotic stresses.10 Some NAC factors have been shown to influence grain yield and seed size in crops like rice.10</span></p><ul class="c19 lst-kix_list_a-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">Tifrunner.gnm1.ann1.CDPA7L:</span><span class="c0">&nbsp;This NAC transcription factor gene emerged as a candidate from the analysis of induced &#39;big seed&#39; mutants (</span><span class="c1">hybs</span><span class="c0">).</span><span class="c12">10</span><span class="c0">&nbsp;It is located on chromosome A08 (at physical position 30,827,924).</span><span class="c12">67</span><span class="c0">&nbsp;Significantly, its expression was found to be </span><span class="c1">down-regulated</span><span class="c0">&nbsp;in the large-seeded </span><span class="c1">hybs</span><span class="c0">&nbsp;mutants compared to the wild-type parent. Furthermore, sequencing revealed a frameshift mutation (C &gt; CA insertion) within this gene in the mutants, which likely leads to a non-functional protein.</span><span class="c12">10</span><span class="c0">&nbsp;Genome-wide studies have confirmed the presence and diversity of the NAC family in peanut.</span><span class="c8">57</span></li></ul><p class="c21"><span class="c0">The finding that reduced expression or mutation of </span><span class="c1">Tifrunner.gnm1.ann1.CDPA7L</span><span class="c0">&nbsp;is associated with </span><span class="c1">larger</span><span class="c0 c9">&nbsp;seeds suggests that this particular NAC factor may function as a negative regulator of seed size in peanut. Identifying such negative regulators is as important as finding positive regulators for developing strategies to precisely modulate seed size according to specific breeding goals.</span></p><p class="c13"><span class="c23 c9">3. Cytochrome P450 (CYP450) Family Genes</span></p><p class="c13"><span class="c23 c9">The Cytochrome P450 superfamily comprises a vast group of enzymes involved in a multitude of oxidative reactions critical for plant metabolism, including the biosynthesis and degradation of hormones (like gibberellins, brassinosteroids), fatty acids, and various secondary metabolites.24 Due to their diverse roles, CYP450s have been implicated in regulating seed development and size in several crop species, potentially by influencing cell proliferation or hormone levels.24</span></p><ul class="c19 lst-kix_list_b-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">Tifrunner.gnm1.ann1.TMG43S:</span><span class="c0">&nbsp;This CYP450 gene was identified through the analysis of the </span><span class="c1">hybs</span><span class="c0">&nbsp;&#39;big seed&#39; mutants.</span><span class="c12">10</span><span class="c0">&nbsp;It is located on chromosome B04 (position 21,074,785).</span><span class="c12">67</span><span class="c0">&nbsp;In contrast to the NAC candidate, this gene was significantly </span><span class="c1">up-regulated</span><span class="c0">&nbsp;in the large-seeded mutants. It also harbors a missense mutation (T &gt; C) resulting in an amino acid substitution in the mutants.</span><span class="c12">10</span><span class="c0">&nbsp;Homologous CYP78A genes in other species have been linked to increased seed size.</span><span class="c8">10</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.QW591P, Arahy.Q9F5SX, Arahy.Z6EAYS, Arahy.4A049R:</span><span class="c0">&nbsp;These four CYP450-encoding genes were identified as candidates within the fine-mapped hotspot region on chromosome A05 (</span><span class="c1">qPLA05.7</span><span class="c0">/</span><span class="c1">qHSWA05.6</span><span class="c0">).</span><span class="c12">24</span><span class="c0">&nbsp;Their candidacy stems from their location within this major QTL region and the known involvement of CYP450s in seed size regulation in other plants, possibly through effects on seed coat cell proliferation.</span><span class="c8">24</span></li></ul><p class="c21"><span class="c0 c9">The identification of CYP450 candidates through both mutant analysis (where upregulation correlates with larger seeds) and QTL mapping reinforces their potential significance in peanut seed size control. Their broad metabolic functions suggest they could act via multiple mechanisms, including modulating hormone homeostasis or pathways affecting cell growth and expansion.</span></p><p class="c13"><span class="c23 c9">4. LEAFY COTYLEDON1 (LEC1) and Related Factors</span></p><p class="c13"><span class="c23 c9">LEC1 is a well-characterized transcription factor in Arabidopsis, belonging to the HAP3 subunit of the CCAAT-binding factor (CBF), also known as Nuclear Factor Y (NF-Y).59 It acts as a master regulator of embryogenesis and seed maturation, controlling processes like storage protein and lipid accumulation, acquisition of desiccation tolerance, and suppression of germination programs.15 Peanut possesses LEC1 homologs (AhNF-YB1 on the A subgenome, AhNF-YB10 on the B subgenome) that have been shown to play conserved roles in embryo development and lipid accumulation when studied through complementation in Arabidopsis.61</span></p><ul class="c19 lst-kix_list_c-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">LEC1 (likely AhNF-YB1/AhNF-YB10):</span><span class="c0">&nbsp;Transcriptome analysis of the </span><span class="c1">ssm1</span><span class="c0">&nbsp;small seed mutant revealed a significant </span><span class="c1">down-regulation</span><span class="c0">&nbsp;of </span><span class="c1">LEC1</span><span class="c0">&nbsp;expression, particularly at the early and mid-stages of seed development (DAP10 and DAP20) compared to the wild-type.</span><span class="c12">15</span><span class="c0">&nbsp;This reduced </span><span class="c1">LEC1</span><span class="c0">&nbsp;expression was accompanied by decreased expression of downstream target genes involved in fatty acid biosynthesis and storage protein accumulation, consistent with the mutant&#39;s phenotype of small, wrinkled seeds with low oil content. This strong correlation makes </span><span class="c1">LEC1</span><span class="c0">&nbsp;a key candidate gene responsible for the </span><span class="c1">ssm1</span><span class="c0">&nbsp;phenotype.</span><span class="c12">15</span><span class="c0">&nbsp;Studies on the </span><span class="c1">AhLEC1A</span><span class="c0">&nbsp;promoter further confirmed its seed-specific expression pattern.</span><span class="c8">59</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.IBM9RL (FIE - Fertilization Independent Endosperm):</span><span class="c0">&nbsp;This gene, encoding a Polycomb group protein, was identified as a candidate within the A08 QTL hotspot associated with HPW/HSW.</span><span class="c12">31</span><span class="c0">&nbsp;FIE proteins are crucial epigenetic regulators involved in repressing gene expression, particularly in controlling endosperm development and the transition phases in seed development. Its presence within a seed weight QTL hotspot suggests that epigenetic mechanisms, potentially modulating the expression of key developmental regulators like </span><span class="c1">LEC1</span><span class="c0 c9">&nbsp;or others, may also contribute to determining final seed size and weight in peanut.</span></li></ul><p class="c21"><span class="c0">The strong evidence linking reduced </span><span class="c1">LEC1</span><span class="c0">&nbsp;function to the severe seed development defects in the </span><span class="c1">ssm1</span><span class="c0 c9">&nbsp;mutant confirms its critical role as a positive regulator of seed maturation and storage deposition in peanut, mirroring its function in model systems. The identification of an epigenetic regulator like FIE within another seed weight QTL hints at the complex layers of regulation involved, potentially linking environmental signals or developmental timing to the control of seed growth via chromatin modifications.</span></p><p class="c13"><span class="c23 c9">5. Metabolic Enzymes</span></p><p class="c13"><span class="c23 c9">The accumulation of storage reserves (primarily oil and protein) is fundamental to seed filling and final weight. Therefore, genes encoding key enzymes in metabolic pathways are expected to influence seed size.</span></p><ul class="c19 lst-kix_list_d-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">PKp2 (Plastidial Pyruvate Kinase):</span><span class="c0">&nbsp;PKp is a rate-limiting enzyme in glycolysis, specifically within plastids, where it generates pyruvate, a crucial precursor for de novo fatty acid synthesis.</span><span class="c12">15</span><span class="c0">&nbsp;Transcriptome analysis of the </span><span class="c1">ssm1</span><span class="c0">&nbsp;small seed mutant showed that the expression of </span><span class="c1">PKp</span><span class="c0">&nbsp;genes, including </span><span class="c1">PKp2</span><span class="c0">, was significantly </span><span class="c1">decreased</span><span class="c0">&nbsp;at all measured developmental stages compared to the wild-type.</span><span class="c12">15</span><span class="c0">&nbsp;This reduction correlates well with the mutant&#39;s lower oil content and overall smaller seed size, leading to the proposal that </span><span class="c1">PKp2</span><span class="c0">&nbsp;(possibly regulated by LEC1 and hormones like ABA) is a key candidate contributing to the </span><span class="c1">ssm1</span><span class="c0">&nbsp;phenotype.</span><span class="c12">15</span><span class="c0">&nbsp;Studies on the peanut </span><span class="c1">PK</span><span class="c0">&nbsp;gene family confirm their existence and potential roles in germination and stress responses.</span><span class="c8">63</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.DY91G5 (SPDS - Spermidine Synthase):</span><span class="c0">&nbsp;This gene, involved in the biosynthesis of polyamines, was identified as a candidate within the major A05 hotspot region (</span><span class="c1">qPLA05.7</span><span class="c0">/</span><span class="c1">qHSWA05.6</span><span class="c0">).</span><span class="c12">24</span><span class="c0">&nbsp;Polyamines are known to play roles in various growth and developmental processes, and studies in rice and wheat have linked </span><span class="c1">SPDS</span><span class="c0">&nbsp;expression levels to grain size and weight.</span><span class="c12">24</span><span class="c0 c9">&nbsp;Its location in a major peanut seed size QTL makes it a plausible candidate.</span></li></ul><p class="c21"><span class="c0">These examples directly link core metabolic functions&mdash;glycolysis providing carbon backbones (</span><span class="c1">PKp2</span><span class="c0">) and polyamine synthesis potentially influencing cell division or differentiation (</span><span class="c1">SPDS</span><span class="c0 c9">)&mdash;to the genetic control of seed size. They highlight how the efficiency of primary metabolism and resource partitioning into storage compounds directly impacts the final seed weight and dimensions.</span></p><p class="c13"><span class="c23 c9">6. Signaling and Cell Cycle Regulators</span></p><p class="c13"><span class="c9 c23">Ultimately, seed size is determined by the coordinated processes of cell proliferation (determining cell number) and cell expansion (determining cell size) within the developing embryo, endosperm, and seed coat.18 Genes involved in signaling pathways and cell cycle control are therefore expected to be key regulators.</span></p><ul class="c19 lst-kix_list_e-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">MPK3 (Mitogen-Activated Protein Kinase 3) (arahy.RQ7QZ9):</span><span class="c0">&nbsp;A component of the MAPK signaling cascade, which is known to regulate seed size in plants.</span><span class="c12">15</span><span class="c0">&nbsp;</span><span class="c1">MPK3</span><span class="c0">&nbsp;expression was found to be significantly lower in large-podded peanut genotypes compared to small-podded ones at an early stage of pod development (Pod-25), suggesting it might act to promote cell division in the integuments or ovary wall early on.</span><span class="c8">35</span></li><li class="c11 c20 li-bullet-0"><span class="c17">GCP4 (Gamma-tubulin complex protein 4) (arahy.D5VDWJ):</span><span class="c0">&nbsp;This gene, involved in microtubule nucleation essential for cell division and organization </span><span class="c12">37</span><span class="c0">, was located within a seed size QTL on chromosome A02.</span><span class="c12">35</span><span class="c0">&nbsp;Its expression was higher in small-podded genotypes, and overexpression of homologs in other species leads to smaller organs, suggesting </span><span class="c1">GCP4</span><span class="c0">&nbsp;might act as a negative regulator of size in peanut.</span><span class="c8">35</span></li><li class="c11 c20 li-bullet-0"><span class="c17">RPPL1 (Ribosomal protein L1) (arahy.2IZ2Y4):</span><span class="c0">&nbsp;Also located within the A02 seed size QTL.</span><span class="c12">35</span><span class="c0">&nbsp;Like </span><span class="c1">GCP4</span><span class="c0">, its expression was higher in small-podded genotypes, hinting at a potential negative regulatory role, possibly related to protein synthesis capacity influencing growth rate.</span><span class="c8">35</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.Q67Z2R (E2F transcription factor):</span><span class="c0">&nbsp;Identified as a candidate within the A05 hotspot region (</span><span class="c1">qPLA05.7</span><span class="c0">/</span><span class="c1">qHSWA05.6</span><span class="c0">).</span><span class="c12">24</span><span class="c0">&nbsp;E2F transcription factors are master regulators of the cell cycle, controlling the transition from G1 to S phase and coordinating cell proliferation with differentiation during development, including in seeds.</span><span class="c12">24</span><span class="c0 c9">&nbsp;Its presence in a major seed size QTL strongly implicates cell cycle control as a key determinant.</span></li></ul><p class="c21"><span class="c0">The identification of candidates involved in MAPK signaling (</span><span class="c1">MPK3</span><span class="c0">), microtubule dynamics (</span><span class="c1">GCP4</span><span class="c0">), ribosome biogenesis (</span><span class="c1">RPPL1</span><span class="c0">), and core cell cycle regulation (</span><span class="c1">E2F</span><span class="c0 c9">) clearly indicates that the genetic control of peanut seed size operates fundamentally at the level of regulating cell number and/or cell size during development. Both positive regulators (like MPK3 and E2F, implied) and negative regulators (like GCP4 and RPPL1, implied) appear to contribute to the final outcome.</span></p><p class="c13"><span class="c23 c9">7. Other Potential Regulators</span></p><p class="c13"><span class="c23 c9">A diverse array of other genes has also been proposed as candidates, highlighting the intricate nature of seed size regulation:</span></p><ul class="c19 lst-kix_list_f-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">AhRUVBL2 (RuvB-like helicase 2):</span><span class="c0">&nbsp;Emerged as the most probable candidate gene underlying the major pod size QTL </span><span class="c1">qAHPS07</span><span class="c0">&nbsp;on chromosome A07, fine-mapped to a 36.46 kb interval.</span><span class="c12">24</span><span class="c0">&nbsp;RuvB-like helicases are ATP-dependent DNA helicases often acting as components of chromatin remodeling complexes (like INO80 or SWR1) or transcription regulation complexes (like TIP60), suggesting </span><span class="c1">AhRUVBL2</span><span class="c0">&nbsp;might influence pod/seed size via epigenetic or transcriptional control. Overexpression of </span><span class="c1">AhRUVBL2</span><span class="c0">&nbsp;in Arabidopsis resulted in larger seeds, supporting a positive regulatory role.</span><span class="c8">53</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Aradu.DN3DB (SAP-like - STERILE APETALA-like):</span><span class="c0">&nbsp;A transcriptional regulator identified within the 168.37 kb fine-mapped seed size QTL on chromosome A07.</span><span class="c12">29</span><span class="c0 c9">&nbsp;SAP domain proteins are often involved in chromatin organization and transcriptional repression. Its precise role in peanut seed size needs further investigation, but its location is compelling.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Aradu.RLZ61 (F-box SNEEZY - SNE):</span><span class="c0">&nbsp;Also located within the 168.37 kb A07 QTL interval.</span><span class="c12">29</span><span class="c0">&nbsp;F-box proteins are substrate-recognition components of SCF ubiquitin E3 ligase complexes, targeting specific proteins for degradation via the proteasome pathway. SNEEZY homologs are known negative regulators of gibberellin (GA) signaling; disrupting them often leads to increased GA responses and enhanced growth. This suggests </span><span class="c1">Aradu.RLZ61</span><span class="c0">&nbsp;might influence seed size by modulating GA signaling.</span><span class="c8">7</span></li><li class="c11 c20 li-bullet-0"><span class="c17">TGA7 (Transcription factor) (EVM0062133):</span><span class="c0">&nbsp;Showed differential expression during seed development and sequence variation between cultivated and wild peanuts.</span><span class="c12">9</span><span class="c0 c9">&nbsp;TGA factors are often involved in responding to hormone signals (like jasmonates or salicylates) and regulating defense or developmental genes.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Topless-related protein 2 (EVM0023227):</span><span class="c0">&nbsp;Differentially expressed during seed development with sequence variations identified.</span><span class="c12">9</span><span class="c0 c9">&nbsp;These proteins act as transcriptional co-repressors, often involved in hormone signaling pathways and developmental patterning.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">ILR1-like 5 (IAA-amino acid hydrolase) (EVM0031048):</span><span class="c0">&nbsp;Differentially expressed with sequence variation found.</span><span class="c12">9</span><span class="c0 c9">&nbsp;These enzymes release active auxin (IAA) from conjugated forms, thereby regulating local auxin levels critical for growth and development.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.D52S1Z (Dioxygenase-like):</span><span class="c0">&nbsp;A candidate gene residing in the A08 QTL hotspot.</span><span class="c12">31</span><span class="c0 c9">&nbsp;2-oxoglutarate/Fe(II)-dependent dioxygenases are involved in various metabolic processes, including hormone biosynthesis (e.g., GAs, ethylene) and modification of proteins or secondary metabolites, potentially impacting growth regulation.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Arahy.14EF4H (Sugar transporter 11):</span><span class="c0">&nbsp;Also located in the A08 QTL hotspot.</span><span class="c12">31</span><span class="c0 c9">&nbsp;Sugar transporters are essential for loading sucrose and other sugars into developing seeds (sink tissues) to provide carbon and energy for storage reserve synthesis, directly impacting seed filling and final weight.</span></li></ul><p class="c21"><span class="c0">This diverse collection of candidates underscores the complexity of seed size regulation, integrating transcriptional control (</span><span class="c1">AhRUVBL2</span><span class="c0">, </span><span class="c1">Aradu.DN3DB</span><span class="c0">, </span><span class="c1">TGA7</span><span class="c0">), post-translational regulation via protein degradation (</span><span class="c1">Aradu.RLZ61</span><span class="c0">), intricate hormone metabolism and signaling networks (</span><span class="c1">ILR1-like 5</span><span class="c0">, </span><span class="c1">Arahy.D52S1Z</span><span class="c0">, </span><span class="c1">TGA7</span><span class="c0">, </span><span class="c1">Aradu.RLZ61</span><span class="c0">), and the fundamental process of nutrient allocation (</span><span class="c1">Arahy.14EF4H</span><span class="c0">). The successful fine-mapping of the A07 QTL, leading to strong candidates like </span><span class="c1">AhRUVBL2</span><span class="c0">, </span><span class="c1">Aradu.DN3DB</span><span class="c0">, and </span><span class="c1">Aradu.RLZ61</span><span class="c0 c9">, exemplifies the power of combining precise genetic localization with functional inference.</span></p><h3 class="c15"><span class="c27 c9">C. Table 2: Summary of Key Candidate Genes for Peanut Seed Size</span></h3><p class="c13 c26"><span class="c27 c9"></span></p><table class="c25"><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Gene Name/ID</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Predicted Function/Family</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Chromosome / QTL Link</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c9 c29">Evidence Type(s)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Phenotypic Association</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c29 c9">Key Reference(s)</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">EVM0025654 / Arahy.TNFT39 (?)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pentatricopeptide Repeat (PPR) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05 (Stable QTL / Hotspot)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, SeqVar (SNP assoc. w/ size), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive (Cultivated allele &gt; Wild)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">14</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.W18Y25</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pentatricopeptide Repeat (PPR) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08 (QTL Hotspot)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ HSW/HPW</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">31</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.CPLC2W</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pentatricopeptide Repeat (PPR-like) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08 (QTL Hotspot)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c9 c10">Associated w/ HSW/HPW</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">31</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.JX1V6X</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pentatricopeptide Repeat (PPR) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy07 (Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Expression, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Kernel Wt/Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">9</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.7TY6PE / Arahy.X5R2L4</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Pentatricopeptide Repeat (PPR) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05 (Hotspot / Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Pod/Seed Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Tifrunner.gnm1.ann1.CDPA7L</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">NAC Domain Transcription Factor</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Mutant (Frameshift), DEG (Down in large seed mutant), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Negative</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">10</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Tifrunner.gnm1.ann1.TMG43S</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Cytochrome P450 (CYP450)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">B04</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Mutant (Missense), DEG (Up in large seed mutant), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">10</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.QW591P / Q9F5SX / Z6EAYS / 4A049R</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Cytochrome P450 (CYP450)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05 (Hotspot / Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Pod/Seed Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">LEC1 (AhNF-YB1 / AhNF-YB10)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">HAP3/NF-YB Transcription Factor</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Unknown (Mutant study)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Mutant (ssm1), DEG (Down in small seed mutant), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">15</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.IBM9RL</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">FIE Polycomb Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08 (QTL Hotspot)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ HSW/HPW</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">31</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">PKp2 (Plastidial Pyruvate Kinase)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Glycolytic Enzyme</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Unknown (Mutant study)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Mutant (ssm1), DEG (Down in small seed mutant), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">15</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.DY91G5</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Spermidine Synthase (SPDS)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05 (Hotspot / Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Pod/Seed Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">MPK3 (arahy.RQ7QZ9)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Mitogen-Activated Protein Kinase</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Unknown (Expression study)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">DEG (Lower in large pod genotype), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive (early stage?)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">35</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">GCP4 (arahy.D5VDWJ)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Microtubule Nucleation Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A02 (QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, DEG (Higher in small pod genotype), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Negative</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">35</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">RPPL1 (arahy.2IZ2Y4)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Ribosomal Protein L1</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A02 (QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, DEG (Higher in small pod genotype), Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Negative</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">35</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.Q67Z2R</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">E2F Transcription Factor</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A05 (Hotspot / Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Pod/Seed Size (likely Positive)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">AhRUVBL2</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">RuvB-like Helicase 2</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A07 (Fine-mapped QTL qAHPS07)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology, Overexpression (Arabidopsis)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Positive</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">24</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Aradu.DN3DB</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">SAP-like Transcriptional Regulator</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A07 (Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Seed Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">29</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Aradu.RLZ61</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">F-box SNEEZY (SNE) Protein</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A07 (Fine-mapped QTL)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ Seed Size (potentially Negative via GA)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">29</span></p></td></tr><tr class="c22"><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Arahy.14EF4H</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Sugar Transporter 11</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">A08 (QTL Hotspot)</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">QTL, Homology</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c10 c9">Associated w/ HSW/HPW</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c11"><span class="c8">31</span></p></td></tr></table><p class="c18"><span class="c1 c9">Note: Evidence Types: QTL = Co-localization with QTL; DEG = Differential Gene Expression; SeqVar = Sequence Variation linked to phenotype; Mutant = Identified via mutant analysis; Homology = Functional inference from homologs. Phenotypic Association indicates whether the gene/allele likely promotes (+) or restricts (-) seed size.</span></p><h2 class="c15"><span class="c4">IV. Synthesis and Future Perspectives</span></h2><h3 class="c15"><span class="c27 c9">A. Integration of Evidence: Connecting QTLs, Gene Expression, and Function</span></h3><p class="c18"><span class="c0">The research landscape investigating the genetic control of peanut seed size reveals a convergence of evidence from multiple experimental approaches. Major QTL hotspots, repeatedly identified on chromosomes A05, A07, A08, B06, and B07, serve as crucial anchors, pinpointing genomic regions with significant and often stable effects on seed weight and dimensions.</span><span class="c12">2</span><span class="c0">&nbsp;Within these regions, candidate genes such as those encoding PPR proteins, CYP450 enzymes, transcription factors (LEC1, NAC, E2F, SAP-like), chromatin remodelers (AhRUVBL2, FIE), signaling components (MPK3, GCP4), metabolic enzymes (PKp2, SPDS), and transporters (Sugar transporter 11) have been proposed based on positional evidence and functional annotations.</span><span class="c8">9</span></p><p class="c18"><span class="c0">The strength of candidacy for any given gene is significantly enhanced when multiple lines of evidence align. For instance, the PPR gene </span><span class="c1">EVM0025654</span><span class="c0">&nbsp;is not only located within a major, stable QTL on chromosome A05 but also possesses a non-synonymous SNP whose allelic state strongly correlates with seed size differences between cultivated and wild peanut, providing compelling evidence for its role in domestication-related seed size increase.</span><span class="c12">14</span><span class="c0">&nbsp;Similarly, the transcription factor </span><span class="c1">LEC1</span><span class="c0">&nbsp;and the metabolic enzyme </span><span class="c1">PKp2</span><span class="c0">&nbsp;were identified as key candidates responsible for the </span><span class="c1">ssm1</span><span class="c0">&nbsp;small seed mutant phenotype, based on their significant down-regulation in the mutant coinciding with defects in seed maturation and oil accumulation.</span><span class="c8">15</span></p><p class="c18"><span class="c0">Transcriptomic studies provide a dynamic layer of information, revealing which genes are active during specific phases of seed development.</span><span class="c12">10</span><span class="c0">&nbsp;Identifying DEGs within QTL intervals, or observing expression changes in mutants that match the phenotype, bridges the gap between genetic locus identification and potential biological function. For example, differential expression observed at early stages (e.g., 15 DPF) might point to genes involved in establishing cell number or organ identity, while changes later in development (e.g., 30-60 DPF) are more likely related to cell expansion, storage accumulation, and maturation.</span><span class="c12">10</span><span class="c0 c9">&nbsp;Thus, integrating QTL mapping (providing location and effect size), transcriptomics (providing activity patterns), mutant analysis (providing causal links), sequence variation (providing potential functional changes), and functional annotation (providing biological context) offers a much more robust and holistic understanding than any single approach can achieve in isolation.</span></p><h3 class="c15"><span class="c27 c9">B. Key Regulatory Pathways Emerging</span></h3><p class="c15"><span class="c0 c9">The candidate genes identified point towards the involvement of several conserved and interconnected biological pathways in the regulation of peanut seed size:</span></p><ul class="c19 lst-kix_list_10-0 start"><li class="c15 c20 li-bullet-0"><span class="c17">Hormone Signaling and Metabolism:</span><span class="c0">&nbsp;Phytohormones are central regulators of plant growth and development. Evidence implicates multiple hormone classes in peanut seed/pod size control. Auxin action is suggested by candidates like </span><span class="c1">ILR1-like 5</span><span class="c0">&nbsp;(auxin activation) and potentially </span><span class="c1">ARF2</span><span class="c0">&nbsp;(negative regulator mentioned in context).</span><span class="c12">14</span><span class="c0">&nbsp;Abscisic acid (ABA) likely plays roles in maturation and storage, potentially influencing </span><span class="c1">LEC1</span><span class="c0">&nbsp;and </span><span class="c1">PKp2</span><span class="c0">&nbsp;expression.</span><span class="c12">15</span><span class="c0">&nbsp;Gibberellins (GAs) are implicated through the potential role of the F-box SNE protein (</span><span class="c1">Aradu.RLZ61</span><span class="c0">) in GA signaling </span><span class="c12">39</span><span class="c0">&nbsp;and the known effects of GA biosynthesis genes (like </span><span class="c1">GA20ox</span><span class="c0">) on seed size in legumes.</span><span class="c12">35</span><span class="c0">&nbsp;Brassinosteroids are generally known to affect seed size </span><span class="c12">15</span><span class="c0">, and cytokinins influence cell division and potentially final seed size.</span><span class="c12">15</span><span class="c0">&nbsp;The presence of CYP450 </span><span class="c12">10</span><span class="c0">&nbsp;and dioxygenase </span><span class="c12">31</span><span class="c0 c9">&nbsp;candidates further suggests roles for hormone metabolism in fine-tuning growth responses.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Cell Cycle and Expansion:</span><span class="c0">&nbsp;The ultimate determinants of size are cell number and cell size. Genes directly involved in these processes are implicated, including MAPK signaling components (</span><span class="c1">MPK3</span><span class="c0">) </span><span class="c12">35</span><span class="c0">, microtubule organization factors (</span><span class="c1">GCP4</span><span class="c0">, </span><span class="c1">Topless-related protein 2</span><span class="c0">) </span><span class="c12">14</span><span class="c0">, and core cell cycle regulators (</span><span class="c1">E2F</span><span class="c0">).</span><span class="c12">24</span><span class="c0 c9">&nbsp;These findings indicate that genetic control operates at the fundamental level of modulating cell division rates and/or the extent of cell expansion during seed and pod growth.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Transcriptional and Epigenetic Regulation:</span><span class="c0">&nbsp;A complex hierarchy of transcriptional control is evident. Key transcription factor families identified include PPRs (organelle gene expression) </span><span class="c12">9</span><span class="c0">, NACs (development/stress) </span><span class="c12">10</span><span class="c0">, HAP3/NF-YB (</span><span class="c1">LEC1</span><span class="c0">, maturation) </span><span class="c12">15</span><span class="c0">, E2F (cell cycle) </span><span class="c12">24</span><span class="c0">, TGA (hormone response) </span><span class="c12">14</span><span class="c0">, and SAP-like factors (</span><span class="c1">Aradu.DN3DB</span><span class="c0">).</span><span class="c12">39</span><span class="c0">&nbsp;Additionally, the identification of a RuvB-like helicase (</span><span class="c1">AhRUVBL2</span><span class="c0">) </span><span class="c12">24</span><span class="c0">&nbsp;and a Polycomb protein (</span><span class="c1">FIE</span><span class="c0">) </span><span class="c12">31</span><span class="c0 c9">&nbsp;suggests that chromatin remodeling and epigenetic mechanisms also play significant roles, potentially by modulating the accessibility and expression of downstream target genes.</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Metabolism and Storage Accumulation:</span><span class="c0">&nbsp;The process of filling the seed with storage reserves is critical for final weight. Key metabolic enzymes like </span><span class="c1">PKp2</span><span class="c0">&nbsp;(providing precursors for fatty acids) </span><span class="c12">15</span><span class="c0">&nbsp;and transporters ensuring nutrient delivery to the seed, such as sugar transporters (</span><span class="c1">Arahy.14EF4H</span><span class="c0">) </span><span class="c12">31</span><span class="c0">, are directly implicated by their location in QTLs or altered expression in mutants. Master regulators like </span><span class="c1">LEC1</span><span class="c0">&nbsp;orchestrate the accumulation of storage products.</span><span class="c8">15</span></li><li class="c11 c20 li-bullet-0"><span class="c17">Ubiquitin-Proteasome Pathway:</span><span class="c0">&nbsp;This pathway, responsible for targeted protein degradation, is known to regulate many developmental processes, including seed size.</span><span class="c12">10</span><span class="c0">&nbsp;The identification of the F-box SNE protein (</span><span class="c1">Aradu.RLZ61</span><span class="c0">) as a candidate within the A07 QTL provides a specific link, suggesting that regulated protein turnover is part of the seed size control network in peanut.</span><span class="c8">39</span></li></ul><p class="c21"><span class="c0 c9">Collectively, these findings suggest that peanut seed size is not governed by a single pathway but rather emerges from the complex interplay of these conserved developmental modules. Hormone signals likely integrate with cell cycle controls and transcriptional networks to regulate cell proliferation and expansion, while metabolic pathways, also under transcriptional control (e.g., by LEC1), determine the extent of storage reserve deposition.</span></p><h3 class="c15"><span class="c27 c9">C. Challenges and Opportunities</span></h3><p class="c18"><span class="c0">Despite significant progress, elucidating the precise genetic mechanisms controlling peanut seed size faces ongoing challenges. The allotetraploid nature of the genome, with its duplicated genes (homoeologs), complicates functional analysis, as redundancy can mask the effects of single gene mutations.</span><span class="c12">1</span><span class="c0">&nbsp;Dissecting the specific contributions of A and B subgenome copies of candidate genes remains difficult. The substantial influence of environmental factors (GxE interaction) means that QTL effects can vary across locations and years, requiring extensive multi-environment testing to identify stable loci suitable for breeding applications.</span><span class="c12">4</span><span class="c0">&nbsp;Furthermore, distinguishing between true pleiotropy (one gene affecting multiple traits) and tight physical linkage of multiple genes within QTL hotspots often requires very high-resolution mapping or functional studies.</span><span class="c12">53</span><span class="c0">&nbsp;Perhaps most critically, definitive functional validation of most candidate genes identified through mapping or expression studies is still lacking in peanut itself.</span><span class="c8">10</span></p><p class="c18"><span class="c0">However, the field is also presented with unprecedented opportunities. The availability of high-quality reference genome sequences for cultivated peanut and its diploid progenitors, coupled with comprehensive genomic databases like PeanutBase, provides essential infrastructure for gene discovery, marker design, and comparative genomics.</span><span class="c12">2</span><span class="c0">&nbsp;Advanced mapping strategies, including GWAS on diverse germplasm panels, QTL-seq which combines bulked segregant analysis with sequencing, and high-resolution fine-mapping using large populations and dense markers, are continually improving the precision of QTL localization.</span><span class="c12">12</span><span class="c0">&nbsp;The advent of efficient genome editing technologies, including CRISPR/Cas9 systems and base editors adapted for peanut </span><span class="c12">8</span><span class="c0">, offers powerful tools for targeted gene knockout or modification, enabling direct functional validation of candidate genes in peanut. Moreover, the systematic exploration of genetic diversity from wild </span><span class="c1">Arachis</span><span class="c0">&nbsp;species, facilitated by the development of synthetic tetraploids and introgression lines (like AB-QTL populations and CSSLs), provides a rich source of novel alleles for traits like seed size that can be introduced into elite breeding material.</span><span class="c12">1</span><span class="c0">&nbsp;The successful validation of QTLs from wild species in CSSL backgrounds demonstrates the utility of this approach.</span><span class="c12">4</span><span class="c0 c9">&nbsp;These converging technological and resource advancements position the field to overcome previous limitations and make significant strides in understanding and manipulating the genes controlling peanut seed size.</span></p><h3 class="c15"><span class="c27 c9">D. Implications for Marker-Assisted Selection (MAS) in Peanut Breeding</span></h3><p class="c18"><span class="c0">The identification and characterization of QTLs and candidate genes for seed size have direct implications for improving peanut breeding efficiency through Marker-Assisted Selection (MAS). Stable, major QTLs identified on chromosomes like A05, A07, A08, B06, and B07, along with the molecular markers tightly linked to them, represent valuable assets for breeders.</span><span class="c12">7</span><span class="c0 c9">&nbsp;MAS allows breeders to select for desirable alleles indirectly using linked markers, which can be particularly advantageous for quantitative traits that are difficult or expensive to phenotype directly, or that exhibit significant GxE interaction.</span></p><p class="c18"><span class="c0">Fine-mapping efforts that narrow QTL intervals </span><span class="c12">24</span><span class="c0">&nbsp;are crucial for developing more accurate and reliable markers for MAS, as markers closer to the causal gene are less likely to become unlinked from the trait during recombination. Ideally, identifying the causal gene itself allows for the development of &#39;perfect&#39; or diagnostic markers based on the functional polymorphism within the gene (e.g., the SNP in </span><span class="c1">EVM0025654</span><span class="c0">&nbsp;</span><span class="c12">14</span><span class="c0 c9">). Such markers are generally more robust across different genetic backgrounds compared to linked markers, which might only be informative within the original mapping population.</span></p><p class="c18"><span class="c0">Furthermore, understanding the biological function of the underlying genes and pathways </span><span class="c12">4</span><span class="c0">&nbsp;provides breeders with knowledge to design more targeted strategies. For example, knowing whether a QTL allele acts positively or negatively, or understanding potential pleiotropic effects or genetic correlations with other traits (like the potential link between seed size and protein/oil content </span><span class="c12">54</span><span class="c0 c9">), allows for more informed decisions about which alleles to select or pyramid into elite cultivars. The ultimate goal is to move beyond using anonymous linked markers towards deploying gene-based markers that directly track the functionally relevant alleles controlling seed size.</span></p><h3 class="c15"><span class="c27 c9">E. Concluding Remarks</span></h3><p class="c18"><span class="c0 c9">Substantial advancements have been achieved in identifying the genetic architecture underlying seed size variation in cultivated peanut. Integrated approaches combining QTL mapping, transcriptomics, mutant analysis, and comparative genomics have successfully pinpointed key genomic regions on chromosomes A05, A07, A08, B06, and B07 that harbor major genetic determinants of seed weight and dimensions. Several candidate genes and gene families, including PPR proteins, NAC transcription factors, CYP450s, and the master regulator LEC1, have emerged as significant players in this complex trait, acting through diverse pathways involving hormone signaling, cell cycle control, transcriptional regulation, and metabolic allocation.</span></p><p class="c34"><span class="c0 c9">While challenges related to peanut&#39;s polyploid genome and the quantitative nature of seed size persist, the growing arsenal of genomic resources and molecular tools provides a clear path forward. Future research priorities should include the rigorous functional validation of the most promising candidate genes, potentially using genome editing techniques, to confirm their roles and elucidate their precise mechanisms of action. Unraveling the intricate regulatory networks that connect hormonal cues, developmental programs, and metabolic fluxes to determine final seed size remains a key objective. Continued efforts in fine-mapping, combined with functional studies, will be essential for developing diagnostic markers based on causal polymorphisms. This deeper mechanistic understanding will ultimately empower breeders to more effectively and efficiently develop improved peanut cultivars with optimized seed size characteristics tailored to specific yield goals and market requirements, contributing to the sustainability and productivity of this vital global crop.</span></p><h4 class="c33"><span class="c9 c30">Works cited</span></h4><ol class="c19 lst-kix_list_11-0 start" start="1"><li class="c2 li-bullet-0"><span class="c5">An Overview of Mapping Quantitative Trait Loci in Peanut (Arachis hypogaea L.) - PMC, accessed April 22, 2025, </span><span class="c5 c14"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10298601/&amp;sa=D&amp;source=editors&amp;ust=1745373066940864&amp;usg=AOvVaw3GoDHjqNHIuqlGTaGnrQIK">https://pmc.ncbi.nlm.nih.gov/articles/PMC10298601/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Pod and Seed Trait QTL Identification To Assist Breeding for Peanut ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://academic.oup.com/g3journal/article/10/7/2297/6026321&amp;sa=D&amp;source=editors&amp;ust=1745373066941462&amp;usg=AOvVaw07x_LPELKA1pRCh6UGjUt5">https://academic.oup.com/g3journal/article/10/7/2297/6026321</a></span></li><li class="c2 li-bullet-0"><span class="c5">Identification of a Major Locus for Flowering Pattern Sheds Light on Plant Architecture Diversification in Cultivated Peanut | bioRxiv, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.biorxiv.org/content/10.1101/2021.03.18.435916.full&amp;sa=D&amp;source=editors&amp;ust=1745373066942044&amp;usg=AOvVaw1trpJ9WLvfZ6UrTHv67-yA">https://www.biorxiv.org/content/10.1101/2021.03.18.435916.full</a></span></li><li class="c2 li-bullet-0"><span class="c5">Assessment of 16 Peanut (Arachis hypogaea L.) CSSLs Derived from an Interspecific Cross for Yield and Yield Component Traits: QTL Validation - MDPI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4395/10/4/583&amp;sa=D&amp;source=editors&amp;ust=1745373066942584&amp;usg=AOvVaw04B8P-8kRLPhu6Vx76tcxK">https://www.mdpi.com/2073-4395/10/4/583</a></span></li><li class="c2 li-bullet-0"><span class="c5">Full article: Groundnut (Arachis hypogaea L.) improvement in sub-Saharan Africa: a review, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.tandfonline.com/doi/full/10.1080/09064710.2019.1601252&amp;sa=D&amp;source=editors&amp;ust=1745373066943092&amp;usg=AOvVaw37z545y3ObCGnGV_82YMWg">https://www.tandfonline.com/doi/full/10.1080/09064710.2019.1601252</a></span></li><li class="c2 li-bullet-0"><span class="c5">Peanut Genome Sequenced with Unprecedent Accuracy - USDA ARS, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.ars.usda.gov/news-events/news/research-news/2019/peanut-genome-sequenced-with-unprecedent-accuracy/&amp;sa=D&amp;source=editors&amp;ust=1745373066943816&amp;usg=AOvVaw0ygK6a2Pc95yH1tvvbyoCz">https://www.ars.usda.gov/news-events/news/research-news/2019/peanut-genome-sequenced-with-unprecedent-accuracy/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Association Analysis Identified Quantitative Trait Loci (QTLs) Underlying Drought-Related Traits in Cultivated Peanut (Arachis hypogaea L.) - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11276114/&amp;sa=D&amp;source=editors&amp;ust=1745373066944412&amp;usg=AOvVaw1CyevmSyAn5pFG87rRX4uj">https://pmc.ncbi.nlm.nih.gov/articles/PMC11276114/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Base Editing in Peanut Using CRISPR/nCas9 - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9133374/&amp;sa=D&amp;source=editors&amp;ust=1745373066944801&amp;usg=AOvVaw1RiB56qiGsR9o72K_lTDGf">https://pmc.ncbi.nlm.nih.gov/articles/PMC9133374/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Summary of identified candidate genes related to the seed size/weight... - ResearchGate, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/figure/Summary-of-identified-candidate-genes-related-to-the-seed-sizeweight-phenotype-based-on_tbl1_351700850&amp;sa=D&amp;source=editors&amp;ust=1745373066945508&amp;usg=AOvVaw22ftjWzI_0alGipSDiOxcD">https://www.researchgate.net/figure/Summary-of-identified-candidate-genes-related-to-the-seed-sizeweight-phenotype-based-on_tbl1_351700850</a></span></li><li class="c2 li-bullet-0"><span class="c5">Multi-Omics Profiling Identifies Candidate Genes Controlling Seed ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2223-7747/11/23/3276&amp;sa=D&amp;source=editors&amp;ust=1745373066945951&amp;usg=AOvVaw0wrhfS5C3-VgwvsW-aMub-">https://www.mdpi.com/2223-7747/11/23/3276</a></span></li><li class="c2 li-bullet-0"><span class="c5">(PDF) Global Transcriptome and Co-Expression Network Analyses Revealed Hub Genes Controlling Seed Size/Weight and/or Oil Content in Peanut - ResearchGate, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/373618956_Global_Transcriptome_and_Co-Expression_Network_Analyses_Revealed_Hub_Genes_Controlling_Seed_SizeWeight_andor_Oil_Content_in_Peanut&amp;sa=D&amp;source=editors&amp;ust=1745373066946840&amp;usg=AOvVaw041Oxtbdkcjh1fvtujg8Pq">https://www.researchgate.net/publication/373618956_Global_Transcriptome_and_Co-Expression_Network_Analyses_Revealed_Hub_Genes_Controlling_Seed_SizeWeight_andor_Oil_Content_in_Peanut</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genetic Diversity and Genome-Wide Association Study of Seed Aspect Ratio Using a High-Density SNP Array in Peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7822046/&amp;sa=D&amp;source=editors&amp;ust=1745373066947415&amp;usg=AOvVaw0vuVCJHlc3zXSGLEbPfctw">https://pmc.ncbi.nlm.nih.gov/articles/PMC7822046/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Coordinated Lipid Mobilization during Seed Development and Germination in Peanut (Arachis hypogaea L.) | Journal of Agricultural and Food Chemistry - ACS Publications, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pubs.acs.org/doi/10.1021/acs.jafc.3c06697&amp;sa=D&amp;source=editors&amp;ust=1745373066948045&amp;usg=AOvVaw1U0vPfdaSDUB0eSV_PIFmy">https://pubs.acs.org/doi/10.1021/acs.jafc.3c06697</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comprehensive Transcriptome Analyses Reveal ... - Frontiers, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2021.666483/full&amp;sa=D&amp;source=editors&amp;ust=1745373066948585&amp;usg=AOvVaw3LcV8DVS2DTBsoEP0rUMC9">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2021.666483/full</a></span></li><li class="c2 li-bullet-0"><span class="c5">Transcriptome Analysis and Gene Expression Profiling of the Peanut Small Seed Mutant Identified Genes Involved in Seed Size Control, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9456316/&amp;sa=D&amp;source=editors&amp;ust=1745373066949299&amp;usg=AOvVaw0PnTiuQHnJMT5DqXXYUzhy">https://pmc.ncbi.nlm.nih.gov/articles/PMC9456316/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Transcriptome Analysis and Gene Expression Profiling of the Peanut Small Seed Mutant Identified Genes Involved in Seed Size Control - PubMed, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/36077124/&amp;sa=D&amp;source=editors&amp;ust=1745373066950061&amp;usg=AOvVaw1Hye9nHCQArzaOSUG7GWHT">https://pubmed.ncbi.nlm.nih.gov/36077124/</a></span></li><li class="c2 li-bullet-0"><span class="c5">QTL identification for seed weight and size based on a high-density ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6892246/&amp;sa=D&amp;source=editors&amp;ust=1745373066950618&amp;usg=AOvVaw3lMZhi7JxjavTyVEbHhx41">https://pmc.ncbi.nlm.nih.gov/articles/PMC6892246/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Transcriptome analysis of pod mutant reveals plant hormones are important regulators in controlling pod size in peanut (Arachis hypogaea L.) - PeerJ, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://peerj.com/articles/12965/&amp;sa=D&amp;source=editors&amp;ust=1745373066951362&amp;usg=AOvVaw0QOccymBbNLipfvAIOpRZr">https://peerj.com/articles/12965/</a></span></li><li class="c2 li-bullet-0"><span class="c5">(PDF) Transcriptome Analysis and Gene Expression Profiling of the Peanut Small Seed Mutant Identified Genes Involved in Seed Size Control - ResearchGate, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/363063765_Transcriptome_Analysis_and_Gene_Expression_Profiling_of_the_Peanut_Small_Seed_Mutant_Identified_Genes_Involved_in_Seed_Size_Control&amp;sa=D&amp;source=editors&amp;ust=1745373066952499&amp;usg=AOvVaw2gaKEut9Z7UYiHOTRNdWD6">https://www.researchgate.net/publication/363063765_Transcriptome_Analysis_and_Gene_Expression_Profiling_of_the_Peanut_Small_Seed_Mutant_Identified_Genes_Involved_in_Seed_Size_Control</a></span></li><li class="c2 li-bullet-0"><span class="c5">Transcriptome Analysis and Gene Expression Profiling of the Peanut Small Seed Mutant Identified Genes Involved in Seed Size Control - MDPI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/1422-0067/23/17/9726&amp;sa=D&amp;source=editors&amp;ust=1745373066953068&amp;usg=AOvVaw3vCaUNS1C47kREUyjjIxN-">https://www.mdpi.com/1422-0067/23/17/9726</a></span></li><li class="c2 li-bullet-0"><span class="c5">GENETIC ANALYSIS OF SEED SIZE AND SEED RELATED TRAITS IN SELECTED GROUNDNUT (Arachis hypogaea L.) GENOTYPES - ResearchGate, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/390617992_GENETIC_ANALYSIS_OF_SEED_SIZE_AND_SEED_RELATED_TRAITS_IN_SELECTED_GROUNDNUT_Arachis_hypogaea_L_GENOTYPES&amp;sa=D&amp;source=editors&amp;ust=1745373066953980&amp;usg=AOvVaw1uqMiuh9UfPv25ETiC0Hyg">https://www.researchgate.net/publication/390617992_GENETIC_ANALYSIS_OF_SEED_SIZE_AND_SEED_RELATED_TRAITS_IN_SELECTED_GROUNDNUT_Arachis_hypogaea_L_GENOTYPES</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comprehensive Transcriptome Analyses Reveal Candidate Genes for Variation in Seed Size/Weight During Peanut (Arachis hypogaea L.) Domestication - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC8170302/&amp;sa=D&amp;source=editors&amp;ust=1745373066954594&amp;usg=AOvVaw1vV6vwMBbA4evVdMGKMb3-">https://pmc.ncbi.nlm.nih.gov/articles/PMC8170302/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Mapping of Quantitative Trait Loci for Yield-Attributing Traits of Peanut, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/377598073_Genome-Wide_Mapping_of_Quantitative_Trait_Loci_for_Yield-Attributing_Traits_of_Peanut&amp;sa=D&amp;source=editors&amp;ust=1745373066955322&amp;usg=AOvVaw22b_NQfAn86rqfPR4Iy-Dy">https://www.researchgate.net/publication/377598073_Genome-Wide_Mapping_of_Quantitative_Trait_Loci_for_Yield-Attributing_Traits_of_Peanut</a></span></li><li class="c2 li-bullet-0"><span class="c5">Identification of Hotspot Regions for Candidate Genes Associated ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2077-0472/14/9/1634&amp;sa=D&amp;source=editors&amp;ust=1745373066955754&amp;usg=AOvVaw13lplbJKJKXQOlxYCgD_RG">https://www.mdpi.com/2077-0472/14/9/1634</a></span></li><li class="c2 li-bullet-0"><span class="c5">Inheritance of traits associated with seed size in groundnut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/225781012_Inheritance_of_traits_associated_with_seed_size_in_groundnut_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1745373066956498&amp;usg=AOvVaw2LNDtb-NSH4m5jYRNhLBqi">https://www.researchgate.net/publication/225781012_Inheritance_of_traits_associated_with_seed_size_in_groundnut_Arachis_hypogaea_L</a></span></li><li class="c2 li-bullet-0"><span class="c5">Combining Ability for Large Pod and Seed Traits in Peanut&#39; - Allen Press, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://meridian.allenpress.com/peanut-science/article-pdf/20/1/49/1855495/i0095-3679-20-1-13.pdf&amp;sa=D&amp;source=editors&amp;ust=1745373066957235&amp;usg=AOvVaw1XSGoWD--w_qOyXYa_Nda6">https://meridian.allenpress.com/peanut-science/article-pdf/20/1/49/1855495/i0095-3679-20-1-13.pdf</a></span></li><li class="c2 li-bullet-0"><span class="c5">Major seed size QTL on chromosome A05 of peanut (Arachis hypogaea) is conserved in the US mini core germplasm collection - ResearchGate, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.researchgate.net/publication/338090845_Major_seed_size_QTL_on_chromosome_A05_of_peanut_Arachis_hypogaea_is_conserved_in_the_US_mini_core_germplasm_collection&amp;sa=D&amp;source=editors&amp;ust=1745373066958297&amp;usg=AOvVaw2K-qJTHHo-WNcmqlOj3UyB">https://www.researchgate.net/publication/338090845_Major_seed_size_QTL_on_chromosome_A05_of_peanut_Arachis_hypogaea_is_conserved_in_the_US_mini_core_germplasm_collection</a></span></li><li class="c2 li-bullet-0"><span class="c5">groundnut improvement network for Africa (GINA) germplasm collection: a unique genetic resource for breeding and gene discovery - Oxford Academic, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://academic.oup.com/g3journal/article/14/1/jkad244/7329280&amp;sa=D&amp;source=editors&amp;ust=1745373066958952&amp;usg=AOvVaw14ivKTv6BKAPqWTjZMxKVS">https://academic.oup.com/g3journal/article/14/1/jkad244/7329280</a></span></li><li class="c2 li-bullet-0"><span class="c5">Fine-Mapping of a Wild Genomic Region Involved in Pod and Seed Size Reduction on Chromosome A07 in Peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7761091/&amp;sa=D&amp;source=editors&amp;ust=1745373066959537&amp;usg=AOvVaw2NlMrdQdEBXCgGb8qkGraG">https://pmc.ncbi.nlm.nih.gov/articles/PMC7761091/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comparative RNA-Seq profiling of a resistant and susceptible peanut (Arachis hypogaea) genotypes in response to leaf rust infection caused by Puccinia arachidis - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7266923/&amp;sa=D&amp;source=editors&amp;ust=1745373066960183&amp;usg=AOvVaw0zOqs4w1zJYOJJIGMZSLa-">https://pmc.ncbi.nlm.nih.gov/articles/PMC7266923/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Mapping Quantitative Trait Loci (QTLs) for Hundred-Pod and Hundred-Seed Weight under Seven Environments in a Recombinant Inbred Line Population of Cultivated Peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10531390/&amp;sa=D&amp;source=editors&amp;ust=1745373066960899&amp;usg=AOvVaw12TMpDU_aEYFDJRpmHrI5w">https://pmc.ncbi.nlm.nih.gov/articles/PMC10531390/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-wide analysis of R2R3-MYB genes in cultivated peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9971814/&amp;sa=D&amp;source=editors&amp;ust=1745373066961407&amp;usg=AOvVaw0ZBx3zjkiT16hTbIiN3bAP">https://pmc.ncbi.nlm.nih.gov/articles/PMC9971814/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Analysis and Expression Profiles of AhCOLs Family in Peanut (Arachis hypogaea L.) - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/&amp;sa=D&amp;source=editors&amp;ust=1745373066961949&amp;usg=AOvVaw3NwaWpB9wiOX0pwz9vlHmm">https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Evaluation of linkage disequilibrium, population structure, and genetic diversity in the U.S. peanut mini core collection - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6558826/&amp;sa=D&amp;source=editors&amp;ust=1745373066962511&amp;usg=AOvVaw2AEDnp-QOHwq4tvKBnB_ym">https://pmc.ncbi.nlm.nih.gov/articles/PMC6558826/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comparative transcriptomics analysis of developing ... - Frontiers, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.958808/full&amp;sa=D&amp;source=editors&amp;ust=1745373066963055&amp;usg=AOvVaw0JibDVDlVpicbVFcVkr5m5">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.958808/full</a></span></li><li class="c2 li-bullet-0"><span class="c5">Phenotyping and genotyping parents of sixteen recombinant inbred peanut populations, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://meridian.allenpress.com/peanut-science/article/45/1/1/108564/Phenotyping-and-genotyping-parents-of-sixteen&amp;sa=D&amp;source=editors&amp;ust=1745373066963920&amp;usg=AOvVaw2vGTj9XeRR63UZLLkdAO0h">https://meridian.allenpress.com/peanut-science/article/45/1/1/108564/Phenotyping-and-genotyping-parents-of-sixteen</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comparative transcriptomics analysis of developing peanut (Arachis hypogaea L.) pods reveals candidate genes affecting peanut seed size - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9511224/&amp;sa=D&amp;source=editors&amp;ust=1745373066964681&amp;usg=AOvVaw2ILbui6Hs9gF33EY06hzx7">https://pmc.ncbi.nlm.nih.gov/articles/PMC9511224/</a></span></li><li class="c2 li-bullet-0"><span class="c5">genetics of arachis hypogaea l. - American Peanut Research and Education Society, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://apresinc.com/images/2015/12/PST-Chapter-3.pdf&amp;sa=D&amp;source=editors&amp;ust=1745373066965294&amp;usg=AOvVaw3RnednmQh7fu_eXA0XNc2h">https://apresinc.com/images/2015/12/PST-Chapter-3.pdf</a></span></li><li class="c2 li-bullet-0"><span class="c5">Fine-Mapping of a Wild Genomic Region Involved in Pod and Seed ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4425/11/12/1402&amp;sa=D&amp;source=editors&amp;ust=1745373066966027&amp;usg=AOvVaw0sPiTpYkFUsIiX3QstpnBx">https://www.mdpi.com/2073-4425/11/12/1402</a></span></li><li class="c2 li-bullet-0"><span class="c5">QTL identification for seed weight and size based on a high-density SLAF-seq genetic map in peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://d-nb.info/1207145467/34&amp;sa=D&amp;source=editors&amp;ust=1745373066967451&amp;usg=AOvVaw1mPoDEww5hBa9EKrG23ksx">https://d-nb.info/1207145467/34</a></span></li><li class="c2 li-bullet-0"><span class="c5">Mapping Quantitative Trait Locus (QTLs) for Hundred Pod and Seed Weight under Seven Environments in a RIL Cultivated Peanut (Arachis - Preprints.org, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.preprints.org/manuscript/202308.1384/v1/download&amp;sa=D&amp;source=editors&amp;ust=1745373066969028&amp;usg=AOvVaw0PyaMJKyueSVJ8mpcfcB2E">https://www.preprints.org/manuscript/202308.1384/v1/download</a></span></li><li class="c2 li-bullet-0"><span class="c5">Mapping of a QTL associated with sucrose content in peanut kernels using BSA-seq, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1089389/full&amp;sa=D&amp;source=editors&amp;ust=1745373066969997&amp;usg=AOvVaw1Ao1SjXO59p1SI2KCI2Zt7">https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.1089389/full</a></span></li><li class="c2 li-bullet-0"><span class="c5">Construction of a genetic linkage map and QTL analysis in bambara groundnut, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://cdnsciencepub.com/doi/10.1139/gen-2015-0153&amp;sa=D&amp;source=editors&amp;ust=1745373066970558&amp;usg=AOvVaw0-Xk-xIjk9qwjp8poIaL4j">https://cdnsciencepub.com/doi/10.1139/gen-2015-0153</a></span></li><li class="c2 li-bullet-0"><span class="c5">Identification of major consensus QTLs for seed size and minor ..., accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6717230/&amp;sa=D&amp;source=editors&amp;ust=1745373066971252&amp;usg=AOvVaw0L5D7ZeinQgSuo3bhfGJYc">https://pmc.ncbi.nlm.nih.gov/articles/PMC6717230/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Data_Sheet_1_Comparative transcriptomics analysis of developing, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://figshare.com/articles/dataset/Data_Sheet_1_Comparative_transcriptomics_analysis_of_developing_peanut_Arachis_hypogaea_L_pods_reveals_candidate_genes_affecting_peanut_seed_size_xlsx/21080389/1&amp;sa=D&amp;source=editors&amp;ust=1745373066972555&amp;usg=AOvVaw0u86i9SvjLbJJufjHXc2D4">https://figshare.com/articles/dataset/Data_Sheet_1_Comparative_transcriptomics_analysis_of_developing_peanut_Arachis_hypogaea_L_pods_reveals_candidate_genes_affecting_peanut_seed_size_xlsx/21080389/1</a></span></li><li class="c2 li-bullet-0"><span class="c5">PeanutBase: Home, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.peanutbase.org/&amp;sa=D&amp;source=editors&amp;ust=1745373066972937&amp;usg=AOvVaw3Rwa8WJgzyimw_H2jXX5ba">https://www.peanutbase.org/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Arachis hypogaea - Search results - Genome - Assembly - NCBI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/assembly/?term%3DArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1745373066973561&amp;usg=AOvVaw0wJyYOleEXUP6CY4ZD48Jy">https://www.ncbi.nlm.nih.gov/assembly/?term=Arachis%20hypogaea</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genomics - PeanutBase, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://dev.peanutbase.org/genomics&amp;sa=D&amp;source=editors&amp;ust=1745373066973996&amp;usg=AOvVaw3BUEN1LtsMv1awXXb06Vpu">https://dev.peanutbase.org/genomics</a></span></li><li class="c2 li-bullet-0"><span class="c5">Arachis duranensis genome assembly Aradu1.1 - NCBI - NLM, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000817695.2/&amp;sa=D&amp;source=editors&amp;ust=1745373066974603&amp;usg=AOvVaw3pmDQuDZDQtF-hanDMC5h1">https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000817695.2/</a></span></li><li class="c2 li-bullet-0"><span class="c5">arahy.Tifrunner.gnm1.KYV3 - Genome - Assembly - NCBI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/assembly/1731041&amp;sa=D&amp;source=editors&amp;ust=1745373066975135&amp;usg=AOvVaw0afYRV32hxFqIs2dx8vau7">https://www.ncbi.nlm.nih.gov/assembly/1731041</a></span></li><li class="c2 li-bullet-0"><span class="c5">Downloads - PeanutBase, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.peanutbase.org/download/&amp;sa=D&amp;source=editors&amp;ust=1745373066975538&amp;usg=AOvVaw0yyMRtkBYbslawgjqzfM9k">https://www.peanutbase.org/download/</a></span></li><li class="c2 li-bullet-0"><span class="c5">News - PeanutBase, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.peanutbase.org/news/&amp;sa=D&amp;source=editors&amp;ust=1745373066975853&amp;usg=AOvVaw3dVzpfo_Nk2OXM2uZwbFEY">https://www.peanutbase.org/news/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Fine mapping of qAHPS07 and functional studies of AhRUVBL2 controlling pod size in peanut (Arachis hypogaea L.), accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10440995/&amp;sa=D&amp;source=editors&amp;ust=1745373066976480&amp;usg=AOvVaw3Esw4hPDxnDKmtZbPuRCrY">https://pmc.ncbi.nlm.nih.gov/articles/PMC10440995/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Identification of a Major QTL for Seed Protein Content in Cultivated Peanut (Arachis hypogaea L.) Using QTL-Seq - MDPI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2223-7747/13/17/2368&amp;sa=D&amp;source=editors&amp;ust=1745373066977107&amp;usg=AOvVaw3JaMUp13s_t7ponor7-Nvw">https://www.mdpi.com/2223-7747/13/17/2368</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Analysis of Pentatricopeptide Repeat Gene Family in Peanut and Identification of AhPPR598 Resistance to Ralstonia solanacearum - MDPI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2077-0472/14/2/195&amp;sa=D&amp;source=editors&amp;ust=1745373066977690&amp;usg=AOvVaw3TvUvcY8s5pHjenoJEOpUV">https://www.mdpi.com/2077-0472/14/2/195</a></span></li><li class="c2 li-bullet-0"><span class="c5">Functioning of PPR Proteins in Organelle RNA Metabolism and Chloroplast Biogenesis, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7900629/&amp;sa=D&amp;source=editors&amp;ust=1745373066978170&amp;usg=AOvVaw2RmMKgTDt_pe_2Xl1x7PH_">https://pmc.ncbi.nlm.nih.gov/articles/PMC7900629/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Comprehensive genomic characterization of NAC transcription factor family and their response to salt and drought stress in peanut - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7532626/&amp;sa=D&amp;source=editors&amp;ust=1745373066978751&amp;usg=AOvVaw06iA6gAGld3Ax2XuUKZgqw">https://pmc.ncbi.nlm.nih.gov/articles/PMC7532626/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Identification of NAC Transcription Factors and Their Functional Prediction of Abiotic Stress Response in Peanut - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7985091/&amp;sa=D&amp;source=editors&amp;ust=1745373066979331&amp;usg=AOvVaw1t0JcpWDCXmwKaYq7AVoTE">https://pmc.ncbi.nlm.nih.gov/articles/PMC7985091/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Cloning and functional characterization of seed-specific LEC1A promoter from peanut (Arachis hypogaea L.) - PMC - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7984638/&amp;sa=D&amp;source=editors&amp;ust=1745373066979881&amp;usg=AOvVaw2Im0FOOx954ILF3j0PWBw5">https://pmc.ncbi.nlm.nih.gov/articles/PMC7984638/</a></span></li><li class="c2 li-bullet-0"><span class="c5">LEAFY COTYLEDON1-LIKE Defines a Class of Regulators Essential for Embryo Development - PMC - PubMed Central, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC143447/&amp;sa=D&amp;source=editors&amp;ust=1745373066980424&amp;usg=AOvVaw1aEzztOfrdDnsZmuwPEWY1">https://pmc.ncbi.nlm.nih.gov/articles/PMC143447/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Peanut LEAFY COTYLEDON1-type genes participate in regulating the embryo development and the accumulation of storage lipids - PubMed, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/38643320/&amp;sa=D&amp;source=editors&amp;ust=1745373066980977&amp;usg=AOvVaw2RlQ_VumEzmcCtuavYpwPe">https://pubmed.ncbi.nlm.nih.gov/38643320/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Cloning and Functional characterization of seed-specific LEC1A promoter from peanut (Arachis hypogaea L.) | bioRxiv, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.biorxiv.org/content/10.1101/2020.11.13.381327.full&amp;sa=D&amp;source=editors&amp;ust=1745373066981610&amp;usg=AOvVaw1P3OAWVY4pQaLoNcggmZEs">https://www.biorxiv.org/content/10.1101/2020.11.13.381327.full</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Identification of Peanut Pyruvate Kinase Gene Family and Their Potential Roles in Seed Germination and Drought Stress Responses - MDPI, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://www.mdpi.com/2311-7524/11/2/200&amp;sa=D&amp;source=editors&amp;ust=1745373066982350&amp;usg=AOvVaw0xbV1HDNNv2ftT1Dg2cs6M">https://www.mdpi.com/2311-7524/11/2/200</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Analysis and Functional Characterization of Pyruvate Kinase (PK) Gene Family Modulating Rice Yield and Quality, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9739881/&amp;sa=D&amp;source=editors&amp;ust=1745373066983096&amp;usg=AOvVaw2deNRV-q1JViE28C57uSx2">https://pmc.ncbi.nlm.nih.gov/articles/PMC9739881/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Genome-Wide Analysis and Functional Characterization of Pyruvate Kinase (PK) Gene Family Modulating Rice Yield and Quality - PubMed, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/36499684/&amp;sa=D&amp;source=editors&amp;ust=1745373066983775&amp;usg=AOvVaw0JVKXny18ECkONE80TNPGQ">https://pubmed.ncbi.nlm.nih.gov/36499684/</a></span></li><li class="c2 li-bullet-0"><span class="c5">PericarpPt5-Seed_Pattee10_down - PeanutBase, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://dev.peanutbase.org/files/expression/arahy_atlas_html/PericarpPt5-Seed_Pattee10_down.html&amp;sa=D&amp;source=editors&amp;ust=1745373066984469&amp;usg=AOvVaw0LF4rCmpVF3dlRMXTxWGy1">https://dev.peanutbase.org/files/expression/arahy_atlas_html/PericarpPt5-Seed_Pattee10_down.html</a></span></li><li class="c2 li-bullet-0"><span class="c5">Multi-Omics Profiling Identifies Candidate Genes Controlling Seed Size in Peanut - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9740956/&amp;sa=D&amp;source=editors&amp;ust=1745373066985015&amp;usg=AOvVaw0-EHYCaaf_Z_iBFpJr6Kma">https://pmc.ncbi.nlm.nih.gov/articles/PMC9740956/</a></span></li><li class="c2 li-bullet-0"><span class="c5">Proteomic Analysis of Arachis hypogaea Seeds from Different Maturity Classes - PMC, accessed April 22, 2025, </span><span class="c14 c5"><a class="c3" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11054812/&amp;sa=D&amp;source=editors&amp;ust=1745373066985618&amp;usg=AOvVaw1TQ2S7O30ILX2txpD2DoRG">https://pmc.ncbi.nlm.nih.gov/articles/PMC11054812/</a></span></li></ol></body></html>