<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-5>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}ol.lst-kix_list_3-0{list-style-type:none}.lst-kix_list_3-0>li:before{content:"" counter(lst-ctn-kix_list_3-0,decimal) ". "}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}ol.lst-kix_list_3-0.start{counter-reset:lst-ctn-kix_list_3-0 0}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_3-1{list-style-type:none}.lst-kix_list_3-5>li:before{content:"\0025a0   "}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_list_3-7>li:before{content:"\0025a0   "}ul.lst-kix_list_2-8{list-style-type:none}ul.lst-kix_list_2-2{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_3-0>li{counter-increment:lst-ctn-kix_list_3-0}.lst-kix_list_1-7>li:before{content:"\0025a0   "}.lst-kix_list_1-5>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"\0025cf   "}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}.lst-kix_list_2-3>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c14{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:66.9pt;border-top-color:#000000;border-bottom-style:solid}.c19{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:156pt;border-top-color:#000000;border-bottom-style:solid}.c6{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:117pt;border-top-color:#000000;border-bottom-style:solid}.c0{color:#575b5f;font-weight:400;text-decoration:none;vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal}.c12{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c9{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c23{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c26{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c16{color:#1b1c1d;font-weight:700;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c1{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c24{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans";font-style:normal}.c15{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c7{font-size:10pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c33{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c22{font-size:10pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c29{font-size:16pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c21{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left;height:11pt}.c8{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c31{border-spacing:0;border-collapse:collapse;margin-right:auto}.c28{padding-top:12pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c30{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c11{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c3{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c5{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c10{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c27{padding-top:0pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c32{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c4{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c2{text-decoration:none;vertical-align:baseline}.c13{color:inherit;text-decoration:inherit}.c25{padding:0;margin:0}.c20{margin-left:23.2pt;padding-left:0pt}.c17{vertical-align:baseline;font-style:normal}.c18{height:0pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c32 doc-content"><h1 class="c10"><span class="c29">Genetic and Locus Control of High Oleic Acid Trait in Peanut (</span><span class="c33">Arachis hypogaea</span><span class="c29 c2">)</span></h1><h2 class="c10"><span class="c24 c2">1. Executive Summary</span></h2><p class="c5"><span class="c1">The pursuit of high oleic acid (OA) in peanut (</span><span class="c8">Arachis hypogaea</span><span class="c1">) represents a pivotal objective in modern agricultural breeding, driven by significant enhancements in oil quality, extended shelf life, and notable health benefits. This report details the genetic underpinnings of this desirable trait, primarily focusing on the critical role of specific genes and loci. The high OA characteristic is predominantly governed by homozygous recessive mutations in the homeologous </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1 c2">&nbsp;genes. These genes encode the fatty acid desaturase 2 (FAD2) enzyme, which is responsible for converting monounsaturated oleic acid to polyunsaturated linoleic acid. While these major genes exert a profound influence, the inheritance pattern is complex, involving quantitative trait loci (QTLs) and intricate epistatic interactions, indicating a polygenic component. Furthermore, the genetic modification leading to high OA can induce pleiotropic effects, notably impacting plant growth and development. To navigate these complexities and integrate the high OA trait with other crucial agronomic characteristics such as yield and disease resistance, advanced molecular breeding strategies, including marker-assisted selection (MAS) and precise gene editing technologies like CRISPR/Cas9, are being extensively employed. These integrated approaches are essential for developing robust and commercially viable high oleic peanut cultivars.</span></p><h2 class="c10"><span class="c24 c2">2. Introduction: The Significance of High Oleic Acid in Peanut</span></h2><p class="c5"><span class="c1">The quality of peanut oil is largely determined by its fatty acid composition, with oleic acid (C18:1) and linoleic acid (C18:2) together constituting approximately 80% of the total oil content.</span><span class="c9">1</span><span class="c1">&nbsp;A high ratio of oleic to linoleic acid (O/L ratio) is a highly sought-after trait due to its multifaceted benefits for both the peanut industry and consumers.</span><span class="c0">1</span></p><p class="c5"><span class="c1">From a commercial perspective, high oleic peanut oil exhibits significantly improved oxidative stability, demonstrating up to a 10-fold higher anti-oxidative stability compared to oil with a conventional fatty acid profile.</span><span class="c9">1</span><span class="c1">&nbsp;This enhanced stability directly translates into an extended shelf life for peanut-derived products, minimizing spoilage and associated economic losses for manufacturers and retailers. Furthermore, oil rich in unsaturated fatty acids, such as oleic acid, can withstand higher temperatures without smoking, leading to faster cooking times and reduced oil absorption by cooked foods, which is advantageous for food processors.</span><span class="c9">1</span><span class="c1">&nbsp;The adoption of high O/L ratio peanuts, especially when combined with high oil content, directly contributes to the profitability of growers and the broader peanut industry, aligning with evolving consumer preferences for healthier and more stable food options.</span><span class="c0">1</span></p><p class="c5"><span class="c1">For consumers, the health benefits associated with high oleic acid consumption are substantial. Oleic acid has been linked to the reduction of serum cholesterol levels, suppression of tumorigenesis, and amelioration of inflammatory diseases.</span><span class="c9">1</span><span class="c1">&nbsp;Notably, while both oleic and linoleic acids are known to lower &quot;bad&quot; cholesterol (low-density lipoprotein, LDL), oleic acid offers the distinct advantage of not negatively affecting &quot;good&quot; cholesterol (high-density lipoprotein, HDL) levels.</span><span class="c9">1</span><span class="c1 c2">&nbsp;This comprehensive impact underscores that the pursuit of high oleic acid in peanut breeding represents a strategic investment in a trait that delivers wide-ranging economic and public health dividends.</span></p><p class="c5"><span class="c1">The current O/L ratio in existing peanut germplasm typically ranges from a low of 1.0 to 2.5. In stark contrast, high oleic mutants can achieve O/L ratios of up to 40.0.</span><span class="c9">1</span><span class="c1">&nbsp;This substantial difference represents a profound, rather than a modest, enhancement in oil quality. Such a significant improvement in a key quality parameter warrants considerable investment in research and development, as it promises to fundamentally alter the market perception and utility of peanut oil, offering a distinct competitive advantage over existing varieties. While the high oleic trait is a primary focus, it is important to acknowledge that total oil content in peanut seeds is a complex, polygenic characteristic influenced by various environmental factors, including temperature, light, water availability, and nutrition.</span><span class="c9">3</span><span class="c1">&nbsp;Despite these environmental influences, significant natural variation in oil content (ranging from 32.35% to 60.26%) exists among peanut germplasm, providing ample genetic resources for breeding efforts aimed at increasing overall oil yield alongside quality.</span><span class="c0">3</span></p><h2 class="c10"><span class="c24 c2">3. Key Genes and Loci Controlling High Oleic Acid Trait</span></h2><p class="c5"><span class="c1 c2">The high oleic acid trait in peanut is primarily controlled by specific genetic loci, with the Fatty Acid Desaturase 2 (FAD2) gene family playing a central role. However, the genetic architecture extends beyond these primary determinants to involve quantitative trait loci (QTLs) and other regulatory elements.</span></p><h3 class="c10"><span class="c23 c2">The FAD2 Gene Family: Primary Determinants</span></h3><p class="c5"><span class="c1">Fatty acid desaturase 2 (FAD2) is a crucial enzyme that catalyzes the conversion of monounsaturated oleic acid (C18:1) to polyunsaturated linoleic acid (C18:2).</span><span class="c9">1</span><span class="c1">&nbsp;This enzymatic step, involving the introduction of a double bond at the &omega;6 position, is a key regulatory point in determining the O/L ratio in peanut oil. In cultivated peanut (</span><span class="c8">Arachis hypogaea</span><span class="c1">), an allotetraploid species, this enzyme is encoded by two homeologous genes: </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1">. These genes are situated on the A and B sub-genomes, respectively, and share a high degree of sequence homology (99%) in their coding regions.</span><span class="c0">1</span></p><p class="c5"><span class="c1">The characteristic high oleic acid phenotype is predominantly achieved through homozygous recessive mutations in both </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1">&nbsp;genes (aabb genotype), which results in the inactivation or significant reduction of both enzyme copies.</span><span class="c9">1</span><span class="c1">&nbsp;While single mutations in either </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;(aaBB) or </span><span class="c8">ahFAD2B</span><span class="c1">&nbsp;(AAbb) can also lead to an increase in oleic acid content, the combined effect of the double mutation is considerably more pronounced, yielding a much higher O/L ratio.</span><span class="c0">7</span></p><p class="c5"><span class="c1">The expression patterns of </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1">&nbsp;are tissue-specific, with primary detection in developing seeds and flowers.</span><span class="c9">7</span><span class="c1">&nbsp;Their expression is barely detectable in root and leaf tissues and is absent in the stem.</span><span class="c9">7</span><span class="c1">&nbsp;Within developing seeds, the expression level of </span><span class="c8">AhFAD2A/B</span><span class="c1">&nbsp;initially increases and then decreases as the seed matures.</span><span class="c9">7</span><span class="c1">&nbsp;Interestingly, despite their established major role in determining the O/L ratio, the expression levels of </span><span class="c8">AhFAD2A/B</span><span class="c1">&nbsp;were found not to be directly correlated with linoleic acid content in AABB genotype peanut seeds at different developmental stages. This suggests that these primary genes might not be directly involved in the maturity-related accumulation of linoleic acid, indicating a more complex regulatory landscape.</span><span class="c0">7</span></p><p class="c5"><span class="c1">This presence of multiple </span><span class="c8">FAD2</span><span class="c1">&nbsp;genes, each with distinct tissue-specific expression patterns (e.g., seed-specific versus constitutive) and varying correlations with fatty acid accumulation at different maturity stages, suggests a sophisticated and potentially redundant regulatory network.</span><span class="c9">7</span><span class="c1">&nbsp;While </span><span class="c8">AhFAD2A/B</span><span class="c1">&nbsp;are the primary targets for increasing seed oleic acid, other </span><span class="c8">FAD2</span><span class="c1">&nbsp;members may fine-tune fatty acid composition, play compensatory roles in vegetative tissues, or become critical under specific environmental cues. This implies that a holistic understanding of the entire </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;gene family is necessary for comprehensive control of fatty acid profiles and to anticipate or mitigate unintended pleiotropic effects in breeding programs.</span></p><h3 class="c10"><span class="c23 c2">Other FAD2 Members and Regulatory Elements</span></h3><p class="c5"><span class="c1">Beyond </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1">, other members of the </span><span class="c8">FAD2</span><span class="c1">&nbsp;gene family have been identified in peanut, each contributing to the overall fatty acid metabolism with distinct expression profiles and potential functional roles. For instance, </span><span class="c8">AhFAD2-7</span><span class="c1">&nbsp;exhibits high expression in young seedling tissues, including the root, stem, and leaf, as well as in flowers.</span><span class="c9">7</span><span class="c1">&nbsp;Its expression in developing seeds decreases with maturity, and this decline is positively correlated with the linoleic acid concentration, indicating a significant role for </span><span class="c8">AhFAD2-7</span><span class="c1">&nbsp;in the maturity-dependent accumulation and regulation of linoleic acid.</span><span class="c9">7</span><span class="c1">&nbsp;Conversely, </span><span class="c8">AhFAD2-8</span><span class="c1">&nbsp;and </span><span class="c8">AhFAD2-9</span><span class="c1">&nbsp;are predominantly expressed in flowers, with relatively lower and increasing expression levels in developing seeds as they mature.</span><span class="c9">7</span><span class="c1">&nbsp;These genes are presumed to play a secondary role in linoleic acid production in seed oil, primarily contributing to overall oleate desaturase activity in vegetative tissues.</span><span class="c9">7</span><span class="c1">&nbsp;The varying contributions of these genes are further highlighted by observations that while one study indicated </span><span class="c8">ahFAD2B</span><span class="c1">&nbsp;had a higher phenotypic effect on oleic acid accumulation than </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;</span><span class="c9">1</span><span class="c1">, other studies have presented inconsistent results.</span><span class="c9">7</span><span class="c1 c2">&nbsp;This variability underscores that the phenotypic effect of a gene is not absolute but can be modulated by factors such as genetic background, specific growing conditions, and seed maturity stages. This highlights the necessity for robust, multi-environment, and multi-genotype studies to draw reliable conclusions about gene contributions and to develop cultivars with stable high oleic phenotypes across diverse agricultural settings.</span></p><h3 class="c10"><span class="c23 c2">Quantitative Trait Loci (QTLs) and Genomic Regions</span></h3><p class="c5"><span class="c1">The genetic control of oleic acid content in peanut is complex, involving intricate quantitative trait loci (QTLs) and multiple genes that govern fatty acid biosynthesis.</span><span class="c9">12</span><span class="c1">&nbsp;Molecular mapping studies utilizing recombinant inbred line (RIL) populations have been instrumental in identifying chromosomal regions associated with oil content and quality traits. For example, one study identified 15 additive QTLs for oil content, explaining between 4.34% and 23.10% of the phenotypic variation (PVE).</span><span class="c9">3</span><span class="c1">&nbsp;A particularly significant finding was the mapping of a novel, stable, and major QTL region, designated qOCB09.1, to chromosome B09. This region spans approximately 1.99 Mb and contains 153 putative genes, notably including the </span><span class="c8">FAD2B</span><span class="c1">&nbsp;gene, which is known to influence oil content.</span><span class="c0">3</span></p><p class="c5"><span class="c1">Genome-Wide Association Studies (GWAS) have further expanded the understanding of the genetic architecture. A GWAS involving 499 peanut accessions identified 87 QTLs across 45 genomic regions that influence oil, oleic acid, linoleic acid, and the O/L ratio.</span><span class="c9">4</span><span class="c1">&nbsp;Among these, eight QTLs were found to explain more than 15% of the phenotypic variation.</span><span class="c9">4</span><span class="c1">&nbsp;A striking observation from this study was that among the 45 potential genes identified as significantly related to these four traits, only three were annotated to the fatty acid pathway.</span><span class="c9">4</span><span class="c1">&nbsp;This suggests that while </span><span class="c8">FAD2</span><span class="c1">&nbsp;genes are clearly major determinants, a substantial portion of the genetic variation in oil content and fatty acid composition is controlled by other, currently uncharacterized, genes or regulatory elements. This implies that while </span><span class="c8">FAD2</span><span class="c1">&nbsp;mutations provide a strong foundational increase in oleic acid, achieving optimal and stable high oleic phenotypes, or modifying other aspects of oil quality, will require further research to identify and understand these additional, less obvious genetic factors and their interactions. To date, a total of 36 major QTLs (PVE &gt; 10%) for oil content have been identified, with 8 of these demonstrating stability across at least three environments on chromosomes A03, A05, and A08.</span><span class="c0">3</span></p><p class="c5"><span class="c1">Table 1 provides a summary of key </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;genes and their mutations that significantly impact the high oleic acid trait in peanut.</span></p><p class="c5"><span class="c15 c2">Table 1: Key FAD2 Genes and Their Mutations Affecting Oleic Acid in Peanut</span></p><p class="c21"><span class="c15 c2"></span></p><table class="c31"><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Gene Name</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Genome Location</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Specific Mutation</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Molecular Effect</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Functional Impact</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Primary Phenotypic Effect</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Expression Pattern</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">ahFAD2A</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">A sub-genome</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">G448A (G:C to A:T substitution)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Missense mutation (D150N)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Eliminates/knocks down desaturase activity</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Increased O/L ratio, High Oleic Acid content, altered palmitic/gadoleic acid</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">Seed/Flower specific </span><span class="c0">7</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">ahFAD2B</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">B sub-genome (also in qOCB09.1)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">442insA (A insertion at +442 bp)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Frameshift, premature stop codon, loss of histidine box</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Loss-of-function desaturase activity, truncated protein</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Increased O/L ratio, High Oleic Acid content, altered palmitic/gadoleic acid</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">Seed/Flower specific </span><span class="c0">7</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">ahFAD2B</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">B sub-genome</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">C301G (C to G substitution at 301 bp)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Amino acid change (H101D)</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Alters enzymatic activity</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Increased O/L ratio, High Oleic Acid content</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">Seed/Flower specific </span><span class="c0">7</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">AhFAD2-7</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Significant role in maturity-dependent linoleic acid accumulation</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Positively correlated with linoleic acid concentration</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">High expression in young seedling tissues (root, stem, leaf), flower; decreases with seed maturity </span><span class="c0">7</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">AhFAD2-8</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Secondary role in seed linoleic acid production, contributes to overall desaturase activity in vegetative tissues</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not primary determinant of high oleic trait</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">Predominantly in flowers; low but increasing expression in developing seeds with maturity </span><span class="c0">7</span></p></td></tr><tr class="c18"><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c22 c2">AhFAD2-9</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not specified</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Secondary role in seed linoleic acid production, contributes to overall desaturase activity in vegetative tissues</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Not primary determinant of high oleic trait</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c3"><span class="c7">Predominantly in flowers; low but increasing expression in developing seeds with maturity </span><span class="c0">7</span></p></td></tr></table><h2 class="c10"><span class="c24 c2">4. Molecular Mechanisms Underlying High Oleic Acid Accumulation</span></h2><p class="c5"><span class="c1 c2">The accumulation of high oleic acid in peanut seeds is a direct consequence of specific molecular alterations that disrupt the normal fatty acid biosynthesis pathway. Understanding these mechanisms is crucial for targeted genetic manipulation.</span></p><h3 class="c10"><span class="c23 c2">Biochemical Pathway: Oleic Acid to Linoleic Acid Conversion</span></h3><p class="c5"><span class="c1">Fatty acid biosynthesis involves a series of enzymatic reactions that build and modify lipid chains. Oleic acid (C18:1), a monounsaturated fatty acid, serves as a pivotal precursor for the synthesis of polyunsaturated fatty acids. The enzymatic step that dictates the balance between oleic and linoleic acid, and thus the O/L ratio, is catalyzed by the Fatty Acid Desaturase 2 (FAD2) enzyme.</span><span class="c9">1</span><span class="c1">&nbsp;FAD2 functions as a microsomal &Delta;12 desaturase (also known as an &omega;6 desaturase), introducing a second double bond at the 12th carbon position of the oleic acid chain. This reaction converts oleic acid into linoleic acid (C18:2).</span><span class="c9">1</span><span class="c1">&nbsp;Therefore, a significant reduction or complete elimination of FAD2 enzyme activity prevents this conversion, leading to the accumulation of oleic acid in peanut seeds.</span><span class="c0">1</span></p><h3 class="c10"><span class="c23 c2">Specific Mutations and Their Functional Impact</span></h3><p class="c5"><span class="c1">The high oleic acid phenotype in peanut is primarily attributed to specific loss-of-function mutations within the homeologous </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1 c2">&nbsp;genes. These mutations are precisely characterized at the nucleotide level, providing a clear molecular blueprint for achieving the high oleic trait.</span></p><p class="c5"><span class="c1">For </span><span class="c8">ahFAD2A</span><span class="c1">, a common functional mutation involves a G:C to A:T substitution.</span><span class="c9">1</span><span class="c1">&nbsp;More precisely, this is a G448A point mutation, representing a guanine to adenine transition at position 448 base pairs after the start codon.</span><span class="c9">13</span><span class="c1">&nbsp;This nucleotide change results in a missense mutation, leading to a critical amino acid substitution from aspartic acid to asparagine (D150N) in the encoded protein.</span><span class="c9">13</span><span class="c1">&nbsp;This specific amino acid alteration is known to eliminate or significantly reduce the desaturase activity of the enzyme.</span><span class="c0">15</span></p><p class="c5"><span class="c1">In the case of </span><span class="c8">ahFAD2B</span><span class="c1">, the most extensively characterized mutation is an insertion of a single base pair, specifically an &#39;A&#39; insertion at position +442 bp (or between 441 and 442 bp) within the coding region.</span><span class="c9">1</span><span class="c1">&nbsp;This insertion causes a frameshift mutation, which subsequently leads to the generation of a premature stop codon further downstream in the gene sequence.</span><span class="c9">13</span><span class="c1">&nbsp;The resulting protein (referred to as AhFAD2B&#39;) is truncated and lacks a crucial conserved histidine box, a motif essential for the enzyme&#39;s catalytic activity.</span><span class="c9">17</span><span class="c1">&nbsp;Experimental validation through yeast expression systems has confirmed that the AhFAD2B&#39; gene product exhibits no significant &Delta;12 fatty acid desaturase activity, demonstrating its loss-of-function nature.</span><span class="c9">17</span><span class="c1">&nbsp;Another identified mutation in </span><span class="c8">ahFAD2B</span><span class="c1">&nbsp;is a C301G substitution (a cytosine to guanine substitution at 301 bp after the start codon), which results in an H101D amino acid change and similarly alters enzymatic activity.</span><span class="c9">13</span><span class="c1 c2">&nbsp;This granular understanding of specific nucleotide changes and their precise molecular consequences on protein structure, such as missense mutations, frameshifts, truncated proteins, and loss of catalytic domains, is invaluable for modern breeding. It enables rational gene editing strategies and the development of highly specific molecular markers for efficient selection, moving beyond traditional phenotype-based breeding to genotype-based precision breeding.</span></p><p class="c5"><span class="c1">An overall reduction in FAD2 activity, typically achieved through homozygous recessive mutations in both </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1">, is necessary to significantly increase the O/L ratio and accumulate high levels of oleic acid.</span><span class="c0">1</span></p><h3 class="c10"><span class="c2 c23">Impact on Overall Fatty Acid Profile</span></h3><p class="c5"><span class="c1">The primary consequence of these </span><span class="c8">FAD2</span><span class="c1">&nbsp;mutations is a substantial accumulation of oleic acid (C18:1) coupled with a concomitant decrease in linoleic acid (C18:2).</span><span class="c9">1</span><span class="c1">&nbsp;The double mutation (ol1ol1ol2ol2) has the most profound effect, leading to the highest increase in the O/L ratio.</span><span class="c0">11</span></p><p class="c5"><span class="c1">Beyond just oleic and linoleic acids, the mutant genotype significantly alters the broader fatty acid profile. It leads to an increase in gadoleic acid and a decrease in palmitic acid (C16:0).</span><span class="c9">11</span><span class="c1">&nbsp;Studies have further shown that oleic and linoleic acids can display five unique phenotypes, depending on the number of </span><span class="c8">ahFAD2</span><span class="c1">&nbsp;mutant alleles present.</span><span class="c9">15</span><span class="c1">&nbsp;This observation indicates a clear dosage effect of the functional alleles on the final fatty acid composition. The observation that </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;mutations not only increase oleic acid but also affect other fatty acids demonstrates a broader pleiotropic effect on the overall lipid profile. This highlights that modifying a central enzyme in a metabolic pathway can have cascading, sometimes unexpected, ripple effects on related biochemical compounds. For breeders, this is a crucial consideration, as desired changes in one fatty acid might inadvertently lead to undesirable changes in others, necessitating comprehensive phenotyping and a balanced approach to trait selection to ensure overall product quality.</span></p><h2 class="c10"><span class="c24 c2">5. Genetic Inheritance of the High Oleic Acid Trait</span></h2><p class="c5"><span class="c1 c2">The genetic inheritance of the high oleic acid trait in peanut is a complex interplay of major gene effects and quantitative genetic contributions, influenced by various gene actions and environmental factors.</span></p><h3 class="c10"><span class="c23 c2">Inheritance Models: Single Gene vs. Polygenic Control</span></h3><p class="c5"><span class="c1">The high oleic acid characteristic in peanut is largely attributed to the homozygous recessive mutation of the two homeologous genes, </span><span class="c8">AhFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">AhFAD2B</span><span class="c1">&nbsp;(aabb genotype).</span><span class="c9">7</span><span class="c1">&nbsp;Early studies frequently reported that this trait in natural mutant peanuts was controlled by one or two recessive genes, often designated </span><span class="c8">ol1</span><span class="c1">&nbsp;and </span><span class="c8">ol2</span><span class="c1">.</span><span class="c9">2</span><span class="c1">&nbsp;Based on </span><span class="c8">FAD2</span><span class="c1">&nbsp;gene mutations, peanut lines can be classified into four genotypes: wild type (Ol1Ol1Ol2Ol2), single functional homozygous mutation on the A genome (ol1ol1Ol2Ol2), single functional homozygous mutation on the B genome (Ol1Ol1ol2ol2), and a double mutation on both A and B genomes (ol1ol1ol2ol2).</span><span class="c9">11</span><span class="c1">&nbsp;Each of these genotypes exhibits a significantly different fatty acid profile, with the double mutation having the most pronounced effect on increasing the O/L ratio.</span><span class="c0">11</span></p><p class="c5"><span class="c1">While the major </span><span class="c8">FAD2</span><span class="c1">&nbsp;genes exert a strong influence, quantitative inheritance has also been reported for oleic acid content in peanut.</span><span class="c9">2</span><span class="c1">&nbsp;This suggests that while the </span><span class="c8">FAD2</span><span class="c1">&nbsp;mutations are primary drivers, other genes and regulatory elements contribute to the continuous variation observed in oleic acid levels. The high oleic trait in peanut is thus characterized by a dual inheritance mechanism: it is predominantly controlled by major recessive genes (</span><span class="c8">ahFAD2A/B</span><span class="c1">), yet it also exhibits significant quantitative inheritance influenced by additive, dominance, and epistatic gene actions.</span><span class="c9">2</span><span class="c1">&nbsp;This creates a breeding challenge: while simple marker-assisted selection for the </span><span class="c8">FAD2</span><span class="c1">&nbsp;mutations can rapidly establish the high oleic phenotype, optimizing the </span><span class="c8">level</span><span class="c1 c2">&nbsp;of oleic acid (e.g., achieving 90% versus 80%) and ensuring its stability across diverse environments requires a more nuanced approach that accounts for the complex quantitative genetic architecture. This implies a strategic, multi-tiered breeding approach: initial rapid selection for major gene fixation, followed by more intricate quantitative trait locus (QTL) pyramiding and rigorous environmental interaction analysis to fine-tune the trait.</span></p><h3 class="c10"><span class="c23 c2">Gene Actions and Interactions</span></h3><p class="c10"><span class="c1 c2">Genetic studies have elucidated various gene actions contributing to the inheritance of the high oleic trait:</span></p><ul class="c25 lst-kix_list_1-0 start"><li class="c10 c20 li-bullet-0"><span class="c15">Additive Effects:</span><span class="c1">&nbsp;Additive gene actions have been consistently found to be significant for oleic acid, linoleic acid, and the O/L ratio in various studies.</span><span class="c9">2</span><span class="c1 c2">&nbsp;This implies that the effects of alleles at different loci sum up to influence the trait, making selection for these alleles effective in breeding programs.</span></li><li class="c3 c20 li-bullet-0"><span class="c15">Dominance Effects:</span><span class="c1">&nbsp;Dominance gene effects are also significant for oleic and linoleic acids.</span><span class="c9">2</span><span class="c1">&nbsp;These effects are often negative and can be greater in magnitude than additive gene effects, suggesting that heterozygosity at certain loci may reduce oleic acid content.</span><span class="c0">2</span></li><li class="c3 c20 li-bullet-0"><span class="c15">Epistatic Interactions:</span><span class="c1">&nbsp;Complex gene interactions, including additive &times; additive, additive &times; dominance, and dominance &times; dominance epistasis, have been identified as significant for oleic and linoleic acids in some genetic crosses and environments.</span><span class="c9">2</span><span class="c1 c2">&nbsp;These interactions highlight the non-additive genetic control contributing to the trait&#39;s variability, making prediction and selection more challenging.</span></li><li class="c3 c20 li-bullet-0"><span class="c15">Pleiotropic Effects:</span><span class="c1">&nbsp;The </span><span class="c8">ahFAD2</span><span class="c1">&nbsp;loci exhibit pleiotropic interactions, influencing not only oleic and linoleic acids but also palmitic acid and the O/L ratio.</span><span class="c9">15</span><span class="c1 c2">&nbsp;This means that selection for high oleic acid will inevitably impact other fatty acid components, requiring breeders to consider the overall fatty acid profile.</span></li></ul><h3 class="c28"><span class="c23 c2">Heritability and Segregation Patterns</span></h3><p class="c5"><span class="c1">The heritability of the high oleic trait is a critical parameter for breeding success. Heritability estimates in the narrow sense for oleic and linoleic acids are intermediate to high (0.63-0.72 and 0.57-0.72, respectively).</span><span class="c9">18</span><span class="c1">&nbsp;High heritability indicates that a large proportion of the phenotypic variation observed is due to genetic factors, making selection for high oleic acid effective, even in early generations like F2.</span><span class="c0">2</span></p><p class="c5"><span class="c1">F2 populations derived from crosses between high-oleic and low-oleic parents typically show clear segregation for the high oleic trait, often in a digenic fashion.</span><span class="c9">15</span><span class="c1">&nbsp;The F2 generation means for oleic acid usually fall between those of the high-oleic and low-oleic parents, confirming Mendelian segregation.</span><span class="c9">2</span><span class="c1">&nbsp;The presence or absence of the G448A mutation in </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;has been shown to be a key factor determining F2 segregation ratios (e.g., 3:1 or 15:1) in specific crosses.</span><span class="c0">16</span></p><p class="c5"><span class="c1">It is important to note that oil content and fatty acid composition are influenced by environmental factors such as temperature, light, and water availability.</span><span class="c9">3</span><span class="c1">&nbsp;Significant seasonal and generation &times; season interactions for oleic acid have been observed, indicating that the phenotypic expression of high oleic acid is not solely determined by genotype but is also modulated by environmental conditions.</span><span class="c9">2</span><span class="c1">&nbsp;This underscores the critical need for multi-environment testing and selection for stable high oleic lines, rather than relying solely on genotypic selection in single environments. This also suggests a future research direction to identify specific environmental response genes or QTLs that interact with the </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;pathway, enabling breeders to develop varieties that are robustly high in oleic acid across varying growing conditions.</span></p><p class="c5"><span class="c1 c2">Table 2 summarizes the key aspects of genetic inheritance for high oleic acid in peanut.</span></p><p class="c5"><span class="c15 c2">Table 2: Summary of Genetic Inheritance Patterns for High Oleic Acid in Peanut</span></p><p class="c21"><span class="c15 c2"></span></p><table class="c31"><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Aspect of Inheritance</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Description and Key Findings</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c2 c16">Implications for Breeding</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Inheritance Model</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Predominantly digenic recessive for </span><span class="c22">ahFAD2A</span><span class="c7">&nbsp;and </span><span class="c22">ahFAD2B</span><span class="c7">&nbsp;(aabb genotype). Also exhibits quantitative inheritance with continuous variation.</span><span class="c0">2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Initial rapid selection for major gene fixation (FAD2 mutations) can establish high oleic phenotype. Fine-tuning and optimizing the </span><span class="c22">level</span><span class="c7 c2">&nbsp;of oleic acid requires accounting for quantitative genetic architecture.</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Key Genes/Loci Involved</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c22">ahFAD2A</span><span class="c7">, </span><span class="c22">ahFAD2B</span><span class="c7">&nbsp;(recessive alleles </span><span class="c22">ol1</span><span class="c7">, </span><span class="c22">ol2</span><span class="c7">). Identified QTLs also contribute to variation.</span><span class="c0">2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Direct targets for molecular breeding and gene editing. Pyramiding desirable QTLs can further enhance the trait.</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Gene Actions Identified</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Significant additive, dominance, and epistatic (additive x additive, additive x dominance, dominance x dominance) effects.</span><span class="c0">2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Additive effects support direct selection. Dominance and epistasis indicate complex interactions, requiring careful population design and selection in later generations to fix desired alleles.</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Heritability (Narrow Sense)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Intermediate to high for oleic acid (0.63-0.72).</span><span class="c0">2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">High heritability means selection for high oleic acid is effective, even in early segregating generations (e.g., F2).</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Typical Segregation Ratios (F2)</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Often digenic (e.g., 15:1 for high oleic if both genes are involved), or monogenic (3:1) if one parent is fixed for a mutation.</span><span class="c0">15</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Predictable patterns aid in identifying desired genotypes in segregating populations.</span></p></td></tr><tr class="c18"><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Environmental Influence</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7">Yes, significant influence from factors like temperature, moisture, and season. Observed generation &times; season interactions.</span><span class="c0">2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">Requires multi-environment testing and selection to identify stable high oleic lines that perform consistently across diverse growing conditions.</span></p></td></tr></table><h2 class="c10"><span class="c24 c2">6. Phenotypic Consequences and Agronomic Considerations of High Oleic Mutants</span></h2><p class="c5"><span class="c1 c2">The genetic modifications leading to high oleic acid in peanut produce a range of phenotypic changes, some highly desirable for quality, while others present significant agronomic challenges due to pleiotropic effects.</span></p><h3 class="c10"><span class="c23 c2">Desired Phenotypes</span></h3><p class="c10"><span class="c1">The primary desired phenotypic outcome of </span><span class="c8">FAD2</span><span class="c1">&nbsp;gene mutations is a substantial increase in oleic acid content and, consequently, a higher O/L ratio in peanut oil.</span><span class="c9">1</span><span class="c1 c2">&nbsp;This foundational change leads to several key benefits:</span></p><ul class="c25 lst-kix_list_2-0 start"><li class="c10 c20 li-bullet-0"><span class="c15">Enhanced Oil Quality and Stability:</span><span class="c1">&nbsp;High oleic peanut oil exhibits superior oxidative stability, resulting in a significantly longer shelf life for various peanut products.</span><span class="c9">1</span><span class="c1 c2">&nbsp;This is a critical advantage for storage, transport, and consumer satisfaction.</span></li><li class="c3 c20 li-bullet-0"><span class="c15">Improved Health Benefits:</span><span class="c1">&nbsp;The consumption of high oleic peanuts has been linked to positive impacts on human health, including the reduction of &quot;bad&quot; cholesterol (LDL) without adversely affecting &quot;good&quot; cholesterol (HDL) levels, and potential anti-inflammatory and anti-tumorigenic properties.</span><span class="c0">1</span></li><li class="c3 c20 li-bullet-0"><span class="c15">Better Frying Characteristics:</span><span class="c1">&nbsp;The altered fatty acid composition leads to a higher smoke point for the oil, allowing for cooking at higher temperatures. This can result in faster cooking and reduced oil absorption by cooked foods, which is beneficial for culinary applications.</span><span class="c0">1</span></li></ul><h3 class="c28"><span class="c23 c2">Undesired Pleiotropic Effects</span></h3><p class="c5"><span class="c1">While the benefits to oil quality are substantial, mutations in </span><span class="c8">FAD2</span><span class="c1">&nbsp;genes that lead to high oleic acid accumulation have been found to exert negative pleiotropic effects on various agronomic traits and overall plant development.</span><span class="c9">5</span><span class="c1 c2">&nbsp;This presents a critical dilemma for breeders: how to effectively decouple the beneficial high oleic trait from these undesirable growth penalties.</span></p><p class="c5"><span class="c1">One prominent consequence is the </span><span class="c15">repression of leaf growth and overall plant dwarfing</span><span class="c1">. High oleic peanut varieties, such as Yueyou271, exhibit a marked retardation in leaf growth, with their leaves remaining tightly closed or only partially expanded for longer periods compared to normal varieties.</span><span class="c9">8</span><span class="c1">&nbsp;Additionally, the leaves of high-OA varieties are significantly smaller in length, width, and area, contributing to an overall dwarfed plant stature.</span><span class="c9">5</span><span class="c1 c2">&nbsp;This reduction in photosynthetic capacity and overall biomass can significantly impact yield.</span></p><p class="c5"><span class="c1">Furthermore, the </span><span class="c8">fad2</span><span class="c1">&nbsp;mutation has been associated with a </span><span class="c15">reduction in pod and seed size</span><span class="c1">&nbsp;</span><span class="c9">5</span><span class="c1 c2">, which directly impacts the economic viability and market value for growers.</span></p><p class="c5"><span class="c1">The molecular mechanism underlying this growth repression involves significant alterations in </span><span class="c15">phytohormone pathways</span><span class="c1">. High-OA peanuts show significantly lowered levels of crucial growth-promoting hormones such as cytokinin, auxin, and gibberellic acid (GA), while levels of jasmonic acid (JA) are elevated.</span><span class="c9">8</span><span class="c1">&nbsp;Specifically, the </span><span class="c8">fad2</span><span class="c1">&nbsp;mutation dramatically impairs the synthesis of active cytokinin forms, </span><span class="c8">cis</span><span class="c1">-Zeatin (cZ) and </span><span class="c8">trans</span><span class="c1">-Zeatin (tZ), and other cytokinin derivatives.</span><span class="c9">8</span><span class="c1">&nbsp;This is linked to the inhibition of </span><span class="c8">LONELY GUY</span><span class="c1">&nbsp;(</span><span class="c8">LOG</span><span class="c1">) genes, which are essential for cytokinin synthesis, particularly in vascular cells.</span><span class="c9">8</span><span class="c1">&nbsp;This hormonal imbalance, especially the reduction in cytokinin, directly represses leaf growth and development.</span><span class="c0">8</span></p><p class="c5"><span class="c1">Beyond hormonal changes, single-cell RNA sequencing (scRNA-seq) studies have revealed that the </span><span class="c8">fad2</span><span class="c1">&nbsp;mutation also affects cell differentiation and </span><span class="c15">perturbs the normal cell cycle</span><span class="c1">.</span><span class="c9">8</span><span class="c1">&nbsp;A significant proportion of cells in the </span><span class="c8">fad2</span><span class="c1">&nbsp;mutant, particularly in epidermal, mesophyll, and guard cells, are arrested or stalled in the S-phase (DNA replication phase) of the cell cycle.</span><span class="c9">8</span><span class="c1">&nbsp;This cell cycle disorder suggests that the excessive accumulation of oleic acid due to the </span><span class="c8">fad2</span><span class="c1">&nbsp;mutation may induce biological stress that suppresses normal cell cycle progression, thereby negatively impacting plant growth.</span><span class="c9">8</span><span class="c1 c2">&nbsp;The detailed elucidation of this mechanism, specifically through the inhibition of cytokinin synthesis and subsequent perturbation of the cell cycle, provides a profound understanding of the underlying physiological consequences. This mechanistic understanding is crucial because it opens new avenues for targeted research and intervention. Instead of merely observing dwarfing, breeders can now explore strategies to restore normal hormone balance or cell cycle progression in high oleic lines, potentially through genetic manipulation of these regulatory pathways, offering a more sophisticated approach to overcoming pleiotropic effects.</span></p><p class="c5"><span class="c1 c2">Table 3 summarizes the phenotypic benefits and challenges associated with high oleic peanut cultivars.</span></p><p class="c5"><span class="c15 c2">Table 3: Phenotypic Benefits and Challenges Associated with High Oleic Peanut Cultivars</span></p><p class="c21"><span class="c15 c2"></span></p><table class="c31"><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Trait/Characteristic</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Benefit</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Challenge/Trade-off</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Relevant Genes/Mechanisms</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Oil Stability</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">10x higher anti-oxidative stability, extended shelf life.</span><span class="c0">1</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c22">FAD2</span><span class="c7">&nbsp;mutations leading to reduced linoleic acid.</span><span class="c0">1</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Health Benefits</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Reduced serum cholesterol (LDL without affecting HDL), anti-tumorigenic, anti-inflammatory.</span><span class="c0">1</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">High oleic acid content due to </span><span class="c22">FAD2</span><span class="c7">&nbsp;mutations.</span><span class="c0">1</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Frying Quality</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Higher smoke point, faster cooking, less oil absorption.</span><span class="c0">1</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Higher unsaturated fatty acids (oleic acid).</span><span class="c0">1</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Plant Growth</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Repressed leaf growth, smaller leaf size, overall dwarfing.</span><span class="c0">5</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c22">FAD2</span><span class="c7">&nbsp;mutations inhibiting cytokinin synthesis (</span><span class="c22">LOG</span><span class="c7">&nbsp;gene), perturbing cell cycle (S-phase arrest).</span><span class="c0">8</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Pod/Seed Size</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Reduced pod and seed size.</span><span class="c0">5</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c22">FAD2</span><span class="c7">&nbsp;mutations and associated growth repression.</span><span class="c0">5</span></p></td></tr><tr class="c18"><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c16 c2">Phytohormone Balance</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7 c2">-</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c7">Lowered cytokinin, auxin, GA; elevated JA.</span><span class="c0">8</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c3"><span class="c22">FAD2</span><span class="c7">&nbsp;mutations disrupting hormonal pathways.</span><span class="c0">8</span></p></td></tr></table><h2 class="c10"><span class="c24 c2">7. Advanced Breeding Strategies for High Oleic Peanut Development</span></h2><p class="c5"><span class="c1">The development of high oleic peanut cultivars that are agronomically competitive requires a sophisticated, multi-pronged approach that integrates traditional breeding methods with cutting-edge molecular technologies. The current landscape of high oleic peanut breeding demonstrates a clear trend towards the synergistic integration of conventional breeding, marker-assisted selection (MAS), and advanced gene editing techniques like CRISPR/Cas9.</span><span class="c9">12</span><span class="c1 c2">&nbsp;This implies that no single approach is sufficient to address the complex breeding objectives of simultaneously enhancing oil quality, yield, and resistance to biotic/abiotic stresses. Instead, a multi-pronged, integrated breeding pipeline is emerging as the most effective strategy for modern plant improvement, leveraging the strengths of each technology to overcome their individual limitations.</span></p><h3 class="c10"><span class="c23 c2">Molecular Breeding and Marker-Assisted Selection (MAS)</span></h3><p class="c5"><span class="c1">Molecular breeding techniques, particularly Marker-Assisted Selection (MAS), are indispensable for accelerating the genetic improvement of oil content and quality traits, including high oleic acid, in peanut.</span><span class="c9">1</span><span class="c1">&nbsp;The identification of molecular markers or Quantitative Trait Loci (QTLs) associated with oil content and fatty acid composition is directly applied in MAS programs.</span><span class="c9">1</span><span class="c1">&nbsp;The rapid advancements in sequencing technologies, such as ddRAD-seq, whole-genome resequencing (WGRS), and specific-locus amplified fragment sequencing (SLAF-seq), have provided an abundance of single nucleotide polymorphisms (SNPs) that enable robust QTL mapping and the development of highly efficient molecular markers.</span><span class="c0">3</span></p><p class="c5"><span class="c1">MAS significantly enhances breeding efficiency by allowing for the precise selection of early generation progeny that are homozygous for desired traits, such as high oleic acid content and resistance to various diseases (e.g., nematodes, leaf spot), a process that is far more rapid and reliable than traditional phenotypic selection alone.</span><span class="c9">12</span><span class="c1">&nbsp;For instance, MAS has been successfully used to identify &quot;true hybrids&quot; in F1 generations and to advance high-oleic, high-pod-yield populations through subsequent generations.</span><span class="c9">12</span><span class="c1">&nbsp;This approach has directly contributed to the development and release of new high oleic peanut cultivars, such as TifNV-HG, which effectively combines a high O/L ratio with resistance to the peanut root-knot nematode and tomato spotted wilt virus (TSWV).</span><span class="c0">20</span></p><h3 class="c10"><span class="c23 c2">Genetic Engineering: CRISPR/Cas9 Applications</span></h3><p class="c5"><span class="c1">Advanced genome editing technologies, particularly the CRISPR/Cas9 system, offer unprecedented precision in genetic manipulation, enabling the enhancement of oleic acid levels with minimal off-target effects.</span><span class="c9">14</span><span class="c1">&nbsp;CRISPR/Cas9 has been successfully employed to transform peanut oil chemistry from a normal to a high oleic acid content.</span><span class="c9">21</span><span class="c1">&nbsp;Specific examples include targeted mutagenesis of the </span><span class="c8">FAD2</span><span class="c1">&nbsp;gene to increase oleic acid content, as demonstrated by Neelakandan et al. (2022).</span><span class="c0">21</span></p><p class="c5"><span class="c1">Further advancements include the use of base editing techniques, such as cytosine base editor in conjunction with CRISPR/nCas9 technology, to induce precise point mutations in the </span><span class="c8">AhFAD2</span><span class="c1">&nbsp;gene.</span><span class="c9">21</span><span class="c1">&nbsp;These methods, demonstrated through hairy root transformation and calyx tube injection, have resulted in an increase in oleic acid content (55-70%) in T0 seeds.</span><span class="c9">21</span><span class="c1">&nbsp;These systems allow for targeted nucleotide conversions (e.g., C to T) within both promoter and coding sequences of </span><span class="c8">AhFAD2</span><span class="c1">&nbsp;genes, effectively mimicking natural mutations that confer the high oleic trait.</span><span class="c0">23</span></p><p class="c5"><span class="c1">Moreover, CRISPR technology is being explored for the simultaneous editing of multiple genes in polyploid oilseed crops like peanut.</span><span class="c9">21</span><span class="c1">&nbsp;For example, research at Clemson University is utilizing CRISPR-Cpf1 gene-editing to develop peanuts with not only enhanced high oleic acid but also reduced allergen content, along with improved shelf-life and tolerance to heat stress.</span><span class="c0">19</span></p><h3 class="c10"><span class="c23 c2">Conventional Breeding Approaches</span></h3><p class="c5"><span class="c1">Despite the advent of sophisticated molecular tools, conventional breeding methods remain fundamental to high oleic peanut development, often working in concert with molecular strategies. Hybridization schemes are routinely employed to introgress the high oleic trait from donor parents (e.g., advanced high-oleic lines like HOG or KN176) into elite, high-yielding, but conventional low-oleic cultivars (e.g., NC-7 or Huayu22).</span><span class="c9">12</span><span class="c1">&nbsp;Backcross breeding is a standard and efficient strategy to transfer the desired </span><span class="c8">FAD2</span><span class="c1">&nbsp;alleles while recovering the desirable genetic background of the recurrent parent.</span><span class="c0">12</span></p><p class="c5"><span class="c1">Breeding populations are strategically developed by crossing high oleic cultivars with lines possessing robust resistance to various biotic stresses (e.g., peanut root-knot nematode, leaf spot) and enhanced tolerance to abiotic stresses (e.g., drought, heat, aflatoxin contamination).</span><span class="c9">20</span><span class="c1">&nbsp;These populations undergo rigorous evaluation under diverse field conditions to select for desired agronomic characteristics, overall yield, and stress tolerance in later generations.</span><span class="c9">20</span><span class="c1">&nbsp;Modern breeding for high oleic acid has evolved beyond simply increasing oleic acid content. It now critically focuses on integrating this trait with other crucial agronomic characteristics, such as high yield, resistance to major diseases, and tolerance to environmental stresses.</span><span class="c9">12</span><span class="c1 c2">&nbsp;This reflects a fundamental shift from single-trait selection to a multi-trait, holistic breeding paradigm, driven by market demands for comprehensive quality and the imperative for agricultural sustainability. This implies that the ultimate value and adoption of a high oleic variety are contingent upon its overall agronomic performance and resilience, necessitating complex selection criteria and integrated genetic solutions.</span></p><h2 class="c10"><span class="c24 c2">8. Challenges and Future Directions in High Oleic Peanut Breeding</span></h2><p class="c5"><span class="c1 c2">The advancement of high oleic peanut cultivars faces several challenges that necessitate continued research and innovative breeding strategies. Addressing these complexities is crucial for developing varieties that are not only rich in oleic acid but also agronomically superior and resilient.</span></p><h3 class="c10"><span class="c23 c2">Addressing Negative Pleiotropic Effects and Balancing Desired Traits</span></h3><p class="c5"><span class="c1">A significant hurdle in high oleic peanut breeding is mitigating the undesirable pleiotropic effects of </span><span class="c8">fad2</span><span class="c1">&nbsp;mutations, such as repressed leaf growth, dwarfing, and reduced pod and seed size.</span><span class="c9">5</span><span class="c1">&nbsp;The detailed understanding that </span><span class="c8">fad2</span><span class="c1">&nbsp;mutations have complex pleiotropic effects, extending to hormonal dysregulation (cytokinin pathway inhibition) and cell cycle perturbation (S-phase arrest) </span><span class="c9">5</span><span class="c1">, highlights that simply modifying the </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;gene is insufficient for optimal cultivar development. This underscores the need for a comprehensive systems biology approach to understand the entire downstream cascade of molecular and physiological changes. This implies a future where breeding programs will increasingly rely on multi-omics data integration and computational modeling to predict, understand, and mitigate unintended consequences, moving towards holistic plant engineering rather than single-gene modification. Future research must therefore focus on identifying and manipulating modifier genes or compensatory pathways that can restore normal plant vigor and yield in high oleic backgrounds without compromising oil quality.</span></p><h3 class="c10"><span class="c23 c2">Integrating High Oleic Trait with Disease Resistance, Yield, and Stress Tolerance</span></h3><p class="c5"><span class="c1">A primary objective for future high oleic peanut breeding is to develop superior cultivars that combine high oleic acid content with other critical agronomic traits. These include high yield, robust resistance to major diseases (e.g., nematodes, leaf spot, white mold, Tomato Spotted Wilt Virus - TSWV), and enhanced tolerance to abiotic stresses (e.g., drought, heat, aflatoxin contamination).</span><span class="c9">20</span><span class="c1">&nbsp;This requires sophisticated genetic strategies, such as pyramiding desirable QTLs for multiple traits into single genotypes.</span><span class="c9">4</span><span class="c1">&nbsp;The identification of stable QTLs across multiple environments for oil content and other traits will be essential for developing broadly adapted high oleic varieties that perform consistently under diverse agricultural conditions.</span><span class="c0">3</span></p><h3 class="c10"><span class="c23 c2">Leveraging Advanced Omics Technologies for Comprehensive Understanding</span></h3><p class="c5"><span class="c1">Breakthroughs in high-throughput sequencing, genotyping, and various omics technologies are pivotal for driving future advancements in high oleic peanut breeding.</span><span class="c9">12</span><span class="c1">&nbsp;Integromics approaches, encompassing transcriptomics, proteomics, and metabolomics, provide a comprehensive, systems-level understanding of the molecular responses and metabolic changes associated with increased oleic acid levels and their pleiotropic effects.</span><span class="c9">14</span><span class="c1">&nbsp;For instance, single-cell RNA sequencing (scRNA-seq) has already provided detailed insights into how </span><span class="c8">fad2</span><span class="c1">&nbsp;mutations inhibit cytokinin synthesis and repress leaf growth.</span><span class="c9">8</span><span class="c1">&nbsp;Continued functional validation of key genes and regulatory elements identified through omics studies is necessary to confirm their roles and explore their potential applications in peanut improvement.</span><span class="c9">21</span><span class="c1">&nbsp;The ongoing identification of QTLs and candidate genes will facilitate the development of more precise and efficient diagnostic markers for marker-assisted breeding.</span><span class="c0">3</span></p><p class="c5"><span class="c1">Furthermore, optimizing peanut transformation techniques and gene editing methodologies (e.g., CRISPR/Cas9, base editing) is crucial to enhance their efficiency and applicability for complex trait manipulation, including simultaneously addressing multiple target genes or regulatory elements.</span><span class="c9">21</span><span class="c1">&nbsp;While cutting-edge tools like CRISPR/Cas9 offer unprecedented precision in genetic modification </span><span class="c9">14</span><span class="c1">, the continued emphasis on conventional breeding and marker-assisted selection </span><span class="c9">12</span><span class="c1 c2">&nbsp;highlights a pragmatic approach to breeding. The core challenge is not merely technological adoption but effectively integrating these high-tech tools into existing, often slower, conventional breeding pipelines to deliver improved varieties that are agronomically viable, stable across environments, and readily accepted by growers. This implies that the &quot;future&quot; of breeding is a hybrid model, combining the precision of molecular tools with the robustness and field-proven efficacy of established breeding practices.</span></p><h2 class="c10"><span class="c2 c24">Conclusions</span></h2><p class="c5"><span class="c1">The high oleic acid trait in peanut is a highly desirable quality attribute, primarily controlled by homozygous recessive mutations in the homeologous </span><span class="c8">ahFAD2A</span><span class="c1">&nbsp;and </span><span class="c8">ahFAD2B</span><span class="c1 c2">&nbsp;genes. These mutations lead to the inactivation of the FAD2 enzyme, preventing the conversion of oleic acid to linoleic acid and significantly enhancing oil stability and health benefits. While these major genes are foundational, the genetic architecture of oleic acid content is complex, involving multiple QTLs and intricate gene interactions (additive, dominance, and epistasis), indicating a significant quantitative component.</span></p><p class="c5"><span class="c1">A critical challenge in developing high oleic varieties stems from the pleiotropic effects of </span><span class="c8">FAD2</span><span class="c1">&nbsp;mutations, which can negatively impact plant growth, leading to dwarfing and reduced pod/seed size. Mechanistic studies reveal that this growth repression is linked to hormonal dysregulation, particularly the inhibition of cytokinin synthesis, and perturbation of the cell cycle. This underscores the need for a systems-level understanding of the downstream molecular and physiological consequences of </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;modification.</span></p><p class="c27"><span class="c1">Future breeding efforts must adopt an integrated, multi-trait paradigm. This involves leveraging the precision of molecular tools like marker-assisted selection for efficient introgression of </span><span class="c8">FAD2</span><span class="c1 c2">&nbsp;mutations and other beneficial QTLs, alongside advanced gene editing techniques such as CRISPR/Cas9 for targeted modifications. Simultaneously, conventional breeding methods remain essential for combining the high oleic trait with other crucial agronomic characteristics, including high yield, disease resistance, and stress tolerance. The ultimate success of high oleic peanut breeding hinges on effectively decoupling the desired oil quality from undesirable agronomic trade-offs, ensuring stable performance across diverse environments, and developing varieties that meet both consumer demand and agricultural sustainability goals.</span></p><h4 class="c30"><span class="c2 c26">Works cited</span></h4><ol class="c25 lst-kix_list_3-0 start" start="1"><li class="c12 li-bullet-0"><span class="c4">openscholar.uga.edu, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://openscholar.uga.edu/record/1300/files/s12863-014-0133-4.pdf&amp;sa=D&amp;source=editors&amp;ust=1749149780457567&amp;usg=AOvVaw03WDcaTm-c5OQnOoEilE-H">https://openscholar.uga.edu/record/1300/files/s12863-014-0133-4.pdf</a></span></li><li class="c12 li-bullet-0"><span class="c4">www.ajol.info, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.ajol.info/index.php/ajb/article/view/129124/118683&amp;sa=D&amp;source=editors&amp;ust=1749149780457840&amp;usg=AOvVaw1EEtqhDyyV69CywAKOmuB8">https://www.ajol.info/index.php/ajb/article/view/129124/118683</a></span></li><li class="c12 li-bullet-0"><span class="c4">Identification of a New Major Oil Content QTL Overlapped with FAD2B in Cultivated Peanut (Arachis hypogaea L.), accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11859173/&amp;sa=D&amp;source=editors&amp;ust=1749149780458107&amp;usg=AOvVaw3-dJMA_UjbUW-S_UmhWGLT">https://pmc.ncbi.nlm.nih.gov/articles/PMC11859173/</a></span></li><li class="c12 li-bullet-0"><span class="c4">(PDF) Genome-wide association analysis in peanut accessions uncovers the genetic basis regulating oil and fatty acid variation - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/391808406_Genome-wide_association_analysis_in_peanut_accessions_uncovers_the_genetic_basis_regulating_oil_and_fatty_acid_variation&amp;sa=D&amp;source=editors&amp;ust=1749149780458559&amp;usg=AOvVaw3bTO_NIkWtJRi5Egw2NXm-">https://www.researchgate.net/publication/391808406_Genome-wide_association_analysis_in_peanut_accessions_uncovers_the_genetic_basis_regulating_oil_and_fatty_acid_variation</a></span></li><li class="c12 li-bullet-0"><span class="c4">scRNA-seq Reveals the Mechanism of Fatty Acid Desaturase 2 Mutation to Repress Leaf Growth in Peanut (Arachis hypogaea L.) - MDPI, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4409/12/18/2305&amp;sa=D&amp;source=editors&amp;ust=1749149780458834&amp;usg=AOvVaw0BJ0CbR-2WUwjwTaYCDYyH">https://www.mdpi.com/2073-4409/12/18/2305</a></span></li><li class="c12 li-bullet-0"><span class="c4">high oleic acid peanuts: potentials in health, products and oil use - Publication : USDA ARS, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/publications/publication/?seqNo115%3D71540&amp;sa=D&amp;source=editors&amp;ust=1749149780459132&amp;usg=AOvVaw0f6iHMgMro7XJYCVBjXYQY">https://www.ars.usda.gov/research/publications/publication/?seqNo115=71540</a></span></li><li class="c12 li-bullet-0"><span class="c4">Insights into the Novel FAD2 Gene Regulating Oleic Acid ..., accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9691258/&amp;sa=D&amp;source=editors&amp;ust=1749149780459359&amp;usg=AOvVaw2fTmETqzjVaQ9fyI5R2Rou">https://pmc.ncbi.nlm.nih.gov/articles/PMC9691258/</a></span></li><li class="c12 li-bullet-0"><span class="c4">scRNA-seq Reveals the Mechanism of Fatty Acid Desaturase 2 Mutation to Repress Leaf Growth in Peanut (Arachis hypogaea L.) - PubMed Central, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10527976/&amp;sa=D&amp;source=editors&amp;ust=1749149780459653&amp;usg=AOvVaw3RVKLGj0UadSJsEt6QdQx6">https://pmc.ncbi.nlm.nih.gov/articles/PMC10527976/</a></span></li><li class="c12 li-bullet-0"><span class="c4">The FAD2 Gene in Plants: Occurrence, Regulation, and Role - Frontiers, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.01789/full&amp;sa=D&amp;source=editors&amp;ust=1749149780459931&amp;usg=AOvVaw1b8MB5TGoNdLpN7g6wyE2w">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.01789/full</a></span></li><li class="c12 li-bullet-0"><span class="c4">Isolation and functional analysis of fatty acid desaturase genes from peanut (Arachis hypogaea L.) - PMC, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5731756/&amp;sa=D&amp;source=editors&amp;ust=1749149780460201&amp;usg=AOvVaw3EMTd0fLxuR4mlKHOxtjLc">https://pmc.ncbi.nlm.nih.gov/articles/PMC5731756/</a></span></li><li class="c12 li-bullet-0"><span class="c4">FAD2 Gene Mutations Significantly Alter Fatty Acid ... - USDA ARS, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/publications/publication/?seqNo115%3D256337&amp;sa=D&amp;source=editors&amp;ust=1749149780460452&amp;usg=AOvVaw1IPARAQIR1HycGKJAeUV3_">https://www.ars.usda.gov/research/publications/publication/?seqNo115=256337</a></span></li><li class="c12 li-bullet-0"><span class="c4">Molecular breeding to develop advanced lines with high oleic acid and pod yield in peanut | Request PDF - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/373195564_Molecular_breeding_to_develop_advanced_lines_with_high_oleic_acid_and_pod_yield_in_peanut&amp;sa=D&amp;source=editors&amp;ust=1749149780460835&amp;usg=AOvVaw2HIwT_aVqFp8iWQTor1hWv">https://www.researchgate.net/publication/373195564_Molecular_breeding_to_develop_advanced_lines_with_high_oleic_acid_and_pod_yield_in_peanut</a></span></li><li class="c12 li-bullet-0"><span class="c4">The high oleate trait in the cultivated peanut [Arachis hypogae L.]. II. Molecular basis and genetics of the trait | Request PDF - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/12414690_The_high_oleate_trait_in_the_cultivated_peanut_Arachis_hypogae_L_II_Molecular_basis_and_genetics_of_the_trait&amp;sa=D&amp;source=editors&amp;ust=1749149780461282&amp;usg=AOvVaw2ORNkkMmhWT-zWsMn0ct8m">https://www.researchgate.net/publication/12414690_The_high_oleate_trait_in_the_cultivated_peanut_Arachis_hypogae_L_II_Molecular_basis_and_genetics_of_the_trait</a></span></li><li class="c12 li-bullet-0"><span class="c4">Molecular insights driving oleic acid improvement in groundnut: A review, accessed June 5, 2025, </span><span class="c4 c11"><a class="c13" href="https://www.google.com/url?q=https://horizonepublishing.com/journals/index.php/PST/article/view/5602?articlesBySameAuthorPage%3D2&amp;sa=D&amp;source=editors&amp;ust=1749149780461571&amp;usg=AOvVaw3Ps4mHb97JgRG-2g7YGLk8">https://horizonepublishing.com/journals/index.php/PST/article/view/5602?articlesBySameAuthorPage=2</a></span></li><li class="c12 li-bullet-0"><span class="c4">Genotypic effect of ahFAD2 on fatty acid profiles in six segregating peanut (Arachis hypogaea L) populations - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/250307478_Genotypic_effect_of_ahFAD2_on_fatty_acid_profiles_in_six_segregating_peanut_Arachis_hypogaea_L_populations&amp;sa=D&amp;source=editors&amp;ust=1749149780461981&amp;usg=AOvVaw2RHptnrfjcROhMjLsu5xkW">https://www.researchgate.net/publication/250307478_Genotypic_effect_of_ahFAD2_on_fatty_acid_profiles_in_six_segregating_peanut_Arachis_hypogaea_L_populations</a></span></li><li class="c12 li-bullet-0"><span class="c4">Genotyping and Fatty Acid Composition Analysis in Segregating Peanut (Arachis hypogaea L.) Populations - Allen Press, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://meridian.allenpress.com/peanut-science/article/38/1/11/108608/Genotyping-and-Fatty-Acid-Composition-Analysis-in&amp;sa=D&amp;source=editors&amp;ust=1749149780462341&amp;usg=AOvVaw3odAaQqXHHdGTU5Cf70ZRZ">https://meridian.allenpress.com/peanut-science/article/38/1/11/108608/Genotyping-and-Fatty-Acid-Composition-Analysis-in</a></span></li><li class="c12 li-bullet-0"><span class="c4">Comparison of the &Delta;12 fatty acid desaturase gene between high-oleic and normal-oleic peanut genotypes | Request PDF - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/23486235_Comparison_of_the_D12_fatty_acid_desaturase_gene_between_high-oleic_and_normal-oleic_peanut_genotypes&amp;sa=D&amp;source=editors&amp;ust=1749149780462757&amp;usg=AOvVaw1Uis-0FbkZCCKiP5xCvqPr">https://www.researchgate.net/publication/23486235_Comparison_of_the_D12_fatty_acid_desaturase_gene_between_high-oleic_and_normal-oleic_peanut_genotypes</a></span></li><li class="c12 li-bullet-0"><span class="c4">Estimation of Heritability by Parent-offspring Regression for High-oleic Acid in Peanut, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/49611667_Estimation_of_Heritability_by_Parent-offspring_Regression_for_High-oleic_Acid_in_Peanut&amp;sa=D&amp;source=editors&amp;ust=1749149780463138&amp;usg=AOvVaw2AdHNGA0wEFQrt78bhmrZv">https://www.researchgate.net/publication/49611667_Estimation_of_Heritability_by_Parent-offspring_Regression_for_High-oleic_Acid_in_Peanut</a></span></li><li class="c12 li-bullet-0"><span class="c4">Clemson unites with FFAR to tackle peanut allergies | Clemson News, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://news.clemson.edu/clemson-unites-with-ffar-to-tackle-peanut-allergies/&amp;sa=D&amp;source=editors&amp;ust=1749149780463395&amp;usg=AOvVaw0AZDfzefdNNwA-szmfS_HA">https://news.clemson.edu/clemson-unites-with-ffar-to-tackle-peanut-allergies/</a></span></li><li class="c12 li-bullet-0"><span class="c4">Development of High-Yielding, High-Oleic Peanut Cultivars or Germplasm with Tolerance to Biotic and Abiotic Stresses - Project : USDA ARS, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/project/?accnNo%3D434346%26fy%3D2023&amp;sa=D&amp;source=editors&amp;ust=1749149780463686&amp;usg=AOvVaw2h1IkklHN4xbKYqHVHAVQR">https://www.ars.usda.gov/research/project/?accnNo=434346&amp;fy=2023</a></span></li><li class="c12 li-bullet-0"><span class="c4">Mutagenesis induced in FAD2 genes by CRISPR/Cas9 medicated gene... - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/figure/Mutagenesis-induced-in-FAD2-genes-by-CRISPR-Cas9-medicated-gene-editing-A-Two_fig2_360771208&amp;sa=D&amp;source=editors&amp;ust=1749149780464022&amp;usg=AOvVaw0bfLXL6Np_bW0b_WhELRr0">https://www.researchgate.net/figure/Mutagenesis-induced-in-FAD2-genes-by-CRISPR-Cas9-medicated-gene-editing-A-Two_fig2_360771208</a></span></li><li class="c12 li-bullet-0"><span class="c4">Molecular insights driving oleic acid improvement in groundnut: A review - ResearchGate, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.researchgate.net/publication/387493196_Molecular_insights_driving_oleic_acid_improvement_in_groundnut_A_review&amp;sa=D&amp;source=editors&amp;ust=1749149780464377&amp;usg=AOvVaw0HxyndOzDFIH93eEQViWjk">https://www.researchgate.net/publication/387493196_Molecular_insights_driving_oleic_acid_improvement_in_groundnut_A_review</a></span></li><li class="c12 li-bullet-0"><span class="c4">Base Editing in Peanut Using CRISPR/nCas9 - Frontiers, accessed June 5, 2025, </span><span class="c11 c4"><a class="c13" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/genome-editing/articles/10.3389/fgeed.2022.901444/full&amp;sa=D&amp;source=editors&amp;ust=1749149780464640&amp;usg=AOvVaw31doYaBlB4uywejzsI2DI7">https://www.frontiersin.org/journals/genome-editing/articles/10.3389/fgeed.2022.901444/full</a></span></li></ol></body></html>