<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);ul.lst-kix_list_1-0{list-style-type:none}ol.lst-kix_list_9-0.start{counter-reset:lst-ctn-kix_list_9-0 0}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}.lst-kix_list_9-0>li{counter-increment:lst-ctn-kix_list_9-0}ul.lst-kix_list_9-6{list-style-type:none}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_k1m9p4de5zmc-1>li:before{content:"\0025cb   "}.lst-kix_k1m9p4de5zmc-0>li:before{content:"\0025cf   "}.lst-kix_k1m9p4de5zmc-2>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-5>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-4>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-6>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-3>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-7>li:before{content:"\0025a0   "}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025a0   "}ul.lst-kix_78vlvctpnz9j-4{list-style-type:none}ul.lst-kix_78vlvctpnz9j-5{list-style-type:none}.lst-kix_list_5-2>li:before{content:"\0025a0   "}.lst-kix_k1m9p4de5zmc-8>li:before{content:"\0025a0   "}ul.lst-kix_78vlvctpnz9j-6{list-style-type:none}ul.lst-kix_78vlvctpnz9j-7{list-style-type:none}.lst-kix_list_5-1>li:before{content:"\0025cb   "}ul.lst-kix_78vlvctpnz9j-8{list-style-type:none}.lst-kix_list_5-7>li:before{content:"\0025a0   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_uixjlpexb0mc-3{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}ul.lst-kix_uixjlpexb0mc-2{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025a0   "}.lst-kix_list_5-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_uixjlpexb0mc-1{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_uixjlpexb0mc-0{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_uixjlpexb0mc-7{list-style-type:none}ul.lst-kix_uixjlpexb0mc-6{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_uixjlpexb0mc-5{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}ul.lst-kix_uixjlpexb0mc-4{list-style-type:none}.lst-kix_list_5-4>li:before{content:"\0025a0   "}ul.lst-kix_uixjlpexb0mc-8{list-style-type:none}.lst-kix_list_5-5>li:before{content:"\0025a0   "}ul.lst-kix_list_8-0{list-style-type:none}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_6-1>li:before{content:"\0025cb   "}.lst-kix_list_6-3>li:before{content:"\0025a0   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025a0   "}.lst-kix_list_6-2>li:before{content:"\0025a0   "}ul.lst-kix_k1m9p4de5zmc-4{list-style-type:none}ul.lst-kix_j2ures5lufxp-6{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-3{list-style-type:none}ul.lst-kix_j2ures5lufxp-7{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-6{list-style-type:none}ul.lst-kix_j2ures5lufxp-8{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-5{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-0{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-2{list-style-type:none}.lst-kix_list_6-8>li:before{content:"\0025a0   "}ul.lst-kix_k1m9p4de5zmc-1{list-style-type:none}ul.lst-kix_j2ures5lufxp-0{list-style-type:none}.lst-kix_list_6-5>li:before{content:"\0025a0   "}.lst-kix_list_6-7>li:before{content:"\0025a0   "}ul.lst-kix_j2ures5lufxp-1{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-8{list-style-type:none}ul.lst-kix_j2ures5lufxp-2{list-style-type:none}ul.lst-kix_k1m9p4de5zmc-7{list-style-type:none}ul.lst-kix_j2ures5lufxp-3{list-style-type:none}ul.lst-kix_j2ures5lufxp-4{list-style-type:none}.lst-kix_list_6-6>li:before{content:"\0025a0   "}ul.lst-kix_j2ures5lufxp-5{list-style-type:none}.lst-kix_list_2-7>li:before{content:"\0025a0   "}.lst-kix_list_7-4>li:before{content:"\0025a0   "}.lst-kix_list_7-6>li:before{content:"\0025a0   "}.lst-kix_list_2-5>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-8>li:before{content:"\0025a0   "}.lst-kix_list_7-2>li:before{content:"\0025a0   "}ul.lst-kix_2bbejj5g047s-1{list-style-type:none}ul.lst-kix_2bbejj5g047s-0{list-style-type:none}ul.lst-kix_2bbejj5g047s-3{list-style-type:none}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_2bbejj5g047s-2{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}.lst-kix_96onxbcsb0ef-6>li:before{content:"\0025a0   "}.lst-kix_96onxbcsb0ef-8>li:before{content:"\0025a0   "}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_2bbejj5g047s-8{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}.lst-kix_list_7-8>li:before{content:"\0025a0   "}ul.lst-kix_list_3-0{list-style-type:none}.lst-kix_96onxbcsb0ef-4>li:before{content:"\0025a0   "}ul.lst-kix_2bbejj5g047s-5{list-style-type:none}ul.lst-kix_list_3-5{list-style-type:none}ol.lst-kix_list_9-0{list-style-type:none}ul.lst-kix_2bbejj5g047s-4{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}ul.lst-kix_2bbejj5g047s-7{list-style-type:none}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_2bbejj5g047s-6{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}ul.lst-kix_c8eslinrrxlh-7{list-style-type:none}ul.lst-kix_c8eslinrrxlh-8{list-style-type:none}.lst-kix_96onxbcsb0ef-0>li:before{content:"\0025cf   "}ul.lst-kix_c8eslinrrxlh-3{list-style-type:none}ul.lst-kix_c8eslinrrxlh-4{list-style-type:none}ul.lst-kix_c8eslinrrxlh-5{list-style-type:none}ul.lst-kix_c8eslinrrxlh-6{list-style-type:none}ul.lst-kix_c8eslinrrxlh-0{list-style-type:none}.lst-kix_j2ures5lufxp-0>li:before{content:"\0025cf   "}.lst-kix_list_4-1>li:before{content:"\0025cb   "}ul.lst-kix_c8eslinrrxlh-1{list-style-type:none}.lst-kix_96onxbcsb0ef-2>li:before{content:"\0025a0   "}ul.lst-kix_c8eslinrrxlh-2{list-style-type:none}.lst-kix_list_9-2>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-4>li:before{content:"\0025a0   "}.lst-kix_list_4-3>li:before{content:"\0025a0   "}.lst-kix_list_4-5>li:before{content:"\0025a0   "}ul.lst-kix_78vlvctpnz9j-0{list-style-type:none}ul.lst-kix_78vlvctpnz9j-1{list-style-type:none}.lst-kix_j2ures5lufxp-2>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-6>li:before{content:"\0025a0   "}ul.lst-kix_78vlvctpnz9j-2{list-style-type:none}ul.lst-kix_78vlvctpnz9j-3{list-style-type:none}.lst-kix_list_9-0>li:before{content:"" counter(lst-ctn-kix_list_9-0,decimal) ". "}.lst-kix_c8eslinrrxlh-1>li:before{content:"\0025cb   "}.lst-kix_c8eslinrrxlh-3>li:before{content:"\0025a0   "}.lst-kix_list_9-6>li:before{content:"\0025a0   "}.lst-kix_list_9-4>li:before{content:"\0025a0   "}ul.lst-kix_list_2-8{list-style-type:none}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_9-8>li:before{content:"\0025a0   "}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}.lst-kix_list_1-7>li:before{content:"\0025a0   "}.lst-kix_list_1-5>li:before{content:"\0025a0   "}ul.lst-kix_5xlsiqxwcoa5-4{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-5{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-6{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-7{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-0{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-1{list-style-type:none}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_c8eslinrrxlh-7>li:before{content:"\0025a0   "}ul.lst-kix_5xlsiqxwcoa5-2{list-style-type:none}ul.lst-kix_5xlsiqxwcoa5-3{list-style-type:none}.lst-kix_list_2-3>li:before{content:"\0025a0   "}.lst-kix_c8eslinrrxlh-5>li:before{content:"\0025a0   "}ul.lst-kix_5xlsiqxwcoa5-8{list-style-type:none}.lst-kix_list_3-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}ul.lst-kix_list_5-5{list-style-type:none}.lst-kix_xd0keoi4jmwp-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-6{list-style-type:none}.lst-kix_list_8-1>li:before{content:"\0025cb   "}.lst-kix_xd0keoi4jmwp-1>li:before{content:"\0025cb   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}ul.lst-kix_list_5-0{list-style-type:none}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_8-7>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-7>li:before{content:"\0025a0   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-6>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-8>li:before{content:"\0025a0   "}.lst-kix_list_8-5>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-5>li:before{content:"\0025a0   "}.lst-kix_list_8-6>li:before{content:"\0025a0   "}.lst-kix_list_8-3>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-3>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-2>li:before{content:"\0025a0   "}.lst-kix_xd0keoi4jmwp-4>li:before{content:"\0025a0   "}.lst-kix_list_3-7>li:before{content:"\0025a0   "}.lst-kix_list_8-4>li:before{content:"\0025a0   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}.lst-kix_78vlvctpnz9j-2>li:before{content:"\0025a0   "}.lst-kix_78vlvctpnz9j-4>li:before{content:"\0025a0   "}.lst-kix_78vlvctpnz9j-1>li:before{content:"\0025cb   "}.lst-kix_78vlvctpnz9j-5>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-0>li:before{content:"\0025cf   "}.lst-kix_2bbejj5g047s-0>li:before{content:"\0025cf   "}.lst-kix_list_4-7>li:before{content:"\0025a0   "}.lst-kix_78vlvctpnz9j-0>li:before{content:"\0025cf   "}.lst-kix_78vlvctpnz9j-6>li:before{content:"\0025a0   "}.lst-kix_78vlvctpnz9j-8>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-2>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-1>li:before{content:"\0025cb   "}.lst-kix_78vlvctpnz9j-7>li:before{content:"\0025a0   "}ul.lst-kix_list_4-8{list-style-type:none}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_4-0{list-style-type:none}ul.lst-kix_list_4-1{list-style-type:none}ul.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}ul.lst-kix_list_4-3{list-style-type:none}.lst-kix_78vlvctpnz9j-3>li:before{content:"\0025a0   "}.lst-kix_5xlsiqxwcoa5-0>li:before{content:"\0025cf   "}.lst-kix_5xlsiqxwcoa5-2>li:before{content:"\0025a0   "}ul.lst-kix_96onxbcsb0ef-0{list-style-type:none}ul.lst-kix_96onxbcsb0ef-1{list-style-type:none}.lst-kix_5xlsiqxwcoa5-1>li:before{content:"\0025cb   "}ul.lst-kix_96onxbcsb0ef-2{list-style-type:none}ul.lst-kix_96onxbcsb0ef-3{list-style-type:none}.lst-kix_5xlsiqxwcoa5-4>li:before{content:"\0025a0   "}ul.lst-kix_96onxbcsb0ef-4{list-style-type:none}ul.lst-kix_96onxbcsb0ef-5{list-style-type:none}ul.lst-kix_96onxbcsb0ef-6{list-style-type:none}.lst-kix_uixjlpexb0mc-8>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-6>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-8>li:before{content:"\0025a0   "}ul.lst-kix_96onxbcsb0ef-7{list-style-type:none}ul.lst-kix_96onxbcsb0ef-8{list-style-type:none}.lst-kix_5xlsiqxwcoa5-3>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-7>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-4>li:before{content:"\0025a0   "}.lst-kix_5xlsiqxwcoa5-8>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-2>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-4>li:before{content:"\0025a0   "}.lst-kix_5xlsiqxwcoa5-6>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-3>li:before{content:"\0025a0   "}.lst-kix_5xlsiqxwcoa5-5>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-7>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-1>li:before{content:"\0025cb   "}.lst-kix_2bbejj5g047s-5>li:before{content:"\0025a0   "}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_uixjlpexb0mc-6>li:before{content:"\0025a0   "}.lst-kix_5xlsiqxwcoa5-7>li:before{content:"\0025a0   "}.lst-kix_uixjlpexb0mc-5>li:before{content:"\0025a0   "}.lst-kix_2bbejj5g047s-3>li:before{content:"\0025a0   "}.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}.lst-kix_list_7-1>li:before{content:"\0025cb   "}.lst-kix_list_7-5>li:before{content:"\0025a0   "}.lst-kix_list_7-3>li:before{content:"\0025a0   "}ul.lst-kix_list_7-5{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-2{list-style-type:none}ul.lst-kix_list_7-6{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-1{list-style-type:none}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-4{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-3{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-6{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-5{list-style-type:none}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-8{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-7{list-style-type:none}.lst-kix_96onxbcsb0ef-7>li:before{content:"\0025a0   "}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_xd0keoi4jmwp-0{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7>li:before{content:"\0025a0   "}.lst-kix_96onxbcsb0ef-5>li:before{content:"\0025a0   "}.lst-kix_96onxbcsb0ef-3>li:before{content:"\0025a0   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}.lst-kix_j2ures5lufxp-1>li:before{content:"\0025cb   "}.lst-kix_96onxbcsb0ef-1>li:before{content:"\0025cb   "}.lst-kix_list_4-4>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-5>li:before{content:"\0025a0   "}.lst-kix_list_4-2>li:before{content:"\0025a0   "}.lst-kix_list_4-6>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-3>li:before{content:"\0025a0   "}.lst-kix_j2ures5lufxp-7>li:before{content:"\0025a0   "}.lst-kix_list_9-3>li:before{content:"\0025a0   "}.lst-kix_list_9-1>li:before{content:"\0025cb   "}.lst-kix_c8eslinrrxlh-2>li:before{content:"\0025a0   "}.lst-kix_c8eslinrrxlh-0>li:before{content:"\0025cf   "}.lst-kix_c8eslinrrxlh-4>li:before{content:"\0025a0   "}.lst-kix_list_9-7>li:before{content:"\0025a0   "}.lst-kix_list_9-5>li:before{content:"\0025a0   "}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_6-2{list-style-type:none}ul.lst-kix_list_6-3{list-style-type:none}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_6-0{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"\0025cf   "}.lst-kix_c8eslinrrxlh-8>li:before{content:"\0025a0   "}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}.lst-kix_c8eslinrrxlh-6>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c2{-webkit-text-decoration-skip:none;color:#0000ee;font-weight:400;text-decoration:underline;vertical-align:baseline;text-decoration-skip-ink:none;font-size:12pt;font-family:"Google Sans";font-style:normal}.c1{margin-left:23.2pt;padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c24{margin-left:72pt;padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c5{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c6{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c19{margin-left:43.5pt;padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c21{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans";font-style:normal}.c36{color:#1b1c1d;font-weight:700;font-size:16pt;font-family:"Google Sans";font-style:normal}.c17{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans Text";font-style:normal}.c34{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left;height:11pt}.c26{color:#1b1c1d;font-weight:700;font-size:14pt;font-family:"Google Sans";font-style:normal}.c22{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c4{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c28{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";color:#575b5f;font-weight:400}.c15{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c0{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c12{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c30{padding-top:0pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c31{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left}.c16{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c35{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c11{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c38{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c32{padding-top:12pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c27{color:#000000;font-weight:400;font-family:"Google Sans Text"}.c9{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c8{color:#1b1c1d;font-weight:400;font-family:"Google Sans Text"}.c37{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c20{color:#1b1c1d;font-weight:700;font-family:"Google Sans Text"}.c25{margin-left:43.5pt;padding-left:0pt}.c33{margin-left:72pt;padding-left:0pt}.c10{color:inherit;text-decoration:inherit}.c3{text-decoration:none;vertical-align:baseline}.c14{margin-left:23.2pt;padding-left:0pt}.c23{font-size:11pt;font-style:normal}.c7{padding:0;margin:0}.c29{font-size:12pt}.c18{font-style:italic}.c13{text-decoration:none}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c37 doc-content"><h1 class="c12"><span class="c3 c36">Molecular Markers for Leaf Spot Resistance Traits in Peanut Breeding</span></h1><h2 class="c12"><span class="c21 c3">Executive Summary</span></h2><p class="c11"><span class="c0">Peanut (</span><span class="c4">Arachis hypogaea L.</span><span class="c0">) is a globally significant crop, providing essential nutrition and oil, but its production is severely hampered by fungal diseases, particularly early leaf spot (ELS) and late leaf spot (LLS).</span><span class="c6">1</span><span class="c0">&nbsp;These diseases cause substantial yield losses, often exceeding 50%, and compromise overall crop quality and economic sustainability.</span><span class="c6">3</span><span class="c0">&nbsp;While traditional management practices, including fungicide applications, are employed, their effectiveness is limited, often leading to overuse and persistent yield constraints.</span><span class="c6">1</span><span class="c0 c3">&nbsp;This ongoing challenge highlights an urgent and growing demand for intrinsic, genetic solutions to enhance resistance. The limitations of chemical controls, whether due to insufficient efficacy, environmental concerns, or economic impracticality in various regions, underscore the imperative for more sustainable and inherent resistance mechanisms within the plant itself.</span></p><p class="c11"><span class="c0">Recent advancements in molecular genetics and genomics have fundamentally transformed peanut breeding, marking a significant shift from traditional, often slow and resource-intensive, phenotypic selection to precise and accelerated genetic improvement.</span><span class="c6">7</span><span class="c0">&nbsp;This evolution is critical for addressing global food security challenges, which are increasingly exacerbated by climate change and heightened disease pressure. The availability of high-resolution genetic maps, the sequencing of the peanut genome, and the development of high-throughput genotyping technologies such as Single Nucleotide Polymorphisms (SNPs) and Kompetitive Allele-Specific PCR (KASP) assays are enabling more efficient identification and deployment of resistance traits.</span><span class="c6">10</span><span class="c0">&nbsp;Key molecular targets include Quantitative Trait Loci (QTLs) and specific Resistance genes (R-genes) that govern defense responses.</span><span class="c6">6</span><span class="c0">&nbsp;Furthermore, wild </span><span class="c4">Arachis</span><span class="c0">&nbsp;species serve as invaluable genetic resources, providing crucial alleles for strong resistance that are often lacking in cultivated varieties.</span><span class="c6">10</span><span class="c0">&nbsp;Genomic databases like PeanutBase play a pivotal role in disseminating this genetic information, thereby accelerating the development of resilient peanut cultivars worldwide.</span><span class="c6 c13">16</span></p><h2 class="c12"><span class="c21 c3">1. Introduction to Peanut Leaf Spot Diseases</span></h2><h3 class="c12"><span class="c15 c3">1.1. Global Significance and Economic Impact</span></h3><p class="c11"><span class="c0">Peanut (</span><span class="c4">Arachis hypogaea L.</span><span class="c0">), also known as groundnut, stands as a cornerstone of global agriculture, widely cultivated in warm climates across over a hundred countries.</span><span class="c6">14</span><span class="c0">&nbsp;It is a critical source of nutritious food, edible oil, and essential vitamins and minerals, playing a vital role in food security and economic sustainability, particularly in developing nations where it often serves as a key cash crop.</span><span class="c6">2</span><span class="c0">&nbsp;The United States, China, India, and Nigeria are among the leading producers, with significant cultivation areas in the South and North Americas, Asia, and Africa.</span><span class="c6 c13">2</span></p><p class="c11"><span class="c0">Despite its global importance, peanut production faces substantial challenges from various biotic and abiotic stresses, with fungal diseases being a primary constraint.</span><span class="c6">1</span><span class="c0">&nbsp;Among these, leaf spot diseases, encompassing Early Leaf Spot (ELS) and Late Leaf Spot (LLS), are the most prevalent and economically devastating.</span><span class="c6">1</span><span class="c0">&nbsp;ELS is caused by </span><span class="c4">Passalora arachidicola</span><span class="c0">, while LLS is caused by </span><span class="c4">Nothopassalora personata</span><span class="c0">.</span><span class="c6">3</span><span class="c0">&nbsp;These diseases lead to significant yield reductions, with estimates ranging from 10% to a staggering 80% depending on environmental conditions and cultivar susceptibility.</span><span class="c6">4</span><span class="c0">&nbsp;In severe epidemics, leaf spot diseases can cause complete defoliation and plant death, resulting in yield losses of 50% or more.</span><span class="c6">3</span><span class="c0">&nbsp;For instance, late leaf spot alone has been reported to reduce yields by 50% to 70% in some regions.</span><span class="c6">5</span><span class="c0">&nbsp;Beyond direct yield loss, the diseases also compromise seed size, oil content, and overall quality, impacting the marketability and economic value of the crop.</span><span class="c6 c13">2</span></p><p class="c11"><span class="c0">Traditional disease management strategies often rely on intensive fungicide spray programs.</span><span class="c6">6</span><span class="c0">&nbsp;However, these methods have inherent limitations, including the overuse and wastage of agrochemicals, as well as reduced effectiveness against certain diseases that affect root systems, where chemicals may not penetrate adequately.</span><span class="c6">1</span><span class="c0">&nbsp;The continued persistence of significant yield losses despite these traditional approaches underscores the pressing need for more efficient and sustainable solutions.</span><span class="c6">1</span><span class="c0">&nbsp;The wide variability in reported yield losses, from 10% to 80%, suggests that the impact of leaf spot diseases is not uniform across all regions or seasons.</span><span class="c6">4</span><span class="c0">&nbsp;This variability is influenced by factors such as local climate (humidity, temperature), soil conditions, the specific pathogen isolates present, and the intensity of fungicide application.</span><span class="c6">1</span><span class="c0">&nbsp;This observation highlights that a single, universal solution for resistance breeding might not be sufficient; instead, breeding programs must develop regionally adapted disease management strategies and resistant varieties that perform robustly across diverse environmental contexts. The substantial economic losses incurred from these diseases, coupled with the environmental and financial burdens of chemical control, create a powerful economic incentive for investing in genetically resistant varieties.</span><span class="c6">1</span><span class="c0">&nbsp;This strategic pivot from chemical dependence to genetic solutions represents a more sustainable, environmentally benign, and economically viable long-term approach for the global peanut industry.</span><span class="c6 c13">2</span></p><h3 class="c12"><span class="c15 c3">1.2. Early Leaf Spot (ELS) and Late Leaf Spot (LLS): Pathogens, Symptoms, and Disease Cycle</span></h3><p class="c11"><span class="c0 c3">Understanding the specific characteristics of ELS and LLS is crucial for effective disease management and resistance breeding. Both are fungal diseases that significantly impact peanut foliage.</span></p><p class="c31"><span class="c27 c3 c23">Pathogens:</span></p><p class="c31"><span class="c27 c3 c23">Early leaf spot is caused by Passalora arachidicola, previously known as Cercospora arachidicola.10 Late leaf spot is caused by Nothopassalora personata, formerly Cercosporidium personatum, and also known as Phaeoisariopsis personata.3 The sexual stage (teleomorph) for N. personata is Mycosphaerella berkeleyi, though its observation in nature is rare.3</span></p><p class="c31"><span class="c27 c3 c23">Symptoms:</span></p><p class="c31"><span class="c27 c3 c23">The symptoms of ELS and LLS, while distinct, can sometimes be challenging to differentiate, especially in early stages or in fields with mixed infections.</span></p><ul class="c7 lst-kix_list_1-0 start"><li class="c12 c14 li-bullet-0"><span class="c17">Early Leaf Spot (ELS):</span><span class="c0">&nbsp;ELS typically presents as penetrating brown lesions or spots visible on both the upper and lower leaf surfaces.</span><span class="c6">14</span><span class="c0">&nbsp;These spots are commonly surrounded by a light yellow halo, although the presence of a halo is not exclusively diagnostic as it can also occur with LLS.</span><span class="c6">18</span><span class="c0">&nbsp;During periods of high humidity, the fungus produces characteristic tufts of silvery, hair-like spores, which can be observed on the upper surface of the lesions, often with the aid of a magnifying glass.</span><span class="c6">18</span><span class="c0">&nbsp;ELS can appear as early as 30 days after planting, though it is usually observed later in well-rotated fields.</span><span class="c6 c13">18</span></li><li class="c1 li-bullet-0"><span class="c17">Late Leaf Spot (LLS):</span><span class="c0">&nbsp;LLS lesions are typically dark brown to black and penetrate through the leaf, making them visible on both upper and lower surfaces.</span><span class="c6">3</span><span class="c0">&nbsp;A distinguishing feature of LLS is the dense mass of dark spores on the underside (abaxial surface) of the leaf, which gives the spot a velvety appearance and can often be seen without magnification.</span><span class="c6">3</span><span class="c0">&nbsp;LLS symptoms generally appear later in the growing season, typically between 30 to 50 days after planting.</span><span class="c6">4</span><span class="c0">&nbsp;Lesions are circular, ranging from 1 to 10 mm in diameter, and can coalesce.</span><span class="c6 c13">3</span></li></ul><p class="c35"><span class="c0">Differentiating between ELS and LLS, particularly early in the season, can be difficult due to their subtle symptomatic differences and the common occurrence of mixed infections.</span><span class="c6">3</span><span class="c0">&nbsp;Visual diagnosis can be confounded by spots caused by pesticide injuries or other irregular leaf spots.</span><span class="c6">18</span><span class="c0">&nbsp;Confirmation of diagnosis relies on the presence and specific location of spores (upper surface for ELS, lower for LLS), which can be encouraged by placing symptomatic leaves in a moist chamber for a day or two to induce sporulation.</span><span class="c6">18</span><span class="c0">&nbsp;The subtle symptomatic differences between ELS and LLS and the common occurrence of mixed infections necessitate careful and often labor-intensive diagnosis.</span><span class="c6">3</span><span class="c0">&nbsp;This diagnostic challenge highlights the significant value of molecular markers, which offer a more precise, rapid, and objective identification method for breeding programs, bypassing the need for subjective visual assessment or specific conditions for sporulation.</span><span class="c6 c13">18</span></p><p class="c31"><span class="c27 c3 c23">Disease Cycle:</span></p><p class="c31"><span class="c3 c23 c27">Both ELS and LLS fungi overwinter in peanut crop debris from previous seasons.3 Old lesions on infected leaves, stems, and petioles serve as the primary source of initial inoculum.3 Spores (conidia) produced from these lesions are then dispersed by air currents, wind, or splashing water onto newly planted peanut leaves, initiating new infection cycles.3 The presence of abundant peanut residue and infected volunteer plants in fields with short rotations significantly contributes to the early and rapid development of leaf spot diseases.3</span></p><p class="c31"><span class="c27 c3 c23">Environmental Conditions:</span></p><p class="c31"><span class="c27 c3 c23">High humidity and warm temperatures are highly conducive to the development and spread of both leaf spot diseases.</span></p><ul class="c7 lst-kix_list_2-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">For ELS, temperatures between approximately 65 and 90 degrees Fahrenheit (18-32&deg;C) are very favorable for infection.</span><span class="c6 c13">18</span></li><li class="c1 li-bullet-0"><span class="c0">For LLS, optimal conditions include temperatures near 20&deg;C and prolonged periods (12 hours or more) of high relative humidity (&gt;93%) or leaf wetness.</span><span class="c6">3</span><span class="c0">&nbsp;Little disease develops if temperatures are outside the 15-28&deg;C range or if leaf wetness periods are insufficient.</span><span class="c6">3</span><span class="c0">&nbsp;Peanut rust, another significant fungal disease, also thrives in warm and humid conditions (20&ndash;30 &deg;C).</span><span class="c6 c13">2</span></li><li class="c1 li-bullet-0"><span class="c0">Under favorable conditions, lesions typically become visible around 10 to 12 days after infection for ELS, with sporulation beginning 2 to 4 days later and continuing for up to two weeks.</span><span class="c6">18</span><span class="c0">&nbsp;For LLS, mature sporulating lesions may appear 14 to 24 days after infection.</span><span class="c6">3</span><span class="c0">&nbsp;Multiple secondary infection cycles can occur within a single growing season, leading to a rapid increase in disease severity.</span><span class="c6">3</span><span class="c0">&nbsp;The consistent environmental conditions favoring leaf spot development, particularly high humidity and warm temperatures, highlight the importance of breeding for broad-spectrum resistance or resistance that is robust across diverse agro-climatic zones.</span><span class="c6">1</span><span class="c0 c3">&nbsp;This also suggests that climate change, potentially leading to more favorable conditions in new or expanded regions, could exacerbate the problem, making genetic resistance even more critical for future peanut production and global food security.</span></li></ul><h2 class="c32"><span class="c21 c3">2. Genetic Architecture of Leaf Spot Resistance</span></h2><h3 class="c12"><span class="c15 c3">2.1. Quantitative Trait Loci (QTLs) Governing ELS and LLS Resistance</span></h3><p class="c11"><span class="c0">Resistance to leaf spot diseases in peanut is a complex genetic trait, characterized by its polygenic nature, meaning it is controlled by multiple major and minor Quantitative Trait Loci (QTLs).</span><span class="c6">6</span><span class="c0">&nbsp;Significant advancements have been made in identifying and precisely mapping these QTLs for both ELS and LLS resistance through the use of various genetic mapping populations, such as Recombinant Inbred Lines (RILs), and the application of advanced molecular marker systems.</span><span class="c6 c13">6</span></p><p class="c12"><span class="c17 c3">Key QTLs identified for LLS resistance include:</span></p><ul class="c7 lst-kix_list_3-0 start"><li class="c12 c14 li-bullet-0"><span class="c20 c29 c18">qLLSB05:</span><span class="c0">&nbsp;This QTL, located on chromosome B05, has demonstrated remarkable consistency across multi-year analyses, explaining a substantial proportion of the phenotypic variation (PVE) ranging from 15% to 41%.</span><span class="c6">6</span><span class="c0">&nbsp;This high PVE indicates a strong genetic effect, meaning that selecting for this QTL will have a significant impact on resistance. Its consistent expression over multiple years further enhances its reliability and utility in breeding programs, suggesting stable genetic control rather than environmentally sensitive expression, which directly translates to more efficient and predictable resistance breeding. Notably, </span><span class="c4">qLLSB05</span><span class="c0">&nbsp;is anchored by two NBS-LRR resistance genes, indicating a direct genetic link to specific defense mechanisms.</span><span class="c6 c13">10</span></li><li class="c1 li-bullet-0"><span class="c20 c29 c18">qLLSA05:</span><span class="c0">&nbsp;Found on chromosome A05, this QTL contributes between 7% and 10% of the phenotypic variation.</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c20 c18 c29">qLLSB03:</span><span class="c0">&nbsp;Located on chromosome B03, this QTL explains 5% to 7% of the phenotypic variation.</span><span class="c6">6</span><span class="c0">&nbsp;The presence of resistance alleles in both </span><span class="c4">qLLSB03</span><span class="c0">&nbsp;and </span><span class="c4">qLLSB05</span><span class="c0">&nbsp;loci has been correlated with protection from yield loss caused by LLS disease damage.</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c0">Other major QTLs for LLS resistance have been reported on linkage groups AhXV (now designated A03) and AhXII (now A02).</span><span class="c6 c13">7</span></li><li class="c1 li-bullet-0"><span class="c0">A major QTL for LLS resistance, </span><span class="c4">qLLSA02</span><span class="c0">&nbsp;on chromosome A02, has been specifically associated with resistance in the CS16 variety, a line known to derive its resistance from the wild species </span><span class="c4">Arachis cardenasii</span><span class="c0">.</span><span class="c6 c13">20</span></li></ul><p class="c32"><span class="c3 c17">Key QTLs identified for ELS resistance include:</span></p><ul class="c7 lst-kix_list_4-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">Three resistance QTLs have been consistently identified across multiple years: two on chromosome A03 (PVE = 8&ndash;12%) and one on chromosome B03 (PVE = 13&ndash;20%).</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c0">The locus </span><span class="c4">qELSA03_1.1</span><span class="c0">&nbsp;on chromosome A03 is particularly noteworthy as it coincides with a previously published genomic region associated with LLS resistance in the GPBD4 variety.</span><span class="c6">6</span><span class="c0 c3">&nbsp;This overlap suggests potential shared genetic pathways or closely linked resistance genes that could confer resistance to both diseases.</span></li><li class="c1 li-bullet-0"><span class="c0">Recent studies using an interspecific AB-QTL population have identified five QTLs for ELS resistance on chromosomes A02, A03, A08, B04, and B09.</span><span class="c6">15</span><span class="c0">&nbsp;A significant finding from these studies is that favorable alleles at all but the B09 QTL were contributed by wild species, highlighting the critical role of genetic diversity from wild relatives. This observation reinforces the necessity of ongoing efforts to introgress these alleles into cultivated peanut, especially given the narrow genetic base of </span><span class="c4">Arachis hypogaea</span><span class="c0 c3">. This strategy is essential for ensuring long-term resistance durability and preventing resistance breakdown, as relying solely on existing cultivated germplasm may not provide sufficient genetic variation for sustainable resistance.</span></li><li class="c1 li-bullet-0"><span class="c0">A specific genomic region on chromosome A02 has been identified as containing several relevant QTLs, contributing not only to ELS resistance but also to desirable agronomic traits such as earliness and increased biomass yield.</span><span class="c6">15</span><span class="c0">&nbsp;The co-localization of QTLs for ELS resistance with other desirable agronomic traits presents a significant opportunity for multi-trait selection in breeding. This &quot;trait stacking&quot; or &quot;pyramiding&quot; using a single genomic region can dramatically improve breeding efficiency by simultaneously selecting for disease resistance and yield-enhancing characteristics, leading to the development of more commercially viable cultivars.</span><span class="c6 c13">21</span></li><li class="c1 li-bullet-0"><span class="c0">Two major QTLs for ELS resistance have been reported on chromosomes A03 and B04, and these are associated with NBS-LRR resistance genes.</span><span class="c6 c13">10</span></li><li class="c1 li-bullet-0"><span class="c0">Other studies have assigned major QTLs for ELS resistance to linkage groups A01 and A03.</span><span class="c6 c13">7</span></li></ul><p class="c35"><span class="c0">Comparative analysis of the genomic regions spanning these QTLs suggests that resistance to ELS and LLS is largely genetically independent.</span><span class="c6">6</span><span class="c0">&nbsp;This finding has significant implications for breeding strategies. It suggests that breeders cannot assume that selecting for resistance to one leaf spot disease will automatically confer resistance to the other. Therefore, pyramiding resistance genes for both diseases might require targeting distinct genomic regions, potentially increasing the complexity but also the specificity of breeding efforts. However, the occasional overlap, such as </span><span class="c4">qELSA03_1.1</span><span class="c0">&nbsp;coinciding with LLS resistance, offers opportunities for selecting for both traits simultaneously, potentially streamlining breeding efforts.</span><span class="c6 c13">6</span></p><h3 class="c12"><span class="c15 c3">2.2. Identification and Role of Resistance Genes (R-genes)</span></h3><p class="c11"><span class="c0">Resistance genes (R-genes) are a fundamental component of the plant immune system, playing a critical role in recognizing pathogen-derived molecules and initiating defense responses. These genes have been successfully identified in peanut, providing specific molecular targets for enhancing disease resistance.</span><span class="c6 c13">14</span></p><p class="c11"><span class="c0">A comprehensive study involving advanced breeding lines evaluated 76 candidate R-genes in peanut. This research revealed that 36 of these R-genes were differentially expressed and significantly correlated with resistant lines.</span><span class="c6">14</span><span class="c0">&nbsp;The majority of these identified R-genes belong to the Receptor-Like Kinases (RLKs) and Receptor-Like Proteins (RLPs) classes. These proteins are crucial for sensing the presence of pathogens at the cell surface, acting as the plant&#39;s first line of defense by recognizing pathogen-associated molecular patterns (PAMPs) and initiating a protective response.</span><span class="c6 c13">14</span></p><p class="c11"><span class="c0">The largest group among the identified R-genes was Receptor-Like Cytoplasmic Kinases (RLCKs) VII. These RLCKs are known to be involved in pattern-triggered kinase signaling, a key pathway in plant immunity that leads to the production of reactive oxygen species (ROS).</span><span class="c6">14</span><span class="c0 c3">&nbsp;ROS generation is a rapid and potent defense mechanism, contributing to localized cell death and strengthening of cell walls to restrict pathogen spread. The identification of specific R-gene classes and their known mechanistic roles in pathogen sensing and defense initiation provides a deeper, mechanistic understanding of peanut resistance. This moves beyond simply identifying a genomic region (QTL) to understanding the underlying biological pathways, which can inform more targeted gene editing or functional genomics approaches in breeding for enhanced and durable resistance.</span></p><p class="c11"><span class="c0">Intriguingly, four of the identified R-genes showed homology to the TMV resistant protein N, which has been demonstrated to confer resistance against tobacco mosaic virus.</span><span class="c6">14</span><span class="c0">&nbsp;This observation suggests the presence of conserved defense mechanisms across different plant species and against diverse pathogens, potentially indicating broad-spectrum resistance capabilities. When these 36 R-genes were mapped to the peanut genomes, they were found to be widely distributed across most chromosomes, with the exceptions of A09 and B09.</span><span class="c6 c13">14</span></p><p class="c11"><span class="c0">A notable observation was the low levels of gene expression in resistant lines for these R-genes.</span><span class="c6">14</span><span class="c0">&nbsp;This suggests a finely tuned regulatory mechanism where the plant balances defense activation with overall growth and productivity. This implies that simply overexpressing R-genes might not be the optimal strategy for breeding; rather, breeders might need to select for alleles that confer effective resistance without imposing a significant fitness cost, ensuring that resistant varieties maintain good agronomic traits and yield potential.</span><span class="c6">14</span><span class="c0">&nbsp;This represents a crucial trade-off in plant breeding, as high resistance is desirable, but if it comes at the expense of yield or other agronomic traits, it may not be practical for farmers. Therefore, the most effective resistance alleles are those that provide adequate defense while minimizing metabolic burden, making it a multi-objective optimization problem in breeding. Furthermore, some major QTLs for resistance, such as the LLS resistance QTL on chromosome B05, are specifically anchored by NBS-LRR resistance genes </span><span class="c6">10</span><span class="c0 c3">, further solidifying the direct link between specific R-genes and phenotypic resistance to leaf spot diseases.</span></p><h2 class="c12"><span class="c21 c3">3. Molecular Markers for Leaf Spot Resistance Traits</span></h2><h3 class="c12"><span class="c3 c15">3.1. Overview of Marker Systems in Peanut (SNPs, SSRs, InDels, RFLPs, etc.)</span></h3><p class="c11"><span class="c0">Molecular markers have become indispensable tools in modern plant breeding, significantly expediting the selection process for desired traits. In peanut, these markers have been successfully employed for improving characteristics such as high oleic acid content and rust resistance.</span><span class="c6">7</span><span class="c0">&nbsp;Over the years, a diverse array of marker systems has been utilized in peanut genetic studies and breeding programs. These include Restriction Fragment Length Polymorphisms (RFLPs), Random Amplified Polymorphic DNAs (RAPDs), Amplified Fragment Length Polymorphisms (AFLPs), Simple Sequence Repeats (SSRs), Inter Simple Sequence Repeats (ISSRs), Single Nucleotide Polymorphisms (SNPs), and Insertion/Deletion (InDel) markers.</span><span class="c6 c13">7</span></p><p class="c11"><span class="c0">Historically, RFLPs were among the first marker systems to reveal a large number of polymorphisms and were widely used for creating linkage maps and implementing indirect selection strategies.</span><span class="c6">24</span><span class="c0">&nbsp;However, cultivated peanut generally exhibited limited molecular variation when analyzed with RFLP technology.</span><span class="c6">24</span><span class="c0">&nbsp;Similarly, while nearly 10,000 SSR markers have been identified globally, their widespread application in peanut breeding has been constrained by low levels of polymorphism, with only about 14.5% showing polymorphism and 6.4% being mapped.</span><span class="c6">23</span><span class="c0">&nbsp;This low polymorphism is primarily attributed to the extremely narrow genetic basis of cultivated peanut.</span><span class="c6">23</span><span class="c0">&nbsp;This persistent low level of polymorphism in cultivated peanut represents a significant bottleneck for marker-assisted breeding. This inherent genetic limitation necessitates the strategic development and adoption of more polymorphic marker systems like SNPs and InDels, and critically, the utilization of wild relatives to introduce greater genetic diversity into breeding populations.</span><span class="c6">9</span><span class="c0 c3">&nbsp;This highlights the importance of leveraging external genetic resources to overcome intrinsic genetic constraints.</span></p><p class="c11"><span class="c0">In recent years, Single Nucleotide Polymorphisms (SNPs) have emerged as a more popular and user-friendly marker type.</span><span class="c6">20</span><span class="c0">&nbsp;SNPs offer significantly higher density and mapping resolution, particularly with the advent of next-generation sequencing (NGS)-based genotyping.</span><span class="c6">9</span><span class="c0">&nbsp;High-density SNP panels, such as those containing 2500 SNP markers distributed across the 20 groundnut chromosomes, are now available, providing powerful tools for breeders to estimate background genome recovery, conduct diversity analyses, and manage germplasm efficiently.</span><span class="c6 c13">12</span></p><p class="c11"><span class="c0">Insertion/Deletion (InDel) markers are also increasingly recognized as an abundant source of genetic markers widely distributed across the genome.</span><span class="c6">9</span><span class="c0">&nbsp;They have been reported to exhibit higher polymorphism than SSRs in some crops and have been successfully developed from functional genes in peanut.</span><span class="c6">9</span><span class="c0">&nbsp;Five InDel markers derived from functional genes have been identified as significantly correlated to leaf spot resistance, demonstrating their potential utility in identifying disease-resistant genotypes within breeding populations.</span><span class="c6 c13">23</span></p><p class="c11"><span class="c0">Kompetitive Allele-Specific PCR (KASP) assays, a fluorescence-based SNP genotyping system, are being developed and validated for their cost-effectiveness and suitability for large-scale genotyping in modern breeding programs.</span><span class="c6">11</span><span class="c0">&nbsp;This clear technological shift towards high-density SNP maps and KASP assays signifies a move towards more efficient, precise, and cost-effective large-scale genotyping. This advancement is not merely about finding more markers, but about making marker-assisted selection (MAS) practical and scalable for routine breeding programs, thereby fundamentally accelerating the rate of cultivar development.</span><span class="c6">8</span><span class="c0 c3">&nbsp;This evolution in marker technology and genotyping methods directly addresses the practical limitations of traditional breeding. High-density maps provide better resolution for QTL mapping, while cost-effective, high-throughput genotyping makes MAS feasible for large breeding populations. This means that molecular breeding is moving from a research-intensive activity to a more routine, industrialized process, democratizing its application in breeding programs worldwide.</span></p><h3 class="c12"><span class="c15 c3">3.2. Specific QTLs and Linked Markers for Early Leaf Spot Resistance</span></h3><p class="c11"><span class="c0 c3">Significant progress has been made in identifying specific QTLs and developing linked molecular markers for Early Leaf Spot (ELS) resistance, which are crucial for accelerating peanut breeding.</span></p><p class="c12"><span class="c17 c3">QTLs for ELS Resistance:</span></p><ul class="c7 lst-kix_list_5-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">Recent studies using interspecific populations have mapped five QTLs for ELS resistance on chromosomes A02, A03, A08, B04, and B09.</span><span class="c6">15</span><span class="c0">&nbsp;A key finding is that favorable alleles for these QTLs were primarily contributed by wild </span><span class="c4">Arachis</span><span class="c0">&nbsp;species, underscoring their importance as genetic resources.</span><span class="c6 c13">15</span></li><li class="c1 li-bullet-0"><span class="c0">A particularly significant genomic region on chromosome A02 has been identified, containing multiple QTLs that contribute not only to ELS resistance but also to desirable agronomic traits such as earliness and increased biomass yield.</span><span class="c6">15</span><span class="c0">&nbsp;This co-localization of QTLs for ELS resistance with other desirable agronomic traits presents a significant opportunity for multi-trait selection in breeding. This &quot;trait stacking&quot; or &quot;pyramiding&quot; using a single genomic region can dramatically improve breeding efficiency by simultaneously selecting for disease resistance and yield-enhancing characteristics, leading to the development of more commercially viable cultivars.</span><span class="c6 c13">21</span></li><li class="c1 li-bullet-0"><span class="c0">Two QTLs on chromosome A03 have been identified, explaining 8&ndash;12% of the phenotypic variation for ELS resistance, along with one on chromosome B03, explaining 13&ndash;20% of the phenotypic variation.</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c0">One major QTL on A03, designated </span><span class="c4">qELSA03_1.1</span><span class="c0">, is notable for its co-occurrence with a genomic region previously associated with LLS resistance, suggesting potential shared genetic pathways or closely linked resistance genes.</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c0">Two major QTLs for ELS resistance have been reported on chromosomes A03 and B04, and these are associated with NBS-LRR resistance genes, indicating a direct genetic basis for the resistance.</span><span class="c6 c13">10</span></li><li class="c1 li-bullet-0"><span class="c0">Other research has assigned major QTLs for ELS resistance to linkage groups A01 and A03.</span><span class="c6 c13">7</span></li></ul><p class="c32"><span class="c17 c3">Molecular Markers for ELS Resistance:</span></p><ul class="c7 lst-kix_list_6-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">While specific SNP marker names for ELS resistance are not explicitly listed as validated markers on the provided PeanutBase&#39;s Marker-Assisted Selection (MAS) page </span><span class="c6">15</span><span class="c0">, the Axiom-Arachis 48K SNPs chip has been extensively utilized for genotyping in studies that successfully mapped ELS QTLs.</span><span class="c6">15</span><span class="c0 c3">&nbsp;This high-density SNP array provides a robust platform for identifying markers linked to resistance.</span></li><li class="c1 li-bullet-0"><span class="c0">New SNP markers are continuously being developed and placed on genetic maps, with ongoing efforts to fill existing gaps and identify additional QTLs for both early and late leaf spot resistance.</span><span class="c6">27</span><span class="c0">&nbsp;For instance, a revised genetic linkage map for an interspecific introgression population of peanut has been developed from SNP markers, with studies underway to identify additional QTLs for ELS and LLS resistance.</span><span class="c6 c13">27</span></li><li class="c1 li-bullet-0"><span class="c0">Five InDel markers derived from functional genes have been identified as significantly correlated to leaf spot, demonstrating their potential utility in identifying disease-resistant genotypes within breeding populations.</span><span class="c6">23</span><span class="c0">&nbsp;These InDels, identified from conserved sequences of functional genes, offer a promising avenue for marker-assisted selection, particularly given their reported higher polymorphism compared to SSRs in some crops.</span><span class="c6 c13">23</span></li></ul><h3 class="c32"><span class="c15 c3">3.3. Specific QTLs and Linked Markers for Late Leaf Spot Resistance</span></h3><p class="c11"><span class="c0 c3">Extensive research has focused on identifying and validating QTLs and their associated molecular markers for Late Leaf Spot (LLS) resistance, a critical step in developing resistant peanut varieties.</span></p><p class="c12"><span class="c17 c3">QTLs for LLS Resistance:</span></p><ul class="c7 lst-kix_list_7-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">Major QTLs for LLS resistance have been consistently identified on chromosome B05, notably </span><span class="c4">qLLSB05</span><span class="c0">, which explains a significant proportion of phenotypic variation (15&ndash;41%) and is anchored by two NBS-LRR resistance genes.</span><span class="c6">6</span><span class="c0">&nbsp;The high phenotypic variation explained by </span><span class="c4">qLLSB05</span><span class="c0">&nbsp;and its consistent expression over multi-year analyses makes it a highly reliable and valuable target for marker-assisted selection in breeding programs.</span><span class="c6">6</span><span class="c0 c3">&nbsp;This indicates a strong and stable genetic control over LLS resistance, allowing for more predictable breeding outcomes.</span></li><li class="c1 li-bullet-0"><span class="c0">Other significant QTLs include </span><span class="c4">qLLSA05</span><span class="c0">&nbsp;on chromosome A05 (PVE = 7&ndash;10%) and </span><span class="c4">qLLSB03</span><span class="c0">&nbsp;on chromosome B03 (PVE = 5&ndash;7%).</span><span class="c6">6</span><span class="c0">&nbsp;The presence of resistance alleles at </span><span class="c4">qLLSB03</span><span class="c0">&nbsp;and </span><span class="c4">qLLSB05</span><span class="c0">&nbsp;loci has been directly correlated with protection from yield loss caused by LLS disease.</span><span class="c6 c13">6</span></li><li class="c1 li-bullet-0"><span class="c0">Additional major QTLs for LLS resistance have been reported on linkage groups AhXV (now designated A03) and AhXII (now A02).</span><span class="c6 c13">7</span></li><li class="c1 li-bullet-0"><span class="c0">The cultivated peanut variety CS16 (ICGV 86855) is a well-known source of LLS resistance, with its resistance primarily traced to the wild diploid peanut </span><span class="c4">Arachis cardenasii</span><span class="c0">.</span><span class="c6">20</span><span class="c0">&nbsp;A major QTL, </span><span class="c4">qLLSA02</span><span class="c0">&nbsp;on chromosome A02, has been identified as conferring LLS resistance in CS16.</span><span class="c6">20</span><span class="c0">&nbsp;</span><span class="c4">A. cardenasii</span><span class="c0">&nbsp;remains the most widely used wild source of resistance to leaf spot diseases.</span><span class="c6">15</span><span class="c0">&nbsp;This highlights the critical role of wild relatives in providing strong resistance alleles that are often limited in cultivated peanut germplasm.</span><span class="c6">10</span><span class="c0">&nbsp;While </span><span class="c4">A. cardenasii</span><span class="c0">&nbsp;has been highly effective, the continuous use of new diploid species as sources of resistance is important to diversify options for breeders and help avoid potential resistance breakdown in the future.</span><span class="c6 c13">15</span></li></ul><p class="c32"><span class="c17 c3">Molecular Markers for LLS Resistance:</span></p><ul class="c7 lst-kix_list_8-0 start"><li class="c12 c14 li-bullet-0"><span class="c0">Several SNP markers have been developed and validated for LLS resistance. A recent study validated three SNP markers&mdash;</span><span class="c17">GKAMA02GL582, GKAMA02GL829, and GKAMA02GL975</span><span class="c0">&mdash;showing strong association with LLS resistance, with adjusted R&sup2; values of 0.758, 0.73, and 0.613, respectively.</span><span class="c6">20</span><span class="c0">&nbsp;These markers were designed from the major QTL </span><span class="c4">qLLSA02</span><span class="c0">&nbsp;on chromosome A02, which is known to confer LLS resistance in the CS16 variety.</span><span class="c6">20</span><span class="c0">&nbsp;The successful validation of these markers means they can be directly applied in future breeding programs to identify LLS-resistant groundnut lines, significantly speeding up the introgression of this trait.</span><span class="c6 c13">20</span></li><li class="c1 li-bullet-0"><span class="c0">Simple Sequence Repeat (SSR) markers have also been associated with LLS resistance. For instance, in a cross between a susceptible genotype (TMV 2) and a resistant genotype (COG 0437), three SSR primers&mdash;</span><span class="c17">PM 375162, pPGPseq5D5220, and PM 384100</span><span class="c0">&mdash;were able to distinguish resistant and susceptible bulks and individuals for LLS.</span><span class="c6">28</span><span class="c0">&nbsp;In single marker analysis, markers such as PM 375, PM 384, pPGPseq5D5, PM 137, PM 3, PMc 588, and Ah 4-26 were linked with LLS severity scores, explaining phenotypic variation ranging from 32% to 59%.</span><span class="c6">28</span><span class="c0">&nbsp;The primer PM 384100 allele, in particular, showed a clear association with resistance during validation across a set of genotypes.</span><span class="c6 c13">28</span></li><li class="c1 li-bullet-0"><span class="c0">The Groundnut DArTag SNP panel, a cost-effective mid-density genotyping panel containing 2500 SNP markers, includes 72 associated markers for 8 key traits, among which late leaf spot resistance is explicitly listed.</span><span class="c6">12</span><span class="c0 c3">&nbsp;This panel provides a powerful tool for groundnut breeders worldwide.</span></li><li class="c1 li-bullet-0"><span class="c0">PeanutBase, a primary genetics and genomics database for cultivated peanut and its wild relatives, lists &quot;Late leaf spot (</span><span class="c4">Phaeoisariopsis personata</span><span class="c0">&nbsp;infection)&quot; as a trait for which Marker-Assisted Selection (MAS) has been used, providing a valuable resource for breeders seeking information on associated markers and germplasm.</span><span class="c6 c13">26 &nbsp;</span></li></ul><h3 class="c12" id="h.8regh5ba3xp9"><span class="c3 c26">Molecular Marker Sequences for Leaf Spot Resistance</span></h3><p class="c11"><span class="c20 c3 c23">1. Single Nucleotide Polymorphism (SNP) Markers for Late Leaf Spot (LLS) Resistance:</span></p><p class="c12"><span class="c8">Three SNP markers have been validated for strong association with LLS resistance, particularly linked to the </span><span class="c8 c18">qLLSA02</span><span class="c8">&nbsp;QTL on chromosome A02.</span><span class="c28">1</span><span class="c8">&nbsp;While their specific primer sequences are proprietary to the LGC Group </span><span class="c28">3</span><span class="c8 c3 c23">, the allele information for these markers is available:</span></p><ul class="c7 lst-kix_xd0keoi4jmwp-0 start"><li class="c12 c14 li-bullet-0"><span class="c20 c3 c23">GKAMA02GL582:</span></li></ul><ul class="c7 lst-kix_j2ures5lufxp-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Position: 1271582 </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Resistance allele: G </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Susceptible allele: C </span><span class="c6 c13">2</span></li></ul><ul class="c7 lst-kix_xd0keoi4jmwp-0"><li class="c1 li-bullet-0"><span class="c20 c3 c23">GKAMA02GL829:</span></li></ul><ul class="c7 lst-kix_c8eslinrrxlh-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Position: 435829 </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Resistance allele: C </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Susceptible allele: A </span><span class="c6 c13">2</span></li></ul><ul class="c7 lst-kix_xd0keoi4jmwp-0"><li class="c1 li-bullet-0"><span class="c20 c3 c23">GKAMA02GL975:</span></li></ul><ul class="c7 lst-kix_78vlvctpnz9j-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Position: 1156975 </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Resistance allele: G </span><span class="c6 c13">2</span></li><li class="c19 li-bullet-0"><span class="c8">Susceptible allele: C </span><span class="c6 c13">2</span></li></ul><p class="c35"><span class="c20 c3 c23">2. Simple Sequence Repeat (SSR) Markers:</span></p><p class="c12"><span class="c8 c3 c23">Several SSR markers have been associated with leaf spot resistance, and some of their primer sequences are available:</span></p><ul class="c7 lst-kix_96onxbcsb0ef-0 start"><li class="c12 c14 li-bullet-0"><span class="c20">IPAHM103:</span><span class="c8">&nbsp;This marker is linked to a major QTL for rust resistance, which often co-occurs with LLS.</span><span class="c6 c13">4</span></li></ul><ul class="c7 lst-kix_uixjlpexb0mc-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Forward primer: GCATTCACCACCATAGTCCA </span><span class="c6 c13">4</span></li><li class="c19 li-bullet-0"><span class="c8 c3 c23">(Reverse primer sequence is not provided in the available information).</span></li></ul><ul class="c7 lst-kix_96onxbcsb0ef-0"><li class="c1 li-bullet-0"><span class="c20 c3 c23">GM2301:</span></li></ul><ul class="c7 lst-kix_k1m9p4de5zmc-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Motif/Pattern: (TGC)7 </span><span class="c6 c13">7</span></li><li class="c19 li-bullet-0"><span class="c8">Forward primer: GTAACCACAGCTGGCATGAAC </span><span class="c6 c13">7</span></li><li class="c19 li-bullet-0"><span class="c8">Reverse primer: TCTTCAAGAACCCACCAACAC </span><span class="c6 c13">7</span></li></ul><ul class="c7 lst-kix_96onxbcsb0ef-0"><li class="c1 li-bullet-0"><span class="c20 c3 c23">GM2079:</span></li></ul><ul class="c7 lst-kix_2bbejj5g047s-1 start"><li class="c12 c33 li-bullet-0"><span class="c8">Motif/Pattern: (CAG)6 </span><span class="c6 c13">8</span></li><li class="c24 li-bullet-0"><span class="c8">Forward primer: GGCCAAGGAGAAGAAGAAAGA </span><span class="c6 c13">8</span></li><li class="c24 li-bullet-0"><span class="c8">Reverse primer: GAAGGAGTAGTGGTGCTGCTG </span><span class="c6 c13">8</span></li></ul><ul class="c7 lst-kix_96onxbcsb0ef-0"><li class="c1 li-bullet-0"><span class="c20">PM 384100:</span><span class="c8">&nbsp;This marker has shown association with LLS resistance.</span><span class="c6 c13">9</span></li></ul><ul class="c7 lst-kix_5xlsiqxwcoa5-1 start"><li class="c12 c25 li-bullet-0"><span class="c8">Forward primer: GGCGTGCCAATAGAGGTTTA </span><span class="c6 c13">10</span></li><li class="c19 li-bullet-0"><span class="c8">Reverse primer: TGAAAACCAACAAGTTTAGTCTCTCT </span><span class="c6 c13">10</span></li></ul><p class="c35"><span class="c20 c3 c23">3. Insertion/Deletion (InDel) Markers:</span></p><p class="c11"><span class="c8">Five InDel markers derived from functional genes have been identified as significantly correlated with leaf spot resistance.</span><span class="c28">11</span><span class="c8">&nbsp;While their utility in identifying disease-resistant genotypes is noted, the specific sequences for these InDel markers or their primers are not provided in the available research material.</span><span class="c6 c13">13</span></p><p class="c34"><span class="c6 c13"></span></p><h2 class="c32"><span class="c3 c21">Conclusion</span></h2><p class="c11"><span class="c0 c3">The persistent and significant yield losses caused by early and late leaf spot diseases underscore the critical need for robust genetic resistance in peanut cultivation. The limitations of traditional disease management, including fungicide applications, further emphasize the urgency of developing inherently resistant varieties. Recent advancements in molecular genetics have ushered in a transformative era for peanut breeding, enabling a more precise and accelerated approach to cultivar development.</span></p><p class="c11"><span class="c0">The identification and mapping of numerous Quantitative Trait Loci (QTLs) for both ELS and LLS resistance represent a cornerstone of this progress. Major QTLs, such as </span><span class="c4">qLLSB05</span><span class="c0 c3">&nbsp;on chromosome B05 for LLS and regions on chromosomes A02 and A03 for ELS, explain substantial phenotypic variation and demonstrate consistent expression, making them highly reliable targets for marker-assisted selection. The detailed understanding of specific Resistance genes (R-genes), including various Receptor-Like Kinases (RLKs) and Receptor-Like Proteins (RLPs), provides a deeper mechanistic insight into peanut&#39;s defense responses, paving the way for more targeted breeding strategies.</span></p><p class="c11"><span class="c0 c3">The evolution of molecular marker systems, particularly the shift towards high-density Single Nucleotide Polymorphism (SNP) maps and cost-effective Kompetitive Allele-Specific PCR (KASP) assays, has overcome historical limitations posed by the narrow genetic base of cultivated peanut. These advanced tools facilitate large-scale, high-throughput genotyping, making marker-assisted selection a practical and scalable component of routine breeding programs. The successful validation of specific SNP markers, such as GKAMA02GL582, GKAMA02GL829, and GKAMA02GL975 for LLS resistance, directly supports their immediate application in identifying resistant lines. Similarly, the identification of promising InDel markers further diversifies the molecular toolkit available to breeders.</span></p><p class="c11"><span class="c0">Crucially, the recognition that wild </span><span class="c4">Arachis</span><span class="c0 c3">&nbsp;species contribute favorable resistance alleles for many QTLs highlights their indispensable role as a genetic reservoir. Ongoing efforts to introgress this rich genetic diversity into cultivated peanut are vital for ensuring long-term resistance durability and preventing resistance breakdown. Furthermore, the discovery of genomic regions that confer resistance to multiple diseases or co-localize with other desirable agronomic traits, such as earliness and increased biomass yield, presents significant opportunities for multi-trait selection and trait pyramiding, leading to the development of more commercially viable and resilient cultivars.</span></p><p class="c30"><span class="c0 c3">In conclusion, the integration of molecular markers and genomic insights into peanut breeding programs is not merely an incremental improvement but a fundamental paradigm shift. This approach offers a sustainable, environmentally sound, and economically viable strategy to combat leaf spot diseases, ensuring enhanced productivity and quality for the global peanut industry in the face of evolving environmental and pathogenic challenges. Continued investment in genomic research, marker development, and the utilization of wild genetic resources will be paramount for future advancements in peanut breeding.</span></p><h4 class="c38"><span class="c3 c22">Works cited</span></h4><ol class="c7 lst-kix_list_9-0 start" start="1"><li class="c5 li-bullet-0"><span class="c9">Technological Progress Toward Peanut Disease Management: A Review - PMC, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11860622/&amp;sa=D&amp;source=editors&amp;ust=1749664151002859&amp;usg=AOvVaw23SUI4LlSHqOtPyEudxPuV">https://pmc.ncbi.nlm.nih.gov/articles/PMC11860622/</a></span></li><li class="c5 li-bullet-0"><span class="c9">Effects of Peanut Rust Disease (Puccinia arachidis Speg.) on Agricultural Production: Current Control Strategies and Progress in Breeding for Resistance - PubMed Central, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10815183/&amp;sa=D&amp;source=editors&amp;ust=1749664151003267&amp;usg=AOvVaw0YQWbO7TRRjEvdl6mnN7XA">https://pmc.ncbi.nlm.nih.gov/articles/PMC10815183/</a></span></li><li class="c5 li-bullet-0"><span class="c9">Passalora personata - Bugwoodwiki, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://wiki.bugwood.org/Passalora_personata&amp;sa=D&amp;source=editors&amp;ust=1749664151003504&amp;usg=AOvVaw29kMDgsLWlFoJVPNPnPKSt">https://wiki.bugwood.org/Passalora_personata</a></span></li><li class="c5 li-bullet-0"><span class="c9">Mycosphaerella berkeleyi - Wikipedia, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://en.wikipedia.org/wiki/Mycosphaerella_berkeleyi&amp;sa=D&amp;source=editors&amp;ust=1749664151003708&amp;usg=AOvVaw2aY_ko9HHc6FeJCXteXArk">https://en.wikipedia.org/wiki/Mycosphaerella_berkeleyi</a></span></li><li class="c5 li-bullet-0"><span class="c9">Impact of Three Isolates of Cercosporidium personatum (Berk. Et M.A Curt.), a Pathogen of Late Leaf Spot, on Defoliation and Yield in Groundnut (Arachis hypogaea L.) under Controlled Conditions, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.scirp.org/journal/paperinformation?paperid%3D128253&amp;sa=D&amp;source=editors&amp;ust=1749664151004065&amp;usg=AOvVaw11EN-vraRcHRqX38l1APo3">https://www.scirp.org/journal/paperinformation?paperid=128253</a></span></li><li class="c5 li-bullet-0"><span class="c9">Major QTLs for Resistance to Early and Late Leaf Spot Diseases Are Identified on Chromosomes 3 and 5 in Peanut (Arachis hypogaea) - Frontiers, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2019.00883/full&amp;sa=D&amp;source=editors&amp;ust=1749664151004415&amp;usg=AOvVaw3G5TeepmE6pFS7Eoeff988">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2019.00883/full</a></span></li><li class="c5 li-bullet-0"><span class="c9">Genetic Dissection of Novel QTLs for Resistance to Leaf Spots and Tomato Spotted Wilt Virus in Peanut (Arachis hypogaea L.) - Frontiers, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.00025/full&amp;sa=D&amp;source=editors&amp;ust=1749664151004760&amp;usg=AOvVaw1kTweVvLK6HDYEBoaLP_9T">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.00025/full</a></span></li><li class="c5 li-bullet-0"><span class="c9">Impact of Molecular Genetic Research on Peanut Cultivar Development - MDPI, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4395/1/1/3&amp;sa=D&amp;source=editors&amp;ust=1749664151005018&amp;usg=AOvVaw0wTpEavj1Dd9k8GsEJzzzq">https://www.mdpi.com/2073-4395/1/1/3</a></span></li><li class="c5 li-bullet-0"><span class="c9">Selective InDel Marker Identification Across the Peanut (Arachis hypogaea L.) Genome Using ddRADSeq - T&Uuml;B&#304;TAK Academic Journals, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://journals.tubitak.gov.tr/cgi/viewcontent.cgi?article%3D2826%26context%3Dbotany&amp;sa=D&amp;source=editors&amp;ust=1749664151005466&amp;usg=AOvVaw2s9Yl-xudUPfi-x7A0tCRi">https://journals.tubitak.gov.tr/cgi/viewcontent.cgi?article=2826&amp;context=botany</a></span></li><li class="c5 li-bullet-0"><span class="c9">A SNP-Based Linkage Map Revealed QTLs for Resistance to Early and Late Leaf Spot Diseases in Peanut (Arachis hypogaea L.), accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6048419/&amp;sa=D&amp;source=editors&amp;ust=1749664151005739&amp;usg=AOvVaw3a6m6AdtNO09bPQcUD6F0W">https://pmc.ncbi.nlm.nih.gov/articles/PMC6048419/</a></span></li><li class="c5 li-bullet-0"><span class="c9">(PDF) Molecular Markers for Assisted Selection in Sclerotinia Blight and Peanut Smut Resistance - ResearchGate, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.researchgate.net/publication/385209138_Molecular_Markers_for_Assisted_Selection_in_Sclerotinia_Blight_and_Peanut_Smut_Resistance&amp;sa=D&amp;source=editors&amp;ust=1749664151006154&amp;usg=AOvVaw27GQ5iJsxoZJXjAPsAM3TK">https://www.researchgate.net/publication/385209138_Molecular_Markers_for_Assisted_Selection_in_Sclerotinia_Blight_and_Peanut_Smut_Resistance</a></span></li><li class="c5 li-bullet-0"><span class="c9">Groundnut mid-density genotyping services - CGIAR Excellence in Breeding, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://excellenceinbreeding.org/toolbox/services/groundnut-mid-density-genotyping-services&amp;sa=D&amp;source=editors&amp;ust=1749664151006445&amp;usg=AOvVaw1hCh6aZrCadNa0Rt60mfEO">https://excellenceinbreeding.org/toolbox/services/groundnut-mid-density-genotyping-services</a></span></li><li class="c5 li-bullet-0"><span class="c9">Mapping QTLs for Leafspot Resistance in Peanut Using SNP-Based Next-Generation Sequencing Markers - Plant Breeding and Biotechnology, accessed June 10, 2025, </span><span class="c9 c16"><a class="c10" href="https://www.google.com/url?q=https://www.plantbreedbio.org/journal/download_pdf.php?doi%3D10.9787/PBB.2017.5.2.115&amp;sa=D&amp;source=editors&amp;ust=1749664151006928&amp;usg=AOvVaw3IvWGueKEjuni1Rk7F-0bv">https://www.plantbreedbio.org/journal/download_pdf.php?doi=10.9787/PBB.2017.5.2.115</a></span></li><li class="c5 li-bullet-0"><span class="c9">Association of differentially expressed R-gene candidates with leaf spot resistance in peanut (Arachis hypogaea L.), accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7884587/&amp;sa=D&amp;source=editors&amp;ust=1749664151007363&amp;usg=AOvVaw0BGHl6q37gUmxihfvF6f9_">https://pmc.ncbi.nlm.nih.gov/articles/PMC7884587/</a></span></li><li class="c5 li-bullet-0"><span class="c9">Mapping QTLs for early leaf spot resistance and yield component traits using an interspecific AB-QTL population in peanut - Frontiers, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1488166/full&amp;sa=D&amp;source=editors&amp;ust=1749664151007839&amp;usg=AOvVaw0hJrhAITU1vjsBhjt_JgDT">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1488166/full</a></span></li><li class="c5 li-bullet-0"><span class="c9">PeanutBase - Ag Data Commons, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://agdatacommons.nal.usda.gov/articles/dataset/PeanutBase/24660672&amp;sa=D&amp;source=editors&amp;ust=1749664151008102&amp;usg=AOvVaw1mbdu5JvbnG_qbxfDV_abA">https://agdatacommons.nal.usda.gov/articles/dataset/PeanutBase/24660672</a></span></li><li class="c5 li-bullet-0"><span class="c9">Dataset - Catalog - Data.gov, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://catalog.data.gov/dataset/?_organization_limit%3D0%26_vocab_category_all_limit%3D0%26_tags_limit%3D0%26organization%3Dusda-gov%26tags%3Dpeanut&amp;sa=D&amp;source=editors&amp;ust=1749664151008525&amp;usg=AOvVaw2d1mMmVTGgAZUZt8SIOfbZ">https://catalog.data.gov/dataset/?_organization_limit=0&amp;_vocab_category_all_limit=0&amp;_tags_limit=0&amp;organization=usda-gov&amp;tags=peanut</a></span></li><li class="c5 li-bullet-0"><span class="c9">Peanut Leaf Spots - NC State Extension Publications, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://content.ces.ncsu.edu/early-leaf-spot-of-peanut-1&amp;sa=D&amp;source=editors&amp;ust=1749664151008815&amp;usg=AOvVaw2VJ7vM9U8lEhxAHIxw5t2t">https://content.ces.ncsu.edu/early-leaf-spot-of-peanut-1</a></span></li><li class="c5 li-bullet-0"><span class="c9">Mapping QTLs for early leaf spot resistance and yield component traits using an interspecific AB-QTL population in peanut - PubMed, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/39886684/&amp;sa=D&amp;source=editors&amp;ust=1749664151009128&amp;usg=AOvVaw2JLAfcL9_wQUnOEJrPXJHX">https://pubmed.ncbi.nlm.nih.gov/39886684/</a></span></li><li class="c5 li-bullet-0"><span class="c9">Validation of SNP markers associated with late leaf spot resistance in groundnut, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.researchgate.net/publication/381489036_Validation_of_SNP_markers_associated_with_late_leaf_spot_resistance_in_groundnut&amp;sa=D&amp;source=editors&amp;ust=1749664151009542&amp;usg=AOvVaw1FLPjA40NMgDYsOjiF-t8K">https://www.researchgate.net/publication/381489036_Validation_of_SNP_markers_associated_with_late_leaf_spot_resistance_in_groundnut</a></span></li><li class="c5 li-bullet-0"><span class="c9">Crop Genetics and Breeding Research: Tifton, GA - Project : USDA ARS, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/project/?accnNo%3D434274&amp;sa=D&amp;source=editors&amp;ust=1749664151009785&amp;usg=AOvVaw1jeCjJRrSQLG-wvcYmsbY7">https://www.ars.usda.gov/research/project/?accnNo=434274</a></span></li><li class="c5 li-bullet-0"><span class="c9">Development of High-Yielding, High-Oleic Peanut Cultivars or Germplasm with Tolerance to Biotic and Abiotic Stresses - Project : USDA ARS, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/project/?accnNo%3D434346%26fy%3D2023&amp;sa=D&amp;source=editors&amp;ust=1749664151010080&amp;usg=AOvVaw31R76DtezOq6WNTsXd0CtX">https://www.ars.usda.gov/research/project/?accnNo=434346&amp;fy=2023</a></span></li><li class="c5 li-bullet-0"><span class="c9">Development and Utilization of InDel Markers to Identify Peanut (Arachis hypogaea) Disease Resistance - Frontiers, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2015.00988/full&amp;sa=D&amp;source=editors&amp;ust=1749664151010407&amp;usg=AOvVaw0q8mUt3kNAc7MyLVQ_80A-">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2015.00988/full</a></span></li><li class="c5 li-bullet-0"><span class="c9">Molecular Markers of Arachis and Marker-Assisted Selection - Allen Press, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://meridian.allenpress.com/peanut-science/article-pdf/28/2/117/1855088/i0095-3679-28-2-13.pdf&amp;sa=D&amp;source=editors&amp;ust=1749664151010712&amp;usg=AOvVaw16rHiExA6t7wTSYsIi3PE3">https://meridian.allenpress.com/peanut-science/article-pdf/28/2/117/1855088/i0095-3679-28-2-13.pdf</a></span></li><li class="c5 li-bullet-0"><span class="c9">Development and Utilization of InDel Markers to Identify Peanut (Arachis hypogaea) Disease Resistance, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC4643128/&amp;sa=D&amp;source=editors&amp;ust=1749664151011002&amp;usg=AOvVaw3TTa5T2G-BSfhrqMpgaWCf">https://pmc.ncbi.nlm.nih.gov/articles/PMC4643128/</a></span></li><li class="c5 li-bullet-0"><span class="c9">Marker-Assisted Selection &ndash; PeanutBase, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.peanutbase.org/mas/&amp;sa=D&amp;source=editors&amp;ust=1749664151011191&amp;usg=AOvVaw1b61n-Wx2VD69dD3gHT9VM">https://www.peanutbase.org/mas/</a></span></li><li class="c5 li-bullet-0"><span class="c9">2025abstract-APRES_2025_Final_abstract_TTK.docx, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://apresinc.com/meetings/apres-meeting-presentation-abstracts/submission-view-file/c96cb10a8c633b4187f5b8f4a3f586ac/88ffa24d1f0990ce3a924ddbabcc6bb1&amp;sa=D&amp;source=editors&amp;ust=1749664151011587&amp;usg=AOvVaw1ghtVWdkyZerMDIyDRhHBC">https://apresinc.com/meetings/apres-meeting-presentation-abstracts/submission-view-file/c96cb10a8c633b4187f5b8f4a3f586ac/88ffa24d1f0990ce3a924ddbabcc6bb1</a></span></li><li class="c5 li-bullet-0"><span class="c9">SSR markers associated for late leaf spot disease resistance by bulked segregant analysis in groundnut (Arachis hypogaea L.) - ResearchGate, accessed June 10, 2025, </span><span class="c16 c9"><a class="c10" href="https://www.google.com/url?q=https://www.researchgate.net/publication/257561307_SSR_markers_associated_for_late_leaf_spot_disease_resistance_by_bulked_segregant_analysis_in_groundnut_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1749664151012019&amp;usg=AOvVaw0n70IRSFMXq-T8wckziEZ-">https://www.researchgate.net/publication/257561307_SSR_markers_associated_for_late_leaf_spot_disease_resistance_by_bulked_segregant_analysis_in_groundnut_Arachis_hypogaea_L</a></span></li></ol></body></html>