<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-5>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}ol.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-8{list-style-type:none}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_2-2{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_1-8{list-style-type:none}ol.lst-kix_list_2-0.start{counter-reset:lst-ctn-kix_list_2-0 0}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_2-0>li{counter-increment:lst-ctn-kix_list_2-0}.lst-kix_list_1-7>li:before{content:"\0025a0   "}.lst-kix_list_1-5>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"" counter(lst-ctn-kix_list_2-0,decimal) ". "}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}.lst-kix_list_2-3>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c22{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:66.9pt;border-top-color:#000000;border-bottom-style:solid}.c12{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:117pt;border-top-color:#000000;border-bottom-style:solid}.c3{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:78pt;border-top-color:#000000;border-bottom-style:solid}.c17{-webkit-text-decoration-skip:none;color:#0000ee;font-weight:400;text-decoration:underline;vertical-align:baseline;text-decoration-skip-ink:none;font-size:12pt;font-family:"Google Sans";font-style:normal}.c2{color:#1b1c1d;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c1{color:#575b5f;font-weight:400;text-decoration:none;vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal}.c6{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Google Sans Text";font-style:normal}.c15{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c8{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c29{font-size:16pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c32{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans";font-style:normal}.c5{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c24{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c35{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c21{color:#1b1c1d;font-weight:400;font-size:10pt;font-family:"Google Sans Text";font-style:italic}.c20{color:#1b1c1d;font-weight:700;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c31{font-size:15pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c14{font-size:10pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c4{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c7{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c28{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c34{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c23{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c0{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c11{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c19{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c9{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c36{padding-top:0pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c25{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left}.c33{border-spacing:0;border-collapse:collapse;margin-right:auto}.c37{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c18{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c26{padding:0;margin:0}.c10{text-decoration:none;vertical-align:baseline}.c16{color:inherit;text-decoration:inherit}.c30{margin-left:23.2pt;padding-left:0pt}.c13{height:0pt}.c27{height:11pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c37 doc-content"><h1 class="c11"><span class="c29">Genetic Control of Flowering Time in Peanut (</span><span class="c28">Arachis hypogaea</span><span class="c29">&nbsp;L.) and Related </span><span class="c28">Arachis</span><span class="c29 c10">&nbsp;Species</span></h1><h2 class="c11"><span class="c32">1. Introduction to Flowering Time Control in Peanut (</span><span class="c31">Arachis hypogaea</span><span class="c32 c10">)</span></h2><p class="c9"><span class="c4">Flowering time is a paramount agronomic trait in cultivated peanut (</span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;L.), serving as a critical determinant for the adaptation of distinct varieties to specific agroecological environments. This trait directly influences the duration of the vegetative growth phase, the window available for pod development, and consequently, the ultimate yield potential. Deviations from optimal flowering periods, whether too early or too late, can result in diminished yields. The transition from vegetative to reproductive growth, known as the floral transition, represents a pivotal stage in the lifecycle of crop plants and is meticulously governed by the timing of flowering. A comprehensive understanding of the regulatory mechanisms underpinning flowering time is therefore indispensable for breeding programs, particularly those aiming to develop early-maturing cultivars. Such cultivars are highly desirable in regions characterized by short growing seasons or those prone to terminal drought stress, where an accelerated lifecycle can mitigate yield losses.</span><span class="c1">1</span></p><p class="c9"><span class="c4">Peanut possesses a unique reproductive biology, often described as &quot;blossoming on the ground and bearing pods underground&quot;. This phenomenon, termed geocarpy, involves aerial flowering followed by the elongation of a specialized structure, the gynophore (or peg), which carries the fertilized ovules into the soil for pod development.</span><span class="c15">2</span><span class="c4">&nbsp;This unusual developmental strategy necessitates a precise coordination between the timing of flowering, environmental conditions, and the subsequent geocarpic development processes.</span><span class="c15">3</span><span class="c4 c10">&nbsp;The success of underground pod formation is contingent upon the gynophore reaching and penetrating the soil, implying that flowering must occur at a juncture that permits adequate peg elongation and soil entry under favorable conditions. This intricate link suggests that genes controlling flowering might be co-regulated with, or exert pleiotropic effects on, genes governing peg development or the light and dark sensory mechanisms essential for subterranean pod maturation. Such a layer of regulatory complexity is not commonly observed in aerial-fruiting crops and indicates that selection pressures on flowering time genes in peanut may be uniquely intertwined with its geocarpic habit. Consequently, breeding efforts targeting flowering time could inadvertently influence linked traits crucial for successful underground fruiting.</span></p><p class="c9"><span class="c4">Similar to other plant species, flowering time in peanut is recognized as a complex quantitative trait. Its expression is orchestrated by the intricate interplay of multiple genes, which in turn interact dynamically with various environmental cues, most notably photoperiod and temperature.</span><span class="c15">5</span><span class="c4">&nbsp;The allotetraploid nature of cultivated peanut (</span><span class="c5">Arachis hypogaea</span><span class="c4">; AABB genomes, 2n=4x=40), which arose from a hybridization event between the diploid progenitors </span><span class="c5">Arachis duranensis</span><span class="c4">&nbsp;(A genome) and </span><span class="c5">Arachis ipaensis</span><span class="c4">&nbsp;(B genome) </span><span class="c15">6</span><span class="c4">, introduces an additional dimension of genetic complexity. This polyploid origin results in the presence of homoeologous gene pairs from the A and B subgenomes for many traits, including those involved in flowering time regulation. Studies on various gene families, such as Phosphatidylethanolamine-Binding Proteins (PEBPs) </span><span class="c15">7</span><span class="c4">, WRKY transcription factors </span><span class="c15">5</span><span class="c4">, R2R3-MYB transcription factors </span><span class="c15">8</span><span class="c4">, and IGT (I ACT IN GRAVITROPISM AND TILLER ANGLE CONTROL 1) genes </span><span class="c15">9</span><span class="c4 c10">, have indicated instances of subgenome dominance or differential expression patterns between homoeologs. This genetic redundancy can provide a buffering capacity against deleterious mutations but also furnishes the raw material for evolutionary divergence through processes like subfunctionalization (partitioning of ancestral functions) or neofunctionalization (acquisition of novel functions). Such evolutionary trajectories could lead to more nuanced and sophisticated control of flowering time, enabling adaptation to a diverse array of environmental conditions. Discerning the relative contributions of alleles from each subgenome is therefore a key consideration for developing targeted and effective breeding strategies in peanut.</span></p><h2 class="c11"><span class="c32 c10">2. Major Gene Families and Key Regulatory Proteins Implicated in Peanut Flowering Time</span></h2><p class="c9"><span class="c4">The regulation of flowering time in peanut involves a diverse array of gene families and regulatory proteins, many of which are conserved across the plant kingdom. Research in </span><span class="c5">Arachis</span><span class="c4 c10">&nbsp;species has begun to elucidate the specific members of these families and their roles in orchestrating the floral transition.</span></p><h3 class="c11"><span class="c24 c10">2.1. CONSTANS-like (COL) Gene Family</span></h3><p class="c9"><span class="c4">The </span><span class="c5">CONSTANS-like (COL)</span><span class="c4">&nbsp;gene family plays a central role in the photoperiod pathway, enabling plants to perceive and respond to changes in day length to initiate flowering. In </span><span class="c5">Arachis hypogaea</span><span class="c4">, a comprehensive analysis has identified 18 </span><span class="c5">AhCOL</span><span class="c4">&nbsp;genes. These genes are characterized by the presence of conserved B-box zinc finger and CCT (CONSTANS, CO-like, and TOC1) domains, and all are predicted to localize to the nucleus.</span><span class="c15">10</span><span class="c4">&nbsp;The specific gene identifiers (e.g., </span><span class="c5">arahy.6J3MKV.1</span><span class="c4">&nbsp;for </span><span class="c5">AhCOL1</span><span class="c4">) and their respective chromosomal locations have been documented (Table 1, derived from). Expression profiling studies have revealed that the majority of </span><span class="c5">AhCOL</span><span class="c4">&nbsp;genes are abundantly expressed in leaves, flowers, and, interestingly, in fruit needles (developing gynophores), suggesting roles beyond floral initiation.</span><span class="c1">3</span></p><p class="c9"><span class="c4">Crucially, protein-protein interaction studies have begun to map the connectivity of AhCOL proteins within the regulatory network. </span><span class="c5">AhCOL1</span><span class="c4">&nbsp;has been shown to interact with </span><span class="c5">AhNF-YC1</span><span class="c4">&nbsp;(Nuclear Factor Y, subunit C1), while </span><span class="c5">AhCOL3</span><span class="c4">&nbsp;interacts with both </span><span class="c5">AhNF-YC1</span><span class="c4">&nbsp;and </span><span class="c5">AhCOP1</span><span class="c4">&nbsp;(CONSTITUTIVE PHOTOMORPHOGENESIS 1).</span><span class="c15">10</span><span class="c4">&nbsp;These interactions are highly significant as NF-YC is a component of the NF-Y transcription factor complex, which, in conjunction with CO/COL proteins, directly binds to the promoter of </span><span class="c5">FLOWERING LOCUS T (FT)</span><span class="c4">&nbsp;to activate its expression in model systems like </span><span class="c5">Arabidopsis</span><span class="c4">. Conversely, COP1, an E3 ubiquitin ligase, targets CO/COL proteins for proteasomal degradation, particularly in darkness, thereby acting as a repressor of flowering. The observed interactions in peanut strongly suggest the conservation of this core regulatory module (CO/COL-NF-YC activation of </span><span class="c5">FT</span><span class="c4 c10">, and COP1-mediated degradation of CO/COL), linking the photoperiod sensing machinery to the downstream activation of florigen signals. This implies that strategies aimed at manipulating these specific protein interactions could offer novel avenues for modulating flowering time in peanut.</span></p><p class="c9"><span class="c4">Furthermore, certain </span><span class="c5">AhCOL</span><span class="c4">&nbsp;genes, including </span><span class="c5">AhCOL4, AhCOL8, AhCOL13</span><span class="c4">, and </span><span class="c5">AhCOL14</span><span class="c4">, exhibit upregulation in response to light induction and mechanical stress.</span><span class="c15">10</span><span class="c4">&nbsp;While pod development is a post-flowering process, its initiation is intrinsically linked to flowering signals. The responsiveness of these </span><span class="c5">AhCOL</span><span class="c4">&nbsp;genes to mechanical stress is particularly intriguing in the context of peanut&#39;s geocarpic reproductive strategy. The penetration of the gynophore into the soil is inherently a mechanical process. If these </span><span class="c5">AhCOL</span><span class="c4 c10">&nbsp;genes, which are potentially involved in light sensing and flowering regulation, are also sensitive to mechanical cues during pegging and subsequent pod development, it suggests a dual functionality or a continued involvement of photoperiod pathway components in the regulation of subterranean fruit development. This could represent an adaptive mechanism for coordinating the unique underground fruiting habit of peanut with environmental signals.</span></p><h3 class="c11"><span class="c24 c10">2.2. Phosphatidylethanolamine-Binding Protein (PEBP) Family (FT, TFL1, MFT-like)</span></h3><p class="c9"><span class="c4">The Phosphatidylethanolamine-Binding Protein (PEBP) family encompasses critical regulators of the floral transition and plant architecture. Within this family, </span><span class="c5">FLOWERING LOCUS T (FT)</span><span class="c4">-like proteins are generally recognized as mobile floral promoting signals (florigen), while </span><span class="c5">TERMINAL FLOWER 1 (TFL1)</span><span class="c4">-like proteins typically act as floral repressors and play roles in maintaining indeterminate inflorescence meristems. </span><span class="c5">MOTHER OF FT AND TFL1 (MFT)</span><span class="c4 c10">-like proteins are considered to represent an ancestral lineage within the PEBP family, often involved in processes such as seed development and germination.</span></p><p class="c9"><span class="c4">In cultivated peanut, 33 </span><span class="c5">PEBP</span><span class="c4">&nbsp;genes have been identified and categorized into FT, TFL, MFT, and FT-like subgroups.</span><span class="c15">7</span><span class="c4">&nbsp;Evidence suggests that some orthologous </span><span class="c5">PEBP</span><span class="c4">&nbsp;genes exhibit subgenome dominance, reflecting the allotetraploid nature of </span><span class="c5">A. hypogaea</span><span class="c4">.</span><span class="c15">7</span><span class="c4">&nbsp;Several key </span><span class="c5">AhPEBP</span><span class="c4">&nbsp;genes have been implicated in controlling the flowering response to different genotypes, photoperiods, and temperatures. These include </span><span class="c5">ArahyM2THPA, ArahyEM6VH3, Arahy4GAQ4U</span><span class="c4">, and </span><span class="c5">ArahyIZ8FG5</span><span class="c4">, which are classified as FT-like and are proposed to promote flowering.</span><span class="c15">7</span><span class="c4">&nbsp;Conversely, </span><span class="c5">ArahyG6F3P2</span><span class="c4">&nbsp;and </span><span class="c5">ArahyLUT2QN</span><span class="c4">, also FT-like, have been suggested to inhibit flowering based on expression patterns.</span><span class="c15">11</span><span class="c4">&nbsp;Within the TFL-like group, </span><span class="c5">ArahyDYRS20</span><span class="c4">&nbsp;and </span><span class="c5">ArahyBBG51B</span><span class="c4">&nbsp;have been identified as key genes, and intriguingly, their elevated expression is associated with the promotion of flowering.</span><span class="c15">11</span><span class="c4">&nbsp;This promoting role is atypical for TFL1 homologs, which are canonical repressors, and suggests a potential functional divergence in peanut. The gene </span><span class="c5">Arahy.UVRI66</span><span class="c4">, annotated as &quot;flowering locus protein T,&quot; is also listed in PeanutBase expression datasets.</span><span class="c15">12</span><span class="c4">&nbsp;The specific locus identifiers for the eight key PEBP genes mentioned above are expected to be found in Supplementary Table S1 of the study by Zhang et al. (2022, PMC9604132).</span><span class="c1">11</span></p><p class="c9"><span class="c4">The significant impact of </span><span class="c5">TFL1</span><span class="c4">-like genes in legumes is further highlighted by studies in common bean, where a </span><span class="c5">TFL1</span><span class="c4">&nbsp;homolog (</span><span class="c5">Phvul.001G189200</span><span class="c4 c10">) was found to exert major effects on flowering time, vegetative growth habit, and overall plant production. This underscores the conserved importance of this PEBP subfamily member in controlling key developmental transitions in related species.</span></p><p class="c9"><span class="c4">The existence of FT-like genes in peanut whose expression appears to inhibit flowering (e.g., </span><span class="c5">ArahyG6F3P2, ArahyLUT2QN</span><span class="c4">&nbsp;</span><span class="c15">11</span><span class="c4">) is a noteworthy finding. While FT proteins are typically potent floral activators, this observation points towards a more sophisticated regulatory system in peanut. It may represent instances of neofunctionalization or subfunctionalization within the </span><span class="c5">FT</span><span class="c4 c10">&nbsp;clade, allowing for a more nuanced modulation of the photoperiodic response across diverse environmental conditions or developmental stages than would be possible with a simple binary (on/off) FT signal. This could involve competitive interactions among different FT-like paralogs for downstream partners or differential targeting to regulatory pathways.</span></p><p class="c9"><span class="c4">Similarly, the atypical flowering-promoting role ascribed to the TFL-like genes </span><span class="c5">ArahyDYRS20</span><span class="c4">&nbsp;and </span><span class="c5">ArahyBBG51B</span><span class="c4">&nbsp;</span><span class="c15">11</span><span class="c4">&nbsp;represents a significant deviation from the well-established repressive function of TFL1 in </span><span class="c5">Arabidopsis</span><span class="c4 c10">&nbsp;and other species. This functional shift, if confirmed, warrants detailed investigation into the specific protein domains of these peanut TFL-like proteins or their interacting partners. It is possible that structural alterations have modified their binding affinities or downstream effects, or they may represent a distinct lineage of PEBPs with a derived, specialized function in peanut. This highlights the critical importance of experimental validation of gene function in the target species, rather than relying solely on homology-based inferences from model plants.</span></p><h3 class="c11"><span class="c10 c24">2.3. WRKY Transcription Factors</span></h3><p class="c9"><span class="c4">WRKY transcription factors constitute a large family in plants, involved in a multitude of physiological processes including growth, development, and responses to biotic and abiotic stresses.</span><span class="c15">13</span><span class="c4">&nbsp;Their involvement in the regulation of flowering time is an area of growing research interest.</span><span class="c15">5</span><span class="c4">&nbsp;In </span><span class="c5">Arachis</span><span class="c4">&nbsp;species, including </span><span class="c5">A. duranensis, A. ipaensis, A. monticola</span><span class="c4">, and the cultivated </span><span class="c5">A. hypogaea</span><span class="c4">, WRKY TFs have been shown to modulate flowering time through their participation in diverse regulatory pathways. These include the aging, autonomous, circadian clock, hormone, photoperiod, sugar, temperature, and vernalization pathways.</span><span class="c1">5</span></p><p class="c9"><span class="c4">Protein interaction network analyses have predicted that these WRKY TFs interact with key flowering regulatory proteins such as FRUITFULL (FUL), a MADS-box transcription factor, and APETALA2 (AP2), a floral repressor and developmental regulator.</span><span class="c15">14</span><span class="c4">&nbsp;This suggests that WRKY TFs may act as crucial nodes integrating signals from various pathways to fine-tune the flowering response. For instance, the interaction with FUL (a downstream target of FT) and AP2 (a floral repressor) could allow WRKYs to modulate the balance between floral promotion and repression, or to mediate crosstalk between the photoperiod/FT pathway and developmental programs. The observation that flowering-implicated WRKY TFs often exhibit low expression levels, while their interacting partners show varied expression patterns, might indicate a stoichiometric sensitivity in these interactions or a role for WRKYs as signaling hubs rather than as bulk transcriptional effectors.</span><span class="c1">14</span></p><p class="c9"><span class="c4">Further evidence for their role comes from homology studies; 16 WRKY TFs identified in </span><span class="c5">Arachis</span><span class="c4">&nbsp;are homologs of the </span><span class="c5">Arabidopsis</span><span class="c4">&nbsp;genes </span><span class="c5">AtWRKY12</span><span class="c4">&nbsp;and </span><span class="c5">AtWRKY75</span><span class="c4">, which are known to participate in flowering time control.</span><span class="c15">5</span><span class="c4">&nbsp;Specific homoeologous gene pairs in </span><span class="c5">A. hypogaea</span><span class="c4">, such as </span><span class="c5">AhTWRKY35/AhTWRKY116</span><span class="c4">&nbsp;and </span><span class="c5">AhTWRKY69/AhTWRKY149</span><span class="c4">, have been implicated in regulating flowering time.</span><span class="c15">5</span><span class="c4">&nbsp;Additionally, a genome-wide association study (GWAS) identified a WRKY TF-encoding gene as one of seven candidate genes significantly associated with flowering time in a diverse panel of peanut germplasm.</span><span class="c15">15</span><span class="c4">&nbsp;PeanutBase expression data also lists </span><span class="c5">Aradu.S8SMF</span><span class="c4">&nbsp;and </span><span class="c5">Arahy.RFW3PD</span><span class="c4">&nbsp;with annotations as WRKY transcription factors.</span><span class="c1">12</span></p><p class="c9"><span class="c4">An interesting evolutionary aspect is the reported asymmetric evolutionary pattern of WRKY TF binding sites within the promoters of homologous flowering-time genes across different </span><span class="c5">Arachis</span><span class="c4">&nbsp;species. Changes in </span><span class="c5">cis</span><span class="c4">-regulatory elements, such as TF binding sites, are a primary mechanism for driving evolutionary changes in gene expression patterns without altering the protein sequence itself. If these binding sites are evolving divergently in different </span><span class="c5">Arachis</span><span class="c4">&nbsp;lineages, it implies that the same WRKY TFs might regulate their target genes with different efficiencies, timing, or in response to different cues in, for example, </span><span class="c5">A. duranensis</span><span class="c4">&nbsp;compared to </span><span class="c5">A. hypogaea</span><span class="c4">. This molecular evolution at the </span><span class="c5">cis</span><span class="c4 c10">-regulatory level could underpin some of the observed variations in flowering behavior and environmental adaptability among wild and cultivated peanut species.</span></p><h3 class="c11"><span class="c24 c10">2.4. MYB Transcription Factors (especially R2R3-MYB)</span></h3><p class="c9"><span class="c4">The R2R3-MYB transcription factors represent one of the largest and most diverse transcription factor families in plants, implicated in a wide array of biological processes, including plant development, primary and secondary metabolism, and responses to environmental stresses.</span><span class="c15">8</span><span class="c4">&nbsp;In </span><span class="c5">Arachis hypogaea</span><span class="c4">, a genome-wide analysis has identified 196 typical R2R3-MYB genes, designated </span><span class="c5">AhMYB001</span><span class="c4">&nbsp;through </span><span class="c5">AhMYB196</span><span class="c4">.</span><span class="c1">8</span></p><p class="c9"><span class="c4">Several </span><span class="c5">AhMYB</span><span class="c4">&nbsp;genes have been linked to traits related to flowering or subsequent reproductive development. For instance, </span><span class="c5">AhMYB51</span><span class="c4">&nbsp;(potentially corresponding to locus </span><span class="c5">Arahy.04JNDX</span><span class="c4">) has emerged as a strong candidate gene underlying quantitative trait loci (QTLs) on chromosome A09 associated with the rate of multiple pods per plant (RMSP), a key yield component influenced by flowering characteristics.</span><span class="c15">18</span><span class="c4">&nbsp;</span><span class="c5">AhMYB51</span><span class="c4">&nbsp;is hypothesized to influence RMSP by affecting brassinosteroid metabolism or signaling within the flowers.</span><span class="c15">18</span><span class="c4 c10">&nbsp;This suggests a connection where hormonal regulation within the floral structures, mediated by an MYB factor, can impact the success of flower development into pods, thereby linking flowering quality to yield.</span></p><p class="c9"><span class="c4">Another R2R3-MYB gene, </span><span class="c5">AhMYB033</span><span class="c4">&nbsp;(orthologous to </span><span class="c5">AdMYB03-18</span><span class="c4">&nbsp;from </span><span class="c5">A. duranensis</span><span class="c4">), has been associated with variations in overall plant architecture. A single nucleotide polymorphism (SNP) within </span><span class="c5">AhMYB033</span><span class="c4">&nbsp;was found to be significantly correlated with total branch number, pod length, and root-shoot ratio, highlighting its role in broader developmental processes that can indirectly affect reproductive success.</span><span class="c15">8</span><span class="c4">&nbsp;While </span><span class="c5">AhTc1</span><span class="c4">&nbsp;(also known as </span><span class="c5">AhMYB099</span><span class="c4">) is well-characterized for its role in regulating anthocyanin biosynthesis, leading to purple testa coloration in peanut </span><span class="c15">8</span><span class="c4 c10">, its direct involvement in flowering time control is less evident from the available data.</span></p><p class="c9"><span class="c4">Phylogenetic and expression analyses provide further clues. Homologs of </span><span class="c5">Arabidopsis</span><span class="c4">&nbsp;R2R3-MYBs in subgroup S18, known to be involved in anther and pollen development, suggest similar roles for their peanut counterparts.</span><span class="c15">8</span><span class="c4">&nbsp;Similarly, </span><span class="c5">AhR2R3-MYB</span><span class="c4">&nbsp;genes belonging to subgroup S14, whose </span><span class="c5">Arabidopsis</span><span class="c4">&nbsp;orthologs control axillary meristem development, show high expression in peanut peg tips, roots, and reproductive shoot tips, indicating potential functions in meristem maintenance and reproductive organ development.</span><span class="c15">20</span><span class="c4">&nbsp;Furthermore, one of the seven candidate genes identified through a GWAS for flowering time in peanut encodes an R2R3-MYB transcription factor.</span><span class="c15">15</span><span class="c4">&nbsp;PeanutBase expression data also includes </span><span class="c5">Aradu.G406S</span><span class="c4">&nbsp;and </span><span class="c5">Arahy.JN4NL9</span><span class="c4">, annotated as MYB transcription factors.</span><span class="c1">12</span></p><p class="c9"><span class="c4">The allotetraploid genome of peanut provides opportunities for functional divergence of homoeologous gene pairs. Studies have reported biased expression of R2R3-MYB homoeologs (derived from the A and B subgenomes) in specific tissues or developmental stages.</span><span class="c15">8</span><span class="c4 c10">&nbsp;This phenomenon indicates that polyploidization has likely facilitated the evolution of specialized roles for these duplicated MYB genes. Such partitioning of functions could contribute to the refined regulatory control necessary for peanut&#39;s complex developmental programs, including the precise timing of flowering and the unique processes involved in geocarpic fruit development. This functional diversification is a common evolutionary outcome of polyploidy and likely plays a significant role in how peanut adapts its reproductive strategies to varying environmental contexts.</span></p><h3 class="c11"><span class="c24 c10">2.5. Basic Helix-Loop-Helix (bHLH) Transcription Factors</span></h3><p class="c9"><span class="c4">Basic Helix-Loop-Helix (bHLH) transcription factors constitute another extensive family in plants, participating in a diverse spectrum of regulatory functions, including the control of flowering time as demonstrated in model species like </span><span class="c5">Arabidopsis thaliana</span><span class="c4">.</span><span class="c15">15</span><span class="c4">&nbsp;In peanut, a gene encoding a bHLH transcription factor was identified as one of seven promising candidate genes emerging from a GWAS focused on flowering time.</span><span class="c15">15</span><span class="c4">&nbsp;This finding directly implicates the bHLH family in the genetic architecture of this crucial agronomic trait in </span><span class="c5">Arachis</span><span class="c4 c10">.</span></p><p class="c9"><span class="c4">PeanutBase expression datasets provide further evidence of bHLH gene activity, listing </span><span class="c5">Arahy.YHH73E</span><span class="c4">&nbsp;and </span><span class="c5">Arahy.N7XQZF</span><span class="c4">&nbsp;with annotations as bHLH transcription factors.</span><span class="c15">12</span><span class="c4">&nbsp;Numerous other </span><span class="c5">Arachis</span><span class="c4">&nbsp;gene identifiers in these datasets are also associated with bHLH domains, although their specific connection to flowering is not always explicit in the provided annotations.</span><span class="c1">12</span></p><p class="c9"><span class="c4">A particularly relevant subgroup of bHLH TFs are the Phytochrome Interacting Factors (PIFs). These proteins are key mediators of light signaling. Research on peanut planting densities has shown that monoseeding (single seed per planting hole) leads to decreased expression of </span><span class="c5">PIF1</span><span class="c4">&nbsp;and </span><span class="c5">PIF4</span><span class="c4">.</span><span class="c15">22</span><span class="c4">&nbsp;This altered expression was associated with modifications in shade-avoidance responses, which are known to influence plant architecture and can, in turn, affect the timing of flowering.</span><span class="c15">22</span><span class="c4 c10">&nbsp;The involvement of bHLH TFs like PIFs in mediating shade-avoidance responses, coupled with the identification of a bHLH gene in a flowering time GWAS, suggests a significant role for this TF family in integrating light quality signals with developmental programs in peanut. Light quality, particularly the red to far-red ratio which PIFs help perceive via phytochromes, is a critical environmental cue. This indicates that peanut&#39;s flowering is not solely dependent on photoperiod duration but also on the spectral quality of light, with bHLH TFs acting as key molecular players in this integration. This is especially pertinent for a crop that develops its fruit underground, where early canopy establishment and light interception efficiency, influenced by shade avoidance, can significantly impact the plant&#39;s overall vigor and resource allocation for reproduction.</span></p><h3 class="c11"><span class="c24 c10">2.6. FAR-RED IMPAIRED RESPONSE 1 (FAR1) Transcription Factors</span></h3><p class="c9"><span class="c4">The FAR-RED IMPAIRED RESPONSE 1 (FAR1) transcription factors and their homologs are recognized as important components of phytochrome A (phyA)-mediated far-red light signaling pathways. These TFs play crucial roles in various aspects of plant growth and development.</span><span class="c15">2</span><span class="c4">&nbsp;A notable feature in </span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;is the significant expansion of the </span><span class="c5">AhFAR1</span><span class="c4">&nbsp;gene family, with 246 members identified.</span><span class="c15">2</span><span class="c4">&nbsp;This expansion is hypothesized to be particularly relevant to peanut&#39;s unique geocarpic reproductive strategy and the development of subterranean pods.</span><span class="c1">2</span></p><p class="c9"><span class="c4">Evidence linking FAR1 TFs to flowering time includes the identification of a </span><span class="c5">FAR1</span><span class="c4">-encoding gene as one of the seven candidate genes from a GWAS for flowering time in peanut.</span><span class="c15">15</span><span class="c4">&nbsp;While this suggests a role in floral transition, much of the research on </span><span class="c5">AhFAR1</span><span class="c4">&nbsp;genes has focused on their post-flowering roles. For example, three specific </span><span class="c5">AhFAR1</span><span class="c4">&nbsp;candidate genes, </span><span class="c5">Ahy_A10g049543, Ahy_A06g026579</span><span class="c4">, and </span><span class="c5">Ahy_A10g048401</span><span class="c4">, exhibit expression patterns indicating their specific involvement in peg and pod development.</span><span class="c15">2</span><span class="c4 c10">&nbsp;Although these are post-flowering events, the initiation and progression of peg and pod development are intrinsically linked to the signals and timing established during flowering.</span></p><p class="c9"><span class="c4">The substantial expansion of the </span><span class="c5">FAR1</span><span class="c4">&nbsp;gene family in </span><span class="c5">Arachis</span><span class="c4">&nbsp;and its strong association with pod development, combined with a GWAS linkage to flowering time, points towards an extended and diversified role for phyA-mediated light signaling in peanut&#39;s reproductive cycle. PhyA is the primary photoreceptor for far-red light. While its influence on flowering initiation is well-established in many plants, its role in peanut appears to be amplified and co-opted for critical post-flowering events, such as the directed growth of the gynophore towards dark soil environments and the subsequent development of the pod. The proliferation of </span><span class="c5">FAR1</span><span class="c4">&nbsp;genes in peanut could provide the genetic toolkit necessary for these specialized light-responsive processes. The appearance of a </span><span class="c5">FAR1</span><span class="c4 c10">&nbsp;gene in a flowering time GWAS could indicate a direct influence on the floral transition itself, or it might relate to an effect on a closely coupled trait, such as the &quot;time to pegging,&quot; which could be phenotypically assessed as part of a broader &quot;flowering time&quot; or &quot;maturity&quot; evaluation. This highlights the often-interconnected nature of flowering and early fruit development stages, particularly in species with unique reproductive strategies like geocarpy.</span></p><h3 class="c11"><span class="c24 c10">2.7. APETALA2/Ethylene Response Factor (AP2/ERF) Family</span></h3><p class="c9"><span class="c4">The APETALA2/Ethylene Response Factor (AP2/ERF) superfamily is one of the largest groups of transcription factors in plants, involved in a wide range of processes including plant growth, development (notably floral organogenesis), and responses to various biotic and abiotic stresses.</span><span class="c15">23</span><span class="c4">&nbsp;Within this superfamily, the AP2 subfamily members are typically characterized by the presence of two AP2 DNA-binding domains. These proteins are known to play significant roles in regulating the development of floral organs, as well as in embryo and seed growth.</span><span class="c1">23</span></p><p class="c9"><span class="c4">In </span><span class="c5">Arachis hypogaea</span><span class="c4">, a genome-wide survey identified 185 </span><span class="c5">AhAP2/ERF</span><span class="c4">&nbsp;members. Among these, 59 genes were classified into the AP2 subfamily.</span><span class="c15">23</span><span class="c4">&nbsp;The specific gene identifiers and chromosomal locations for these </span><span class="c5">AhAP2</span><span class="c4">&nbsp;genes are detailed in the supplementary tables of the original research publication by JGJ et al. (2021, Frontiers in Genetics).</span><span class="c1">23</span></p><p class="c9"><span class="c4">An important regulatory connection has been established through the interaction of WRKY transcription factors with AP2 proteins, which collectively modulate flowering time in </span><span class="c5">Arachis</span><span class="c4">&nbsp;species.</span><span class="c15">14</span><span class="c4 c10">&nbsp;AP2 is a well-characterized floral repressor and also plays a role in defining floral organ identity. The interaction between WRKY TFs (often involved in stress signaling) and AP2 proteins suggests a sophisticated regulatory module. This connection provides a direct mechanism through which environmental stress signals, often transduced by WRKYs, can impinge upon floral developmental programs and the timing of flowering, which are partly controlled by AP2. Such a link could represent an adaptive strategy for peanut, allowing the plant to adjust its reproductive investment and timing based on prevailing environmental conditions, thereby optimizing its chances for successful reproduction.</span></p><h3 class="c11"><span class="c24 c10">2.8. Phytochrome (PHY) Genes</span></h3><p class="c9"><span class="c4">Phytochromes are a family of photoreceptors that plants use to sense red (R) and far-red (FR) light, playing indispensable roles in regulating a multitude of developmental processes. These include seed germination, de-etiolation, shade avoidance responses, and, critically, the photoperiodic control of flowering.</span><span class="c15">2</span><span class="c4">&nbsp;In </span><span class="c5">Arabidopsis</span><span class="c4">, five distinct phytochrome genes (</span><span class="c5">PHYA</span><span class="c4">&nbsp;through </span><span class="c5">PHYE</span><span class="c4 c10">) have been characterized.</span></p><p class="c9"><span class="c4">Research in peanut has led to the identification of several phytochrome genes, including </span><span class="c5">AhphyA, AhphyA-like, AhphyB</span><span class="c4">, and </span><span class="c5">AhphyE</span><span class="c4">&nbsp;in cultivated peanut (</span><span class="c5">Arachis hypogaea</span><span class="c4">).</span><span class="c15">4</span><span class="c4">&nbsp;The full coding sequences (CDS) for these cultivated peanut phytochromes have been obtained.</span><span class="c15">4</span><span class="c4">&nbsp;For the diploid wild relatives, specific gene IDs such as </span><span class="c5">Aradu.G6IAK</span><span class="c4">&nbsp;(</span><span class="c5">A. duranensis phyA</span><span class="c4">) and </span><span class="c5">Araip.K62H2</span><span class="c4">&nbsp;(</span><span class="c5">A. ipaensis phyA</span><span class="c4">) have also been reported.</span><span class="c1">4</span></p><p class="c9"><span class="c4">Studies on planting density in peanut have revealed differential expression of phytochrome genes; for instance, </span><span class="c5">PhyA</span><span class="c4">&nbsp;expression was observed to decrease while </span><span class="c5">PhyB</span><span class="c4">&nbsp;expression increased under monoseeding conditions compared to traditional planting patterns.</span><span class="c15">22</span><span class="c4">&nbsp;These changes in phytochrome gene expression were linked to alterations in shade-avoidance responses.</span><span class="c15">22</span><span class="c4">&nbsp;</span><span class="c5">PhyB</span><span class="c4">&nbsp;is the primary sensor for changes in the R:FR ratio, which signals the proximity of neighboring plants. An increase in </span><span class="c5">PhyB</span><span class="c4">&nbsp;expression (and a concomitant decrease in </span><span class="c5">PhyA</span><span class="c4 c10">) under monoseeding could modify the plant&#39;s perception of competition, leading to a less pronounced shade-avoidance syndrome (e.g., reduced stem elongation, increased branching). Given that phytochrome signaling is intrinsically linked to the molecular pathways controlling flowering time (for instance, by influencing the stability of CONSTANS protein), such agronomic practice-induced alterations in phytochrome gene expression could have consequential effects on the timing of floral initiation and the overall duration of maturity in peanut.</span></p><p class="c9"><span class="c4">While much of the research on peanut phytochromes has focused on their role in the unique geocarpic development of pods </span><span class="c15">4</span><span class="c4">, their fundamental role in light perception implies an inherent involvement in flowering time regulation, as documented extensively in other plant species.</span><span class="c1">25</span></p><h3 class="c11"><span class="c24 c10">2.9. GIGANTEA (GI)</span></h3><p class="c9"><span class="c5">GIGANTEA (GI)</span><span class="c4">&nbsp;encodes a plant-specific nuclear protein that is a key component of the circadian clock machinery and plays a crucial role in integrating light signals for the photoperiodic control of flowering.</span><span class="c15">26</span><span class="c4">&nbsp;GI typically acts upstream of, or in concert with, other central flowering regulators like CONSTANS (CO) and FLOWERING LOCUS T (FT).</span><span class="c15">29</span><span class="c4">&nbsp;The expression of </span><span class="c5">GI</span><span class="c4">&nbsp;itself is under circadian clock control, exhibiting rhythmic accumulation with peaks occurring at specific times of the day, influenced by day length.</span><span class="c1">26</span></p><p class="c9"><span class="c4">In </span><span class="c5">Arachis hypogaea</span><span class="c4">, the ortholog </span><span class="c5">Arahy.SYK0BG</span><span class="c4">&nbsp;has been identified and annotated as encoding the GIGANTEA protein, as evidenced by PeanutBase expression data.</span><span class="c15">12</span><span class="c4">&nbsp;The presence of a </span><span class="c5">GIGANTEA</span><span class="c4">&nbsp;ortholog in peanut strongly implies that the core circadian clock mechanism, which gates photoperiodic responses primarily through the GI-CO-FT regulatory module, is conserved and operative in this legume species. The intricate molecular clockwork detailed in </span><span class="c5">Arabidopsis</span><span class="c4">, such as GI&#39;s involvement in the light-dependent degradation of CYCLING DOF FACTORs (CDFs) which are repressors of </span><span class="c5">CO</span><span class="c4">&nbsp;transcription </span><span class="c15">27</span><span class="c4">, is likely to be functionally mirrored in peanut. Consequently, natural or induced variations within the </span><span class="c5">Arahy.SYK0BG</span><span class="c4 c10">&nbsp;gene or its regulatory regions could be significant sources of the diversity observed in flowering time and photoperiod sensitivity among different peanut cultivars.</span></p><h3 class="c11"><span class="c24 c10">2.10. Other Key Flowering Regulators (CO, FUL, Chlorophyll a-b binding protein)</span></h3><p class="c25"><span class="c6">Beyond the major families detailed above, several other proteins are integral to the flowering process.</span></p><p class="c25"><span class="c6">CONSTANS (CO), and its homologs the CONSTANS-like (COL) proteins (discussed in section 2.1), are central integrators of light and circadian clock signals, acting to promote flowering primarily through the activation of FT.27</span></p><p class="c25"><span class="c6">FRUITFULL (FUL) is a MADS-box transcription factor that functions downstream of FT. It is involved in establishing floral meristem identity and promoting the transition to flowering. In Arachis, WRKY transcription factors have been shown to interact with FUL, suggesting a modulation of FUL&#39;s activity or integration of other signals through this interaction.14</span></p><p class="c25"><span class="c6">A Chlorophyll a-b binding protein encoding gene was identified as one of seven candidate genes from a GWAS for flowering time in peanut.15 While not a direct component of canonical flowering pathways, its association is intriguing. Such proteins are essential for light harvesting and photosynthesis. An association with flowering time could reflect the plant&#39;s energy status, as adequate photosynthetic output is often a prerequisite for the energetically demanding process of reproduction. Alternatively, chlorophyll-binding proteins or chloroplast-derived signals can sometimes participate in retrograde signaling from the chloroplast to the nucleus, influencing gene expression related to development in response to light conditions or metabolic status. This finding opens an avenue to explore the roles of metabolic signals or chloroplast-to-nucleus communication in the fine-tuning of flowering time in peanut.</span></p><p class="c9"><span class="c4">The following table (Table 1) summarizes the key gene families discussed, providing examples of </span><span class="c5">Arachis</span><span class="c4 c10">&nbsp;genes and their general roles in flowering.</span></p><p class="c9"><span class="c7 c10">Table 1: Key Gene Families Implicated in Peanut Flowering Time.</span></p><p class="c25 c27"><span class="c7 c10"></span></p><table class="c33"><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Gene Family</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Example Arachis Gene(s)/ID(s)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">General Role in Flowering</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Key Snippet Reference(s)</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like (COL)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhCOL1</span><span class="c14">&nbsp;(arahy.6J3MKV.1), </span><span class="c21">AhCOL3</span><span class="c2">&nbsp;(arahy.S3IANQ.1), 16 others</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Photoperiod sensing, activation of FT, integration of light and circadian signals.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">S3, S13, S49</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phosphatidylethanolamine-Binding Protein (PEBP)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyM2THPA, ArahyEM6VH3</span><span class="c14">&nbsp;(FT-like, promote); </span><span class="c21">ArahyG6F3P2</span><span class="c14">&nbsp;(FT-like, inhibit); </span><span class="c21">ArahyDYRS20</span><span class="c14">&nbsp;(TFL-like, promote); </span><span class="c21">Arahy.UVRI66</span><span class="c2">&nbsp;(FT-like)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like proteins act as florigens (promoters); TFL1-like proteins act as repressors or control inflorescence architecture.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">WRKY Transcription Factors</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhTWRKY35/AhTWRKY116</span><span class="c14">; </span><span class="c21">Arahy.RFW3PD</span><span class="c2">; GWAS candidate</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Modulate flowering via multiple pathways (photoperiod, autonomous, hormone, stress); interact with FUL, AP2.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, S10, </span><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">MYB Transcription Factors (R2R3-MYB)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhMYB51</span><span class="c14">&nbsp;(Arahy.04JNDX?); </span><span class="c21">AhMYB033</span><span class="c14">; </span><span class="c21">AhMYB001-AhMYB196</span><span class="c14">; </span><span class="c21">Arahy.JN4NL9</span><span class="c2">; GWAS candidate</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Diverse roles in development, hormone signaling impacting flowers (e.g., brassinosteroids), anther/pollen development, meristem identity.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S1, S4, </span><span class="c1">8</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Basic Helix-Loop-Helix (bHLH) TFs</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">Arahy.YHH73E</span><span class="c14">; </span><span class="c21">PIF1, PIF4</span><span class="c2">; GWAS candidate</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Regulation of light signaling (PIFs), potential direct role in flowering pathways.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR-RED IMPAIRED RESPONSE 1 (FAR1) TFs</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">Ahy_A10g049543</span><span class="c14">; 245 other </span><span class="c21">AhFAR1s</span><span class="c2">; GWAS candidate</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">PhyA-mediated far-red light signaling; roles in pod development, potential link to flowering initiation via light perception.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">2</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">APETALA2/Ethylene Response Factor (AP2/ERF)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">AP2 subfamily (59 members in </span><span class="c21">A. hypogaea</span><span class="c2">)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">AP2 subfamily involved in floral organ development, floral repression; interacts with WRKYs.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S10, </span><span class="c1">23</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome (PHY)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhphyA, AhphyA-like, AhphyB, AhphyE</span><span class="c14">; </span><span class="c21">Aradu.G6IAK</span><span class="c2">&nbsp;(AdphyA)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Photoreceptors for red/far-red light, crucial for photoperiodic flowering, shade avoidance.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c1">4</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">GIGANTEA (GI)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c21 c10">Arahy.SYK0BG</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Circadian clock component, regulation of photoperiodic flowering upstream of/with CO and FT.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">FRUITFULL (FUL)</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">Interacts with WRKY TFs in </span><span class="c21 c10">Arachis</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">MADS-box TF, floral meristem identity, downstream of FT.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S10, </span><span class="c1">30</span></p></td></tr><tr class="c13"><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chlorophyll a-b binding protein</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS candidate</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c2">Light harvesting, potential link to energy status or retrograde signaling for flowering.</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr></table><h2 class="c11"><span class="c32 c10">3. Genomic Regions and Quantitative Trait Loci (QTLs) Associated with Flowering Time and Related Traits</span></h2><p class="c9"><span class="c4 c10">The identification of genomic regions harboring genes that control flowering time and related developmental traits is paramount for understanding the genetic architecture of these complex characteristics and for facilitating molecular breeding. Quantitative Trait Locus (QTL) mapping and Genome-Wide Association Studies (GWAS) are powerful approaches extensively used in peanut genetics to pinpoint such regions.</span></p><p class="c9"><span class="c4">Several significant QTLs influencing flowering time or closely associated traits have been reported in </span><span class="c5">Arachis</span><span class="c4">. A notable QTL located on linkage group 06 (LG06) was identified from an interspecific cross between </span><span class="c5">A. duranensis</span><span class="c4">&nbsp;(wild A-genome progenitor) and </span><span class="c5">A. cardenasii</span><span class="c4">&nbsp;(another wild diploid). This single QTL was found to explain a substantial 44% of the phenotypic variation for main stem flowering time.</span><span class="c15">3</span><span class="c4">&nbsp;This finding underscores the rich reservoir of potentially valuable alleles for flowering time modification that resides within wild </span><span class="c5">Arachis</span><span class="c4 c10">&nbsp;germplasm, which could be harnessed for cultivated peanut improvement. The magnitude of this QTL&#39;s effect suggests that introgression from wild relatives can be a potent strategy for achieving significant alterations in flowering characteristics.</span></p><p class="c9"><span class="c4">In cultivated peanut, studies focusing on yield components that are intrinsically linked to flowering have also identified relevant QTLs. For instance, two stable, major-effect QTLs, designated </span><span class="c5">qRMPA09.1</span><span class="c4">&nbsp;and </span><span class="c5">qRMPA09.2</span><span class="c4">, were detected on chromosome A09.</span><span class="c15">18</span><span class="c4">&nbsp;These QTLs are associated with the rate of multiple pods per plant (RMSP), a trait influenced by the efficiency and characteristics of flowering. The R2R3-MYB transcription factor gene </span><span class="c5">AhMYB51</span><span class="c4">&nbsp;has been proposed as a candidate gene residing within this genomic region, potentially influencing RMSP through its effects on brassinosteroid levels in flowers.</span><span class="c15">18</span><span class="c4">&nbsp;These QTLs were mapped to chromosomal intervals of approximately 400 Kb and 600 Kb, respectively.</span><span class="c15">18</span><span class="c4">&nbsp;The co-localization of a QTL for a post-flowering yield trait (RMSP) with a candidate gene (</span><span class="c5">AhMYB51</span><span class="c4 c10">) implicated in floral biology highlights either pleiotropic effects of the underlying gene(s) or tight genetic linkage between genes controlling floral initiation/development and those governing subsequent fruit set. This implies that selection for flowering time might also lead to correlated responses in these downstream yield components.</span></p><p class="c9"><span class="c4">GWAS approaches, leveraging natural diversity, have further expanded the inventory of genomic regions associated with flowering time. A study involving 390 diverse peanut germplasm accessions, evaluated across five different environments, identified 259 Single Nucleotide Polymorphisms (SNPs) significantly associated with flowering time.</span><span class="c15">3</span><span class="c4">&nbsp;Within the confidence intervals of recurrently identified loci, seven candidate genes were predicted. These genes encode various types of proteins, including an R2R3-MYB transcription factor, a chlorophyll a-b binding protein, a bHLH transcription factor, a WRKY transcription factor, and a FAR1 transcription factor.</span><span class="c15">15</span><span class="c4">&nbsp;These associated loci were distributed across multiple chromosomes (A01, A02, A03, A04, A05, A07, A10, B01, B02, B03, B04, B06, B07, and B09) and individually explained a relatively small proportion of the phenotypic variation, ranging from 3.47% to 8.58%.</span><span class="c15">15</span><span class="c4">&nbsp;Specific gene identifiers for these seven candidates were not provided in the primary publication.</span><span class="c15">15</span><span class="c4 c10">&nbsp;The identification of numerous loci with minor effects through this large-scale GWAS confirms the highly polygenic nature of flowering time in peanut. This genetic architecture suggests that achieving substantial modifications in flowering time via breeding will likely necessitate the pyramiding of multiple favorable alleles from these minor-effect QTLs/genes, making genomic selection strategies potentially more effective than marker-assisted selection (MAS) targeting only a few major genes.</span></p><p class="c9"><span class="c4">Research has also focused on &quot;time-to-maturation&quot; (TTM), a trait closely related to flowering time, especially in specific market types like Virginia peanuts. Two consistent QTLs for TTM were identified in a Recombinant Inbred Line (RIL) population derived from two Virginia-type cultivars. These are </span><span class="c5">qMIA04a,b</span><span class="c4">, located on chromosome A04 within a 6.9 Mbp region (flanked by markers AX-176819644_A04 and AX-176815499_A04), and </span><span class="c5">qMIB03a,b</span><span class="c4">, on chromosome B03 spanning a 1.8 Mbp region (flanked by markers AX-176807311_B03 and AX-176806413_B03).</span><span class="c15">1</span><span class="c4">&nbsp;These QTLs accounted for 9.9&ndash;11.9% and 9.3&ndash;9.9% of the phenotypic variance for maturity index, respectively.</span><span class="c15">1</span><span class="c4">&nbsp;Candidate genes within these specific TTM QTL regions were not explicitly detailed in the cited source.</span><span class="c15">1</span><span class="c4">&nbsp;Additionally, other studies have reported QTLs associated with seed number, another yield component influenced by flowering, on chromosomes A05 and B06.</span><span class="c1">18</span></p><p class="c9"><span class="c4 c10">The collective findings from these QTL mapping and GWAS efforts are summarized in Table 2, providing a consolidated view of genomic regions influencing flowering and related traits in peanut.</span></p><p class="c9"><span class="c7 c10">Table 2: Identified QTLs for Flowering Time and Related Traits in Peanut.</span></p><p class="c25 c27"><span class="c7 c10"></span></p><table class="c33"><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">QTL Name/Region</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Chromosomal Location/Markers</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Associated Trait</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c10 c20">Key Candidate Gene(s) within QTL (if identified)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">PVE (%) (if available)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Source Population/Study</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Key Snippet Reference(s)</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">LG06 QTL</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Linkage Group 06</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Main stem flowering time</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Not specified</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">44%</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c21">A. duranensis</span><span class="c14">&nbsp;x </span><span class="c21">A. cardenasii</span><span class="c2">&nbsp;interspecific population</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c14">S3, S13, </span><span class="c1">3</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">qRMPA09.1</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA09 (~400Kb interval)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Rate of Multiple Pods per Plant (RMSP)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhMYB51</span><span class="c14">&nbsp;(potentially </span><span class="c21">Arahy.04JNDX</span><span class="c2">)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Major effect</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">R45 x JNH3 BC1F2 population</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c14">S1, </span><span class="c1">18</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">qRMPA09.2</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA09 (~600Kb interval)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Rate of Multiple Pods per Plant (RMSP)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhMYB51</span><span class="c14">&nbsp;(potentially </span><span class="c21">Arahy.04JNDX</span><span class="c2">)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Major effect</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">R45 x JNH3 BC1F2 population</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c14">S1, </span><span class="c1">18</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS Loci</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01, A02, A03, A04, A05, A07, A10, B01, B02, B03, B04, B06, B07, B09 (259 SNPs)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Flowering Time</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF, Chl a-b binding protein, bHLH TF, WRKY TF, FAR1 TF (among 7 candidates)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">3.47 - 8.58% (per locus)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">390 peanut germplasm accessions (GWAS)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">3</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">qMIA04a,b</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA04 (118.7-125.6 Mbp); Flanking markers: AX-176819644_A04, AX-176815499_A04</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Time-to-Maturation (TTM) (Maturity Index)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Not specified</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">9.9 - 11.9%</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">&#39;Hanoch&#39; x &#39;Harari&#39; RIL population (Virginia-type)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c1">1</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">qMIB03a,b</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrB03 (2.8-4.7 Mbp); Flanking markers: AX-176807311_B03, AX-176806413_B03</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Time-to-Maturation (TTM) (Maturity Index)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Not specified</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">9.3 - 9.9%</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">&#39;Hanoch&#39; x &#39;Harari&#39; RIL population (Virginia-type)</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c1">1</span></p></td></tr><tr class="c13"><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Seed Number QTLs</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA05, ChrB06</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Seed Number per Pod</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Not specified</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Not specified</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c2">Various populations</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c0"><span class="c14">S1, </span><span class="c1">18</span></p></td></tr></table><h2 class="c11"><span class="c10 c32">4. Key Regulatory Pathways Orchestrating Flowering in Peanut</span></h2><p class="c9"><span class="c4">The transition from vegetative growth to flowering in plants is a complex process governed by an intricate network of interacting genetic pathways that integrate both endogenous cues and external environmental signals. Evidence gathered from research in </span><span class="c5">Arachis</span><span class="c4 c10">&nbsp;species suggests that the core components and logic of these pathways are largely conserved, although potentially adapted to peanut&#39;s specific biology.</span></p><p class="c9"><span class="c4">The </span><span class="c7">Photoperiod Pathway</span><span class="c4">&nbsp;is particularly prominent, enabling plants to synchronize their flowering with favorable seasons by sensing changes in day length. Key molecular players identified or inferred in peanut&#39;s photoperiod pathway include members of the </span><span class="c5">CONSTANS (CO)</span><span class="c4">&nbsp;/ </span><span class="c5">CONSTANS-like (COL)</span><span class="c4">&nbsp;gene family, the circadian clock-associated protein </span><span class="c5">GIGANTEA (GI)</span><span class="c4">&nbsp;</span><span class="c15">12</span><span class="c4">, photoreceptors like Phytochromes (</span><span class="c5">PHY</span><span class="c4">) </span><span class="c15">2</span><span class="c4">, and the downstream mobile signal </span><span class="c5">FLOWERING LOCUS T (FT)</span><span class="c4">. Interactions within this pathway are being uncovered in peanut, such as </span><span class="c5">AhCOL1</span><span class="c4">&#39;s interaction with </span><span class="c5">AhNF-YC1</span><span class="c4">&nbsp;and </span><span class="c5">AhCOL3</span><span class="c4">&#39;s interaction with both </span><span class="c5">AhNF-YC1</span><span class="c4">&nbsp;and </span><span class="c5">AhCOP1</span><span class="c4">, mirroring regulatory modules known from </span><span class="c5">Arabidopsis</span><span class="c4">. The model derived from </span><span class="c5">Arabidopsis</span><span class="c4">&nbsp;suggests GI interacts with FLAVIN-BINDING, KELCH REPEAT, F BOX protein 1 (FKF1) to mediate the degradation of CO repressors (CDFs), thus allowing CO accumulation and FT activation under inductive long days.</span><span class="c15">27</span><span class="c4">&nbsp;The presence of </span><span class="c5">GI</span><span class="c4">&nbsp;and </span><span class="c5">COL</span><span class="c4 c10">&nbsp;homologs in peanut points to the likely operation of this mechanism.</span></p><p class="c9"><span class="c4">Beyond photoperiod, several other pathways contribute significantly to flowering time regulation in </span><span class="c5">Arachis</span><span class="c4">. These include the </span><span class="c7">Autonomous Pathway</span><span class="c4">, the </span><span class="c7">Aging Pathway</span><span class="c4">, the </span><span class="c7">Vernalization Pathway</span><span class="c4">&nbsp;(response to cold), the </span><span class="c7">Gibberellin Pathway</span><span class="c4">, the </span><span class="c7">Temperature Pathway</span><span class="c4">, and </span><span class="c7">Sugar Signaling Pathways</span><span class="c4">.</span><span class="c15">3</span><span class="c4">&nbsp;Phytohormones, in general, play multifaceted roles in plant development, including the regulation of flowering time. Specific hormonal influences are emerging in peanut; for example, brassinosteroids are implicated in the floral effects of the </span><span class="c5">AhMYB51</span><span class="c4">&nbsp;gene.</span><span class="c15">18</span><span class="c4">&nbsp;Furthermore, analysis of </span><span class="c5">AhCOL</span><span class="c4">&nbsp;gene promoters revealed an abundance of </span><span class="c5">cis</span><span class="c4">-regulatory elements responsive to abscisic acid (ABA) and methyl jasmonate (MeJA).</span><span class="c15">10</span><span class="c4">&nbsp;This finding suggests a direct molecular mechanism by which stress hormones, which accumulate under adverse conditions like drought or pathogen attack, could modulate the expression of key photoperiod pathway genes (</span><span class="c5">AhCOLs</span><span class="c4 c10">). Such a connection would allow environmental stresses to directly influence or override the primary photoperiodic signal, potentially serving as an adaptive strategy to adjust flowering time under unfavorable conditions to enhance reproductive success.</span></p><p class="c9"><span class="c4">WRKY transcription factors appear to function as crucial integrators within this network. Studies indicate their involvement across a remarkable breadth of flowering pathways in </span><span class="c5">Arachis</span><span class="c4">, encompassing aging, autonomous, circadian clock, hormone, photoperiod, sugar, temperature, and vernalization pathways.</span><span class="c15">5</span><span class="c4">&nbsp;Flowering is not merely a response to day length but is modulated by the plant&#39;s internal state (age, nutrient levels) and external conditions (temperature, stress). WRKY TFs, known for their roles in stress responses and development, seem ideally positioned to act as convergence points for these diverse signals. Their interaction with downstream regulators like FUL (involved in floral promotion) and AP2 (a floral repressor) </span><span class="c15">14</span><span class="c4 c10">&nbsp;further supports this integrative role, potentially allowing WRKYs to fine-tune the final decision to flower based on a holistic assessment of internal and external cues. This positions WRKY TFs as key modulators capable of integrating information from multiple signaling streams to orchestrate the appropriate flowering response in peanut.</span></p><p class="c9"><span class="c4">Key protein-protein interactions that form the nodes of these regulatory networks are beginning to be identified in peanut. These include the aforementioned interactions involving AhCOL1, AhCOL3, AhNF-YC1, AhCOP1, WRKY TFs, FUL, and AP2.</span><span class="c15">14</span><span class="c4 c10">&nbsp;Understanding these interactions is crucial for deciphering the precise molecular mechanisms controlling the floral transition.</span></p><h2 class="c11"><span class="c32 c10">5. Consolidated List of Candidate Genes with Available Identifiers</span></h2><p class="c9"><span class="c4 c10">Significant progress in peanut genomics has enabled the identification of numerous candidate genes associated with flowering time control. Compiling a list of these genes, particularly those with stable locus identifiers (e.g., the arahy.XXXXXX format used in PeanutBase and NCBI), is essential for facilitating further research and application in breeding. The availability of such identifiers, stemming directly from genome sequencing and annotation efforts, is transformative. It allows for precise gene targeting, marker development, functional characterization using tools like CRISPR/Cas9, comparative genomic analyses, and more accurate QTL mapping, representing a major advancement from relying solely on anonymous genomic regions. However, it is important to note that accessing specific gene lists, particularly from supplementary materials of publications, can sometimes present challenges, highlighting the need for continued improvements in data accessibility and standardization [Insight 23].</span></p><p class="c9"><span class="c4">Table 3 provides a consolidated list of selected candidate genes in </span><span class="c5">Arachis hypogaea</span><span class="c4 c10">&nbsp;implicated in flowering time or related developmental processes, based on the information available in the reviewed sources.</span></p><p class="c9"><span class="c7">Table 3: Selected Candidate Genes in </span><span class="c5">Arachis hypogaea</span><span class="c7 c10">&nbsp;Associated with Flowering Time Control.</span></p><p class="c25 c27"><span class="c7 c10"></span></p><table class="c33"><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Gene Name/Symbol</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Locus ID (arahy.XXXXXX or other)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Chromosome (if known)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Brief Functional Annotation (related to flowering/development)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Method of Identification</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c20 c10">Key Snippet Reference(s)</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.6J3MKV.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr01</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Photoperiod pathway; Interacts with AhNF-YC1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Interaction</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL3</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.S3IANQ.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr04</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Photoperiod pathway; Interacts with AhNF-YC1, AhCOP1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Interaction</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL4</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.7A5VUD.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr06</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Upregulated by light/mechanical stress; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL8</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.ZS9VWH.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr10</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Upregulated by light/mechanical stress; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL13</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.WEZ503.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr15</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Upregulated by light/mechanical stress; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AhCOL14</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">arahy.N2A79V.1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr16</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">CONSTANS-like; Upregulated by light/mechanical stress; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S13, </span><span class="c1">10</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyM2THPA</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyEM6VH3</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">Arahy4GAQ4U</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyIZ8FG5</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyG6F3P2</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Inhibits flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyLUT2QN</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Inhibits flowering; Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">TFL-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyDYRS20</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering (atypical); Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">TFL-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">ArahyBBG51B</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">PEBP family; Promotes flowering (atypical); Temp/photoperiod response</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S15, </span><span class="c1">7</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FT-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.UVRI66</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Flowering locus protein T (PEBP family)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">WRKY TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.RFW3PD</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">WRKY family transcription factor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">WRKY TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(Candidate gene)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01-A10, B01-B09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">WRKY transcription factor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhMYB51</span><span class="c2">&nbsp;(Arahy.04JNDX?)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">MYB TF; Affects pod number via floral brassinosteroids</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">QTL Candidate</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S1, </span><span class="c1">18</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c10 c21">AhMYB033</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF; Associated with branch number, pod length, root-shoot ratio</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Association Analysis</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">8</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhTc1</span><span class="c2">&nbsp;(AhMYB099)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF; Regulates anthocyanin in testa</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Functional Characterization</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">8</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.JN4NL9</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">MYB family transcription factor APL-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(Candidate gene)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01-A10, B01-B09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">R2R3-MYB transcription factor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">bHLH TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.YHH73E</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Basic helix-loop-helix (bHLH) domain TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">bHLH TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.N7XQZF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Transcription factor bHLH68 isoform 1</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">bHLH TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(Candidate gene)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01-A10, B01-B09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">bHLH transcription factor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Ahy_A10g049543</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA10</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF; Expressed in peg/pod; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">2</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Ahy_A06g026579</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA06</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF; Expressed in peg/pod; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">2</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Ahy_A10g048401</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">ChrA10</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF; Expressed in peg/pod; Pod development</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Expression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">2</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(Candidate gene)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01-A10, B01-B09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">FAR1 transcription factor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">AP2 TF</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(59 members)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Various</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">APETALA2 subfamily; Floral organ development, floral repression</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">23</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome A</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhphyA</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome A photoreceptor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Cloning</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">4</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome A-like</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhphyA-like</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome A-like photoreceptor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Cloning</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">4</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome B</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhphyB</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome B photoreceptor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Cloning</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">4</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome E</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c21">AhphyE</span><span class="c2">&nbsp;(ID not in snippets)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Phytochrome E photoreceptor</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Homology, Cloning</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">4</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GIGANTEA</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Arahy.SYK0BG</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Unknown</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GIGANTEA protein; Circadian clock, photoperiod pathway</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Expression Annotation</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c1">12</span></p></td></tr><tr class="c13"><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chlorophyll a-b binding protein</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">(Candidate gene)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chr A01-A10, B01-B09</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">Chlorophyll a-b binding protein</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c2">GWAS</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c0"><span class="c14">S4, </span><span class="c1">15</span></p></td></tr></table><p class="c9"><span class="c5 c10">Note: Locus IDs for some key PEBP genes and cultivated peanut phytochromes were not directly available in the reviewed snippets and may require accessing supplementary data from cited publications.</span></p><h2 class="c11"><span class="c32 c10">6. Essential Databases and Resources for Peanut Flowering Time Research</span></h2><p class="c11"><span class="c4 c10">Research into the genetic control of flowering time in peanut relies heavily on access to comprehensive and well-curated genomic and genetic databases. Several key resources serve the peanut research community:</span></p><ul class="c26 lst-kix_list_1-0 start"><li class="c11 c30 li-bullet-0"><span class="c7">PeanutBase (peanutbase.org):</span><span class="c4">&nbsp;This database stands as the central, dedicated repository for genetic and genomic information pertaining to cultivated peanut (</span><span class="c5">Arachis hypogaea</span><span class="c4">) and its wild relatives.</span><span class="c15">31</span><span class="c4">&nbsp;It integrates a wide array of data types, including reference genome sequences, gene predictions with functional annotations, genetic linkage maps, molecular markers (such as SNPs), QTL data, germplasm information, and extensive gene expression datasets.</span><span class="c15">12</span><span class="c4">&nbsp;PeanutBase offers valuable bioinformatics tools, such as gene search capabilities (by keyword or identifier, e.g., arahy.XXXXXX), sequence similarity searching (BLAST), synteny viewers, and integrated genome browsers.</span><span class="c15">33</span><span class="c4">&nbsp;Notably, it hosts RNA-Seq data from 22 different tissues of the reference cultivar &#39;Tifrunner&#39;, providing a rich resource for examining tissue-specific gene expression patterns.</span><span class="c1">34</span></li><li class="c0 c30 li-bullet-0"><span class="c7">NCBI (National Center for Biotechnology Information):</span><span class="c4">&nbsp;NCBI provides a broader biological context through its suite of databases. The NCBI Gene database offers detailed information on specific genes, including orthologs in other species, functional summaries, links to relevant literature (PubMed), and associated sequences.</span><span class="c15">29</span><span class="c4">&nbsp;NCBI maintains reference genome assemblies and annotations for </span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;and related species, which undergo periodic updates (e.g., Annotation Release 100).</span><span class="c15">38</span><span class="c4">&nbsp;The Sequence Read Archive (SRA) provides access to raw sequencing data from various experiments, including transcriptomic studies relevant to flowering or stress responses.</span><span class="c1">36</span></li><li class="c0 c30 li-bullet-0"><span class="c7">Plant Flowering Gene Database (PFGD) (pfgd.bio2db.com):</span><span class="c4">&nbsp;This specialized database focuses specifically on flowering-time genes across a wide range of angiosperm species.</span><span class="c15">40</span><span class="c4">&nbsp;It organizes genes based on their involvement in established flowering pathways (e.g., photoperiod, autonomous, vernalization, hormone pathways).</span><span class="c15">40</span><span class="c4">&nbsp;As of early 2024, PFGD reportedly contained entries for 576 flowering-time associated genes from </span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;cv. Tifrunner.</span><span class="c15">5</span><span class="c4">&nbsp;The database structure allows users to browse by species or pathway and potentially download gene lists.</span><span class="c1">40</span></li><li class="c0 c30 li-bullet-0"><span class="c7">Other Resources:</span><span class="c4">&nbsp;Additional specialized resources include Ensembl Genomes, which also hosts the </span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;genome </span><span class="c15">41</span><span class="c4">, and dedicated expression atlases like the Peanut Gene Expression Atlas (AhGEA) </span><span class="c15">42</span><span class="c4">&nbsp;and the </span><span class="c5">Arachis hypogaea</span><span class="c4">&nbsp;Gene Expression database (AhNGE).</span><span class="c1">43</span></li></ul><p class="c34"><span class="c4">Effective research into a complex trait like flowering time necessitates the synergistic use of these diverse resources. PeanutBase provides the critical species-specific genomic framework and curated data. NCBI offers broader comparative context, literature access, and raw data repositories. PFGD delivers a valuable functional overlay by organizing genes within known flowering pathways. Researchers must often navigate and integrate information from these multiple platforms to construct a comprehensive understanding. Furthermore, the dynamic nature of genomics research means these databases are continually updated with new genome assemblies, refined gene annotations, and additional experimental data.</span><span class="c15">36</span><span class="c4 c10">&nbsp;Consequently, researchers must remain cognizant of database versions and annotation releases to ensure they are utilizing the most current and accurate information available.</span></p><h2 class="c11"><span class="c32 c10">7. Implications for Peanut Improvement and Future Research Directions</span></h2><p class="c9"><span class="c4 c10">The expanding knowledge of the genes and genomic regions controlling flowering time in peanut holds significant implications for crop improvement strategies and guides future research endeavors.</span></p><p class="c25"><span class="c6">Breeding Applications:</span></p><p class="c25"><span class="c6">The identification of candidate genes and validated QTLs associated with flowering time and related maturity traits provides tangible targets for molecular breeding. SNPs closely linked to major QTLs, such as those identified on chromosome A09 for RMSP 18 or the TTM QTLs on A04 and B03 1, can be converted into user-friendly molecular markers. These markers enable Marker-Assisted Selection (MAS), allowing breeders to screen progeny more efficiently and accurately for desirable flowering characteristics, thereby accelerating the development of improved cultivars.1 However, the polygenic nature of flowering time, evidenced by GWAS studies identifying numerous loci with small individual effects 15, suggests that relying solely on MAS for a few major genes may not be sufficient to achieve substantial shifts in flowering behavior. In such cases, Genomic Selection (GS), which utilizes genome-wide marker data to predict the breeding value of individuals, may prove more powerful for accumulating favorable alleles from multiple minor loci simultaneously. The complex genetic architecture therefore necessitates a combined approach, integrating traditional breeding with advanced molecular tools like MAS and GS, underpinned by robust phenotyping and functional gene knowledge.</span></p><p class="c9"><span class="c4">Developing varieties with optimized flowering times, particularly early-maturing types, is a key objective for enhancing peanut adaptation, especially in regions with limited water availability where early maturity facilitates drought escape.</span><span class="c15">1</span><span class="c4">&nbsp;Genes identified through GWAS or those underlying TTM QTLs </span><span class="c15">1</span><span class="c4">&nbsp;represent prime targets for breeding efforts aimed at achieving earliness. Furthermore, understanding how stress response pathways, potentially mediated by WRKY TFs or involving ABA/MeJA-responsive </span><span class="c5">AhCOL</span><span class="c4 c10">&nbsp;genes, intersect with flowering control pathways is crucial. This knowledge can guide the development of cultivars that maintain stable flowering patterns even under challenging environmental conditions.</span></p><p class="c25"><span class="c6">Exploiting Genetic Diversity:</span></p><p class="c25"><span class="c6">The cultivated peanut germplasm suffers from a relatively narrow genetic base. Wild Arachis species, however, harbor significant genetic diversity, including potentially novel alleles influencing flowering time, as demonstrated by the major QTL identified on LG06 from an interspecific cross. Continued exploration of wild relatives and the strategic introgression of beneficial alleles into elite breeding lines remain valuable avenues for broadening the genetic toolkit available for flowering time manipulation.</span></p><p class="c25"><span class="c6">Future Research Directions:</span></p><p class="c25"><span class="c6">Despite considerable progress, many identified genes remain candidates requiring functional validation. Techniques such as gene editing (e.g., CRISPR/Cas9), overexpression studies, and gene silencing are essential to definitively confirm the roles of genes like AhMYB51, the GWAS candidates, and the atypically behaving TFL1-like PEBPs.8 Elucidating the complete molecular mechanisms, including characterizing protein-protein interaction networks (e.g., extending work on COL, WRKY interactions) and identifying key cis-regulatory elements controlling the expression of major flowering genes, is critical.</span></p><p class="c9"><span class="c4">Further investigation into the intricate links between flowering time regulation and peanut&#39;s unique geocarpic development is warranted. Understanding the specific roles of gene families like </span><span class="c5">AhFAR1</span><span class="c4">&nbsp;</span><span class="c15">2</span><span class="c4">&nbsp;or light-responsive </span><span class="c5">AhCOL</span><span class="c4 c10">&nbsp;genes in coordinating aerial flowering with subsequent subterranean podding could reveal novel regulatory checkpoints.</span></p><p class="c9"><span class="c4 c10">A major frontier lies in dissecting the molecular basis of Genotype x Environment (GxE) interactions for flowering time. Peanut is cultivated across diverse latitudes and climates, experiencing varied photoperiod and temperature regimes. Understanding which specific genes and alleles confer broad adaptation versus niche-specific flowering responses is crucial for developing varieties tailored to target environments. This requires studying allele-specific responses to environmental cues and characterizing the functional consequences of allelic variation within key flowering time genes across diverse germplasm. Such knowledge is fundamental for enabling precision breeding strategies that optimize flowering time for specific production scenarios.</span></p><h2 class="c11"><span class="c32 c10">8. Conclusion</span></h2><p class="c9"><span class="c4">The timing of flowering in peanut (</span><span class="c5">Arachis hypogaea</span><span class="c4 c10">) is a complex, quantitatively inherited trait of profound agronomic importance, governed by an intricate network of genes interacting with environmental signals. Research efforts employing QTL mapping, GWAS, transcriptomics, and comparative genomics have made substantial strides in identifying key genetic players and pathways involved in this process.</span></p><p class="c9"><span class="c4">Major gene families, including </span><span class="c5">CONSTANS-like (COL)</span><span class="c4">, Phosphatidylethanolamine-Binding Proteins (</span><span class="c5">PEBP</span><span class="c4">, encompassing </span><span class="c5">FT</span><span class="c4">&nbsp;and </span><span class="c5">TFL1</span><span class="c4">&nbsp;homologs), </span><span class="c5">WRKY</span><span class="c4">, </span><span class="c5">R2R3-MYB</span><span class="c4">, </span><span class="c5">bHLH</span><span class="c4">, </span><span class="c5">FAR1</span><span class="c4">, </span><span class="c5">AP2/ERF</span><span class="c4">, Phytochromes (</span><span class="c5">PHY</span><span class="c4">), and </span><span class="c5">GIGANTEA (GI)</span><span class="c4 c10">, have been implicated in peanut flowering control. These families contribute to various regulatory pathways, with the photoperiod pathway being particularly prominent, alongside autonomous, hormone-mediated, and stress-responsive pathways. The identification of interactions between proteins from different families (e.g., WRKY-AP2/FUL, COL-NF-YC/COP1) underscores the highly interconnected nature of the flowering regulatory network, suggesting complex signal integration rather than simple linear cascades.</span></p><p class="c9"><span class="c4 c10">Genomic studies have pinpointed specific QTLs and candidate genes associated with flowering time and related traits like time-to-maturation and pod number. While some major QTLs, potentially introgressed from wild relatives, offer targets for marker-assisted selection, the overall polygenic architecture highlights the likely need for genomic selection approaches to effectively manipulate this trait. The allotetraploid nature of peanut adds complexity, with evidence for subgenome dominance and functional divergence among homoeologous flowering genes providing both challenges and opportunities for breeding. Furthermore, peanut&#39;s unique geocarpy introduces specific selective pressures and likely involves co-option or modification of flowering-related pathways (e.g., light signaling via PHY and FAR1) for subterranean fruit development.</span></p><p class="c36"><span class="c4 c10">The availability of genomic resources like PeanutBase and NCBI, coupled with specialized databases like PFGD, provides an essential foundation for ongoing research. Future efforts should focus on the functional validation of candidate genes, particularly those with potentially novel roles in peanut, and the detailed characterization of allelic variation within these genes across diverse germplasm. Understanding the functional impact of different alleles and their interaction with environmental factors (GxE) remains a critical frontier. This knowledge is essential for developing precision breeding strategies to optimize flowering time, enhance adaptation to diverse environments, improve stress resilience, and ultimately increase the yield and stability of this vital global crop.</span></p><h4 class="c23"><span class="c10 c35">Works cited</span></h4><ol class="c26 lst-kix_list_2-0 start" start="1"><li class="c8 li-bullet-0"><span class="c18">d-nb.info, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://d-nb.info/1234869217/34&amp;sa=D&amp;source=editors&amp;ust=1746682144722982&amp;usg=AOvVaw0TW8XnTc4qOGyvojcX-QoP">https://d-nb.info/1234869217/34</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Identification and Expression of FAR1 ... - Frontiers, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.893278/full&amp;sa=D&amp;source=editors&amp;ust=1746682144723936&amp;usg=AOvVaw2Be9nCooALqfQJfeNZR7Ka">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.893278/full</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Analysis and Expression Profiles of AhCOLs Family in Peanut (Arachis hypogaea L.) - MDPI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.mdpi.com/1422-0067/26/7/3404&amp;sa=D&amp;source=editors&amp;ust=1746682144724712&amp;usg=AOvVaw2cSb1XNWb71y2o3pnEIzcG">https://www.mdpi.com/1422-0067/26/7/3404</a></span></li><li class="c8 li-bullet-0"><span class="c18">Characterization of peanut phytochromes and their possible ..., accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5969742/&amp;sa=D&amp;source=editors&amp;ust=1746682144725461&amp;usg=AOvVaw18wR5b-woCWoyJxAT8CxUn">https://pmc.ncbi.nlm.nih.gov/articles/PMC5969742/</a></span></li><li class="c8 li-bullet-0"><span class="c18">WRKY transcription factors modulate flowering time in four Arachis ..., accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11212391/&amp;sa=D&amp;source=editors&amp;ust=1746682144726179&amp;usg=AOvVaw2lZHpCfoVd1sFVjYXO46_I">https://pmc.ncbi.nlm.nih.gov/articles/PMC11212391/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome Sequencing and Analysis of the Peanut B-Genome Progenitor (Arachis ipaensis) - PMC - PubMed Central, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5943715/&amp;sa=D&amp;source=editors&amp;ust=1746682144726942&amp;usg=AOvVaw2S87U4m5hQsU5x0TyTJ5Ag">https://pmc.ncbi.nlm.nih.gov/articles/PMC5943715/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Comparative Genomic and Expression Analysis Insight into Evolutionary Characteristics of PEBP Genes in Cultivated Peanuts and Their Roles in Floral Induction - MDPI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.mdpi.com/1422-0067/23/20/12429&amp;sa=D&amp;source=editors&amp;ust=1746682144727865&amp;usg=AOvVaw1KGgwKPLUpThFoAaXoBkfz">https://www.mdpi.com/1422-0067/23/20/12429</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-wide analysis of R2R3-MYB genes in cultivated ... - Frontiers, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2023.1102174/full&amp;sa=D&amp;source=editors&amp;ust=1746682144728532&amp;usg=AOvVaw1RNmtMLQu5aoJMswU0cpZU">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2023.1102174/full</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-wide identification of peanut IGT family genes and their potential roles in the development of plant architecture, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10663514/&amp;sa=D&amp;source=editors&amp;ust=1746682144729074&amp;usg=AOvVaw2CihEt83NjRZ0yZb4OGqaX">https://pmc.ncbi.nlm.nih.gov/articles/PMC10663514/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Analysis and Expression Profiles of AhCOLs Family in Peanut (Arachis hypogaea L.) - PubMed Central, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/&amp;sa=D&amp;source=editors&amp;ust=1746682144729584&amp;usg=AOvVaw2_OR2d5BLIFtfZVyx7UCCf">https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Comparative Genomic and Expression Analysis Insight into ..., accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9604132/&amp;sa=D&amp;source=editors&amp;ust=1746682144730040&amp;usg=AOvVaw2m2mhAZ46qGDT9BzYp9AOw">https://pmc.ncbi.nlm.nih.gov/articles/PMC9604132/</a></span></li><li class="c8 li-bullet-0"><span class="c18">www.peanutbase.org, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.peanutbase.org/files/expression/arahy_atlas_html/PodPt1-Seed_Pattee8_down.html&amp;sa=D&amp;source=editors&amp;ust=1746682144730726&amp;usg=AOvVaw2atWF8Vcf0gsr1wtfNkH_-">https://www.peanutbase.org/files/expression/arahy_atlas_html/PodPt1-Seed_Pattee8_down.html</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Investigation of Apyrase (APY) Genes in Peanut (Arachis hypogaea L.) and Functional Characterization of a Pod-Abundant Expression Promoter AhAPY2-1p - MDPI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.mdpi.com/1422-0067/24/5/4622&amp;sa=D&amp;source=editors&amp;ust=1746682144731601&amp;usg=AOvVaw0NXj78ykEyWeEMM1w_g5dE">https://www.mdpi.com/1422-0067/24/5/4622</a></span></li><li class="c8 li-bullet-0"><span class="c18">WRKY transcription factors modulate flowering time in four Arachis ..., accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/38943100/&amp;sa=D&amp;source=editors&amp;ust=1746682144732056&amp;usg=AOvVaw1g_rocMedi2ghlac1VexVW">https://pubmed.ncbi.nlm.nih.gov/38943100/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-wide associate analysis of flowering traits and identification ..., accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://zwxb.chinacrops.org/EN/10.3724/SP.J.1006.2025.44145&amp;sa=D&amp;source=editors&amp;ust=1746682144732484&amp;usg=AOvVaw0z3JlzilzPYnlVPs1Fnx5u">https://zwxb.chinacrops.org/EN/10.3724/SP.J.1006.2025.44145</a></span></li><li class="c8 li-bullet-0"><span class="c18">peanutbase.org, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://peanutbase.org/files/expression/express_pairs_html/MainStem_Leaves-Flowers_down.html&amp;sa=D&amp;source=editors&amp;ust=1746682144732907&amp;usg=AOvVaw3l0ygdcwksE8h0iQj_u-x1">https://peanutbase.org/files/expression/express_pairs_html/MainStem_Leaves-Flowers_down.html</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-wide analysis of R2R3-MYB genes in cultivated peanut (Arachis hypogaea L.), accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9971814/&amp;sa=D&amp;source=editors&amp;ust=1746682144733330&amp;usg=AOvVaw3hm7Lqyes7_VbJajy1gfK7">https://pmc.ncbi.nlm.nih.gov/articles/PMC9971814/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Fine mapping and candidate gene analysis of major QTLs for number of seeds per pod in Arachis hypogaea L - PubMed Central, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11998195/&amp;sa=D&amp;source=editors&amp;ust=1746682144733807&amp;usg=AOvVaw1qCk6rmzPF6Uvq0ZmREDrC">https://pmc.ncbi.nlm.nih.gov/articles/PMC11998195/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Whole&#8208;genome resequencing&#8208;based QTL&#8208;seq identified AhTc1 gene encoding a R2R3&#8208;MYB transcription factor controlling peanut purple testa colour - PMC, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6920131/&amp;sa=D&amp;source=editors&amp;ust=1746682144734402&amp;usg=AOvVaw1hT-clE_OV9HNvOvUGwW3S">https://pmc.ncbi.nlm.nih.gov/articles/PMC6920131/</a></span></li><li class="c8 li-bullet-0"><span class="c18">CN - MGI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://en.mgi-tech.com/Download/download_file_new/id/414&amp;sa=D&amp;source=editors&amp;ust=1746682144734788&amp;usg=AOvVaw1qntenWQbgDRk16fDBxjTt">https://en.mgi-tech.com/Download/download_file_new/id/414</a></span></li><li class="c8 li-bullet-0"><span class="c18">MainStem_Leaves-Seed_Pattee8_down - PeanutBase, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.peanutbase.org/files/expression/arahy_atlas_html/MainStem_Leaves-Seed_Pattee8_down.html&amp;sa=D&amp;source=editors&amp;ust=1746682144735291&amp;usg=AOvVaw1AaTN0Bck7X90BJ3bP36pL">https://www.peanutbase.org/files/expression/arahy_atlas_html/MainStem_Leaves-Seed_Pattee8_down.html</a></span></li><li class="c8 li-bullet-0"><span class="c18">Monoseeding Increases Peanut (Arachis hypogaea L.) Yield by Regulating Shade-Avoidance Responses and Population Density - MDPI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.mdpi.com/2223-7747/10/11/2405&amp;sa=D&amp;source=editors&amp;ust=1746682144735739&amp;usg=AOvVaw0DJaHaDsfzE-uStWffEXCv">https://www.mdpi.com/2223-7747/10/11/2405</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Identification and Expression Analysis of ... - Frontiers, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2021.750761/full&amp;sa=D&amp;source=editors&amp;ust=1746682144736217&amp;usg=AOvVaw0nGOpn457TopsvbJzkeJds">https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2021.750761/full</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Identification and Expression Analysis of ... - Frontiers, accessed May 7, 2025, </span><span class="c18 c19"><a class="c16" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2021.750761/full%23SM1&amp;sa=D&amp;source=editors&amp;ust=1746682144736758&amp;usg=AOvVaw1pAhd-k-avQasRmrv6JurY">https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2021.750761/full#SM1</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genetic characterization of flowering and phytochrome genes in peanut (Arachis hypogaea L.) for early maturity Sammyia Ja - Lancaster University, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.research.lancs.ac.uk/portal/services/downloadRegister/358056102/Jannat_et_al_2022.pdf&amp;sa=D&amp;source=editors&amp;ust=1746682144737475&amp;usg=AOvVaw0IPt9VfHGPFciRl88YUI-B">https://www.research.lancs.ac.uk/portal/services/downloadRegister/358056102/Jannat_et_al_2022.pdf</a></span></li><li class="c8 li-bullet-0"><span class="c18">GIGANTEA &ndash; an emerging story - Frontiers, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2015.00008/full&amp;sa=D&amp;source=editors&amp;ust=1746682144738209&amp;usg=AOvVaw36PJgYPzLsYEFUy3ZOAvSw">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2015.00008/full</a></span></li><li class="c8 li-bullet-0"><span class="c18">Gigantea: Uncovering New Functions in Flower Development - PMC - PubMed Central, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7600796/&amp;sa=D&amp;source=editors&amp;ust=1746682144738658&amp;usg=AOvVaw2pm-8dUO0cHRgIF_HsqlIg">https://pmc.ncbi.nlm.nih.gov/articles/PMC7600796/</a></span></li><li class="c8 li-bullet-0"><span class="c18">GIGANTEA: a circadian clock-controlled gene that regulates photoperiodic flowering in Arabidopsis and encodes a protein with several possible membrane-spanning domains - PubMed, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/10469647/&amp;sa=D&amp;source=editors&amp;ust=1746682144739310&amp;usg=AOvVaw2l8LMg4b3BzfSkd333XfSx">https://pubmed.ncbi.nlm.nih.gov/10469647/</a></span></li><li class="c8 li-bullet-0"><span class="c18">GI gigantea protein (GI) [Arabidopsis thaliana (thale cress)] - Gene - NCBI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene?cmd%3DDetailsSearch%26term%3DAT1G22770&amp;sa=D&amp;source=editors&amp;ust=1746682144739812&amp;usg=AOvVaw2NxDXXtJGq37blV5O0K6N1">https://www.ncbi.nlm.nih.gov/gene?cmd=DetailsSearch&amp;term=AT1G22770</a></span></li><li class="c8 li-bullet-0"><span class="c18">Flowering time, assayed by leaf numbers of the basal rosette - ResearchGate, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.researchgate.net/figure/Flowering-time-assayed-by-leaf-numbers-of-the-basal-rosette_tbl1_23462298&amp;sa=D&amp;source=editors&amp;ust=1746682144740448&amp;usg=AOvVaw2KtAGkruXjz4jrAnq649z3">https://www.researchgate.net/figure/Flowering-time-assayed-by-leaf-numbers-of-the-basal-rosette_tbl1_23462298</a></span></li><li class="c8 li-bullet-0"><span class="c18">PeanutBase - Ag Data Commons, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://agdatacommons.nal.usda.gov/articles/dataset/PeanutBase/24660672&amp;sa=D&amp;source=editors&amp;ust=1746682144740897&amp;usg=AOvVaw27yOi6FdFLmnjaLzSNSDo7">https://agdatacommons.nal.usda.gov/articles/dataset/PeanutBase/24660672</a></span></li><li class="c8 li-bullet-0"><span class="c18">PeanutBase - Dataset - Catalog - Data.gov, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://catalog.data.gov/dataset/peanutbase-9d5e9&amp;sa=D&amp;source=editors&amp;ust=1746682144741265&amp;usg=AOvVaw3SG72yWB0r7bWnPSrJLiK3">https://catalog.data.gov/dataset/peanutbase-9d5e9</a></span></li><li class="c8 li-bullet-0"><span class="c18">Gene Search &ndash; PeanutBase, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.peanutbase.org/tools/search/gene.html&amp;sa=D&amp;source=editors&amp;ust=1746682144741591&amp;usg=AOvVaw0JuISxf_PaO2rTa6zQi7VP">https://www.peanutbase.org/tools/search/gene.html</a></span></li><li class="c8 li-bullet-0"><span class="c18">Expression - PeanutBase, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.peanutbase.org/expression/&amp;sa=D&amp;source=editors&amp;ust=1746682144741881&amp;usg=AOvVaw3_2etvAGjazIdHrpXU0uem">https://www.peanutbase.org/expression/</a></span></li><li class="c8 li-bullet-0"><span class="c18">PeanutBase: Home, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.peanutbase.org/&amp;sa=D&amp;source=editors&amp;ust=1746682144742132&amp;usg=AOvVaw27hfCeB0ie3QSbpSKmMqCu">https://www.peanutbase.org/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Genome-Wide Identification of Peanut KCS Genes Reveals That AhKCS1 and AhKCS28 Are Involved in Regulating VLCFA Contents in Seeds - PubMed Central, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7221192/&amp;sa=D&amp;source=editors&amp;ust=1746682144742647&amp;usg=AOvVaw3B9jdKgo4zy22yCeqcii39">https://pmc.ncbi.nlm.nih.gov/articles/PMC7221192/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Taxonomy browser (Arachis hypogaea) - NCBI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode%3Dinfo%26id%3D3818&amp;sa=D&amp;source=editors&amp;ust=1746682144743125&amp;usg=AOvVaw2Q6_K7iyweMUkcypkdiCmx">https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=info&amp;id=3818</a></span></li><li class="c8 li-bullet-0"><span class="c18">Arachis hypogaea Annotation Report - NCBI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_hypogaea/100/&amp;sa=D&amp;source=editors&amp;ust=1746682144743613&amp;usg=AOvVaw3y0NRp60Bk14bw1EWMyp9J">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_hypogaea/100/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Arachis hypogaea - Search results - Genome - Assembly - NCBI, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/assembly/?term%3DArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1746682144744093&amp;usg=AOvVaw3rs04GHORJhcc5ub4d8f8y">https://www.ncbi.nlm.nih.gov/assembly/?term=Arachis%20hypogaea</a></span></li><li class="c8 li-bullet-0"><span class="c18">Plant Flowering-time Gene Database, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=http://pfgd.bio2db.com/&amp;sa=D&amp;source=editors&amp;ust=1746682144744400&amp;usg=AOvVaw0s6p74q961-pGWReYUkipa">http://pfgd.bio2db.com/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Arachis hypogaea - Ensembl Genomes 60, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://plants.ensembl.org/Arachis_hypogaea/Info/Index&amp;sa=D&amp;source=editors&amp;ust=1746682144744887&amp;usg=AOvVaw0KdQY8IAHvTeQX7TaGGBd2">https://plants.ensembl.org/Arachis_hypogaea/Info/Index</a></span></li><li class="c8 li-bullet-0"><span class="c18">Arachis hypogaea gene expression atlas for fastigiata subspecies of cultivated groundnut to accelerate functional and translational genomics applications - PMC, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC7589347/&amp;sa=D&amp;source=editors&amp;ust=1746682144745448&amp;usg=AOvVaw3ECLvKZCP3oiaN5YLHOwrc">https://pmc.ncbi.nlm.nih.gov/articles/PMC7589347/</a></span></li><li class="c8 li-bullet-0"><span class="c18">Arachis hypogaea Gene Expression, accessed May 7, 2025, </span><span class="c19 c18"><a class="c16" href="https://www.google.com/url?q=https://nipgr.ac.in/AhNGE/index.php&amp;sa=D&amp;source=editors&amp;ust=1746682144745869&amp;usg=AOvVaw2d0D4AuuZD0PLmOEvW7dHa">https://nipgr.ac.in/AhNGE/index.php</a></span></li></ol></body></html>