<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);.lst-kix_list_c-0>li:before{content:"\0025cf   "}ul.lst-kix_list_b-8{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_c-1>li:before{content:"\0025cb   "}ul.lst-kix_list_b-3{list-style-type:none}ul.lst-kix_list_b-2{list-style-type:none}.lst-kix_list_c-3>li:before{content:"\0025a0   "}.lst-kix_list_c-4>li:before{content:"\0025a0   "}ul.lst-kix_list_b-1{list-style-type:none}ul.lst-kix_list_b-0{list-style-type:none}ul.lst-kix_list_b-7{list-style-type:none}ul.lst-kix_list_b-6{list-style-type:none}.lst-kix_list_c-2>li:before{content:"\0025a0   "}ul.lst-kix_list_b-5{list-style-type:none}ul.lst-kix_list_b-4{list-style-type:none}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}.lst-kix_list_11-0>li{counter-increment:lst-ctn-kix_list_11-0}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_9-0{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025a0   "}ul.lst-kix_list_a-4{list-style-type:none}ul.lst-kix_list_a-3{list-style-type:none}.lst-kix_list_5-2>li:before{content:"\0025a0   "}ul.lst-kix_list_a-2{list-style-type:none}ul.lst-kix_list_a-1{list-style-type:none}.lst-kix_list_5-1>li:before{content:"\0025cb   "}ul.lst-kix_list_a-8{list-style-type:none}ul.lst-kix_list_a-7{list-style-type:none}ul.lst-kix_list_a-6{list-style-type:none}ul.lst-kix_list_a-5{list-style-type:none}.lst-kix_list_5-7>li:before{content:"\0025a0   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025a0   "}.lst-kix_list_5-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_d-6>li:before{content:"\0025a0   "}.lst-kix_list_5-4>li:before{content:"\0025a0   "}.lst-kix_list_5-5>li:before{content:"\0025a0   "}ul.lst-kix_list_8-0{list-style-type:none}.lst-kix_list_d-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_d-7>li:before{content:"\0025a0   "}.lst-kix_list_d-2>li:before{content:"\0025a0   "}.lst-kix_list_6-1>li:before{content:"\0025cb   "}.lst-kix_list_6-3>li:before{content:"\0025a0   "}.lst-kix_list_d-1>li:before{content:"\0025cb   "}.lst-kix_list_d-5>li:before{content:"\0025a0   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025a0   "}.lst-kix_list_d-4>li:before{content:"\0025a0   "}.lst-kix_list_d-3>li:before{content:"\0025a0   "}.lst-kix_list_6-2>li:before{content:"\0025a0   "}.lst-kix_list_c-7>li:before{content:"\0025a0   "}ul.lst-kix_list_a-0{list-style-type:none}.lst-kix_list_c-5>li:before{content:"\0025a0   "}.lst-kix_list_c-8>li:before{content:"\0025a0   "}.lst-kix_list_6-8>li:before{content:"\0025a0   "}.lst-kix_list_6-5>li:before{content:"\0025a0   "}.lst-kix_list_6-7>li:before{content:"\0025a0   "}.lst-kix_list_d-0>li:before{content:"\0025cf   "}.lst-kix_list_c-6>li:before{content:"\0025a0   "}.lst-kix_list_6-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_d-8{list-style-type:none}ul.lst-kix_list_d-7{list-style-type:none}.lst-kix_list_7-4>li:before{content:"\0025a0   "}.lst-kix_list_7-6>li:before{content:"\0025a0   "}ul.lst-kix_list_d-6{list-style-type:none}.lst-kix_list_2-5>li:before{content:"\0025a0   "}ul.lst-kix_list_d-1{list-style-type:none}ul.lst-kix_list_d-0{list-style-type:none}.lst-kix_list_b-8>li:before{content:"\0025a0   "}.lst-kix_list_7-2>li:before{content:"\0025a0   "}ul.lst-kix_list_d-5{list-style-type:none}ul.lst-kix_list_d-4{list-style-type:none}ul.lst-kix_list_d-3{list-style-type:none}ul.lst-kix_list_d-2{list-style-type:none}.lst-kix_list_b-4>li:before{content:"\0025a0   "}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}.lst-kix_list_10-1>li:before{content:"\0025cb   "}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}.lst-kix_list_7-8>li:before{content:"\0025a0   "}ul.lst-kix_list_3-0{list-style-type:none}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}.lst-kix_list_b-6>li:before{content:"\0025a0   "}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}.lst-kix_list_10-7>li:before{content:"\0025a0   "}.lst-kix_list_10-5>li:before{content:"\0025a0   "}.lst-kix_list_10-3>li:before{content:"\0025a0   "}ul.lst-kix_list_11-7{list-style-type:none}ul.lst-kix_list_11-6{list-style-type:none}.lst-kix_list_b-0>li:before{content:"\0025cf   "}.lst-kix_list_4-1>li:before{content:"\0025cb   "}ul.lst-kix_list_11-5{list-style-type:none}ul.lst-kix_list_11-4{list-style-type:none}ul.lst-kix_list_11-3{list-style-type:none}ul.lst-kix_list_11-2{list-style-type:none}.lst-kix_list_b-2>li:before{content:"\0025a0   "}ul.lst-kix_list_11-1{list-style-type:none}.lst-kix_list_9-2>li:before{content:"\0025a0   "}.lst-kix_list_4-3>li:before{content:"\0025a0   "}.lst-kix_list_4-5>li:before{content:"\0025a0   "}ul.lst-kix_list_11-8{list-style-type:none}.lst-kix_list_9-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-8{list-style-type:none}ul.lst-kix_list_c-7{list-style-type:none}.lst-kix_list_9-6>li:before{content:"\0025a0   "}ul.lst-kix_list_c-2{list-style-type:none}ul.lst-kix_list_c-1{list-style-type:none}ul.lst-kix_list_c-0{list-style-type:none}.lst-kix_list_9-4>li:before{content:"\0025a0   "}ul.lst-kix_list_c-6{list-style-type:none}.lst-kix_list_a-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-5{list-style-type:none}.lst-kix_list_11-3>li:before{content:"\0025a0   "}ul.lst-kix_list_c-4{list-style-type:none}ul.lst-kix_list_c-3{list-style-type:none}ul.lst-kix_list_2-8{list-style-type:none}.lst-kix_list_11-5>li:before{content:"\0025a0   "}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_9-8>li:before{content:"\0025a0   "}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}ul.lst-kix_list_2-7{list-style-type:none}.lst-kix_list_11-7>li:before{content:"\0025a0   "}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}ul.lst-kix_list_10-0{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}ul.lst-kix_list_10-8{list-style-type:none}ul.lst-kix_list_10-7{list-style-type:none}.lst-kix_list_1-7>li:before{content:"\0025a0   "}ul.lst-kix_list_10-6{list-style-type:none}ul.lst-kix_list_10-5{list-style-type:none}ul.lst-kix_list_10-4{list-style-type:none}ul.lst-kix_list_10-3{list-style-type:none}.lst-kix_list_1-5>li:before{content:"\0025a0   "}ul.lst-kix_list_10-2{list-style-type:none}ul.lst-kix_list_10-1{list-style-type:none}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_list_2-3>li:before{content:"\0025a0   "}ul.lst-kix_list_f-7{list-style-type:none}ul.lst-kix_list_f-6{list-style-type:none}ul.lst-kix_list_f-5{list-style-type:none}ul.lst-kix_list_f-4{list-style-type:none}ul.lst-kix_list_f-8{list-style-type:none}ul.lst-kix_list_f-3{list-style-type:none}.lst-kix_list_3-0>li:before{content:"\0025cf   "}ul.lst-kix_list_f-2{list-style-type:none}ul.lst-kix_list_f-1{list-style-type:none}ul.lst-kix_list_f-0{list-style-type:none}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}ul.lst-kix_list_5-5{list-style-type:none}ol.lst-kix_list_11-0{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}.lst-kix_list_8-1>li:before{content:"\0025cb   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}ul.lst-kix_list_5-0{list-style-type:none}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_8-7>li:before{content:"\0025a0   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_list_f-6>li:before{content:"\0025a0   "}.lst-kix_list_f-8>li:before{content:"\0025a0   "}.lst-kix_list_8-5>li:before{content:"\0025a0   "}.lst-kix_list_a-7>li:before{content:"\0025a0   "}.lst-kix_list_8-6>li:before{content:"\0025a0   "}.lst-kix_list_f-7>li:before{content:"\0025a0   "}.lst-kix_list_8-3>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_list_f-4>li:before{content:"\0025a0   "}.lst-kix_list_3-7>li:before{content:"\0025a0   "}.lst-kix_list_a-8>li:before{content:"\0025a0   "}.lst-kix_list_8-4>li:before{content:"\0025a0   "}.lst-kix_list_f-5>li:before{content:"\0025a0   "}.lst-kix_list_11-2>li:before{content:"\0025a0   "}.lst-kix_list_a-1>li:before{content:"\0025cb   "}.lst-kix_list_a-2>li:before{content:"\0025a0   "}.lst-kix_list_f-0>li:before{content:"\0025cf   "}.lst-kix_list_a-3>li:before{content:"\0025a0   "}.lst-kix_list_11-1>li:before{content:"\0025cb   "}.lst-kix_list_f-3>li:before{content:"\0025a0   "}.lst-kix_list_a-5>li:before{content:"\0025a0   "}.lst-kix_list_a-6>li:before{content:"\0025a0   "}.lst-kix_list_f-2>li:before{content:"\0025a0   "}.lst-kix_list_11-0>li:before{content:"" counter(lst-ctn-kix_list_11-0,decimal) ". "}.lst-kix_list_a-4>li:before{content:"\0025a0   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}.lst-kix_list_f-1>li:before{content:"\0025cb   "}.lst-kix_list_e-5>li:before{content:"\0025a0   "}ul.lst-kix_list_e-8{list-style-type:none}ul.lst-kix_list_e-7{list-style-type:none}ul.lst-kix_list_e-6{list-style-type:none}ul.lst-kix_list_e-5{list-style-type:none}.lst-kix_list_e-3>li:before{content:"\0025a0   "}.lst-kix_list_e-7>li:before{content:"\0025a0   "}.lst-kix_list_e-2>li:before{content:"\0025a0   "}.lst-kix_list_e-6>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-0{list-style-type:none}.lst-kix_list_4-7>li:before{content:"\0025a0   "}ul.lst-kix_list_e-4{list-style-type:none}ul.lst-kix_list_e-3{list-style-type:none}ul.lst-kix_list_e-2{list-style-type:none}.lst-kix_list_e-4>li:before{content:"\0025a0   "}ul.lst-kix_list_e-1{list-style-type:none}ul.lst-kix_list_4-8{list-style-type:none}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_4-0{list-style-type:none}.lst-kix_list_e-1>li:before{content:"\0025cb   "}ul.lst-kix_list_4-1{list-style-type:none}ul.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}.lst-kix_list_e-0>li:before{content:"\0025cf   "}ul.lst-kix_list_4-3{list-style-type:none}ol.lst-kix_list_11-0.start{counter-reset:lst-ctn-kix_list_11-0 0}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}.lst-kix_list_7-1>li:before{content:"\0025cb   "}.lst-kix_list_7-5>li:before{content:"\0025a0   "}.lst-kix_list_7-3>li:before{content:"\0025a0   "}ul.lst-kix_list_7-5{list-style-type:none}.lst-kix_list_10-0>li:before{content:"\0025cf   "}ul.lst-kix_list_7-6{list-style-type:none}.lst-kix_list_b-5>li:before{content:"\0025a0   "}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}.lst-kix_list_b-3>li:before{content:"\0025a0   "}.lst-kix_list_b-7>li:before{content:"\0025a0   "}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7>li:before{content:"\0025a0   "}.lst-kix_list_10-4>li:before{content:"\0025a0   "}.lst-kix_list_10-8>li:before{content:"\0025a0   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}.lst-kix_list_b-1>li:before{content:"\0025cb   "}.lst-kix_list_10-2>li:before{content:"\0025a0   "}.lst-kix_list_4-4>li:before{content:"\0025a0   "}.lst-kix_list_4-2>li:before{content:"\0025a0   "}.lst-kix_list_4-6>li:before{content:"\0025a0   "}.lst-kix_list_9-3>li:before{content:"\0025a0   "}.lst-kix_list_10-6>li:before{content:"\0025a0   "}.lst-kix_list_9-1>li:before{content:"\0025cb   "}.lst-kix_list_e-8>li:before{content:"\0025a0   "}.lst-kix_list_9-7>li:before{content:"\0025a0   "}.lst-kix_list_11-4>li:before{content:"\0025a0   "}.lst-kix_list_9-5>li:before{content:"\0025a0   "}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_11-6>li:before{content:"\0025a0   "}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_6-2{list-style-type:none}.lst-kix_list_11-8>li:before{content:"\0025a0   "}ul.lst-kix_list_6-3{list-style-type:none}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_6-0{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"\0025cf   "}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c17{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:78pt;border-top-color:#000000;border-bottom-style:solid}.c8{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:66.9pt;border-top-color:#000000;border-bottom-style:solid}.c2{color:#1b1c1d;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c4{color:#575b5f;font-weight:400;text-decoration:none;vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal}.c14{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c23{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c21{font-size:15pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c0{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c11{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c29{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c26{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c30{font-size:16pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c31{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c32{color:#1b1c1d;font-weight:700;font-size:12pt;font-family:"Google Sans Text";font-style:italic}.c19{color:#1b1c1d;font-weight:400;font-size:10pt;font-family:"Google Sans Text";font-style:italic}.c18{color:#1b1c1d;font-weight:700;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c27{font-size:10pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c16{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c35{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left;height:11pt}.c24{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans";font-style:normal}.c36{padding-top:12pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c34{padding-top:12.8pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c5{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c28{border-spacing:0;border-collapse:collapse;margin-right:auto}.c6{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c10{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c9{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c33{padding-top:24pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c20{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c3{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c38{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c7{margin-left:23.2pt;padding-left:0pt}.c25{padding:0;margin:0}.c1{color:inherit;text-decoration:inherit}.c12{text-decoration:none;vertical-align:baseline}.c15{margin-left:43.5pt;padding-left:0pt}.c13{vertical-align:baseline;font-style:normal}.c37{font-style:italic}.c22{height:0pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c38 doc-content"><h1 class="c10"><span class="c30">Molecular Markers and Their Sequences Associated with the High Oleic Acid Trait in Peanut (</span><span class="c31">Arachis hypogaea</span><span class="c30 c12">&nbsp;L.)</span></h1><h2 class="c10"><span class="c24 c12">1. Introduction to High Oleic Acid in Peanut</span></h2><p class="c5"><span class="c0">Peanut (</span><span class="c16">Arachis hypogaea</span><span class="c0">&nbsp;L.) stands as a globally significant oilseed crop, valued for its substantial protein and oil content.</span><span class="c14">1</span><span class="c0">&nbsp;Within its seeds, the fatty acid composition is a critical determinant of quality, with oleic acid (OA) being a particularly important monounsaturated omega-9 fatty acid.</span><span class="c14">3</span><span class="c0 c12">&nbsp;The presence of high oleic acid content in peanut oil confers numerous advantages, both for industrial applications and human health.</span></p><p class="c5"><span class="c0">From an industrial perspective, high oleic acid significantly enhances the oxidative stability of peanut oil, thereby extending its shelf life and improving flavor.</span><span class="c14">1</span><span class="c0">&nbsp;This superior stability is attributed to oleic acid&#39;s resistance to auto-oxidation, which is reported to be 10-fold higher than that of polyunsaturated linoleic acid (LA).</span><span class="c14">2</span><span class="c0">&nbsp;The oleic-to-linoleic acid (O/L) ratio serves as a key indicator of this quality. While normal peanut genotypes typically exhibit an O/L ratio ranging from 1.0 to 4.0, high O/L mutant lines can achieve ratios as high as 30 to 40.</span><span class="c14">5</span><span class="c0">&nbsp;An O/L ratio exceeding 9, or an oleic acid content above 70%, is generally considered industrially desirable.</span><span class="c4">8</span></p><p class="c5"><span class="c0">Beyond its industrial utility, high oleic acid offers notable health benefits. Consumption of oleic acid has been linked to improved human health outcomes, including the reduction of low-density lipoprotein (LDL) cholesterol while maintaining high-density lipoprotein (HDL) levels, anti-inflammatory effects, and a reduced risk of coronary heart disease, inflammatory diseases, obesity, type 2 diabetes, and hypertension.</span><span class="c14">2</span><span class="c0">&nbsp;These health advantages contribute to the growing consumer demand for healthier edible oils. Consequently, increasing oleic acid levels in peanut has become a central objective in breeding programs, driven by both market demands for high-quality vegetable oil and public health considerations.</span><span class="c14">1</span><span class="c0">&nbsp;The economic implications are substantial, with a mere 1% increase in peanut oil content potentially leading to a 7% surge in oil processing revenue.</span><span class="c14">4</span><span class="c0 c12">&nbsp;This underscores that the scientific pursuit of molecular markers and gene editing is not purely academic but is fundamentally driven by significant economic and public health imperatives, highlighting the translational value of this research.</span></p><p class="c5"><span class="c0">Peanut seeds typically contain approximately 50% oil, with oleic and linoleic acids constituting about 80% of the total fatty acids.</span><span class="c14">5</span><span class="c0">&nbsp;The biosynthesis of fatty acids involves a crucial step where oleic acid (C18:1) is converted to linoleic acid (C18:2). This conversion is catalyzed by &Delta;12 fatty acid desaturase (FAD2) enzymes, also known as oleate dehydrogenase, which introduce a second double bond at the C12 position of oleic acid.</span><span class="c14">5</span><span class="c0">&nbsp;Mutations within the </span><span class="c16">FAD2</span><span class="c0">&nbsp;gene lead to a decrease in the activity of this enzyme, thereby impeding the synthesis of linoleic acid and resulting in an accumulation of oleic acid in peanut seeds.</span><span class="c4">6</span></p><p class="c5"><span class="c0">The high oleic acid trait in cultivated peanut is predominantly controlled by recessive alleles of two homeologous genes: </span><span class="c16">ahFAD2A</span><span class="c0">&nbsp;(also referred to as </span><span class="c16">ol1</span><span class="c0">) and </span><span class="c16">ahFAD2B</span><span class="c0">&nbsp;(also referred to as </span><span class="c16">ol2</span><span class="c0">).</span><span class="c14">5</span><span class="c0">&nbsp;These two genes share a high degree of sequence homology, approximately 99% in their coding regions.</span><span class="c14">8</span><span class="c0">&nbsp;Achieving the high oleic acid phenotype typically requires loss-of-function mutations or a significant reduction in the transcript levels of both </span><span class="c16">ahFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">ahFAD2B</span><span class="c0">.</span><span class="c14">5</span><span class="c0">&nbsp;This necessity for dual gene inactivation explains why high oleic acid peanuts are often characterized as &quot;double mutants&quot;.</span><span class="c4">5</span></p><h2 class="c10"><span class="c24">2. Genetic Basis: The </span><span class="c21">AhFAD2</span><span class="c24 c12">&nbsp;Genes</span></h2><h3 class="c10"><span class="c29">2.1. Origin and Structure of </span><span class="c32">AhFAD2</span><span class="c29 c12">&nbsp;Genes</span></h3><p class="c5"><span class="c0">Cultivated peanut, </span><span class="c16">Arachis hypogaea</span><span class="c0">&nbsp;L., is an allotetraploid species with a genome designated as AABB (2n = 4x = 40).</span><span class="c14">9</span><span class="c0">&nbsp;This genetic constitution is a result of a natural hybridization event between two wild diploid progenitor species: </span><span class="c16">Arachis duranensis</span><span class="c0">, which contributed the A genome, and </span><span class="c16">Arachis ipaensis</span><span class="c0">, which contributed the B genome.</span><span class="c14">9</span><span class="c0 c12">&nbsp;The evolutionary history of peanut as an allotetraploid explains the presence of duplicated genes, known as homeologs, derived from these ancestral genomes.</span></p><p class="c5"><span class="c0">Among these homeologous genes are </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">, which are derived from the A and B subgenomes, respectively.</span><span class="c14">12</span><span class="c0">&nbsp;These two genes exhibit remarkable sequence similarity, sharing approximately 99% homology in their coding regions, with only about 11 base pair differences.</span><span class="c14">8</span><span class="c0">&nbsp;Their coding sequences (CDS) span 1140 base pairs, encoding 379 amino acids, and notably, they lack introns in their coding regions.</span><span class="c14">8</span><span class="c0 c12">&nbsp;This high degree of similarity underscores their shared evolutionary origin and highly conserved functional roles in fatty acid metabolism.</span></p><h3 class="c10"><span class="c29 c12">2.2. Functional Role in Oleic Acid to Linoleic Acid Conversion</span></h3><p class="c5"><span class="c0">The primary function of </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;is to encode microsomal oleoyl-PC desaturase enzymes.</span><span class="c14">9</span><span class="c0">&nbsp;These enzymes are integral membrane proteins located within the endoplasmic reticulum of plant cells.</span><span class="c14">9</span><span class="c0">&nbsp;Their critical role in fatty acid biosynthesis involves catalyzing the desaturation of oleic acid (C18:1) to linoleic acid (C18:2) by introducing a second double bond.</span><span class="c4">3</span></p><p class="c5"><span class="c0">In wild-type or &quot;normal&quot; peanut genotypes, where both </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;genes are fully functional, the efficient conversion of oleic acid to linoleic acid is maintained, resulting in a typical O/L ratio ranging from 1.0 to 4.0.</span><span class="c14">5</span><span class="c0">&nbsp;However, when mutations occur in these genes, they lead to altered gene expression and/or a reduction in the enzymatic activity of the oleoyl-PC desaturase.</span><span class="c14">3</span><span class="c0">&nbsp;This diminished enzymatic function impedes the conversion of oleic acid, causing it to accumulate and consequently increasing the O/L ratio, thereby conferring the high oleic acid phenotype.</span><span class="c4">3</span></p><p class="c5"><span class="c0">The necessity for inactivating both </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;genes to produce high O/L genotypes </span><span class="c14">5</span><span class="c0 c12">&nbsp;highlights a critical aspect of peanut genetics: functional redundancy. Given their high sequence homology, if only one gene were mutated, the other functional copy would likely compensate for the loss of enzyme activity, preventing the significant accumulation of oleic acid. This inherent genetic redundancy explains why the high oleic trait is a &quot;double mutant&quot; phenomenon, requiring breeders to target both homeologous genes simultaneously for complete inactivation and the desired high oleic acid accumulation.</span></p><h3 class="c10"><span class="c29 c12">2.3. Chromosomal Locations</span></h3><p class="c10"><span class="c0">The precise chromosomal localization of the </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;genes is fundamental for molecular breeding efforts, enabling targeted genetic manipulation.</span></p><ul class="c25 lst-kix_list_1-0 start"><li class="c10 c7 li-bullet-0"><span class="c16">AhFAD2A</span><span class="c0">&nbsp;has been consistently mapped to the A sub-genome, specifically on linkage group a09 (chromosome A09).</span><span class="c4">20</span></li><li class="c6 c7 li-bullet-0"><span class="c16">AhFAD2B</span><span class="c0">&nbsp;is located on the B sub-genome, specifically on linkage group b09 (chromosome B09).</span><span class="c4">20</span></li></ul><p class="c20"><span class="c0">Both genes are therefore situated on chromosome 9 of their respective subgenomes.</span><span class="c14">24</span><span class="c0">&nbsp;For instance, a novel and stable quantitative trait locus (QTL) for oil content, designated qOCB09.1, was mapped to chromosome B09. This QTL region spans approximately 1.99 Mb and includes the </span><span class="c16">FAD2B</span><span class="c0">&nbsp;gene, indicating its influence on oil content and fatty acid composition.</span><span class="c14">26</span><span class="c0 c12">&nbsp;This precise mapping provides invaluable genetic targets for breeders and molecular biologists, streamlining the process of marker development and gene editing efforts.</span></p><h2 class="c10"><span class="c24 c12">3. Specific Molecular Mutations Conferring High Oleic Acid</span></h2><p class="c5"><span class="c0">The high oleic acid trait in peanut is a direct consequence of specific loss-of-function mutations in the </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0 c12">&nbsp;genes. These mutations either reduce or completely eliminate the activity of the oleoyl-PC desaturase enzyme, thereby preventing the conversion of oleic acid to linoleic acid.</span></p><h3 class="c10"><span class="c29">3.1. Mutations in </span><span class="c32 c12">AhFAD2A</span></h3><p class="c5"><span class="c0">The most widely reported and characterized mutation in </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;(ol1) is a </span><span class="c11">single nucleotide substitution of Guanine (G) to Adenine (A) at the 448 base pair (bp) position</span><span class="c0">&nbsp;after the start codon (G448A).</span><span class="c14">7</span><span class="c0">&nbsp;This specific G448A transition leads to a </span><span class="c11">missense amino acid change</span><span class="c0">, where Aspartic acid (D) is replaced by Asparagine (N) at position 150 (D150N).</span><span class="c14">8</span><span class="c0">&nbsp;This amino acid alteration is critical as it impacts the enzyme&#39;s structural integrity and catalytic efficiency, leading to reduced desaturase activity.</span><span class="c14">33</span><span class="c0">&nbsp;This mutation is commonly found in many natural high O/L lines.</span><span class="c4">29</span></p><h3 class="c10"><span class="c29">3.2. Mutations in </span><span class="c12 c32">AhFAD2B</span></h3><p class="c10"><span class="c0">The </span><span class="c16">AhFAD2B</span><span class="c0 c12">&nbsp;gene (ol2) exhibits a greater diversity of mutations that confer the high oleic acid phenotype, indicating multiple origins for this desirable trait.</span></p><ul class="c25 lst-kix_list_2-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Natural Mutation (F435 type):</span><span class="c0">&nbsp;The most prevalent natural mutation identified in </span><span class="c16">AhFAD2B</span><span class="c0">, particularly in the widely utilized high O/L line F435, is a </span><span class="c11">single nucleotide &#39;A&#39; insertion between base pairs 441 and 442 (441_442insA)</span><span class="c0">.</span><span class="c14">3</span><span class="c0">&nbsp;This insertion causes a </span><span class="c11">frameshift mutation</span><span class="c0">&nbsp;in the coding sequence.</span><span class="c14">3</span><span class="c0">&nbsp;The frameshift subsequently leads to the generation of a </span><span class="c11">premature stop codon</span><span class="c0">, resulting in the production of a truncated, non-functional protein.</span><span class="c14">3</span><span class="c0">&nbsp;This type of mutation is highly effective in causing a complete loss of enzyme function, which is why the F435 line and its derivatives (e.g., &#39;Georgia-02C&#39;, &#39;Georgia Hi-O/L&#39;) are extensively used as donor parents in breeding programs.</span><span class="c4">7</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Other Reported Mutations:</span><span class="c0">&nbsp;Beyond the F435 type, several other mutations in </span><span class="c16">AhFAD2B</span><span class="c0 c12">&nbsp;have been identified, highlighting the diverse genetic origins of the high oleic trait:</span></li></ul><ul class="c25 lst-kix_list_3-1 start"><li class="c10 c15 li-bullet-0"><span class="c0">A </span><span class="c11">transposable element insertion</span><span class="c0">&nbsp;has been characterized in the </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;gene of the cultivar &#39;Flavorunner 458&#39;.</span><span class="c4">7</span></li><li class="c6 c15 li-bullet-0"><span class="c0">A novel </span><span class="c11">Guanine (G) to Thymine (T) substitution at position 451 (G451T)</span><span class="c0">&nbsp;has been identified in </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;through CRISPR/Cas9 gene editing.</span><span class="c14">16</span><span class="c0">&nbsp;This mutation was observed independently and not in conjunction with the 441_442insA mutation, suggesting it represents a distinct mechanism for loss of function.</span><span class="c4">16</span></li><li class="c6 c15 li-bullet-0"><span class="c0">A </span><span class="c11">Cytosine (C) to Guanine (G) substitution at position 301 (C301G)</span><span class="c0">&nbsp;in </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;was found in natural mutant lines such as PI342666. This mutation results in a </span><span class="c11">histidine (H) to aspartic acid (D) amino acid change at position 101 (H101D)</span><span class="c0">.</span><span class="c4">8</span></li><li class="c6 c15 li-bullet-0"><span class="c0">Chemically induced mutations of </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;in lines M2-225 and C458 involved </span><span class="c11">miniature inverted transposable element (MITE) insertions</span><span class="c0">&nbsp;at 997 bp and 665 bp, respectively, leading to the loss of </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;function.</span><span class="c4">24</span></li></ul><p class="c20"><span class="c0">The identification of multiple distinct </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;mutations (e.g., 441_442insA, transposable element insertion, G451T, C301G, MITE insertions) that all lead to the high oleic phenotype demonstrates that the trait can arise from various genetic origins. This genetic diversity is crucial for broadening the genetic base of high oleic cultivars. Given the often narrow genetic background of many existing high-oleic donor parents </span><span class="c14">24</span><span class="c0">, breeders can leverage these different mutations, or induce new ones via gene editing technologies like CRISPR/Cas9 </span><span class="c14">16</span><span class="c0">, to introduce the high oleic trait into a wider range of elite cultivars without necessarily inheriting undesirable linked traits from a limited donor pool. The observation that natural mutations appear more frequently in the </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;gene than in the </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;gene </span><span class="c14">16</span><span class="c0 c12">&nbsp;also provides valuable information that can guide targeted gene editing strategies.</span></p><p class="c5"><span class="c11">Table 1: Summary of Key </span><span class="c16">AhFAD2</span><span class="c11 c12">&nbsp;Mutations and Their Molecular Effects</span></p><table class="c28"><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Gene</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Mutation Type</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Specific Nucleotide Change/Location</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Amino Acid Change</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Effect on Protein/Gene Product</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Origin/Source (Examples)</span></p></td></tr><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c19 c12">AhFAD2A</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Substitution</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">G448A (G to A at 448 bp)</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">D150N</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Missense change; Reduced enzyme activity</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">F435, Many natural high O/L lines</span></p></td></tr><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c12 c19">AhFAD2B</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Insertion</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">441_442insA (A insertion between bp 441 and 442)</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Frameshift</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Frameshift; Premature stop codon; Truncated, non-functional protein; Loss of enzyme activity</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">F435, &#39;Georgia-02C&#39;, &#39;Georgia Hi-O/L&#39;</span></p></td></tr><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c19 c12">AhFAD2B</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Transposable Element Insertion</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified (e.g., at 997 bp, 665 bp)</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Loss of function</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">&#39;Flavorunner 458&#39;, M2-225, C458</span></p></td></tr><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c19 c12">AhFAD2B</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Substitution</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">G451T (G to T at 451 bp)</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Altered protein sequence; Reduced/Loss of enzyme activity</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">CRISPR/Cas9 induced</span></p></td></tr><tr class="c22"><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c19 c12">AhFAD2B</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Substitution</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">C301G (C to G at 301 bp)</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">H101D</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">Missense change; Reduced/Loss of enzyme activity</span></p></td><td class="c17" colspan="1" rowspan="1"><p class="c6"><span class="c2">PI342666 (natural mutant)</span></p></td></tr></table><h2 class="c33"><span class="c12 c24">4. Molecular Markers for High Oleic Acid Trait Detection</span></h2><p class="c5"><span class="c0">Molecular markers are indispensable tools in modern peanut breeding, enabling the rapid and efficient selection for the high oleic acid trait by circumventing the labor-intensive and time-consuming process of traditional fatty acid composition analyses (phenotyping).</span><span class="c14">3</span><span class="c0">&nbsp;These markers accelerate breeding cycles by allowing early identification and elimination of undesirable lines, and facilitating the introgression of desired traits.</span><span class="c4">3</span></p><h3 class="c10"><span class="c29 c12">4.1. Cleaved Amplified Polymorphic Sequence (CAPS) Markers</span></h3><p class="c5"><span class="c0">CAPS markers are PCR-based molecular markers that detect genetic polymorphisms by exploiting the presence or absence of a restriction endonuclease recognition site.</span><span class="c14">30</span><span class="c0">&nbsp;The principle involves PCR amplification of a specific genomic region, followed by digestion of the PCR product with a chosen restriction enzyme. Differences in fragment sizes after digestion, visualized via gel electrophoresis, indicate the presence of specific alleles.</span><span class="c14">30</span><span class="c0">&nbsp;CAPS markers are a robust and preferred choice for routine screening of a large number of samples in breeding programs.</span><span class="c14">36</span><span class="c0">&nbsp;They have been successfully developed and applied for detecting allele polymorphisms in both </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">.</span><span class="c4">8</span></p><p class="c5"><span class="c0">For the </span><span class="c11">441_442insA mutation in </span><span class="c16">AhFAD2B</span><span class="c0">, a specific CAPS marker has been designed. This marker targets the insertion site </span><span class="c14">7</span><span class="c0">&nbsp;and utilizes the restriction enzyme Hpy188I.</span><span class="c14">15</span><span class="c0">&nbsp;The Hpy188I enzyme recognizes the sequence 5&#39;-TCNGA-3&#39;.</span><span class="c14">30</span><span class="c0">&nbsp;The 441_442insA insertion in the mutant </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;allele specifically creates a &quot;TCAGA&quot; site, which is then recognized and cleaved by Hpy188I, allowing for differentiation from the wild-type allele.</span><span class="c4">30</span></p><p class="c10"><span class="c0 c12">The primer sequences for amplifying the region containing this mutation are:</span></p><ul class="c25 lst-kix_list_4-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Forward primer (1344):</span><span class="c0">&nbsp;5&prime;-ggagctttaacaacacaa-3&prime; </span><span class="c4">15</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Reverse primer (1345):</span><span class="c0">&nbsp;5&prime;-atatgggagcataagggt-3&prime; </span><span class="c4">15</span></li></ul><p class="c20"><span class="c0">When these primers are used, the PCR product amplified from the </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;mutant allele (containing 441_442insA) is digested by Hpy188I into two fragments of </span><span class="c11">521 bp and 137 bp</span><span class="c0">. The wild-type allele&#39;s amplicon remains undigested.</span><span class="c4">15</span></p><p class="c10"><span class="c0">An alternative primer pair, bF19/R1(FAD), is also available for the </span><span class="c16">AhFAD2B</span><span class="c0 c12">&nbsp;441_442insA mutation:</span></p><ul class="c25 lst-kix_list_5-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Forward primer (bF19):</span><span class="c0">&nbsp;5&prime;-cagaaccattagctttg-3&prime; </span><span class="c4">15</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Reverse primer (R1/FAD):</span><span class="c0">&nbsp;5&prime;-ctctgactatgcatcag-3&prime; </span><span class="c14">15</span><span class="c0">&nbsp;Using these primers, digestion of the mutant allele amplicon with Hpy188I yields fragments of </span><span class="c11">505, 263, 213, 171, 32, and 12 bp</span><span class="c0">. The wild-type allele amplicon yields fragments of </span><span class="c11">736, 263, 171, 32, and 12 bp</span><span class="c0">.</span><span class="c14">15</span><span class="c0">&nbsp;These bF19/R1(FAD) primers are </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;specific and can be used to detect heterozygosity, which is valuable in breeding programs.</span><span class="c4">15</span></li></ul><p class="c20"><span class="c0">For the </span><span class="c11">G448A mutation in </span><span class="c16">AhFAD2A</span><span class="c0">, the restriction enzyme Hpy99I is used.</span><span class="c14">15</span><span class="c0">&nbsp;The wild-type </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;gene contains a Hpy99I recognition site (5&#39;-CGWCG-3&#39;), which, upon digestion, yields 598 bp and 228 bp fragments. In contrast, the </span><span class="c16">Ahfad2a</span><span class="c0">&nbsp;mutant allele (G448A) lacks this recognition site, and its PCR product remains undigested.</span><span class="c4">30</span></p><p class="c5"><span class="c0">The precise design of CAPS markers leverages the direct causal link between specific nucleotide changes (such as the &#39;A&#39; insertion or G448A substitution) and the presence or absence of a restriction enzyme recognition site. This molecular interaction enables the marker to differentiate between wild-type and mutant alleles. The continued utility and development of CAPS markers, even with the advent of newer technologies, highlight their reliability and cost-effectiveness for certain breeding applications, particularly for laboratories with limited access to advanced equipment like qPCR machines.</span><span class="c14">34</span><span class="c0 c12">&nbsp;This demonstrates the importance of a diverse toolkit of markers to suit varying resource levels and throughput requirements.</span></p><h3 class="c10"><span class="c29 c12">4.2. Kompetitive Allele Specific PCR (KASP) Markers</span></h3><p class="c5"><span class="c0">KASP technology represents a significant advancement in molecular breeding, offering high accuracy, time-saving capabilities, high-throughput genotyping, and a relatively low cost per data point.</span><span class="c14">22</span><span class="c0">&nbsp;A key advantage of KASP is its ability to detect both alleles in a single reaction, making it exceptionally well-suited for large-scale screening in peanut breeding programs.</span><span class="c14">22</span><span class="c0">&nbsp;The reliability of the KASP method has been rigorously validated through comparisons with CAPS analysis, demonstrating consistent results.</span><span class="c4">22</span></p><p class="c5"><span class="c0">KASP assays operate on the principle of competitive binding. Two allele-specific forward primers are designed, each carrying a unique tail sequence that corresponds to a universal FRET (fluorescence resonant energy transfer) cassette. One primer is typically labeled with a FAM&trade; dye (designed for the mutant allele), and the other with a HEX&trade; dye (designed for the wild-type allele). The differential binding and subsequent fluorescent signaling allow for clear discrimination of homozygous wild-type, homozygous mutant, and heterozygous genotypes.</span><span class="c4">30</span></p><p class="c10"><span class="c0">The specific KASP primer sequences for </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;are as follows </span><span class="c14">30</span><span class="c0 c12">:</span></p><ul class="c25 lst-kix_list_6-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">For </span><span class="c16">AhFAD2A</span><span class="c11 c12">&nbsp;(targeting G448A mutation):</span></li></ul><ul class="c25 lst-kix_list_7-1 start"><li class="c10 c15 li-bullet-0"><span class="c11">AhFAD2A-Allele-l (mutant allele, FAM&trade; dye):</span><span class="c0 c12">&nbsp;5&prime;-GTTTTGGGACAAACACTTCGTT-3&prime;</span></li><li class="c6 c15 li-bullet-0"><span class="c11">AhFAD2A-Allele-2 (wild-type allele, HEX&trade; dye):</span><span class="c0 c12">&nbsp;5&prime;-GTTTTGGGACAAACACTTCGTC-3&prime;</span></li><li class="c6 c15 li-bullet-0"><span class="c11">AhFAD2A-common (reverse primer):</span><span class="c0 c12">&nbsp;5&prime;-CGCCACCACTCCAACACC-3&prime;</span></li></ul><ul class="c25 lst-kix_list_6-0"><li class="c6 c7 li-bullet-0"><span class="c11">For </span><span class="c16">AhFAD2B</span><span class="c11 c12">&nbsp;(targeting 441_442insA mutation):</span></li></ul><ul class="c25 lst-kix_list_8-1 start"><li class="c10 c15 li-bullet-0"><span class="c11">AhFAD2B-Allele-l (mutant allele, FAM&trade; dye):</span><span class="c0 c12">&nbsp;5&prime;-CAAACACTTCGTCGCGGTCT-3&prime;</span></li><li class="c6 c15 li-bullet-0"><span class="c11">AhFAD2B-Allele-2 (wild-type allele, HEX&trade; dye):</span><span class="c0 c12">&nbsp;5&prime;-CAAACACTTCGTCGCGGTCG-3&prime;</span></li><li class="c6 c15 li-bullet-0"><span class="c11">AhFAD2B-common (reverse primer):</span><span class="c0 c12">&nbsp;5&prime;-CCGCCACCACTCCAACACA-3&prime;</span></li></ul><ul class="c25 lst-kix_list_6-0"><li class="c6 c7 li-bullet-0"><span class="c11 c12">Universal FRET Tail Sequences:</span></li></ul><ul class="c25 lst-kix_list_9-1 start"><li class="c10 c15 li-bullet-0"><span class="c11">Allele-l Tail (FAM tail):</span><span class="c0 c12">&nbsp;5&prime;-GAAGGTGACCAAGTTCATGCT-3&prime;</span></li><li class="c6 c15 li-bullet-0"><span class="c11">Allele-2 Tail (HEX tail):</span><span class="c0 c12">&nbsp;5&prime;-GAAGGTCGGAGTCAACGGATT-3&prime;</span></li></ul><p class="c20"><span class="c0">A notable feature of the KASP primer design for peanut is the strategic placement of the common reverse primer&#39;s 3&#39; end at an SNP position (Cytosine in </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and Adenine in </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;at position 432 from the 5&#39; end of the open reading frame). This allows for discrimination between the A and B subgenomes, effectively handling the homeologous nature of the genes and ensuring accurate and rapid genotype determination.</span><span class="c4">30</span></p><p class="c5"><span class="c0">The widespread adoption and successful application of KASP markers signify a broader trend towards more efficient, high-throughput, and cost-effective marker-assisted breeding strategies in peanut.</span><span class="c14">2</span><span class="c0">&nbsp;This technology is instrumental in accelerating the development and commercial release of new high oleic cultivars, as evidenced by its use in major peanut-producing regions like China and India.</span><span class="c14">2</span><span class="c0 c12">&nbsp;The development of KASP markers represents a significant leap, enabling faster and more economical screening of large breeding populations, which directly translates to accelerated breeding cycles and more rapid development of improved high oleic cultivars to meet the growing demand for healthier edible oils.</span></p><h3 class="c10"><span class="c29 c12">4.3. Other Marker Technologies</span></h3><p class="c10"><span class="c0">In addition to CAPS and KASP, other PCR-based marker technologies have been developed and utilized for </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;mutation detection:</span></p><ul class="c25 lst-kix_list_a-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Allele-Specific PCR (AS-PCR):</span><span class="c0">&nbsp;This method uses primers designed to specifically amplify only one allele (wild-type or mutant) based on a single nucleotide difference at the 3&#39; end of the primer.</span><span class="c14">8</span><span class="c0">&nbsp;It has been used to confirm true hybrids.</span><span class="c4">1</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Real-time PCR (qRT-PCR):</span><span class="c0">&nbsp;This quantitative method allows for the rapid and precise identification of both wild-type and mutant alleles by measuring DNA amplification in real-time.</span><span class="c14">3</span><span class="c0">&nbsp;It is particularly effective for the early elimination of undesirable non-high oleic acid lines in segregating populations, thereby streamlining breeding efforts.</span><span class="c4">3</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Simple Sequence Repeats (SSR) Markers:</span><span class="c0">&nbsp;While not specific to high oleic acid, SSR markers are broadly used for evaluating molecular diversity in groundnut genotypes.</span><span class="c4">1</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Single Nucleotide Polymorphism (SNP) Markers:</span><span class="c0">&nbsp;KASP is based on SNP markers.</span><span class="c14">37</span><span class="c0">&nbsp;Genome-Wide Association Studies (GWAS) have identified numerous SNPs associated with seed quality traits, including loci near </span><span class="c16">FAD2A</span><span class="c0">&nbsp;and </span><span class="c16">FAD2B</span><span class="c0">.</span><span class="c14">4</span><span class="c0">&nbsp;The development of SNP array chips is essential for high-throughput genotyping in cultivated peanuts, particularly given their large genome size and limited genetic diversity.</span><span class="c4">4</span></li></ul><p class="c20"><span class="c0">The availability of multiple PCR-based marker types provides flexibility for breeding programs, allowing them to choose the most suitable method based on their specific needs, resources, and desired throughput. The co-existence and continued use of various marker technologies imply that each method possesses unique strengths that make it optimal for different stages of a breeding program or for laboratories with varying levels of technological infrastructure. For instance, real-time PCR is valuable for early, rapid screening, while KASP excels in high-throughput applications.</span><span class="c14">3</span><span class="c0 c12">&nbsp;This indicates that breeders employ a strategic approach, selecting the marker technology that best fits the specific objective and scale of their genotyping efforts at different points in the breeding cycle.</span></p><p class="c5"><span class="c11 c12">Table 2: Molecular Markers for High Oleic Acid Trait in Peanut Breeding</span></p><p class="c35"><span class="c11 c12"></span></p><table class="c28"><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Marker Type</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Target Gene &amp; Mutation</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Primer Name/ID</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c12 c18">Forward Primer Sequence (5&#39;-3&#39;)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Reverse Primer Sequence (5&#39;-3&#39;)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Additional Components/Notes</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c18 c12">Expected Results (for CAPS)</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">CAPS</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2B</span><span class="c2">&nbsp;441_442insA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">1344/1345</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-ggagctttaacaacacaa-3&prime; </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-atatgggagcataagggt-3&prime; </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Restriction Enzyme: Hpy188I </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Mutant: 521 bp, 137 bp fragments; Wild-type: Undigested </span><span class="c4">15</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">CAPS</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2B</span><span class="c2">&nbsp;441_442insA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">bF19/R1(FAD)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-cagaaccattagctttg-3&prime; </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-ctctgactatgcatcag-3&prime; </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Restriction Enzyme: Hpy188I </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Mutant: 505, 263, 213, 171, 32, 12 bp fragments; Wild-type: 736, 263, 171, 32, 12 bp fragments </span><span class="c4">15</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">CAPS</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2A</span><span class="c2">&nbsp;G448A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Not specified</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Restriction Enzyme: Hpy99I </span><span class="c4">15</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">Wild-type: 598 bp, 228 bp fragments; Mutant: Undigested </span><span class="c4">30</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">KASP</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2A</span><span class="c2">&nbsp;G448A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">AhFAD2A-Allele-1 (mutant)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-GTTTTGGGACAAACACTTCGTT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CGCCACCACTCCAACACC-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">FAM&trade; dye, Allele-1 Tail: 5&prime;-GAAGGTGACCAAGTTCATGCT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Fluorescent signal (FAM) for mutant allele</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">KASP</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2A</span><span class="c2">&nbsp;G448A</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">AhFAD2A-Allele-2 (wild-type)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-GTTTTGGGACAAACACTTCGTC-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CGCCACCACTCCAACACC-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">HEX&trade; dye, Allele-2 Tail: 5&prime;-GAAGGTCGGAGTCAACGGATT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Fluorescent signal (HEX) for wild-type allele</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">KASP</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2B</span><span class="c2">&nbsp;441_442insA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">AhFAD2B-Allele-1 (mutant)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CAAACACTTCGTCGCGGTCT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CCGCCACCACTCCAACACA-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">FAM&trade; dye, Allele-1 Tail: 5&prime;-GAAGGTGACCAAGTTCATGCT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Fluorescent signal (FAM) for mutant allele</span></p></td></tr><tr class="c22"><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">KASP</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c19">AhFAD2B</span><span class="c2">&nbsp;441_442insA</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">AhFAD2B-Allele-2 (wild-type)</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CAAACACTTCGTCGCGGTCG-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">5&prime;-CCGCCACCACTCCAACACA-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c27">HEX&trade; dye, Allele-2 Tail: 5&prime;-GAAGGTCGGAGTCAACGGATT-3&prime; </span><span class="c4">30</span></p></td><td class="c8" colspan="1" rowspan="1"><p class="c6"><span class="c2">Fluorescent signal (HEX) for wild-type allele</span></p></td></tr></table><h2 class="c10"><span class="c24 c12">5. Gene Sequence Information and Public Accessions</span></h2><p class="c5"><span class="c0 c12">Access to publicly available genomic and gene sequence databases is paramount for molecular breeding, enabling researchers to elucidate the genetic underpinnings of traits, design novel molecular markers, and develop advanced gene-editing strategies.</span></p><h3 class="c10"><span class="c29">5.1. </span><span class="c32">Arachis hypogaea</span><span class="c29 c12">&nbsp;Genome Assemblies</span></h3><p class="c5"><span class="c0">The NCBI Assembly database serves as a comprehensive repository for numerous genome assemblies of </span><span class="c16">Arachis hypogaea</span><span class="c0">&nbsp;(peanut), encompassing both complete genome and chromosome-level assemblies.</span><span class="c14">38</span><span class="c0">&nbsp;Examples of GenBank assembly accessions include GCA_039854485.1, GCA_039853315.1, GCA_003086295.3 (for the cultivar Tifrunner), and GCA_028451205.1 (for the cultivar BaileyII).</span><span class="c14">38</span><span class="c0">&nbsp;These assemblies provide the essential genomic context required for identifying and characterizing genes such as </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">.</span><span class="c14">4</span><span class="c0">&nbsp;For instance, the Tifrunner v1.0 assembly (PI 644011) was constructed using a combination of PacBio long reads and Illumina HiSeq reads, scaffolded with Hi-C technology, and spans approximately 2.54 Gb.</span><span class="c4">21</span></p><p class="c5"><span class="c0">Beyond NCBI, PeanutBase (dev.peanutbase.org) functions as a valuable specialized resource, offering extensive genomic data, gene search tools, and trait-specific information pertinent to </span><span class="c16">Arachis</span><span class="c0">&nbsp;species.</span><span class="c14">5</span><span class="c0">&nbsp;The presence of these numerous genome assemblies and specific gene accessions on public platforms signifies a robust public data infrastructure. This accessibility is crucial for researchers to decode the genetic makeup of peanut </span><span class="c14">4</span><span class="c0">&nbsp;and to design new markers or gene-editing strategies, such as sgRNAs for CRISPR/Cas9.</span><span class="c4">10</span></p><h3 class="c10"><span class="c29">5.2. Accession Numbers for </span><span class="c32">AhFAD2</span><span class="c29 c12">&nbsp;Genes</span></h3><p class="c10"><span class="c0">The </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;genes and their corresponding protein products are extensively characterized and publicly available in databases like NCBI and UniProt.</span></p><ul class="c25 lst-kix_list_b-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">For </span><span class="c16">AhFAD2A</span><span class="c11 c12">&nbsp;(Ol1):</span></li></ul><ul class="c25 lst-kix_list_c-1 start"><li class="c10 c15 li-bullet-0"><span class="c11">Protein Accession:</span><span class="c0">&nbsp;AAB84262.</span><span class="c14">9</span><span class="c0">&nbsp;UniProtKB also lists C8CK29 for Oleate desaturase (FAD2A) in </span><span class="c16">Arachis hypogaea</span><span class="c0">.</span><span class="c4">39</span></li><li class="c6 c15 li-bullet-0"><span class="c11">Nucleotide Accession:</span><span class="c0">&nbsp;GQ412349.</span><span class="c14">39</span><span class="c0">&nbsp;Other related accessions mentioned for </span><span class="c16">AhFAD2</span><span class="c0">&nbsp;(which would include </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;homologs) are AF030319.1 (omega-6 desaturase, FadS) </span><span class="c14">40</span><span class="c0">, DQ019933.1 (delta-12 fatty acid desaturase, Ah1FAD2) </span><span class="c14">40</span><span class="c0">, and AF248739.1 (delta-12 fatty acid desaturase, Ah2FAD2).</span><span class="c4">40</span></li></ul><ul class="c25 lst-kix_list_b-0"><li class="c6 c7 li-bullet-0"><span class="c11">For </span><span class="c16">AhFAD2B</span><span class="c11 c12">&nbsp;(Ol2):</span></li></ul><ul class="c25 lst-kix_list_d-1 start"><li class="c10 c15 li-bullet-0"><span class="c11">Protein Accession:</span><span class="c0">&nbsp;AAF82293.</span><span class="c4">9</span></li><li class="c6 c15 li-bullet-0"><span class="c11">Nucleotide Accession:</span><span class="c0">&nbsp;AF272950.</span><span class="c4">34</span></li></ul><p class="c20"><span class="c0">It is important to note that direct access to some specific NCBI Nucleotide URLs was reported as inaccessible during the research phase. Users may need to navigate the NCBI Nucleotide database </span><span class="c14">42</span><span class="c0 c12">&nbsp;using the provided accession numbers to retrieve full sequence data.</span></p><p class="c5"><span class="c0 c12">The accessibility of publicly available genomic and gene sequence data is foundational to accelerating research and breeding efforts. These resources reduce the need for redundant sequencing across different projects, foster global collaboration, and facilitate the rapid translation of basic genetic discoveries into practical applications for crop improvement. This open access to data is a cornerstone of modern agricultural biotechnology.</span></p><h2 class="c10"><span class="c24 c12">6. Application in Peanut Breeding Programs</span></h2><p class="c5"><span class="c0">The precise identification of specific </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;mutations and the subsequent development of robust molecular markers have profoundly transformed peanut breeding strategies for enhancing oleic acid content.</span></p><h3 class="c10"><span class="c12 c29">6.1. Marker-Assisted Selection (MAS) and Marker-Assisted Backcrossing (MABC)</span></h3><p class="c5"><span class="c0">Molecular markers are routinely integrated into marker-assisted selection (MAS) protocols to screen for known </span><span class="c16">ahFAD2</span><span class="c0">&nbsp;mutations.</span><span class="c14">7</span><span class="c0">&nbsp;This approach significantly expedites breeding cycles by enabling the rapid identification of progeny carrying the desired high oleic acid alleles at early developmental stages, allowing for the efficient elimination of undesirable non-high oleic acid lines from segregating populations.</span><span class="c4">3</span></p><p class="c5"><span class="c0">Marker-Assisted Backcrossing (MABC) is a particularly crucial strategy employed to introgress the high oleic trait from donor parents into elite, yet low-oleic, recipient cultivars.</span><span class="c14">1</span><span class="c0">&nbsp;This method is designed to ensure the recovery of the recurrent parent&#39;s desirable agronomic traits, such as high pod yield, optimal seed size, and disease resistance, while simultaneously incorporating the high oleic characteristic.</span><span class="c14">1</span><span class="c0 c12">&nbsp;This approach addresses the challenge of combining specific quality traits with overall agronomic performance.</span></p><p class="c10"><span class="c0 c12">Successful implementations of these molecular breeding strategies are evident in various peanut improvement programs:</span></p><ul class="c25 lst-kix_list_e-0 start"><li class="c10 c7 li-bullet-0"><span class="c0">High oleic lines have been successfully developed through crosses between elite cultivars (e.g., NC7) and advanced high oleic lines (HOG).</span><span class="c4">1</span></li><li class="c6 c7 li-bullet-0"><span class="c0">The </span><span class="c16">ahFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">ahFAD2B</span><span class="c0">&nbsp;mutant alleles originating from &#39;SunOleic 95R&#39; have been effectively introgressed into popular Indian peanut cultivars like GG-7 and TKG19A, resulting in advanced lines with significantly increased oleic acid content (78&ndash;80%).</span><span class="c4">2</span></li><li class="c6 c7 li-bullet-0"><span class="c0">In India, the first marker-assisted high oleic peanut varieties, Girnar 4 and Girnar 5, were released for commercial cultivation in 2020. These varieties boast an oleic acid content exceeding 78% and an O/L ratio of 17, demonstrating the tangible impact of MAS on cultivar development.</span><span class="c4">2</span></li><li class="c6 c7 li-bullet-0"><span class="c0">Marker-assisted backcross selection has yielded high oleic acid lines (e.g., 24 lines from BC4F5 and BC5F5 populations) that exhibit morphological and agronomic traits comparable to their recurrent parents, with genetic background recovery rates ranging from 79.49% to 92.31%.</span><span class="c4">24</span></li></ul><h3 class="c36"><span class="c29 c12">6.2. Impact on Cultivar Development and Market Demands</span></h3><p class="c5"><span class="c0">The integration of molecular markers has significantly reduced the time required for the development and release of new high oleic cultivars.</span><span class="c14">36</span><span class="c0">&nbsp;This acceleration is vital for meeting the increasing global demand for healthier and more stable peanut oil products.</span><span class="c14">3</span><span class="c0">&nbsp;Molecular breeding further enables the pyramiding of the high oleic trait with other critical characteristics, such as resistance to foliar diseases (e.g., leaf spot and rust) and nematodes.</span><span class="c14">1</span><span class="c0">&nbsp;This allows for the creation of advanced lines that combine superior oil quality with robust agronomic performance, addressing multiple breeding objectives simultaneously.</span><span class="c14">1</span><span class="c0">&nbsp;Furthermore, genetic markers play a crucial role in quality control, facilitating the detection of seed mixtures and heterozygotes, thereby ensuring genetic purity in seed production.</span><span class="c4">7</span></p><h3 class="c10"><span class="c29 c12">6.3. Challenges and Considerations</span></h3><p class="c10"><span class="c0 c12">While the benefits of molecular breeding for high oleic acid are substantial, certain challenges and considerations must be addressed:</span></p><ul class="c25 lst-kix_list_f-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Pleiotropic Effects:</span><span class="c0">&nbsp;A critical observation is that while </span><span class="c16">fad2</span><span class="c0">&nbsp;mutations are essential for increasing oleic acid content, they have been noted to negatively affect peanut growth and development. This can manifest as dwarfing and a reduction in pod and seed size.</span><span class="c14">6</span><span class="c0">&nbsp;This presents a significant trade-off that breeders must carefully manage. The goal is to develop varieties that not only possess high oleic acid but also maintain or improve crucial agronomic performance indicators such as yield and plant vigor.</span><span class="c14">1</span><span class="c0 c12">&nbsp;This necessitates a holistic breeding approach that balances oil quality with overall agronomic viability.</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Narrow Genetic Base of Donors:</span><span class="c0">&nbsp;The genetic background of existing high oleic acid donor parents is often limited, with many derived from small-seeded and late-maturing cultivars.</span><span class="c14">24</span><span class="c0 c12">&nbsp;This narrow genetic base can restrict the genetic diversity available for further breeding. Consequently, careful MABC strategies are essential not just for introgression but also for efficiently recovering the favorable genetic background and diverse traits of elite recurrent parents.</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Environmental Influences:</span><span class="c0">&nbsp;Intermittent failures of high oleic cultivars to consistently meet self-imposed oleic acid content thresholds have been observed.</span><span class="c14">3</span><span class="c0">&nbsp;This suggests that environmental factors or other unidentified genetic elements may influence fatty acid composition.</span><span class="c14">3</span><span class="c0">&nbsp;The overall genetics of oleic acid content can involve intricate quantitative trait loci (QTL) and multiple genes governing fatty acid biosynthesis beyond the primary </span><span class="c16">AhFAD2</span><span class="c0">&nbsp;genes.</span><span class="c4">3</span></li></ul><p class="c20"><span class="c0 c12">Effective high oleic peanut breeding is a complex endeavor that extends beyond single-trait selection. It requires a comprehensive approach that balances the desired oil quality with crucial agronomic performance indicators (yield, plant vigor, disease resistance). Advanced breeding techniques like MABC are essential not only for trait introgression but also for minimizing undesirable pleiotropic effects and efficiently recovering the elite genetic background of recurrent parents, ultimately ensuring the commercial viability of new high oleic cultivars.</span></p><h2 class="c10"><span class="c24 c12">7. Conclusion</span></h2><h3 class="c10"><span class="c29 c12">7.1. Recap of Critical Molecular Markers and Genetic Insights</span></h3><p class="c5"><span class="c0">The high oleic acid trait in peanut (</span><span class="c16">Arachis hypogaea</span><span class="c0">) is fundamentally governed by recessive, loss-of-function mutations within two homeologous genes, </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">. These genes encode &Delta;12 fatty acid desaturase enzymes, which are responsible for converting oleic acid to linoleic acid. The primary mutation in </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;is a G448A substitution, resulting in a D150N missense amino acid change. In </span><span class="c16">AhFAD2B</span><span class="c0 c12">, a greater diversity of mutations has been identified, including the prevalent 441_442insA insertion (causing a frameshift and premature stop codon, notably from the F435 line), as well as other substitutions like G451T and C301G (leading to H101D), and various transposable element or MITE insertions. The necessity of inactivating both homeologous genes for high oleic acid expression underscores the inherent genetic redundancy in this allotetraploid species.</span></p><p class="c5"><span class="c0">The advent and continuous refinement of molecular markers, particularly Cleaved Amplified Polymorphic Sequence (CAPS) and Kompetitive Allele Specific PCR (KASP) assays, have been instrumental in revolutionizing high oleic peanut breeding. CAPS markers leverage restriction enzyme polymorphisms to detect specific mutations, with established primer sequences and digestion patterns for key </span><span class="c16">AhFAD2</span><span class="c0">&nbsp;alleles. KASP markers represent a significant leap forward, utilizing allele-specific primers and fluorescent dyes for high-throughput, cost-effective genotyping. The precise primer sequences for both </span><span class="c16">AhFAD2A</span><span class="c0">&nbsp;and </span><span class="c16">AhFAD2B</span><span class="c0">&nbsp;KASP assays are well-defined, enabling accurate and rapid detection of the underlying mutations. The availability of extensive </span><span class="c16">Arachis hypogaea</span><span class="c0">&nbsp;genome assemblies and specific </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;gene accession numbers in public databases such as NCBI and PeanutBase provides a critical foundational resource for ongoing molecular research and breeding innovation.</span></p><h3 class="c10"><span class="c29 c12">7.2. Future Prospects for Enhancing High Oleic Acid Trait in Peanut</span></h3><p class="c5"><span class="c0 c12">The trajectory of peanut breeding for high oleic acid points towards continued advancements in molecular tools and integrated breeding strategies. The ongoing development and widespread application of high-throughput genotyping technologies like KASP, alongside emerging gene-editing technologies such as CRISPR/Cas9, will further enhance the efficiency and precision of high oleic peanut breeding. These tools facilitate targeted mutagenesis and the rapid introgression of desirable alleles into diverse genetic backgrounds, overcoming limitations imposed by narrow genetic diversity in existing donor pools.</span></p><p class="c10"><span class="c0 c12">Future research efforts will likely focus on several key areas:</span></p><ul class="c25 lst-kix_list_10-0 start"><li class="c10 c7 li-bullet-0"><span class="c11">Elucidating Complex Genetic Interactions:</span><span class="c0 c12">&nbsp;A deeper understanding of additional genetic factors or quantitative trait loci (QTLs) that influence fatty acid composition will be crucial. This will enable more precise fine-tuning of oleic acid levels and assist in mitigating any negative pleiotropic effects on essential agronomic traits such as yield and plant vigor.</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Broadening Genetic Diversity:</span><span class="c0">&nbsp;Exploring and utilizing novel </span><span class="c16">AhFAD2</span><span class="c0 c12">&nbsp;mutant alleles, whether naturally occurring or induced through advanced gene-editing techniques, will be vital for broadening the genetic base of high oleic germplasm. This diversification is essential for developing robust cultivars adaptable to various environments and agricultural systems.</span></li><li class="c6 c7 li-bullet-0"><span class="c11">Multi-Trait Breeding:</span><span class="c0 c12">&nbsp;The emphasis will increasingly be on developing comprehensive multi-trait breeding strategies. These strategies aim to combine high oleic acid content with other critical traits, including high yield potential, enhanced disease resistance, and improved stress tolerance. Such holistic improvement of peanut cultivars is necessary to ensure their sustainable production and meet evolving market demands for healthier, more stable, and agriculturally resilient peanut varieties globally.</span></li></ul><h4 class="c34"><span class="c12 c26">Works cited</span></h4><ol class="c25 lst-kix_list_11-0 start" start="1"><li class="c23 li-bullet-0"><span class="c3">Molecular breeding to develop advanced lines with high oleic acid ..., accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/373195564_Molecular_breeding_to_develop_advanced_lines_with_high_oleic_acid_and_pod_yield_in_peanut&amp;sa=D&amp;source=editors&amp;ust=1749151369075719&amp;usg=AOvVaw1Shfi3-pPeU67NVHZd34ih">https://www.researchgate.net/publication/373195564_Molecular_breeding_to_develop_advanced_lines_with_high_oleic_acid_and_pod_yield_in_peanut</a></span></li><li class="c23 li-bullet-0"><span class="c3">Does improved oleic acid content due to marker-assisted introgression of ahFAD2 mutant alleles in peanuts alter its mineral and vitamin composition? - Frontiers, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.942617/pdf&amp;sa=D&amp;source=editors&amp;ust=1749151369076315&amp;usg=AOvVaw07eruRIU9impXbM4vTKw1I">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2022.942617/pdf</a></span></li><li class="c23 li-bullet-0"><span class="c3">Development of a real-time PCR genotyping assay to identify high oleic acid peanuts (Arachis hypogaea L.) - ResearchGate, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/47538832_Development_of_a_real-time_PCR_genotyping_assay_to_identify_high_oleic_acid_peanuts_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1749151369077015&amp;usg=AOvVaw2WT54RuG_0ghKarn4LWbG4">https://www.researchgate.net/publication/47538832_Development_of_a_real-time_PCR_genotyping_assay_to_identify_high_oleic_acid_peanuts_Arachis_hypogaea_L</a></span></li><li class="c23 li-bullet-0"><span class="c3">Genome-wide association analysis in peanut accessions uncovers the genetic basis regulating oil and fatty acid variation - PMC, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC12082984/&amp;sa=D&amp;source=editors&amp;ust=1749151369077514&amp;usg=AOvVaw135Ru_VRoZ0tJH1SpyQ95g">https://pmc.ncbi.nlm.nih.gov/articles/PMC12082984/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Seed oleic acid to linoleic acid ratio &ndash; PeanutBase, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://dev.peanutbase.org/mas/seed_oleic_to_linoleic_ratio_xa&amp;sa=D&amp;source=editors&amp;ust=1749151369077931&amp;usg=AOvVaw0aISkjHqv37pqYo_7Wvbfr">https://dev.peanutbase.org/mas/seed_oleic_to_linoleic_ratio_xa</a></span></li><li class="c23 li-bullet-0"><span class="c3">scRNA-seq Reveals the Mechanism of Fatty Acid Desaturase 2 Mutation to Repress Leaf Growth in Peanut (Arachis hypogaea L.) - MDPI, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4409/12/18/2305&amp;sa=D&amp;source=editors&amp;ust=1749151369078252&amp;usg=AOvVaw0b3XKnZh2zGAyOKyy9gf_a">https://www.mdpi.com/2073-4409/12/18/2305</a></span></li><li class="c23 li-bullet-0"><span class="c3">Crop Genetics and Breeding Research: Tifton, GA - Publication : USDA ARS, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.ars.usda.gov/research/publications/publication/?seqNo115%3D248235&amp;sa=D&amp;source=editors&amp;ust=1749151369078692&amp;usg=AOvVaw33UHM2bUsmOf710rUYrUT4">https://www.ars.usda.gov/research/publications/publication/?seqNo115=248235</a></span></li><li class="c23 li-bullet-0"><span class="c3">Development of high oleic peanut lines through marker-assisted introgression of mutant ahFAD2 alleles and its fatty acid profiles under open-field and controlled conditions - PMC, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6545185/&amp;sa=D&amp;source=editors&amp;ust=1749151369079145&amp;usg=AOvVaw3CVHioByYE1lgFS6ylByTY">https://pmc.ncbi.nlm.nih.gov/articles/PMC6545185/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Isolation and functional analysis of fatty acid desaturase genes from peanut (Arachis hypogaea L.) - PMC, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5731756/&amp;sa=D&amp;source=editors&amp;ust=1749151369079607&amp;usg=AOvVaw2lOBOXSBKZ8SZwhDtBGPAp">https://pmc.ncbi.nlm.nih.gov/articles/PMC5731756/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Preliminary study of genome editing of peanut FAD2 genes by CRISPR/Cas9 - &#20316;&#29289;&#23398;&#25253;, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://zwxb.chinacrops.org/EN/10.3724/SP.J.1006.2021.04214&amp;sa=D&amp;source=editors&amp;ust=1749151369079969&amp;usg=AOvVaw0MK1HbU422kz9TYszJZyzW">https://zwxb.chinacrops.org/EN/10.3724/SP.J.1006.2021.04214</a></span></li><li class="c23 li-bullet-0"><span class="c3">The FAD2 Gene in Plants: Occurrence, Regulation, and Role - Frontiers, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.01789/full&amp;sa=D&amp;source=editors&amp;ust=1749151369080290&amp;usg=AOvVaw11UdAmt5-HZaY1mOBoTn2l">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2017.01789/full</a></span></li><li class="c23 li-bullet-0"><span class="c3">Molecular Mapping of Oil Content and Fatty Acids Using Dense Genetic Maps in Groundnut (Arachis hypogaea L.), accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC5438992/&amp;sa=D&amp;source=editors&amp;ust=1749151369080632&amp;usg=AOvVaw350X7aKLAZzsAvXO5D-S9_">https://pmc.ncbi.nlm.nih.gov/articles/PMC5438992/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Insights into the Indian Peanut Genotypes for ahFAD2 Gene Polymorphism Regulating Its Oleic and Linoleic Acid Fluxes - Frontiers, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2016.01271/pdf&amp;sa=D&amp;source=editors&amp;ust=1749151369081126&amp;usg=AOvVaw2YFtwYEtBUBBDCwaL5k98o">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2016.01271/pdf</a></span></li><li class="c23 li-bullet-0"><span class="c3">Genotypic effect of ahFAD2 on fatty acid profiles in six segregating peanut (Arachis hypogaea L) populations - PubMed Central, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC3722029/&amp;sa=D&amp;source=editors&amp;ust=1749151369081489&amp;usg=AOvVaw0RUeES28fWQ0CF-1i3b4w1">https://pmc.ncbi.nlm.nih.gov/articles/PMC3722029/</a></span></li><li class="c23 li-bullet-0"><span class="c3">(PDF) Two Alleles of ahFAD2B Control the High Oleic Acid Trait in Cultivated Peanut, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/43288506_Two_Alleles_of_ahFAD2B_Control_the_High_Oleic_Acid_Trait_in_Cultivated_Peanut&amp;sa=D&amp;source=editors&amp;ust=1749151369082062&amp;usg=AOvVaw0DwHxyPDXbOjJCTMZ2Au7O">https://www.researchgate.net/publication/43288506_Two_Alleles_of_ahFAD2B_Control_the_High_Oleic_Acid_Trait_in_Cultivated_Peanut</a></span></li><li class="c23 li-bullet-0"><span class="c3">Mutagenesis of FAD2 genes in peanut with CRISPR/Cas9 based gene editing, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://d-nb.info/1190619156/34&amp;sa=D&amp;source=editors&amp;ust=1749151369082545&amp;usg=AOvVaw3K3dE1SH0rp8gDIB11kXiE">https://d-nb.info/1190619156/34</a></span></li><li class="c23 li-bullet-0"><span class="c3">Mutagenesis of FAD2 genes in peanut with CRISPR/Cas9 based gene editing - PMC, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6489235/&amp;sa=D&amp;source=editors&amp;ust=1749151369083018&amp;usg=AOvVaw33uFSV5aheNa4O1CP7_phb">https://pmc.ncbi.nlm.nih.gov/articles/PMC6489235/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Insights into the Indian Peanut Genotypes for ahFAD2 Gene Polymorphism Regulating Its Oleic and Linoleic Acid Fluxes - Frontiers, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2016.01271/full&amp;sa=D&amp;source=editors&amp;ust=1749151369083498&amp;usg=AOvVaw3ln-RemGr78LRaMSTKC234">https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2016.01271/full</a></span></li><li class="c23 li-bullet-0"><span class="c3">Isolation and functional analysis of fatty acid desaturase genes from peanut (Arachis hypogaea L.) | PLOS One, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://journals.plos.org/plosone/article?id%3D10.1371/journal.pone.0189759&amp;sa=D&amp;source=editors&amp;ust=1749151369084060&amp;usg=AOvVaw0sv7Lm6AFlN1Lvmo164DAY">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0189759</a></span></li><li class="c23 li-bullet-0"><span class="c3">Fatty acid desaturase-2 (ahFAD2) mutant alleles in peanut (Arachis hypogaea L.) pre-breeding lines - OAR@ICRISAT, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://oar.icrisat.org/id/eprint/11618/contents&amp;sa=D&amp;source=editors&amp;ust=1749151369084398&amp;usg=AOvVaw0DskTSg4Rya3CHkxNya9GY">https://oar.icrisat.org/id/eprint/11618/contents</a></span></li><li class="c23 li-bullet-0"><span class="c3">Arachis hypogaea v1.0 - Phytozome, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://phytozome-next.jgi.doe.gov/info/Ahypogaea_v1_0&amp;sa=D&amp;source=editors&amp;ust=1749151369084688&amp;usg=AOvVaw0CZPHjSMrd9TpYVz-amD2M">https://phytozome-next.jgi.doe.gov/info/Ahypogaea_v1_0</a></span></li><li class="c23 li-bullet-0"><span class="c3">Development and application of KASP marker for high throughput ..., accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.ejbiotechnology.info/index.php/ejbiotechnology/article/view/2016.10.010&amp;sa=D&amp;source=editors&amp;ust=1749151369085184&amp;usg=AOvVaw0zbY9UgHPAoFx7VT6KkKzb">https://www.ejbiotechnology.info/index.php/ejbiotechnology/article/view/2016.10.010</a></span></li><li class="c23 li-bullet-0"><span class="c3">Identification of QTLs associated with oil content and mapping FAD2 genes and their relative contribution to oil quality in peanut (Arachis hypogaea L.), accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://openscholar.uga.edu/record/1300/files/s12863-014-0133-4.pdf&amp;sa=D&amp;source=editors&amp;ust=1749151369085853&amp;usg=AOvVaw2CN2WT-YyCDeqy76Ul9tvU">https://openscholar.uga.edu/record/1300/files/s12863-014-0133-4.pdf</a></span></li><li class="c23 li-bullet-0"><span class="c3">Marker-assisted backcrossing to improve seed oleic acid content in four elite and popular peanut (</span><span class="c3 c37">Arachis hypogaea</span><span class="c3">&nbsp;L.) cultivars with high oil content - J-Stage, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.jstage.jst.go.jp/article/jsbbs/69/2/69_18107/_html/-char/en&amp;sa=D&amp;source=editors&amp;ust=1749151369086500&amp;usg=AOvVaw2G1Rn4vfkeGHSv83HY7k7G">https://www.jstage.jst.go.jp/article/jsbbs/69/2/69_18107/_html/-char/en</a></span></li><li class="c23 li-bullet-0"><span class="c3">Comparison of the Delta(12) fatty acid desaturase gene between high-oleic and normal-oleic peanut genotypes - PubMed, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/19022202/&amp;sa=D&amp;source=editors&amp;ust=1749151369086961&amp;usg=AOvVaw0WXG6EHVm4cO86bUcgPsjP">https://pubmed.ncbi.nlm.nih.gov/19022202/</a></span></li><li class="c23 li-bullet-0"><span class="c3">(PDF) Identification of a New Major Oil Content QTL Overlapped with FAD2B in Cultivated Peanut (Arachis hypogaea L.) - ResearchGate, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/389096067_Identification_of_a_New_Major_Oil_Content_QTL_Overlapped_with_FAD2B_in_Cultivated_Peanut_Arachis_hypogaea_L&amp;sa=D&amp;source=editors&amp;ust=1749151369087535&amp;usg=AOvVaw3OszAzzQaO_OlBRYm2J39y">https://www.researchgate.net/publication/389096067_Identification_of_a_New_Major_Oil_Content_QTL_Overlapped_with_FAD2B_in_Cultivated_Peanut_Arachis_hypogaea_L</a></span></li><li class="c23 li-bullet-0"><span class="c3">Identification of Putative Quantitative Trait Loci for Improved Seed Oil Quality in Peanuts, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC10815147/&amp;sa=D&amp;source=editors&amp;ust=1749151369087953&amp;usg=AOvVaw08IQM-Rgch7Q3ADSyQLvYh">https://pmc.ncbi.nlm.nih.gov/articles/PMC10815147/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Identification of a New Major Oil Content QTL Overlapped with FAD2B in Cultivated Peanut (Arachis hypogaea L.) - MDPI, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.mdpi.com/2223-7747/14/4/615&amp;sa=D&amp;source=editors&amp;ust=1749151369088360&amp;usg=AOvVaw3AjY_lTAK3Gy3Iqu3nl2fG">https://www.mdpi.com/2223-7747/14/4/615</a></span></li><li class="c23 li-bullet-0"><span class="c3">Mutagenesis of FAD2 Genes in Peanut with CRISPR/Cas9 Based Gene Editing, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/347689814_Mutagenesis_of_FAD2_Genes_in_Peanut_with_CRISPRCas9_Based_Gene_Editing&amp;sa=D&amp;source=editors&amp;ust=1749151369088914&amp;usg=AOvVaw1FtR8bcrHrnHw9lPtyrQOO">https://www.researchgate.net/publication/347689814_Mutagenesis_of_FAD2_Genes_in_Peanut_with_CRISPRCas9_Based_Gene_Editing</a></span></li><li class="c23 li-bullet-0"><span class="c3">(PDF) Development and application of KASP marker for high ..., accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/310472960_Development_and_application_of_KASP_marker_for_high_throughput_detection_of_AhFAD2_mutation_in_peanut&amp;sa=D&amp;source=editors&amp;ust=1749151369089529&amp;usg=AOvVaw0of4w4it--KTqKur2FKCiH">https://www.researchgate.net/publication/310472960_Development_and_application_of_KASP_marker_for_high_throughput_detection_of_AhFAD2_mutation_in_peanut</a></span></li><li class="c23 li-bullet-0"><span class="c3">Mutagenesis of FAD2 genes in peanut with CRISPR/Cas9 based gene editing, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.researchgate.net/publication/332745444_Mutagenesis_of_FAD2_genes_in_peanut_with_CRISPRCas9_based_gene_editing&amp;sa=D&amp;source=editors&amp;ust=1749151369090061&amp;usg=AOvVaw3FVOpOCjdcaXNfvsuAnuYO">https://www.researchgate.net/publication/332745444_Mutagenesis_of_FAD2_genes_in_peanut_with_CRISPRCas9_based_gene_editing</a></span></li><li class="c23 li-bullet-0"><span class="c3">Marker-assisted backcrossing to improve seed oleic acid content in four elite and popular peanut (Arachis hypogaea L.) cultivars with high oil content, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC6711728/&amp;sa=D&amp;source=editors&amp;ust=1749151369090500&amp;usg=AOvVaw1Ix_Hhqb7_ZsJz8JhKzzUU">https://pmc.ncbi.nlm.nih.gov/articles/PMC6711728/</a></span></li><li class="c23 li-bullet-0"><span class="c3">The high oleate trait in the cultivated peanut [Arachis hypogaea L]. II. Molecular basis and genetics of the trait - PubMed, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pubmed.ncbi.nlm.nih.gov/10905348/&amp;sa=D&amp;source=editors&amp;ust=1749151369090895&amp;usg=AOvVaw2JleWVGoH9mPGuBzAZcA78">https://pubmed.ncbi.nlm.nih.gov/10905348/</a></span></li><li class="c23 li-bullet-0"><span class="c3">Novel protocol to identify true hybrids in normal oleate x high oleate crosses in peanut - Bioline International, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=http://www.bioline.org.br/pdf?ej10062&amp;sa=D&amp;source=editors&amp;ust=1749151369091277&amp;usg=AOvVaw0kkRiXIpDdU_pj58o-_xmk">http://www.bioline.org.br/pdf?ej10062</a></span></li><li class="c23 li-bullet-0"><span class="c3">Development and application of KASP marker for high throughput detection of AhFAD2 mutation in peanut, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.bioline.org.br/pdf?ej17002&amp;sa=D&amp;source=editors&amp;ust=1749151369091690&amp;usg=AOvVaw19LCznXxuBb-NPby1VA8mX">https://www.bioline.org.br/pdf?ej17002</a></span></li><li class="c23 li-bullet-0"><span class="c3">Cleaved Amplified Polymorphic Sequence Markers in Horticultural Crops: Current Status and Future Perspectives - MDPI, accessed June 5, 2025, </span><span class="c3 c9"><a class="c1" href="https://www.google.com/url?q=https://www.mdpi.com/2073-4395/14/11/2598&amp;sa=D&amp;source=editors&amp;ust=1749151369091991&amp;usg=AOvVaw0lYXdnd_SB47sGWFHL3S6n">https://www.mdpi.com/2073-4395/14/11/2598</a></span></li><li class="c23 li-bullet-0"><span class="c3">Marker-assisted backcrossing to improve seed oleic acid content in four elite and popular peanut (Arachis hypogaea L.) cultivars with high oil content - Descubridor, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://descubridor.banrepcultural.org/discovery/fulldisplay?docid%3Dcdi_pubmedcentral_primary_oai_pubmedcentral_nih_gov_6711728%26context%3DPC%26vid%3D57BDLRDC_INST:57BDLRDC_INST%26lang%3Des%26search_scope%3DMyInst_and_CI%26adaptor%3DPrimo%2BCentral%26tab%3DALL%26query%3Dcreator,exact,Huang,%2BBingyan%2B%26facet%3Dcreator,exact,Huang,%2BBingyan%2B%26offset%3D0&amp;sa=D&amp;source=editors&amp;ust=1749151369092827&amp;usg=AOvVaw0Q6e6mAjUYGca5afKfFiCQ">https://descubridor.banrepcultural.org/discovery/fulldisplay?docid=cdi_pubmedcentral_primary_oai_pubmedcentral_nih_gov_6711728&amp;context=PC&amp;vid=57BDLRDC_INST:57BDLRDC_INST&amp;lang=es&amp;search_scope=MyInst_and_CI&amp;adaptor=Primo%20Central&amp;tab=ALL&amp;query=creator%2Cexact%2CHuang%2C%20Bingyan%20&amp;facet=creator%2Cexact%2CHuang%2C%20Bingyan%20&amp;offset=0</a></span></li><li class="c23 li-bullet-0"><span class="c3">Arachis hypogaea - Search results - Genome - Assembly - NCBI, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/assembly/?term%3DArachis%2Bhypogaea&amp;sa=D&amp;source=editors&amp;ust=1749151369093270&amp;usg=AOvVaw3lMNLfOZNgOAbIBuOoA2LB">https://www.ncbi.nlm.nih.gov/assembly/?term=Arachis%20hypogaea</a></span></li><li class="c23 li-bullet-0"><span class="c3">FAD2A - Oleate desaturase - Arachis hypogaea (Peanut) | UniProtKB | UniProt, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.uniprot.org/uniprotkb/C8CK29/entry&amp;sa=D&amp;source=editors&amp;ust=1749151369093669&amp;usg=AOvVaw0J83lTGIU6WRF2W6iOwEsB">https://www.uniprot.org/uniprotkb/C8CK29/entry</a></span></li><li class="c23 li-bullet-0"><span class="c3">Comparative Analysis and Structural Modeling of Elaeis oleifera FAD2, a Fatty Acid Desaturase Involved in Unsaturated Fatty Acid Composition of American Oil Palm - PubMed Central, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9032008/&amp;sa=D&amp;source=editors&amp;ust=1749151369094257&amp;usg=AOvVaw1dqMS53iBZ_9olKOFfrnpe">https://pmc.ncbi.nlm.nih.gov/articles/PMC9032008/</a></span></li><li class="c23 li-bullet-0"><span class="c3">2613050 - Nucleotide Result - NCBI, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/nuccore/AF030319&amp;sa=D&amp;source=editors&amp;ust=1749151369094608&amp;usg=AOvVaw18i9DJvvG_o5VwUIAOn-1m">https://www.ncbi.nlm.nih.gov/nuccore/AF030319</a></span></li><li class="c23 li-bullet-0"><span class="c3">Home - Nucleotide - NCBI, accessed June 5, 2025, </span><span class="c9 c3"><a class="c1" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/nucleotide/&amp;sa=D&amp;source=editors&amp;ust=1749151369094922&amp;usg=AOvVaw2xv5xgALHBfUTUe7hPJwFX">https://www.ncbi.nlm.nih.gov/nucleotide/</a></span></li></ol></body></html>