<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=DFQxm4rd7fRHgM9OTejWVT5Vho6BE7M80rHXEVKqXWegg2XYR88pwOsaJkfiF7cJu5e0vFtnyLdhsxviZUUN-U0KZVwUvSK-LyXz4qcE1hc);.lst-kix_list_c-0>li:before{content:"\0025cf   "}ul.lst-kix_list_b-8{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_c-1>li:before{content:"\0025cb   "}ul.lst-kix_list_b-3{list-style-type:none}ul.lst-kix_list_b-2{list-style-type:none}.lst-kix_list_c-3>li:before{content:"\0025a0   "}.lst-kix_list_c-4>li:before{content:"\0025a0   "}ul.lst-kix_list_b-1{list-style-type:none}ul.lst-kix_list_b-0{list-style-type:none}ul.lst-kix_list_b-7{list-style-type:none}ul.lst-kix_list_b-6{list-style-type:none}.lst-kix_list_c-2>li:before{content:"\0025a0   "}ul.lst-kix_list_b-5{list-style-type:none}ul.lst-kix_list_b-4{list-style-type:none}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_9-0{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_list_5-0>li:before{content:"\0025cf   "}.lst-kix_list_5-3>li:before{content:"\0025a0   "}ul.lst-kix_list_a-4{list-style-type:none}ul.lst-kix_list_a-3{list-style-type:none}.lst-kix_list_5-2>li:before{content:"\0025a0   "}ul.lst-kix_list_a-2{list-style-type:none}ul.lst-kix_list_a-1{list-style-type:none}.lst-kix_list_5-1>li:before{content:"\0025cb   "}ul.lst-kix_list_a-8{list-style-type:none}ul.lst-kix_list_a-7{list-style-type:none}ul.lst-kix_list_a-6{list-style-type:none}ul.lst-kix_list_a-5{list-style-type:none}.lst-kix_list_5-7>li:before{content:"\0025a0   "}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025a0   "}.lst-kix_list_5-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_d-6>li:before{content:"\0025a0   "}.lst-kix_list_5-4>li:before{content:"\0025a0   "}.lst-kix_list_5-5>li:before{content:"\0025a0   "}ul.lst-kix_list_8-0{list-style-type:none}.lst-kix_list_d-8>li:before{content:"\0025a0   "}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_d-7>li:before{content:"\0025a0   "}.lst-kix_list_d-2>li:before{content:"\0025a0   "}.lst-kix_list_6-1>li:before{content:"\0025cb   "}.lst-kix_list_6-3>li:before{content:"\0025a0   "}.lst-kix_list_d-1>li:before{content:"\0025cb   "}.lst-kix_list_d-5>li:before{content:"\0025a0   "}.lst-kix_list_6-0>li:before{content:"\0025cf   "}.lst-kix_list_6-4>li:before{content:"\0025a0   "}.lst-kix_list_d-4>li:before{content:"\0025a0   "}.lst-kix_list_d-3>li:before{content:"\0025a0   "}.lst-kix_list_6-2>li:before{content:"\0025a0   "}.lst-kix_list_c-7>li:before{content:"\0025a0   "}ul.lst-kix_list_a-0{list-style-type:none}.lst-kix_list_c-5>li:before{content:"\0025a0   "}.lst-kix_list_c-8>li:before{content:"\0025a0   "}.lst-kix_list_6-8>li:before{content:"\0025a0   "}.lst-kix_list_6-5>li:before{content:"\0025a0   "}.lst-kix_list_6-7>li:before{content:"\0025a0   "}.lst-kix_list_d-0>li:before{content:"\0025cf   "}.lst-kix_list_c-6>li:before{content:"\0025a0   "}.lst-kix_list_6-6>li:before{content:"\0025a0   "}.lst-kix_list_2-7>li:before{content:"\0025a0   "}ul.lst-kix_list_d-8{list-style-type:none}ul.lst-kix_list_d-7{list-style-type:none}.lst-kix_list_7-4>li:before{content:"\0025a0   "}.lst-kix_list_7-6>li:before{content:"\0025a0   "}ul.lst-kix_list_d-6{list-style-type:none}.lst-kix_list_2-5>li:before{content:"\0025a0   "}ul.lst-kix_list_d-1{list-style-type:none}ul.lst-kix_list_d-0{list-style-type:none}.lst-kix_list_b-8>li:before{content:"\0025a0   "}.lst-kix_list_7-2>li:before{content:"\0025a0   "}ul.lst-kix_list_d-5{list-style-type:none}ul.lst-kix_list_d-4{list-style-type:none}ul.lst-kix_list_d-3{list-style-type:none}ul.lst-kix_list_d-2{list-style-type:none}.lst-kix_list_b-4>li:before{content:"\0025a0   "}ul.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_list_3-8{list-style-type:none}.lst-kix_list_10-1>li:before{content:"\0025cb   "}ul.lst-kix_list_3-1{list-style-type:none}ul.lst-kix_list_3-2{list-style-type:none}.lst-kix_list_7-8>li:before{content:"\0025a0   "}ul.lst-kix_list_3-0{list-style-type:none}ul.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_list_3-6{list-style-type:none}.lst-kix_list_b-6>li:before{content:"\0025a0   "}ul.lst-kix_list_3-3{list-style-type:none}ul.lst-kix_list_3-4{list-style-type:none}.lst-kix_list_10-7>li:before{content:"\0025a0   "}.lst-kix_list_10-5>li:before{content:"\0025a0   "}.lst-kix_list_10-3>li:before{content:"\0025a0   "}.lst-kix_list_b-0>li:before{content:"\0025cf   "}.lst-kix_list_4-1>li:before{content:"\0025cb   "}.lst-kix_list_b-2>li:before{content:"\0025a0   "}.lst-kix_list_9-2>li:before{content:"\0025a0   "}.lst-kix_list_4-3>li:before{content:"\0025a0   "}.lst-kix_list_4-5>li:before{content:"\0025a0   "}.lst-kix_list_9-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-8{list-style-type:none}ul.lst-kix_list_c-7{list-style-type:none}.lst-kix_list_9-6>li:before{content:"\0025a0   "}ul.lst-kix_list_c-2{list-style-type:none}ul.lst-kix_list_c-1{list-style-type:none}ul.lst-kix_list_c-0{list-style-type:none}.lst-kix_list_9-4>li:before{content:"\0025a0   "}ul.lst-kix_list_c-6{list-style-type:none}.lst-kix_list_a-0>li:before{content:"\0025cf   "}ul.lst-kix_list_c-5{list-style-type:none}ul.lst-kix_list_c-4{list-style-type:none}ul.lst-kix_list_c-3{list-style-type:none}ol.lst-kix_list_10-0{list-style-type:none}ul.lst-kix_list_2-8{list-style-type:none}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_list_9-8>li:before{content:"\0025a0   "}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1>li:before{content:"\0025cb   "}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0   "}ul.lst-kix_list_10-8{list-style-type:none}ul.lst-kix_list_10-7{list-style-type:none}.lst-kix_list_1-7>li:before{content:"\0025a0   "}ul.lst-kix_list_10-6{list-style-type:none}ul.lst-kix_list_10-5{list-style-type:none}ul.lst-kix_list_10-4{list-style-type:none}ul.lst-kix_list_10-3{list-style-type:none}.lst-kix_list_1-5>li:before{content:"\0025a0   "}ul.lst-kix_list_10-2{list-style-type:none}ul.lst-kix_list_10-1{list-style-type:none}.lst-kix_list_2-1>li:before{content:"\0025cb   "}.lst-kix_list_2-3>li:before{content:"\0025a0   "}ul.lst-kix_list_f-7{list-style-type:none}ul.lst-kix_list_f-6{list-style-type:none}ul.lst-kix_list_f-5{list-style-type:none}ul.lst-kix_list_f-4{list-style-type:none}ul.lst-kix_list_f-8{list-style-type:none}ul.lst-kix_list_f-3{list-style-type:none}.lst-kix_list_3-0>li:before{content:"\0025cf   "}ul.lst-kix_list_f-2{list-style-type:none}ul.lst-kix_list_f-1{list-style-type:none}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}.lst-kix_list_3-1>li:before{content:"\0025cb   "}.lst-kix_list_3-2>li:before{content:"\0025a0   "}ul.lst-kix_list_5-5{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}.lst-kix_list_8-1>li:before{content:"\0025cb   "}.lst-kix_list_8-2>li:before{content:"\0025a0   "}.lst-kix_list_3-5>li:before{content:"\0025a0   "}ul.lst-kix_list_5-0{list-style-type:none}.lst-kix_list_10-0>li{counter-increment:lst-ctn-kix_list_10-0}.lst-kix_list_3-4>li:before{content:"\0025a0   "}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3>li:before{content:"\0025a0   "}ul.lst-kix_list_5-4{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0>li:before{content:"\0025cf   "}ul.lst-kix_list_5-2{list-style-type:none}.lst-kix_list_8-7>li:before{content:"\0025a0   "}.lst-kix_list_3-8>li:before{content:"\0025a0   "}.lst-kix_list_f-6>li:before{content:"\0025a0   "}.lst-kix_list_f-8>li:before{content:"\0025a0   "}.lst-kix_list_8-5>li:before{content:"\0025a0   "}.lst-kix_list_a-7>li:before{content:"\0025a0   "}.lst-kix_list_8-6>li:before{content:"\0025a0   "}.lst-kix_list_f-7>li:before{content:"\0025a0   "}.lst-kix_list_8-3>li:before{content:"\0025a0   "}.lst-kix_list_3-6>li:before{content:"\0025a0   "}.lst-kix_list_f-4>li:before{content:"\0025a0   "}.lst-kix_list_3-7>li:before{content:"\0025a0   "}.lst-kix_list_a-8>li:before{content:"\0025a0   "}.lst-kix_list_8-4>li:before{content:"\0025a0   "}.lst-kix_list_f-5>li:before{content:"\0025a0   "}.lst-kix_list_a-1>li:before{content:"\0025cb   "}.lst-kix_list_a-2>li:before{content:"\0025a0   "}.lst-kix_list_f-0>li:before{content:"" counter(lst-ctn-kix_list_f-0,decimal) ". "}.lst-kix_list_a-3>li:before{content:"\0025a0   "}.lst-kix_list_f-3>li:before{content:"\0025a0   "}.lst-kix_list_a-5>li:before{content:"\0025a0   "}.lst-kix_list_a-6>li:before{content:"\0025a0   "}.lst-kix_list_f-2>li:before{content:"\0025a0   "}.lst-kix_list_a-4>li:before{content:"\0025a0   "}.lst-kix_list_8-8>li:before{content:"\0025a0   "}.lst-kix_list_f-1>li:before{content:"\0025cb   "}.lst-kix_list_e-5>li:before{content:"\0025a0   "}ul.lst-kix_list_e-8{list-style-type:none}ul.lst-kix_list_e-7{list-style-type:none}ul.lst-kix_list_e-6{list-style-type:none}ul.lst-kix_list_e-5{list-style-type:none}.lst-kix_list_e-3>li:before{content:"\0025a0   "}.lst-kix_list_e-7>li:before{content:"\0025a0   "}.lst-kix_list_e-2>li:before{content:"\0025a0   "}.lst-kix_list_e-6>li:before{content:"\0025a0   "}.lst-kix_list_4-8>li:before{content:"\0025a0   "}ul.lst-kix_list_e-0{list-style-type:none}.lst-kix_list_4-7>li:before{content:"\0025a0   "}ul.lst-kix_list_e-4{list-style-type:none}ul.lst-kix_list_e-3{list-style-type:none}ul.lst-kix_list_e-2{list-style-type:none}.lst-kix_list_e-4>li:before{content:"\0025a0   "}ul.lst-kix_list_e-1{list-style-type:none}ul.lst-kix_list_4-8{list-style-type:none}ul.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_4-0{list-style-type:none}.lst-kix_list_e-1>li:before{content:"\0025cb   "}ul.lst-kix_list_4-1{list-style-type:none}ul.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_4-2{list-style-type:none}.lst-kix_list_e-0>li:before{content:"\0025cf   "}ul.lst-kix_list_4-3{list-style-type:none}.lst-kix_list_7-0>li:before{content:"\0025cf   "}.lst-kix_list_2-6>li:before{content:"\0025a0   "}.lst-kix_list_2-4>li:before{content:"\0025a0   "}.lst-kix_list_2-8>li:before{content:"\0025a0   "}.lst-kix_list_7-1>li:before{content:"\0025cb   "}.lst-kix_list_7-5>li:before{content:"\0025a0   "}.lst-kix_list_7-3>li:before{content:"\0025a0   "}ul.lst-kix_list_7-5{list-style-type:none}.lst-kix_list_10-0>li:before{content:"" counter(lst-ctn-kix_list_10-0,decimal) ". "}ul.lst-kix_list_7-6{list-style-type:none}.lst-kix_list_b-5>li:before{content:"\0025a0   "}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}.lst-kix_list_b-3>li:before{content:"\0025a0   "}.lst-kix_list_b-7>li:before{content:"\0025a0   "}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7>li:before{content:"\0025a0   "}ol.lst-kix_list_10-0.start{counter-reset:lst-ctn-kix_list_10-0 0}.lst-kix_list_10-4>li:before{content:"\0025a0   "}.lst-kix_list_10-8>li:before{content:"\0025a0   "}.lst-kix_list_4-0>li:before{content:"\0025cf   "}.lst-kix_list_b-1>li:before{content:"\0025cb   "}.lst-kix_list_10-2>li:before{content:"\0025a0   "}.lst-kix_list_4-4>li:before{content:"\0025a0   "}ol.lst-kix_list_f-0{list-style-type:none}.lst-kix_list_4-2>li:before{content:"\0025a0   "}.lst-kix_list_4-6>li:before{content:"\0025a0   "}.lst-kix_list_9-3>li:before{content:"\0025a0   "}.lst-kix_list_10-6>li:before{content:"\0025a0   "}.lst-kix_list_9-1>li:before{content:"\0025cb   "}.lst-kix_list_e-8>li:before{content:"\0025a0   "}.lst-kix_list_9-7>li:before{content:"\0025a0   "}.lst-kix_list_9-5>li:before{content:"\0025a0   "}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf   "}ul.lst-kix_list_6-2{list-style-type:none}ul.lst-kix_list_6-3{list-style-type:none}.lst-kix_list_1-2>li:before{content:"\0025a0   "}ul.lst-kix_list_6-0{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_list_f-0>li{counter-increment:lst-ctn-kix_list_f-0}.lst-kix_list_1-4>li:before{content:"\0025a0   "}.lst-kix_list_1-6>li:before{content:"\0025a0   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_list_2-0>li:before{content:"\0025cf   "}ol.lst-kix_list_f-0.start{counter-reset:lst-ctn-kix_list_f-0 0}.lst-kix_list_1-8>li:before{content:"\0025a0   "}.lst-kix_list_2-2>li:before{content:"\0025a0   "}ol{margin:0;padding:0}table td,table th{padding:0}.c5{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:52pt;border-top-color:#000000;border-bottom-style:solid}.c3{border-right-style:solid;padding:6pt 9pt 6pt 9pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f8fafd;border-left-style:solid;border-bottom-width:1pt;width:46.8pt;border-top-color:#000000;border-bottom-style:solid}.c6{color:#1b1c1d;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c2{color:#1b1c1d;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Google Sans";font-style:normal}.c8{color:#1b1c1d;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Google Sans Text";font-style:italic}.c18{margin-left:43.5pt;padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c39{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#575b5f;font-weight:400}.c31{margin-left:30pt;padding-top:0pt;padding-bottom:0pt;line-height:1.0;padding-left:0pt;text-align:left}.c16{margin-left:23.2pt;padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;padding-left:0pt;text-align:left}.c10{vertical-align:super;font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#575b5f;font-weight:400}.c17{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;vertical-align:baseline;text-decoration-skip-ink:none;font-style:normal}.c38{color:#000000;font-weight:700;font-size:12pt;font-family:"Google Sans";font-style:normal}.c14{color:#1b1c1d;font-weight:700;font-size:10pt;font-family:"Google Sans Text";font-style:normal}.c27{font-size:10pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c29{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:left;height:11pt}.c9{font-size:12pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:400}.c36{padding-top:0pt;padding-bottom:0pt;line-height:1.149999976158142;text-align:left;height:11pt}.c22{color:#1b1c1d;font-weight:700;font-size:15pt;font-family:"Google Sans";font-style:normal}.c7{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:700}.c37{font-size:16pt;font-family:"Google Sans";font-style:normal;color:#1b1c1d;font-weight:700}.c40{font-size:16pt;font-family:"Google Sans Text";font-style:italic;color:#1b1c1d;font-weight:700}.c0{font-size:12pt;font-family:"Google Sans Text";font-style:normal;color:#1b1c1d;font-weight:400}.c19{-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline;text-decoration-skip-ink:none}.c34{padding-top:12pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c1{padding-top:6pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c23{padding-top:0pt;padding-bottom:12.8pt;line-height:1.0;text-align:left}.c4{padding-top:0pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c20{padding-top:0pt;padding-bottom:6pt;line-height:1.149999976158142;text-align:left}.c24{border-spacing:0;border-collapse:collapse;margin-right:auto}.c30{padding-top:12pt;padding-bottom:12pt;line-height:1.149999976158142;text-align:left}.c28{padding-top:0pt;padding-bottom:12.8pt;line-height:1.149999976158142;text-align:left}.c35{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c13{font-size:12pt;font-weight:400;font-family:"Google Sans"}.c26{padding:0;margin:0}.c11{text-decoration:none;vertical-align:baseline}.c25{margin-left:23.2pt;padding-left:0pt}.c21{margin-left:24pt;padding-left:0pt}.c33{margin-left:43.5pt;padding-left:0pt}.c32{color:inherit;text-decoration:inherit}.c15{height:0pt}.c12{text-decoration:none}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c35 doc-content"><h1 class="c20"><span class="c37">Molecular Markers and Genomic Regions Associated with Flowering Time and Maturity in Cultivated Peanut (</span><span class="c40">Arachis hypogaea</span><span class="c37 c11">&nbsp;L.)</span></h1><h2 class="c20"><span class="c22 c11">1. Introduction</span></h2><h3 class="c20"><span class="c2">1.1. Significance of Flowering Time and Time-to-Maturation (TTM) in Peanut</span></h3><p class="c4"><span class="c0">Cultivated peanut (</span><span class="c9">Arachis hypogaea</span><span class="c0 c11">&nbsp;L.) is a globally significant legume crop, valued for its high oil and protein content. The timing of flowering initiation and the subsequent duration required to reach physiological maturity, often collectively referred to as Time-to-Maturation (TTM), are paramount traits influencing the crop&#39;s adaptability, productivity, and quality. Peanut exhibits a unique geocarpic reproductive strategy, producing aerial flowers but developing fruits (pods) underground after fertilization and the elongation of a specialized structure called the peg or gynophore. The duration from planting to maturity dictates the suitability of a cultivar for specific agro-ecological zones, particularly those defined by limited growing seasons or the prevalence of end-of-season stresses like drought. Early maturation (e.g., 90-120 days post-planting) is a critical adaptive feature in regions with short rainy seasons, such as West Africa and parts of India, enabling drought avoidance. Conversely, longer-maturing varieties may have higher yield potential in environments with extended favorable conditions.</span></p><p class="c4"><span class="c0 c11">The TTM significantly impacts peanut yield and economic value. Optimizing the match between a cultivar&#39;s maturity period and the available growing season is essential for maximizing pod development, seed filling, and overall harvest index (HI) &ndash; the ratio of pod weight to total biomass. Consequently, understanding and manipulating TTM is a primary objective in peanut breeding programs worldwide, aiming to develop cultivars with appropriate maturity durations for diverse production systems. The economic viability of peanut cultivation relies heavily on aligning planting and harvesting schedules with the specific maturation characteristics of the chosen cultivar to achieve optimal yield and grade.</span></p><p class="c4"><span class="c0 c11">It is important to distinguish between &quot;flowering time&quot; (often measured as days to first flower or days to 50% flowering) and the broader concept of &quot;Time-to-Maturation&quot; (TTM) or &quot;Maturity Index&quot; (MI), which integrates the entire period from planting to harvest readiness, encompassing flowering, pegging, and pod/seed development. While flowering time is a component of TTM, the overall maturation period is influenced by the duration of subsequent developmental stages as well. Some genetic factors might specifically affect flowering initiation, while others could influence the pace of pod development or overall plant senescence, thereby impacting TTM/MI. Therefore, markers associated with TTM or MI, while highly relevant to the overall goal of managing crop duration, may not exclusively control flowering initiation itself but rather the integrated process leading to maturity. Recognizing this distinction is crucial when interpreting the function and application of identified molecular markers.</span></p><h3 class="c20"><span class="c2">1.2. Genetic Complexity and Environmental Influences</span></h3><p class="c4"><span class="c0 c11">Flowering time and TTM in peanut are recognized as complex quantitative traits, meaning their expression is governed by the interplay of multiple genes (polygenic control) and is significantly modulated by environmental conditions. Factors such as photoperiod, temperature, soil moisture, altitude, and latitude can exert considerable influence on the phenology of peanut development. This complexity presents challenges for breeders attempting to select for specific maturity types consistently across different locations and years. Some studies have reported low heritability estimates for TTM, suggesting a strong environmental component relative to the genetic contribution in those specific populations and environments. For instance, the broad-sense heritability for MI in a Virginia-type RIL population was estimated at 0.39, indicating moderate-to-low but significant genetic influence. However, other studies investigating related yield and quality traits in diverse germplasm have reported high broad-sense heritability (e.g., 88-98%), and the identification of major Quantitative Trait Loci (QTLs) for various traits, including TTM/MI, demonstrates that a substantial and selectable genetic component underlies these traits. The observed heritability can depend heavily on the genetic diversity within the population studied, the environmental conditions, and the precision of phenotyping.</span></p><p class="c4"><span class="c0">The genetic architecture of peanut is further complicated by its evolutionary history. Cultivated peanut is a relatively recent allotetraploid (AABB genome, 2n = 4x = 40), originating from a single hybridization event between two wild diploid species, </span><span class="c9">Arachis duranensis</span><span class="c0">&nbsp;(A genome) and </span><span class="c9">Arachis ipa&euml;nsis</span><span class="c0 c11">&nbsp;(B genome), followed by spontaneous chromosome doubling approximately 9,000-10,000 years ago. This recent polyploidization event created a significant genetic bottleneck, resulting in a remarkably narrow genetic base within the cultivated species compared to its wild relatives. This limited genetic diversity historically hampered efforts in genetic mapping and marker development, as finding sufficient polymorphism between parental lines was often difficult. Furthermore, the presence of two similar, yet distinct, subgenomes (A and B) complicates genetic analyses, particularly the accurate identification and scoring of molecular markers like Single Nucleotide Polymorphisms (SNPs), requiring specialized bioinformatic approaches to distinguish between homoeologous variants (variants at corresponding locations on the A and B subgenomes) and true allelic variation. Significant genotype &times; environment (GxE) interactions are also commonly observed for peanut traits, implying that the effects of specific genes or QTLs on flowering time or TTM might vary depending on the environmental context, further complicating breeding efforts. Understanding these inherent biological complexities &ndash; the quantitative nature of the trait, environmental sensitivity, allotetraploidy, and narrow genetic base &ndash; is essential for interpreting genetic studies and devising effective breeding strategies.</span></p><h3 class="c20"><span class="c2">1.3. Role of Molecular Markers and Genomic Approaches</span></h3><p class="c4"><span class="c0 c11">Despite the challenges, significant progress has been made in dissecting the genetic basis of flowering time, TTM, and other important agronomic traits in peanut through the application of molecular markers and advanced genomic techniques. QTL mapping in biparental populations (such as Recombinant Inbred Lines - RILs) and Genome-Wide Association Studies (GWAS) in diverse germplasm panels have emerged as powerful strategies for identifying genomic regions associated with phenotypic variation. These approaches allow researchers to link specific DNA sequence variations, such as Simple Sequence Repeats (SSRs), SNPs, or Insertions/Deletions (InDels), to differences in traits like flowering time or maturity.</span></p><p class="c4"><span class="c0">The identification of robust molecular markers tightly linked to genes controlling TTM is highly valuable for practical breeding applications. Marker-Assisted Selection (MAS) utilizes these DNA markers to predict the phenotypic potential of individual plants early in the breeding cycle, often at the seedling stage. This allows breeders to select desirable genotypes more efficiently, accurately, and rapidly compared to relying solely on traditional phenotypic selection, which can be time-consuming, labor-intensive, and confounded by environmental effects. The development of peanut genomics has seen a transition from using relatively low-density marker systems like SSRs to employing high-throughput SNP genotyping arrays and whole-genome resequencing approaches. These advancements provide much greater marker density and genome coverage, significantly enhancing the resolution and power of genetic mapping studies, even within the context of peanut&#39;s low polymorphism and polyploid complexity. This report synthesizes current knowledge on molecular markers and genomic regions associated with flowering time and TTM in </span><span class="c9">Arachis hypogaea</span><span class="c0 c11">, drawing upon findings from QTL mapping and GWAS studies, and discusses their potential application in peanut improvement.</span></p><h2 class="c20"><span class="c22 c11">2. Genetic Architecture of Flowering Time in Peanut</span></h2><h3 class="c20"><span class="c2">2.1. Subspecies Differentiation</span></h3><p class="c20"><span class="c0">The cultivated peanut species, </span><span class="c9">A. hypogaea</span><span class="c0">, is primarily classified into two major subspecies: ssp. </span><span class="c9">hypogaea</span><span class="c0">&nbsp;and ssp. </span><span class="c9">fastigiata</span><span class="c0 c11">. These subspecies represent distinct evolutionary lineages that likely arose during the processes of domestication and diversification in South America. They exhibit contrasting characteristics, particularly in growth habit, flowering patterns, and maturity duration.</span></p><ul class="c26 lst-kix_list_1-0 start"><li class="c16 li-bullet-0"><span class="c9">A. hypogaea</span><span class="c7">&nbsp;ssp. </span><span class="c9">fastigiata</span><span class="c0">: This subspecies includes the Spanish and Valencia market types. Genotypes within </span><span class="c9">fastigiata</span><span class="c0 c11">&nbsp;are generally characterized by an erect growth habit, determinate shoot formation (the main stem terminates in an inflorescence), sequential flowering (presence of flowers on the main stem, often denoted as MSF+), and typically early fruit maturation (often 90-120 days).</span></li><li class="c1 c25 li-bullet-0"><span class="c9">A. hypogaea</span><span class="c7">&nbsp;ssp. </span><span class="c9">hypogaea</span><span class="c0">: This subspecies encompasses the Virginia and Runner market types. Genotypes within </span><span class="c9">hypogaea</span><span class="c0 c11">&nbsp;usually display a spreading or bunch growth habit, indeterminate shoot formation (the main stem remains vegetative), an alternative flowering pattern (absence of flowers on the main stem, MSF-), and are generally later maturing (often 130-170 days).</span></li></ul><p class="c30"><span class="c0 c11">This fundamental divergence in flowering pattern (sequential vs. alternative) and associated maturity level represents a major axis of genetic variation for phenological traits in peanut. The genetic control of these subspecies-defining traits, particularly the presence or absence of mainstem flowering, has been a subject of significant research interest.</span></p><h3 class="c20"><span class="c2">2.2. Inheritance Patterns</span></h3><p class="c4"><span class="c0 c11">Flowering time and TTM are generally considered quantitative traits in peanut, controlled by multiple genes (polygenes), each potentially contributing a small to moderate effect on the overall phenotype. This polygenic control, combined with the significant environmental influence discussed earlier, contributes to the continuous variation observed for these traits in segregating populations and germplasm collections.</span></p><p class="c4"><span class="c0 c11">While some studies reported low heritability for TTM, suggesting challenges in selection, numerous QTL mapping and GWAS investigations have successfully identified specific genomic regions (QTLs) that exert a measurable influence on flowering time, TTM, or related traits. The identification of major QTLs, defined as those explaining a substantial portion of the phenotypic variance (e.g., &gt;10% PVE), has been reported not only for TTM but also for other complex traits in peanut, such as pod and seed characteristics, shelling percentage, and disease resistance. This indicates that despite the polygenic nature, genes with larger effects do exist and can be targeted for selection. For example, the locus controlling mainstem flowering (MSF) acts like a major gene differentiating the subspecies. Studies investigating the inheritance of related traits, like growth habit (lateral branch angle - LBA), suggested control by multiple genes, as segregation ratios in an F2 population did not fit a simple single-gene model.</span></p><p class="c4"><span class="c0">It is plausible, and indeed likely, that the specific genetic architecture underlying flowering time variation differs depending on the genetic backgrounds being compared. Studies involving crosses </span><span class="c9">between</span><span class="c0">&nbsp;the two subspecies (</span><span class="c9">hypogaea</span><span class="c0">&nbsp;x </span><span class="c9">fastigiata</span><span class="c0">) are likely to capture the effects of major genes differentiating them, such as the MSF locus. In contrast, studies focusing on variation </span><span class="c9">within</span><span class="c0 c11">&nbsp;a single subspecies, such as the analysis of TTM variation among Virginia-type cultivars, may primarily identify QTLs representing modifiers of maturity within that specific genetic background. These intra-subspecific QTLs might have smaller individual effects compared to the major subspecies-differentiating loci but are nonetheless crucial for fine-tuning maturity within a given market class. Therefore, comparing QTL results across different studies requires careful consideration of the parental materials and the type of genetic variation being investigated (inter- vs. intra-subspecific).</span></p><h2 class="c20"><span class="c11 c22">3. Quantitative Trait Loci (QTLs) Associated with Flowering Time / Time-to-Maturation</span></h2><h3 class="c20"><span class="c2">3.1. Overview of QTL Mapping Studies</span></h3><p class="c4"><span class="c0 c11">QTL mapping has been extensively applied in peanut genetics and breeding to identify genomic regions controlling economically important traits. Despite the inherent challenges posed by the crop&#39;s narrow genetic base and allotetraploid genome, numerous studies have successfully mapped QTLs for a wide range of characteristics, including yield components, oil and protein quality, drought tolerance, disease resistance, and pod/seed morphology. Several studies have specifically focused on identifying QTLs associated with flowering time, TTM, or related phenological traits.</span></p><p class="c4"><span class="c0 c11">These studies typically employ structured populations, most commonly RIL populations derived from crosses between parents exhibiting contrasting phenotypes for the trait of interest. Genetic linkage maps are constructed using molecular markers (historically SSRs, more recently high-density SNPs), and phenotypic data collected across multiple environments (locations and/or years) are analyzed to detect statistical associations between marker genotypes and trait variation. Early studies were sometimes limited by the low density and polymorphism of available markers, but the advent of high-throughput SNP genotyping has enabled the construction of much more saturated maps, improving the precision and power of QTL detection.</span></p><h3 class="c20"><span class="c2">3.2. Key QTLs for TTM / Maturity Index (MI)</span></h3><p class="c20"><span class="c0">A significant study investigating the genetic control of TTM within the late-maturing Virginia market type (ssp. </span><span class="c9">hypogaea</span><span class="c0">) utilized an RIL population derived from a cross between cultivars &#39;Hanoch&#39; and &#39;Harari&#39;, which differed in TTM by 12-15 days. Phenotyping for Maturity Index (MI) over two years (2018 and 2019) and genotyping with a high-density SNP array led to the identification of four QTL regions for MI. Notably, two of these regions showed consistent effects across both years, representing robust QTLs for TTM in this genetic background.</span><span class="c10 c12">1</span></p><ul class="c26 lst-kix_list_2-0 start"><li class="c16 li-bullet-0"><span class="c7">qMIA04a,b (Chromosome A04):</span><span class="c0">&nbsp;This consistent QTL was detected on chromosome A04 in both 2018 (</span><span class="c9">qMIA04a</span><span class="c0">) and 2019 (</span><span class="c9">qMIA04b</span><span class="c0 c11">).</span></li></ul><ul class="c26 lst-kix_list_3-1 start"><li class="c18 li-bullet-0"><span class="c7">Location:</span><span class="c0">&nbsp;Chromosome A04. The physical interval identified spanned approximately from 117.6 Mbp to 125.6 Mbp in 2018 and 118.6 Mbp to 125.6 Mbp in 2019, based on the flanking SNP markers mapped to the Tifrunner reference genome. This corresponds to a genomic region of roughly 6.9 to 8 Mbp.</span><span class="c10 c12">1</span></li><li class="c1 c33 li-bullet-0"><span class="c7">Flanking Markers:</span><span class="c0">&nbsp;The interval was defined by SNP markers AX-176802283_A04 and AX-176815499_A04 (2018), and AX-176819644_A04 and AX-176815499_A04 (2019).</span><span class="c10 c12">1</span></li><li class="c1 c33 li-bullet-0"><span class="c7">Statistical Significance:</span><span class="c0">&nbsp;The QTL exhibited Logarithm of Odds (LOD) scores of 5.33 (2018) and 6.45 (2019), explaining 9.9% and 11.9% of the phenotypic variance (PVE) for MI in the respective years.</span><span class="c10 c12">1</span></li></ul><ul class="c26 lst-kix_list_2-0"><li class="c1 c25 li-bullet-0"><span class="c7">qMIB03a,b (Chromosome B03):</span><span class="c0">&nbsp;A second consistent QTL was identified on chromosome B03 in both years (</span><span class="c9">qMIB03a</span><span class="c0">&nbsp;in 2018, </span><span class="c9">qMIB03b</span><span class="c0 c11">&nbsp;in 2019).</span></li></ul><ul class="c26 lst-kix_list_4-1 start"><li class="c18 li-bullet-0"><span class="c7">Location:</span><span class="c0">&nbsp;Chromosome B03. The physical interval was mapped from approximately 2.8 Mbp to 4.6 Mbp (2018) and 2.8 Mbp to 5.7 Mbp (2019), representing a region of about 1.8 to 2.9 Mbp.</span><span class="c10 c12">1</span></li><li class="c1 c33 li-bullet-0"><span class="c7">Flanking Markers:</span><span class="c0">&nbsp;Defined by SNP markers AX-176807311_B03 and AX-176806413_B03 (2018), and AX-176807311_B03 and AX-176801237_B03 (2019).</span><span class="c10 c12">1</span></li><li class="c1 c33 li-bullet-0"><span class="c7">Statistical Significance:</span><span class="c0">&nbsp;This QTL showed LOD scores of 5.0 (2018) and 5.35 (2019), explaining 9.3% and 9.9% of the PVE for MI, respectively.</span><span class="c10 c12">1</span></li></ul><p class="c30"><span class="c0 c11">Interestingly, the study also found that QTLs for Harvest Index (HI) co-localized with the MI QTLs on chromosomes B03, as well as on B05 and B06. This co-localization suggests either pleiotropy (where the same gene influences both traits) or tight linkage of separate genes controlling MI and HI within these genomic regions. This finding underscores the intricate relationship between plant development duration and yield accumulation efficiency.</span></p><h3 class="c20"><span class="c2">3.3. QTL for Mainstem Flowering (MSF) / Flowering Pattern</span></h3><p class="c20"><span class="c0">As previously discussed, the presence or absence of flowers on the main stem (MSF) is a key morphological trait distinguishing ssp. </span><span class="c9">fastigiata</span><span class="c0">&nbsp;(MSF+) from ssp. </span><span class="c9">hypogaea</span><span class="c0">&nbsp;(MSF-). A study specifically aimed at mapping this trait utilized SNP array genotyping in populations segregating for MSF.</span><span class="c10 c12">2</span></p><ul class="c26 lst-kix_list_5-0 start"><li class="c16 li-bullet-0"><span class="c7">Location:</span><span class="c0">&nbsp;The major locus controlling MSF was mapped to the distal end of chromosome B02.</span><span class="c10">2</span><span class="c0">&nbsp;Initial mapping placed it within a 3.1 Mbp interval (between markers AX-176791582_B02 and AX-176803063_B02 based on the genetic map), which was refined using the physical Tifrunner genome sequence to a 1.7 Mbp segment defined by markers AX-176805026_B02 and AX-176803063_B02.</span><span class="c10 c12">2</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Significance:</span><span class="c0">&nbsp;This locus represents a major genetic determinant responsible for the differing flowering patterns observed between the two primary subspecies and likely played a significant role during peanut domestication and diversification. Its large effect makes it a prime target for understanding the evolution of flowering habits in </span><span class="c9">Arachis</span><span class="c0 c11">.</span></li></ul><h3 class="c34"><span class="c2">3.4. Other Potential QTLs/Associations</span></h3><p class="c4"><span class="c0 c11">Other studies have also reported QTLs or associations related to flowering time, although sometimes with less detail provided in the available summaries. Wang et al. (2020), cited in, identified three main-effect QTLs for the initial flowering date using an RIL population, but specific genomic locations and marker details were not provided in the reference snippet.</span></p><p class="c4"><span class="c0">Genome-wide association studies (GWAS) offer a complementary approach to QTL mapping, leveraging historical recombination in diverse germplasm panels to identify marker-trait associations. While the provided materials do not contain detailed results from a GWAS specifically focused on flowering time, reference is made to a study that identified 259 SNP loci significantly associated with flowering time in a large panel of peanut cultivars, predicting seven candidate genes in associated regions.</span><span class="c10">3</span><span class="c0 c11">&nbsp;This highlights the utility of GWAS for exploring the genetic basis of this trait. Furthermore, GWAS studies on related traits have identified significant loci: pleiotropic QTLs on chromosome 16 (Chr16) were associated with multiple yield components, while loci on Chr08 were linked to protein and oil content. A GWAS for drought tolerance identified significant QTLs on multiple chromosomes (A03, A05, A06, A07, A08, B04, B05, B06, B09, B10) associated with pod weight and count under drought stress. Although not directly for flowering time under optimal conditions, these loci influencing yield and stress response could potentially interact with maturity pathways. A study on Groundnut Rosette Disease (GRD) resistance identified multiple significant SNPs on chromosome A04, a chromosome also harboring a major TTM QTL (qMIA04). While these GRD SNPs are linked to a different trait, their location warrants investigation for potential linkage or pleiotropic effects related to plant development or stress response influencing maturity.</span></p><h3 class="c20"><span class="c2">Table 1: Summary of Key QTLs for Flowering Time/TTM/Pattern in Peanut</span></h3><p class="c36"><span class="c2"></span></p><table class="c24"><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">QTL Name/ID</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Trait Measured</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Population / Parents</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Chromosome</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Genomic Region / Position</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Flanking/Peak Markers (SNP unless noted)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Peak LOD</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">PVE (%)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Marker Type</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Source Study Reference</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIA04a</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Maturity Index (MI)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">RIL / Hanoch x Harari</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">A04</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">~117.6 - 125.6 Mbp (2018)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176802283_A04 - AX-176815499_A04</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">5.33</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">9.9</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIA04b</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Maturity Index (MI)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">RIL / Hanoch x Harari</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">A04</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">~118.6 - 125.6 Mbp (2019)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176819644_A04 - AX-176815499_A04</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">6.45</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">11.9</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIB03a</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Maturity Index (MI)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">RIL / Hanoch x Harari</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">B03</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">~2.8 - 4.6 Mbp (2018)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176807311_B03 - AX-176806413_B03</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">5.00</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">9.3</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIB03b</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Maturity Index (MI)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">RIL / Hanoch x Harari</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">B03</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">~2.8 - 5.7 Mbp (2019)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176807311_B03 - AX-176801237_B03</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">5.35</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">9.9</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">MSF Locus</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Mainstem Flowering</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Segregating Populations</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">B02</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">~1.7 Mbp (Physical Map)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176805026_B02 - AX-176803063_B02</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">High*</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Major</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP, InDel</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">2</span></p></td></tr><tr class="c15"><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">QTL (unnamed)</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">Initial Flowering Date</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">RIL / Silihong x Jinonghei3</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c1"><span class="c6">SSR?</span></p></td><td class="c3" colspan="1" rowspan="1"><p class="c29"><span class="c6"></span></p></td></tr></table><p class="c4"><span class="c9">Note: Specific LOD for MSF locus varied depending on marker; significantly higher LOD obtained using the AhTFL1 InDel marker.</span><span class="c39">2</span><span class="c8">&nbsp;PVE considered major as it differentiates subspecies.</span></p><h2 class="c20"><span class="c22 c11">4. Molecular Markers (SNPs, SSRs, InDels) for Peanut Flowering Time</span></h2><h3 class="c20"><span class="c2">4.1. Overview of Marker Types Used</span></h3><p class="c4"><span class="c0 c11">The exploration of peanut genetics has utilized various types of molecular markers over time. Early studies employed techniques like Random Amplified Polymorphic DNA (RAPD), Amplified Fragment Length Polymorphism (AFLP), and Inter Simple Sequence Repeat (ISSR) markers. However, these markers often suffered from limitations in reproducibility or genome coverage.</span></p><p class="c4"><span class="c0 c11">Simple Sequence Repeat (SSR) markers, also known as microsatellites, became more prevalent due to their co-dominant nature and relative abundance. Thousands of SSRs were developed from SSR-enriched genomic libraries and Expressed Sequence Tags (ESTs) in peanut. SSRs were instrumental in constructing early genetic linkage maps and assessing genetic diversity. However, a major limitation encountered in cultivated peanut was the low level of polymorphism detected with SSRs; estimates suggested less than 15% of developed SSRs showed polymorphism between cultivated peanut lines, with an even smaller fraction being mapped, largely due to the crop&#39;s narrow genetic base resulting from the polyploidization bottleneck. For instance, a genetic map used for QTL mapping of gynophore-pod structure traits utilized 830 polymorphic SSR markers.</span></p><p class="c4"><span class="c0">More recently, Single Nucleotide Polymorphisms (SNPs) have become the dominant marker type in peanut genomics, driven by advancements in sequencing and genotyping technologies. High-density SNP arrays, such as the Affymetrix Axiom_Arachis array series (e.g., Axiom_Arachis2 used in), allow for the simultaneous genotyping of tens of thousands of SNPs across the genome.</span><span class="c10">1</span><span class="c0 c11">&nbsp;Furthermore, whole-genome resequencing (WGRS) of parental lines or entire populations enables the discovery of millions of SNPs and other variants. These high-density SNP datasets have facilitated the construction of highly saturated genetic maps (e.g., 1833 SNPs, 5120 SNPs) and powered GWAS studies, significantly improving the resolution for locating trait-associated loci.</span></p><p class="c4"><span class="c0">Insertion-Deletion (InDel) markers represent another valuable source of polymorphism. InDels arise from the insertion or deletion of one or more nucleotides at a specific genomic location. Studies have suggested that InDels might exhibit higher levels of polymorphism than SSRs in some crops, and they can be readily detected. Research in peanut has involved the development of gene-specific InDel markers, with some showing utility in diversity analysis and potential for trait association. A functionally significant 1492 bp deletion within the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;gene serves as a powerful marker for the MSF trait.</span><span class="c10 c12">2</span></p><h3 class="c20"><span class="c2">4.2. Specific Markers Associated with Key QTLs</span></h3><p class="c20"><span class="c0 c11">Based on the QTL mapping studies discussed, specific markers have been identified within or flanking the key genomic regions associated with TTM/MI and MSF.</span></p><ul class="c26 lst-kix_list_6-0 start"><li class="c16 li-bullet-0"><span class="c7">Markers for qMIA04a,b (TTM QTL on A04):</span><span class="c0 c11">&nbsp;The consistent QTL for Maturity Index on chromosome A04, identified in the Hanoch x Harari RIL population, was flanked by SNP markers derived from the Axiom_Arachis array. Key markers defining this region include:</span></li></ul><ul class="c26 lst-kix_list_7-1 start"><li class="c18 li-bullet-0"><span class="c0 c11">AX-176802283_A04</span></li><li class="c1 c33 li-bullet-0"><span class="c0 c11">AX-176819644_A04</span></li><li class="c1 c33 li-bullet-0"><span class="c0">AX-176815499_A04 These markers delimit the ~117.6-125.6 Mbp region associated with TTM variation in Virginia-type peanuts.</span><span class="c10 c12">1</span></li></ul><ul class="c26 lst-kix_list_6-0"><li class="c1 c25 li-bullet-0"><span class="c7">Markers for qMIB03a,b (TTM QTL on B03):</span><span class="c0 c11">&nbsp;Similarly, the consistent TTM QTL on chromosome B03 in the same population was associated with the following Axiom array SNP markers:</span></li></ul><ul class="c26 lst-kix_list_8-1 start"><li class="c18 li-bullet-0"><span class="c0 c11">AX-176807311_B03</span></li><li class="c1 c33 li-bullet-0"><span class="c0 c11">AX-176806413_B03</span></li><li class="c1 c33 li-bullet-0"><span class="c0">AX-176801237_B03 These markers define the ~2.8-5.7 Mbp interval linked to MI variation.</span><span class="c10 c12">1</span></li></ul><ul class="c26 lst-kix_list_6-0"><li class="c1 c25 li-bullet-0"><span class="c7">Markers for MSF Locus (Flowering Pattern on B02):</span><span class="c0 c11">&nbsp;The major locus controlling mainstem flowering at the end of chromosome B02 was mapped using SNP markers and further validated with a gene-specific InDel:</span></li></ul><ul class="c26 lst-kix_list_9-1 start"><li class="c18 li-bullet-0"><span class="c0 c11">AX-176805026_B02 (SNP)</span></li><li class="c1 c33 li-bullet-0"><span class="c0 c11">AX-176803063_B02 (SNP)</span></li><li class="c1 c33 li-bullet-0"><span class="c0 c11">AX-176791582_B02 (SNP)</span></li><li class="c1 c33 li-bullet-0"><span class="c0">A 1492 bp deletion within the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;gene (InDel). This deletion serves as a highly diagnostic marker for the presence/absence of mainstem flowering, showing a very strong association with the trait.</span><span class="c10 c12">2</span></li></ul><ul class="c26 lst-kix_list_6-0"><li class="c1 c25 li-bullet-0"><span class="c7">Markers from GWAS and Other Studies:</span><span class="c0">&nbsp;While specific sequences are often not provided directly for GWAS hits in summary reports, marker identifiers are usually given. For example, GWAS for GRD resistance listed SNP IDs like AX-147219775, AX-147219783, etc., on chromosome A04. Markers developed from resequencing data for KASP assays are often named based on their genomic position, such as A12.4097252 (Chromosome A12, position 4,097,252 bp), which was developed for bacterial wilt resistance. Such position-based nomenclature, tied to a reference genome assembly (like Tifrunner </span><span class="c10">2</span><span class="c0 c11">), provides a standardized way to identify and locate markers discovered through sequencing, overcoming the variability seen in older marker naming conventions (e.g., SSR prefixes like &#39;Ah-&#39;, &#39;TC-&#39;, &#39;Seq-&#39; seen in). Careful tracking of marker IDs and their corresponding genomic coordinates is essential when comparing results across different studies and platforms.</span></li></ul><h3 class="c34"><span class="c2">4.3. Marker Sequences and Accessibility</span></h3><p class="c4"><span class="c0">Obtaining the exact DNA sequences for molecular markers can vary depending on the marker type and origin. For SNP markers identified using commercial arrays (e.g., Affymetrix Axiom &quot;AX-&quot; markers), the specific sequence information surrounding the SNP might be proprietary or require access to manufacturer-provided annotation files. However, the genomic coordinates (chromosome and base pair position) provided </span><span class="c10">1</span><span class="c0 c11">&nbsp;allow researchers to locate the marker position within publicly available reference genome sequences (e.g., Tifrunner genome available via PeanutBase or NCBI). This enables retrieval of the flanking sequence context necessary for designing alternative assays or further investigation.</span></p><p class="c4"><span class="c0">For markers developed from WGRS, such as SNPs targeted for KASP assays or specific InDels like the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;deletion </span><span class="c10">2</span><span class="c0 c11">, the sequence variation itself and the flanking sequences required for primer/probe design are typically provided in the supplementary materials of the original publication or can be readily extracted from the reference genome using the reported coordinates.</span></p><p class="c4"><span class="c0">For SSR markers, the original publications usually report the forward and reverse primer sequences needed to amplify the microsatellite region. Databases like PeanutBase </span><span class="c10">4</span><span class="c0 c11">&nbsp;serve as central repositories aiming to consolidate marker information, including sequences and genomic locations where available. Researchers seeking to utilize specific markers should consult the primary literature and relevant databases for the most accurate and complete sequence information.</span></p><h3 class="c20"><span class="c2">4.4. Statistical Validation</span></h3><p class="c4"><span class="c0">The reliability of a molecular marker for research or breeding purposes depends on the statistical strength of its association with the trait of interest. For QTLs identified through linkage mapping, the LOD score is a key statistic, indicating the likelihood of linkage versus chance association. A commonly accepted threshold for declaring a significant QTL is a LOD score of 2.5 or 3.0, although this can vary. The TTM QTLs qMIA04a,b and qMIB03a,b exhibited peak LOD scores ranging from 5.0 to 6.45, well above typical significance thresholds, indicating strong statistical support for their association with Maturity Index.</span><span class="c10">1</span><span class="c0">&nbsp;The MSF locus on B02 also showed high LOD scores, particularly when the functional </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;deletion marker was used, confirming its major role.</span><span class="c10 c12">2</span></p><p class="c4"><span class="c0">The proportion of phenotypic variance explained (PVE) by a QTL provides an estimate of its effect size. The consistent TTM QTLs qMIA04 and qMIB03 explained approximately 9-12% of the variation in MI each.</span><span class="c10">1</span><span class="c0 c11">&nbsp;While statistically significant, these represent moderate effects, consistent with the quantitative nature of TTM within the Virginia market class. In contrast, major QTLs for other traits, like rust resistance or pod weight, have been reported with PVE values exceeding 20%, 30%, or even 80% in some cases, indicating genes with larger impacts on those specific traits. The MSF locus, differentiating subspecies, effectively explains a very large proportion of the variance for the mainstem flowering trait.</span></p><p class="c4"><span class="c0 c11">For GWAS, significance is typically assessed using p-values, often adjusted for multiple testing (e.g., using False Discovery Rate - FDR). Stringent p-value thresholds (e.g., p&lt;10&minus;5 or 10&minus;6) or corresponding -log10(p) values (e.g., &gt; 4.5 or 5.0) are used to declare significant associations. PVE estimates can also be calculated for significant GWAS loci.</span></p><h3 class="c20"><span class="c2">Table 2: Detailed Molecular Markers Associated with Peanut Flowering Time/TTM/Pattern</span></h3><p class="c36"><span class="c2"></span></p><table class="c24"><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Marker ID</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Marker Type</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Chromosome</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c11 c14">Position (bp) (Tifrunner Ref.)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Associated QTL/Trait</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Sequence/Context/Link</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Significance (LOD / p-value)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">PVE (%)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c14 c11">Source Study Reference</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176802283_A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~117,600,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIA04a (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">LOD ~5.33 (Peak)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">9.9</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176819644_A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~118,600,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIA04b (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">LOD ~6.45 (Peak)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">11.9</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176815499_A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">A04</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~125,600,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIA04a/b (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Flanking marker</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176807311_B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~2,800,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIB03a/b (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">LOD ~5.0-5.35 (Peak)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">9.3-9.9</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176806413_B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~4,600,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIB03a (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Flanking marker</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176801237_B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B03</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">~5,700,000</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">qMIB03b (MI)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Flanking marker</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">-</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">1</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176805026_B02</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B02</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">End of Chromosome</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">MSF Locus</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Flanking marker</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Major</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">2</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">AX-176803063_B02</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">SNP</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B02</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">End of Chromosome</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">MSF Locus</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Position provided; Sequence context via genome browser</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Flanking marker</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Major</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">2</span></p></td></tr><tr class="c15"><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c27">AhTFL1</span><span class="c6">&nbsp;deletion</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">InDel</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">B02</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Within MSF Locus</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">MSF Locus</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">1492 bp deletion; Sequence in source publication/database</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">High LOD</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c6">Major</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c1"><span class="c10 c12">2</span></p></td></tr></table><p class="c4"><span class="c8">Note: Positions are approximate based on Mbp ranges provided in sources and mapped to the Tifrunner reference genome. Precise bp coordinates may require consulting source data or databases like PeanutBase.</span></p><h2 class="c20"><span class="c22 c11">5. Candidate Genes Implicated in Flowering Time Regulation</span></h2><p class="c4"><span class="c0 c11">Identifying the specific genes underlying QTLs is a crucial step towards understanding the biological mechanisms controlling traits and developing highly accurate markers. While QTL mapping points to genomic regions, these regions can contain numerous genes. Further fine-mapping or candidate gene approaches are needed to pinpoint the causal variants.</span></p><h3 class="c20"><span class="c2">5.1. Genes within QTL Regions</span></h3><ul class="c26 lst-kix_list_a-0 start"><li class="c16 li-bullet-0"><span class="c9">AhTFL1</span><span class="c7">&nbsp;(Terminal Flower 1 homolog):</span><span class="c0">&nbsp;The most compelling candidate gene identified in the context of peanut flowering pattern is </span><span class="c9">AhTFL1</span><span class="c0">, located within the major MSF locus on chromosome B02.</span><span class="c10">2</span><span class="c0">&nbsp;Homologs of </span><span class="c9">TFL1</span><span class="c0">&nbsp;in model plants like Arabidopsis typically act as floral repressors, maintaining indeterminate growth at the shoot apex and delaying flowering. The presence or absence of mainstem flowering in peanut subspecies aligns conceptually with differential regulation or function of a </span><span class="c9">TFL1</span><span class="c0">-like gene. The identification of a significant deletion within </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;that strongly correlates with the MSF phenotype provides strong evidence for its causal role in differentiating the alternative (MSF-, </span><span class="c9">hypogaea</span><span class="c0">) and sequential (MSF+, </span><span class="c9">fastigiata</span><span class="c0">) flowering patterns.</span><span class="c10">2</span><span class="c0">&nbsp;Sequence analysis suggested gain/loss events of this deletion contributed to the evolution of MSF in </span><span class="c9">Arachis</span><span class="c0 c11">. This represents a successful example of moving from a mapped locus to a highly plausible candidate gene with functional implications.</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Genes in TTM QTL regions (A04, B03):</span><span class="c0">&nbsp;For the consistent TTM QTLs identified on chromosomes A04 (qMIA04a,b) and B03 (qMIB03a,b) </span><span class="c10">1</span><span class="c0 c11">, the associated genomic intervals are relatively large (~1.8-8 Mbp). Such regions inevitably contain hundreds of genes. The source publications identifying these QTLs did not report specific candidate genes within these intervals responsible for the observed effects on Maturity Index. Identifying the causal genes within qMIA04 and qMIB03 will require further research, potentially involving fine-mapping with higher marker density in larger populations (as demonstrated for a bacterial wilt resistance QTL, qBWA12, which was narrowed down to a 216.7 kb region containing candidate genes), comparative genomics, and transcriptomic analyses (like RNA-seq) to identify differentially expressed genes between lines with contrasting alleles. While not directly related to flowering, studies on other traits have successfully identified candidate genes within QTL regions, such as seed maturation protein-coding genes for hundred-seed weight (HSW), highlighting the feasibility of this approach in peanut.</span></li></ul><h3 class="c34"><span class="c2">5.2. Gene Families Involved in Flowering Pathways</span></h3><p class="c20"><span class="c0 c11">Beyond genes located within specific QTL intervals, studies investigating key gene families known to regulate flowering time in other plant species provide additional candidates for controlling this trait in peanut.</span></p><ul class="c26 lst-kix_list_b-0 start"><li class="c16 li-bullet-0"><span class="c7">PEBP Family (FT/TFL1 homologs):</span><span class="c0">&nbsp;The Phosphatidylethanolamine-Binding Protein (PEBP) family includes critical regulators of the transition to flowering, notably the </span><span class="c9">FLOWERING LOCUS T</span><span class="c0">&nbsp;(</span><span class="c9">FT</span><span class="c0">) and </span><span class="c9">TERMINAL FLOWER 1</span><span class="c0">&nbsp;(</span><span class="c9">TFL1</span><span class="c0">) subfamilies. </span><span class="c9">FT</span><span class="c0">&nbsp;homologs generally act as floral promoters (florigen), moving from leaves to the shoot apex to induce flowering, while </span><span class="c9">TFL1</span><span class="c0">&nbsp;homologs act as repressors, maintaining vegetative growth. A study systematically identified PEBP family members in the peanut genome and analyzed their expression patterns.</span><span class="c10">5</span><span class="c0">&nbsp;Several specific peanut PEBP genes were highlighted as potentially key regulators responding to photoperiod and temperature cues, suggesting their involvement in controlling flowering time: </span><span class="c9">ArahyM2THPA</span><span class="c0">, </span><span class="c9">ArahyEM6VH3</span><span class="c0">, </span><span class="c9">Arahy4GAQ4U</span><span class="c0">, </span><span class="c9">ArahyIZ8FG5</span><span class="c0">, </span><span class="c9">ArahyG6F3P2</span><span class="c0">, </span><span class="c9">ArahyLUT2QN</span><span class="c0">, </span><span class="c9">ArahyDYRS20</span><span class="c0">, and </span><span class="c9">ArahyBBG51B</span><span class="c0">.</span><span class="c10">5</span><span class="c0">&nbsp;While </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;(likely </span><span class="c9">Arahy.TFL1b</span><span class="c0 c11">&nbsp;based on) is implicated in the MSF locus, the roles and genomic locations of these other PEBP candidates relative to the TTM QTLs on A04 and B03 require further investigation.</span></li><li class="c1 c25 li-bullet-0"><span class="c7">AhCOL Family (CONSTANS-like):</span><span class="c0">&nbsp;The </span><span class="c9">CONSTANS</span><span class="c0">&nbsp;(</span><span class="c9">CO</span><span class="c0">) gene and its homologs (</span><span class="c9">CONSTANS-like</span><span class="c0">, </span><span class="c9">COL</span><span class="c0">) play a central role in the photoperiodic flowering pathway, typically acting upstream of </span><span class="c9">FT</span><span class="c0">&nbsp;to regulate its expression in response to day length. An analysis of the peanut genome identified 18 </span><span class="c9">AhCOL</span><span class="c0">&nbsp;genes.</span><span class="c10">3</span><span class="c0">&nbsp;Expression profiling revealed that many </span><span class="c9">AhCOL</span><span class="c0">&nbsp;genes are expressed in leaves and flowers. Notably, </span><span class="c9">AhCOL4</span><span class="c0">, </span><span class="c9">AhCOL8</span><span class="c0">, </span><span class="c9">AhCOL13</span><span class="c0">, and </span><span class="c9">AhCOL14</span><span class="c0">&nbsp;were implicated in light response and potentially pod development.</span><span class="c10">3</span><span class="c0">&nbsp;Furthermore, protein interaction studies suggested </span><span class="c9">AhCOL1</span><span class="c0">&nbsp;and </span><span class="c9">AhCOL3</span><span class="c0">&nbsp;might play roles in flowering regulation.</span><span class="c10">3</span><span class="c0">&nbsp;These </span><span class="c9">AhCOL</span><span class="c0 c11">&nbsp;genes represent potential upstream regulators of flowering time in peanut, although their specific contributions and linkage to known QTLs need to be established.</span></li></ul><p class="c30"><span class="c0">Pinpointing causal genes within QTL intervals, especially large ones identified in initial mapping studies, remains a significant challenge in polyploid crops like peanut. The successful identification of </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;as the likely gene underlying the major MSF locus demonstrates feasibility.</span><span class="c10">2</span><span class="c0 c11">&nbsp;However, for quantitative traits influenced by multiple loci with smaller effects, like TTM within a subspecies, integrating fine-mapping, transcriptomics, and candidate gene approaches based on known flowering pathways (involving PEBP, COL, and other families) will be necessary to unravel the precise molecular mechanisms.</span></p><h2 class="c20"><span class="c22 c11">6. Bioinformatic Resources and Databases</span></h2><p class="c4"><span class="c0 c11">Access to centralized and well-curated bioinformatic resources is essential for modern plant genetics research and breeding. Several key databases provide valuable information for peanut researchers studying flowering time and other traits.</span></p><h3 class="c20"><span class="c2">6.1. PeanutBase</span></h3><p class="c20"><span class="c0">PeanutBase (peanutbase.org) stands as the primary, community-driven online resource dedicated to peanut genomics and genetics. It serves as a vital hub integrating diverse data types relevant to the study of flowering time and other traits. Key features include </span><span class="c10">4</span><span class="c0 c11">:</span></p><ul class="c26 lst-kix_list_c-0 start"><li class="c16 li-bullet-0"><span class="c7">Genome Sequences:</span><span class="c0">&nbsp;Access to reference genome assemblies for cultivated peanut (e.g., </span><span class="c9">A. hypogaea</span><span class="c0">&nbsp;cv. Tifrunner </span><span class="c10">2</span><span class="c0">) and its diploid progenitors (</span><span class="c9">A. duranensis</span><span class="c0">, </span><span class="c9">A. ipa&euml;nsis</span><span class="c0 c11">).</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Genetic Maps and Markers:</span><span class="c0 c11">&nbsp;Collections of genetic linkage maps and information on molecular markers (SSRs, SNPs, etc.), including their sequences and positions where available.</span></li><li class="c1 c25 li-bullet-0"><span class="c7">QTL and GWAS Data:</span><span class="c0">&nbsp;Dedicated tools and browsers (e.g., Trait Search Tool, QTL/GWAS viewers) allow users to search for and visualize QTLs and GWAS hits associated with specific traits, including potentially flowering time or TTM.</span><span class="c10 c12">4</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Gene Information:</span><span class="c0 c11">&nbsp;Gene models, functional annotations, gene family classifications, and orthology relationships with other species.</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Trait Descriptions:</span><span class="c0 c11">&nbsp;Information on various peanut traits and phenotypic characteristics.</span></li></ul><p class="c30"><span class="c0">PeanutBase provides the necessary framework for researchers to locate markers like those listed in Table 2 within the genomic context, retrieve flanking sequences, explore genes within QTL regions, and access curated trait association data.</span><span class="c10 c12">4</span></p><h3 class="c20"><span class="c2">6.2. NCBI Databases</span></h3><p class="c20"><span class="c0 c11">The National Center for Biotechnology Information (NCBI) hosts fundamental biological databases widely used by the scientific community.</span></p><ul class="c26 lst-kix_list_d-0 start"><li class="c16 li-bullet-0"><span class="c7">PubMed:</span><span class="c0 c11">&nbsp;Provides access to the vast majority of peer-reviewed scientific literature, including the publications cited throughout this report (e.g.,).</span></li><li class="c1 c25 li-bullet-0"><span class="c7">GenBank:</span><span class="c0 c11">&nbsp;A comprehensive public database of nucleotide sequences. Researchers can deposit and retrieve sequences for genes, markers, and entire genomes. Marker sequences or flanking regions, if made public by the original researchers, can often be found here.</span></li></ul><h3 class="c34"><span class="c2">6.3. Other Resources</span></h3><p class="c4"><span class="c0 c11">Other specialized databases, such as the Legume Information System (LIS), also aggregate genomic data for legume species and may contain relevant information or tools for comparative genomics involving peanut.</span></p><p class="c4"><span class="c0 c11">Leveraging these bioinformatic resources is crucial for accessing marker sequences, validating genomic locations, identifying candidate genes within QTL intervals, and staying updated on the latest research findings related to peanut flowering time genetics.</span></p><h2 class="c20"><span class="c22 c11">7. Application in Marker-Assisted Selection (MAS) and Future Prospects</span></h2><h3 class="c20"><span class="c2">7.1. MAS Potential</span></h3><p class="c4"><span class="c0 c11">A primary motivation for identifying QTLs and associated molecular markers is their potential application in Marker-Assisted Selection (MAS) to enhance breeding efficiency. By using DNA markers linked to desirable alleles for flowering time or TTM, breeders can screen large populations of plants at an early stage, selecting individuals with the desired genetic makeup without needing to wait for laborious and environmentally sensitive field phenotyping. This can significantly shorten breeding cycles and increase the accuracy of selection, particularly for complex traits like TTM.</span></p><p class="c4"><span class="c0">The SNP markers identified for the TTM QTLs (qMIA04, qMIB03) and the MSF locus (especially the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;deletion) hold promise for MAS.</span><span class="c10">1</span><span class="c0 c11">&nbsp;SNPs are particularly amenable to high-throughput, cost-effective genotyping platforms like Kompetitive Allele-Specific PCR (KASP). Successful development and validation of KASP markers for yield, quality, and disease resistance traits in peanut have already been demonstrated, providing a clear pathway for converting the identified flowering time/TTM markers into practical breeding tools. For instance, the KASP marker A12.4097252, developed for a major bacterial wilt resistance QTL, was validated in a large collection of breeding lines, proving its utility for MAS. Similar validation efforts would be necessary for the TTM markers.</span></p><h3 class="c20"><span class="c2">7.2. Challenges and Future Directions</span></h3><p class="c20"><span class="c0 c11">Despite the potential, several challenges need to be addressed for the effective implementation of MAS for flowering time/TTM in peanut.</span></p><ul class="c26 lst-kix_list_e-0 start"><li class="c16 li-bullet-0"><span class="c7">Marker Validation:</span><span class="c0">&nbsp;QTL effects can sometimes be specific to the population in which they were identified or influenced by the environment (GxE interactions). Therefore, markers linked to qMIA04 and qMIB03 need rigorous validation across diverse genetic backgrounds and multiple environments relevant to target breeding programs before widespread deployment. The consistency of these QTLs across two years in the original study is encouraging, but broader validation is crucial. In contrast, the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;deletion marker for MSF, being directly linked to a likely causal gene with a large, qualitative effect differentiating subspecies, is expected to be more robust and immediately applicable in breeding scenarios involving crosses between </span><span class="c9">fastigiata</span><span class="c0">&nbsp;and </span><span class="c9">hypogaea</span><span class="c0">&nbsp;types.</span><span class="c10 c12">2</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Fine-Mapping and Gene Identification:</span><span class="c0">&nbsp;The TTM QTL intervals on A04 and B03 are still relatively large.</span><span class="c10">1</span><span class="c0 c11">&nbsp;Fine-mapping these regions to narrow down the location of the causal genes would increase marker accuracy (by reducing the chance of recombination between the marker and the gene) and provide deeper biological insights.</span></li><li class="c1 c25 li-bullet-0"><span class="c7">Polyploid Complexity:</span><span class="c0 c11">&nbsp;Continued development of analytical tools and strategies specifically designed for polyploid genomes will aid in more accurate marker discovery and genotyping.</span></li></ul><p class="c34"><span class="c0 c11">Future research should focus on:</span></p><ol class="c26 lst-kix_list_f-0 start" start="1"><li class="c20 c21 li-bullet-0"><span class="c7">Fine-mapping</span><span class="c0 c11">&nbsp;the consistent TTM QTLs (qMIA04, qMIB03) to identify candidate genes and develop markers closer to the causal variants.</span></li><li class="c1 c21 li-bullet-0"><span class="c7">Functional characterization</span><span class="c0">&nbsp;of candidate genes, including </span><span class="c9">AhTFL1</span><span class="c0 c11">&nbsp;and members of the PEBP and COL families, to confirm their roles in regulating peanut flowering and maturity pathways.</span></li><li class="c1 c21 li-bullet-0"><span class="c7">Expanded GWAS</span><span class="c0 c11">&nbsp;using larger, more diverse germplasm panels and higher-density marker data to uncover additional loci contributing to flowering time variation across the species.</span></li><li class="c1 c21 li-bullet-0"><span class="c7">Validation</span><span class="c0 c11">&nbsp;of promising markers (like those flanking qMIA04 and qMIB03) in relevant breeding populations and target environments.</span></li><li class="c1 c21 li-bullet-0"><span class="c7">Integration</span><span class="c0 c11">&nbsp;of marker information into genomic selection (GS) models, which can simultaneously account for numerous small-effect loci across the genome, potentially offering advantages over traditional MAS for highly complex traits like TTM.</span></li></ol><h2 class="c34"><span class="c22 c11">8. Conclusion</span></h2><h3 class="c20"><span class="c2">8.1. Synthesis of Findings</span></h3><p class="c4"><span class="c0">Significant strides have been made in elucidating the genetic basis of flowering time and time-to-maturation in cultivated peanut, leveraging advances in molecular marker technology and genomic resources. QTL mapping and GWAS studies have successfully identified key genomic regions associated with these critical agronomic traits. Notably, consistent QTLs for TTM/Maturity Index have been mapped to chromosomes A04 (qMIA04a,b) and B03 (qMIB03a,b) in Virginia-type peanuts, explaining roughly 9-12% of the phenotypic variance each.</span><span class="c10">1</span><span class="c0">&nbsp;Furthermore, a major locus controlling the mainstem flowering pattern (MSF), a key trait differentiating ssp. </span><span class="c9">fastigiata</span><span class="c0">&nbsp;and ssp. </span><span class="c9">hypogaea</span><span class="c0">, has been pinpointed to the distal end of chromosome B02.</span><span class="c10 c12">2</span></p><p class="c4"><span class="c0">Specific molecular markers, primarily SNPs derived from array-based genotyping or resequencing, have been linked to these loci.</span><span class="c10">1</span><span class="c0">&nbsp;The identification of the </span><span class="c9">AhTFL1</span><span class="c0">&nbsp;gene, harboring a functional deletion within the MSF locus, represents a significant breakthrough in connecting a mapped QTL to a causal gene.</span><span class="c10">2</span><span class="c0">&nbsp;Additionally, studies exploring key flowering gene families have identified potential candidates from the PEBP (</span><span class="c9">FT/TFL1</span><span class="c0">-like) and COL (</span><span class="c9">CONSTANS</span><span class="c0">-like) families that likely contribute to the regulation of flowering time in response to environmental cues in peanut.</span><span class="c10">3</span><span class="c0">&nbsp;Bioinformatic resources like PeanutBase provide essential platforms for accessing and integrating this rapidly growing body of genetic and genomic information.</span><span class="c10 c12">4</span></p><h3 class="c20"><span class="c2">8.2. Implications for Research and Breeding</span></h3><p class="c4"><span class="c0 c11">The identified QTLs, markers, and candidate genes offer valuable resources for both fundamental research and applied peanut breeding. They provide insights into the genetic architecture of adaptation in peanut, the molecular pathways controlling flowering and maturity, and the evolutionary consequences of domestication and polyploidization.</span></p><p class="c28"><span class="c0">From a breeding perspective, the molecular markers associated with qMIA04, qMIB03, and the MSF locus hold considerable potential for use in Marker-Assisted Selection (MAS). The </span><span class="c9">AhTFL1</span><span class="c0 c11">&nbsp;deletion marker appears particularly robust for manipulating flowering patterns in inter-subspecific crosses. The TTM markers require further validation across diverse germplasm and environments but offer promising avenues for accelerating the development of cultivars with optimized maturity durations tailored to specific production regions and cropping systems. Continued research focused on fine-mapping existing QTLs, identifying causal genes, discovering novel loci through GWAS, and validating markers for breeding utility will further enhance our ability to understand and manipulate flowering time and maturity in peanut, ultimately contributing to improved crop productivity and resilience.</span></p><h4 class="c23"><span class="c11 c38">Works cited</span></h4><ol class="c26 lst-kix_list_10-0 start" start="1"><li class="c31 li-bullet-0"><span class="c13">Identification of consistent QTL for time to maturation in Virginia-type ..., accessed April 22, 2025, </span><span class="c19 c13"><a class="c32" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC8054412/&amp;sa=D&amp;source=editors&amp;ust=1745373148037777&amp;usg=AOvVaw2RcGChxdl9g3PskvCsDBKu">https://pmc.ncbi.nlm.nih.gov/articles/PMC8054412/</a></span></li><li class="c31 li-bullet-0"><span class="c13">Identification of a Major Locus for Flowering Pattern Sheds Light on ..., accessed April 22, 2025, </span><span class="c13 c19"><a class="c32" href="https://www.google.com/url?q=https://www.biorxiv.org/content/10.1101/2021.03.18.435916.full&amp;sa=D&amp;source=editors&amp;ust=1745373148038129&amp;usg=AOvVaw3pjNC7LgxvfwBHCuYYTEDu">https://www.biorxiv.org/content/10.1101/2021.03.18.435916.full</a></span></li><li class="c31 li-bullet-0"><span class="c13">Genome-Wide Analysis and Expression Profiles of AhCOLs Family ..., accessed April 22, 2025, </span><span class="c19 c13"><a class="c32" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/&amp;sa=D&amp;source=editors&amp;ust=1745373148038390&amp;usg=AOvVaw2lRgndhGzxl2wQJ_WRV1Ii">https://pmc.ncbi.nlm.nih.gov/articles/PMC11989928/</a></span></li><li class="c31 li-bullet-0"><span class="c13">PeanutBase: Home, accessed April 22, 2025, </span><span class="c19 c13"><a class="c32" href="https://www.google.com/url?q=https://www.peanutbase.org/&amp;sa=D&amp;source=editors&amp;ust=1745373148038590&amp;usg=AOvVaw0kmsxJKqSRV4LRIQjpoac1">https://www.peanutbase.org/</a></span></li><li class="c31 li-bullet-0"><span class="c13">Comparative Genomic and Expression Analysis Insight into ..., accessed April 22, 2025, </span><span class="c19 c13"><a class="c32" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC9604132/&amp;sa=D&amp;source=editors&amp;ust=1745373148038943&amp;usg=AOvVaw1R4u1bLyZs54yxnCca8Pit">https://pmc.ncbi.nlm.nih.gov/articles/PMC9604132/</a></span></li></ol></body></html>