---
layout:     news-item
title:      (February 2022) A new and improved Arachis ipaensis genome; Peanut expression atlas mapped to genome2 assembly
author:     Sudhansu Dash
date:       2022-02-14
summary:    New Arachis ipaensis assembly; Peanut comparative expression data
categories: jekyll pixyll
---

**New and improved assembly of <i>Arachis ipaensis</i>**

A completely new PacBio and Hi-C sequence and assembly of the genome of <i>Arachis ipaensis</i> is now available (with some usage restrictions) at the PeanutBase [Data Store](https://data.legumeinfo.org/Arachis/ipaensis/genomes/K30076.gnm2.1GWY/) as a result of a collaboration between USDA, University of Georgia, and Hudson Alpha. <i>Arachis ipaensis</i> is the B subgenome donor of cultivated peanut (<i>A. hypogaea</i>) and its wild counterpart <i>A. monticola</i>. This remarkable accession, K 30076, is the only collection of this species ever made. Its location, far from that of other B genome <i>Arachis</i> species, is incompatible with natural dispersal, indicating past transport by humans. Its extraordinarily high DNA identity with peanutâ€™s B subgenome and location, close to Villa Montes in Bolivia where the landraces of peanut that combine the most primitive, or wild-like, characteristics are grown indicate that it is descended from the very same population that gave rise to the tetraploid species.

**Peanut comparative expression data with other legumes**

The peanut expression atlas data ([Clevenger, et al. 2016](https://www.frontiersin.org/articles/10.3389/fpls.2016.01446/full)) after remapping the reads to gnm2.ann1 is now in the [CoNekT](https://conekt.legumeinfo.org/species/) tool where the expression patterns can be compared with their orthologs in other legumes at LIS. You can
  - [Browse](https://conekt.legumeinfo.org/species/view/5) the list of peanut genes with expression profiles in this dataset; click on the gene to visualize the expression profile with links to other features.
  - Visualize [expression profile of a single peanut gene](https://conekt.legumeinfo.org/sequence/view/150167).
  - [Compare the profiles of a group of genes](https://conekt.legumeinfo.org/profile_comparison/) of interest. (Paste this list as an example after choosing peanut for species: **arahy.Tifrunner.gnm2.ann1.0FWC6J arahy.Tifrunner.gnm2.ann1.5ZI49M arahy.Tifrunner.gnm2.ann1.61UWXT arahy.Tifrunner.gnm2.ann1.6T97B arahy.Tifrunner.gnm2.ann1.72B7PK** )
  - Look at the available legume orthologs ([gene family members](https://conekt.legumeinfo.org/family/find_forLIS/legfed_v1_0.L_BQRRPW)) and from there the comparative expression pattern ([tabular view](https://conekt.legumeinfo.org/heatmap/comparative/family/80617/row), [tree view](https://conekt.legumeinfo.org/tree/find_forLIS/legfed_v1_0.L_BQRRPW_tree)) in those genes.
  - Search for a [specific expression profile](https://conekt.legumeinfo.org/search/specific/profiles) at Conekt.
  - Coexpressed profile neighbors of a gene presented as a [limited network](https://conekt.legumeinfo.org/network/graph/190540) (accessible from the [profile page](https://conekt.legumeinfo.org/sequence/view/154464)).
  - An example of a peanut gene [arahy.Tifrunner.gnm2.ann1.T0173K](https://conekt.legumeinfo.org/sequence/view/194988) specifically expressed in roots like its orthologs in <i>Glycine</i> but showing [divergence in function](https://conekt.legumeinfo.org/tree/find_forLIS/legfed_v1_0.L_B3L2R6_tree) (expression pattern in non-root tissues) as a result of speciation in <i>Phaseolus</i> and <i>Vigna</i>.

