<!-- 
### JBrowse2: View Variant Data 
-->
**View Variant Data in JBrowse2**
(**Work in progress)
* <a href="https://dev.peanutbase.org/tools/jbrowse2/" target="_blank">Start a new session</a>  
* Select Linear Genome view and launch.  
* At Select the assembly (variant data must exist for this genome).  
  * Ardu.gnm1. Then, the feature (Chromosome, Aradu.A01). Open.  
  * A featureless viewer appears and tracks need to be selected.  
* At viewer.  
  * Open track selecter(&#9776;).  
  * In the "Available tracks panel" (right), choose 'Arachis duranensis V14167 Annotations'
  * In the "Available tracks panel" (right) under diversity, choose "aradu1_araip1.gnm1.div.Otyama_Kulkarni_2020.main.vcf".  
  

* Go to your favorite genic region (example: aradu.V14167.gnm1.Aradu.A01:300,002..528,417) 
  * In the "Available tracks panel" (right) under diversity, choose "aradu1_araip1.gnm1.div.Otyama_Kulkarni_2020.main.vcf".  
  * If needed zoom to a level where the features are available (may need to try a few zoom levels).  
  * Now one can visualize if region of interest has features in the proximity from the variant viewer.


<!--
-----
  * Zoom until the two needed Chrs are visible.  
  * Select the entire rectangle(click-drag) representing the two Chrs for comparison.  
  * Select zoom in.  An empty chart appears.
  * Click tract selector(&#9776;) 
  * In the "Available tracks panel" (right) choose aradu.gnm2-x-araca.gnm1.
* Dotplot of the selected region or Chr appears.  
**Next: (Optional)if need to add synteny view of the region**
* Select a region from dotplot (click-drag).
  * Select open linear synteny viewi from popup options.
  * A new view window appears below with synteny view.
-->

