---
layout: default
title: The Peanut Genome
---
<h1>The Peanut Genome</h1>

<p>This page describes the high-quality genome assembly for peanut (<i>Arachis hypogaea</i>), cultivar Tifrunner, which was released in December, 2017 (<a href="https://www.nature.com/articles/s41588-019-0405-z">Bertioli et al., 2019</a>), and the improved version 2.0. An explanation of the reason for the corrected version and a list of differences is available <a href="/genome/peanut_genome_v1_v2">here</a>.

<p><b>IMPORTANT NOTE:</b> the version 2.0 assembly (gnm2) for <i>Arachis hypogaea</i> var. Tifrunner has not been submitted to GenBank.

<p>Tifrunner is an important U.S. variety, with good market and growth characteristics and resistance to several peanut diseases (early and late leaf spot and TSWV/spotted wilt).

<p>This is a project of the <a href="/genome/IPGI">International Peanut Genome Initiative</a>, in order to accelerate breeding progress and get more productive, disease-resistant, stress-tolerant varieties to farmers. The IPGI project has sequenced the genomes of the <span style="color: red">*</span> <a href="https://peanutbase.org/browse_search#browse_genomes">two diploid progenitors of cultivated peanut</a>, as well as the genome of cultivated peanut itself.

<h4>Cultivated peanut</h4>
<p><b>Version 2 (gnm2):</b> <i>Arachis hypogaea</i> cv. Tifrunner: <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/genomes/Tifrunner.gnm2.J5K5/">assembly</a>, <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/annotations/Tifrunner.gnm2.ann1.4K0L/">annotation</a> (gnm2.ann1)
<br/>Genome browser: <span style="color: red">*</span> <a href="https://peanutbase.org/gb2/gbrowse/arahy.Tifrunner.gnm2/">GBrowse</a>
<p><b>Version 1 (gnm1):</b> <i>Arachis hypogaea</i> cv. Tifrunner: <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/genomes/Tifrunner.gnm1.KYV3/">assembly</a>, <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/annotations/Tifrunner.gnm1.ann1.CCJH/">annotation</a> (gnm1.ann1)
<br/>Genome browser: <span style="color: red">*</span> <a href="https://peanutbase.org/gbrowse_peanut1.0">GBrowse</a> and <span style="color: red">*</span> <a href="https://peanutbase.org/genomes/jbrowse?data=arahy.Tifrunner.gnm1">JBrowse</a>
<br/>GenBank: <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_003086295.2">assembly GCA_003086295.2</a>

<p><a href="/genome/peanut_genome_v1_v2">Changes between the genome assemblies version 1 and version 2</a>
<br/><b>Note:</b> The gene models on gnm2.ann1 are the result of liftover, not a new annotation procedure; ie the gene model structures and basic names stay the same, though some coordinates changed and a few gene models got duplicates.

<p>Download folders for <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/hypogaea/"><i>Arachis hypogaea</i> at data store</a> (includes genomes and other available data types).

<h4>Diploid progenitors</h4>
<p><b><i>Arachis duranensis</i></b>
<br/><span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/duranensis/genomes/V14167.gnm1.SWBf/">assembly</a> (gnm1), <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/duranensis/annotations/V14167.gnm1.ann1.cxSM/">annotation</a> (gnm1.ann1)
<br/>Genome browser: <span style="color: red">*</span> <a href="https://peanutbase.org/gbrowse_aradu1.0">GBrowse</a> and <span style="color: red">*</span> <a href="https://peanutbase.org/genomes/jbrowse?data=Aradu1.0">JBrowse</a>
<br/>Download folders for <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/duranensis/"><i>Arachis duranensis</i> at data store</a> (includes genomes and other available data types).
<br/>GenBank: Assembly <a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000817695.2">GCF_000817695.2</a>

<p><b><i>Arachis ipaensis</i></b>
<br/><b>Version 2 (gnm2):</b> <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/ipaensis/genomes/K30076.gnm2.1GWY/">assembly</a> (gnm2), no annotation yet.
<br/>This is a high-quality chromosome-scale genome sequence, based on PacBio RSII and PacBio Sequel read data, with ordering into pseudomolecules by Dovetail using Hi-C Chicago libraries and the HiRise and SNAP software. See more information about the assembly in the assembly directory.
<br/>Genome browser: <span style="color: red">*</span> <a href="https://peanutbase.org/gb2/gbrowse/araip.K30076.gnm2/">GBrowse</a> and <span style="color: red">*</span> <a href="https://peanutbase.org/genomes/jbrowse?data=Araip1.0">JBrowse</a>
<br/><b>Version 1 (gnm1):</b> <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/ipaensis/genomes/K30076.gnm1.bXJ8/">assembly</a> (gnm1), <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/ipaensis/annotations/K30076.gnm1.ann1.J37m/">annotation</a> (gnm1.ann1)
<br/>Genome browser: <span style="color: red">*</span> <a href="https://peanutbase.org/gbrowse_araip1.0">GBrowse</a> and <span style="color: red">*</span> <a href="https://peanutbase.org/genomes/jbrowse?data=Araip1.0">JBrowse</a>
<br/>Download folders for <span style="color: red">*</span> <a href="https://peanutbase.org/data/v2/Arachis/ipaensis/"><i>Arachis ipaensis</i> at data store</a> (includes genomes and other available data types).
<br/>GenBank: Assembly <a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000816755.2">GCF_000816755.2</a>

<h4>Additional details about the <i>A. hypogaea</i> assembly</h4>
<p>The assembly size is 2,556 Mbp, which we estimate to span more than 99% of the actual genome. The scaffold N50 (a measure of the assembly contiguity) is 135.2 MB (the scale of the complete peanut chromosomes). A total of 48.25x of PACBIO sequence (avg. read length of 11,525) was used to generate the initial assembly, which was subsequently polished using Illumina sequences and ARROW. Homozygous SNPs and INDELs were corrected in the release sequence using ~40x of Illumina reads (2x250, 800bp insert, library ID ICIH and ICID). Synteny with the diploid <i>A. duranensis</i> and <i>A. ipaensis</i>, along with 1 genetic map and 2 synthetic maps (provided by David Bertioli) were used to identify misjoins in the raw assembly. The resulting assembly was then scaffolded using HiC data. Post scaffolding, 6 additional breaks were made to resolve misjoins introduced during the scaffolding procedure.

<p>The original sequences were combined with the duplicated tetrasomic regions and joined together using 26 joins to create the 20 <i>A. hypogaea</i> chromosomes. During the construction of the chromosomes, all 500bp scaffolded gaps were converted to 1,000 bp gaps, and the map joins that were added consisted of 10,000 bp gaps. Chromosomes were numbered as Arahy.01-Arahy.20, where the A genome is represented as Arahy.01-Arahy.10 and the B genome is represented as Arahy.11-Arahy.20. 99.3% of the assembled sequence is contained in the chromosomes.

