---
layout: default
title: The Peanut Genome
---
<h1>The Peanut Genome</h1>

<p>
This page describes the high-quality genome assembly for peanut (<i>Arachis hypogaea</i>), cultivar Tifrunner, which was released in December, 2017 (<a href="https://www.nature.com/articles/s41588-019-0405-z">Bertioli et al., 2019</a>), and the improved version 2.0. An explanation of the reason for the corrected version and a list of differences is available <a href="https://www.peanutbase.org/genome/peanut_genome_v1_v2">here</a>.
<p>
<b>IMPORTANT NOTE</b>: the version 2.0 assembly (gnm2) for <i>Arachis hypogaea</i> var. Tifrunner has not been submitted to GenBank.
<p>
Tifrunner is an important U.S. variety, with good market and growth characteristics and resistance to several peanut diseases (early and late leaf spot and TSWV/spotted wilt).
<p>
This is a project of the <a href="https://www.peanutbase.org/genome/IPGI">International Peanut Genome Initiative</a>, in order to accelerate breeding progress and get more productive, disease-resistant, stress-tolerant varieties to farmers. The IPGI project has sequenced the genomes of the <a href="https://legacy.peanutbase.org/browse_search#browse_genomes">two diploid progenitors of cultivated peanut</a>, as well as the genome of cultivated peanut itself.

{% if site.data.taxa['Arachis'] %}

<hr>
{% for sp in site.data.taxa['Arachis'].species_resources.species %}
  {% assign genus_species = sp.genus | append: " " | append: sp.species %}
  {% assign nc = sp.commonName | size | minus: 1 %}
  <h3><i>{{ genus_species }}</i>: {{ sp.commonName | slice: 0 | capitalize }}{{ sp.commonName | slice: 1, nc }}</h3>
  {% assign datastore_url = "https://data.legumeinfo.org/" | append: genus_species | replace: " ", "/" %}
  <p>Download folders for <a href="{{ datastore_url }}/">{{ genus_species }} at data store</a> (includes genomes and other available data types)
  {% if sp.strains %}
  {% for str in sp.strains %}
    <p><b><i>{{ genus_species }}</i> cv. {{ str.name }}</b>
    <!-- TODO: split by version, not just strain -->
    {% assign spc = site.data.taxa['Arachis'].species_collections.species | where: "name", sp.species | first %}

    <!-- Genome assemblies -->
    <p>Assemblies:
    {% assign gnms = spc.genomes | where_exp: "gnm", "gnm.collection contains str.identifier" %}
    {% for gnm in gnms %}
      <br><a href="{{ datastore_url }}/genomes/{{ gnm.collection }}/">{{ gnm.synopsis }}</a>
    {% endfor %} <!-- gnm -->

    <!-- Annotations, if any -->
    <p>Annotations:
    {% assign anns = spc.annotations | where_exp: "ann", "ann.collection contains str.identifier" %}
    {% if anns.size > 0 %}
      {% for ann in anns %}
        <br><a href="{{ datastore_url }}/annotations/{{ ann.collection }}/">{{ ann.synopsis }}</a>
      {% endfor %} <!-- ann -->
    {% else %}
      <br>No annotations yet.
    {% endif %} <!-- anns -->

    {% if str.resources %}
      <!-- Genome browsers, if any -->
      {% assign gbs = str.resources | where_exp: "rs", "rs.description contains 'Browse'" %}
      {% if gbs.size > 0 %}
        <p>Genome browsers:
        {% for gb in gbs %}
          <br><a href="{{ gb.URL }}">{{ gb.name }}</a>: {{ gb.description }}
        {% endfor %} <!-- gb -->
      {% endif %} <!-- gbs.size > 0 -->

      <!-- Other resources, if any -->
      {% assign nother = str.resources.size | minus: gbs.size %}
      {% if nother > 0 %}
        <p>Other resources:
        {% for rs in str.resources %}
         {% unless rs.description contains "Browse" %}
            <br><a href="{{ rs.URL }}">{{ rs.name }}</a>: {{ rs.description }}
          {% endunless %} <!-- rs.description contains "Browse" -->
        {% endfor %} <!-- rs -->
      {% endif %} <!-- nother > 0 -->
    {% endif %} <!-- str.resources -->

    <!-- These A. hypogaea cv. Tifrunner details are not in the metadata, so include them here -->
    {% if sp.species == "hypogaea" and str.name == "Tifrunner" %}
      <p>
        <a href='https://dev.peanutbase.org/genome/peanut_genome_v1_v2'>Changes between the genome assemblies version 1 and version 2</a>
      <p>
        <b>Note:</b> The gene models on gnm2.ann1 are the result of liftover, not a new annotation procedure; ie the gene model structures and basic names stay the same, though some coordinates changed and a few gene models got duplicates.
      <h4>Additional details about the <i>A. hypogaea</i> assembly</h4>
      <p>
        The assembly size is 2,556 Mbp, which we estimate to span more than 99% of the actual genome. The scaffold N50 (a measure of the assembly contiguity) is 135.2 MB (the scale of the complete peanut chromosomes). A total of 48.25x of PACBIO sequence (avg. read length of 11,525) was used to generate the initial assembly, which was subsequently polished using Illumina sequences and ARROW. Homozygous SNPs and INDELs were corrected in the release sequence using ~40x of Illumina reads (2x250, 800bp insert, library ID ICIH and ICID). Synteny with the diploid <i>A. duranensis</i> and <i>A. ipaensis</i>, along with 1 genetic map and 2 synthetic maps (provided by David Bertioli) were used to identify misjoins in the raw assembly. The resulting assembly was then scaffolded using HiC data. Post scaffolding, 6 additional breaks were made to resolve misjoins introduced during the scaffolding procedure.
      <p>
        The original sequences were combined with the duplicated tetrasomic regions and joined together using 26 joins to create the 20 <i>A. hypogaea</i> chromosomes. During the construction of the chromosomes, all 500bp scaffolded gaps were converted to 1,000 bp gaps, and the map joins that were added consisted of 10,000 bp gaps. Chromosomes were numbered as Arahy.01-Arahy.20, where the A genome is represented as Arahy.01-Arahy.10 and the B genome is represented as Arahy.11-Arahy.20. 99.3% of the assembled sequence is contained in the chromosomes.
    {% endif %} <!-- sp.species == "hypogaea" and str.name == "Tifrunner" -->

  {% endfor %} <!-- str -->
  {% endif %} <!-- sp.strains -->

  <hr>
{% endfor %} <!-- sp -->
{% endif %} <!-- site.data.taxa['Arachis'] -->

