<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN"
  "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" version="XHTML+RDFa 1.0" dir="ltr">

<head profile="http://www.w3.org/1999/xhtml/vocab">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="shortcut icon" href="https://legacy.peanutbase.org/sites/default/files/favicon.ico" type="image/vnd.microsoft.icon" />
  <title>NEWS | PeanutBase(Legacy)</title>
  <style type="text/css" media="all">
@import url("https://legacy.peanutbase.org/modules/system/system.base.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/system/system.menus.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/system/system.messages.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/system/system.theme.css?rgf2aa");
</style>
<style type="text/css" media="all">
@import url("https://legacy.peanutbase.org/modules/comment/comment.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/field/theme/field.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/filebrowser/css/filebrowser_style.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/legume/legume_featuremap/theme/css/legume_featuremap.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/legume/legume_marker/theme/css/legume_marker.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/legume/legume_qtl/theme/css/legume_qtl.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/legume/legume_stock/theme/css/legume_stock.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/node/node.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/ontology_search/theme/css/ontology_search_css.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/search/search.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_core/theme/css/tripal.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_feature/theme/css/tripal_feature.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/tripal_gene/theme/css/tripal_gene.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_organism/theme/css/tripal_organism.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_pub/theme/css/tripal_pub.css?rgf2aa");
@import url("https://legacy.peanutbase.org/modules/user/user.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/extlink/css/extlink.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/views/css/views.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/ckeditor/css/ckeditor.css?rgf2aa");
</style>
<style type="text/css" media="all">
@import url("https://legacy.peanutbase.org/sites/all/modules/ctools/css/ctools.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/lightbox2/css/lightbox.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/nice_menus/css/nice_menus.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/modules/nice_menus/css/nice_menus_default.css?rgf2aa");
</style>
<style type="text/css" media="all">
@import url("https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/css/layout.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/css/style.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/css/colors.css?rgf2aa");
@import url("https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/css/nicemenus_override.css?rgf2aa");
</style>
<style type="text/css" media="print">
@import url("https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/css/print.css?rgf2aa");
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://legacy.peanutbase.org/themes/bartik/css/ie.css?rgf2aa" media="all" />
<![endif]-->

<!--[if IE 6]>
<link type="text/css" rel="stylesheet" href="https://legacy.peanutbase.org/themes/bartik/css/ie6.css?rgf2aa" media="all" />
<![endif]-->
  <script type="text/javascript" src="https://legacy.peanutbase.org/misc/jquery.js?v=1.4.4"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/misc/jquery-extend-3.4.0.js?v=1.4.4"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/misc/jquery-html-prefilter-3.5.0-backport.js?v=1.4.4"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/misc/jquery.once.js?v=1.2"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/misc/drupal.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/nice_menus/js/jquery.bgiframe.js?v=2.1"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/nice_menus/js/jquery.hoverIntent.js?v=0.5"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/nice_menus/js/superfish.js?v=1.4.8"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/nice_menus/js/nice_menus.js?v=1.0"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/legume/legume_qtl/theme/js/legume_qtl.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/ontology_search/theme/js/ontology_search_scripts.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_core/theme/js/tripal.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/tripal/tripal_feature/theme/js/tripal_feature.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/lis_tours/build/bundle.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/lightbox2/js/lightbox.js?rgf2aa"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
 var baseurl  = 'https://legacy.peanutbase.org';
      var isClean  =  1;
//--><!]]>
</script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/google_analytics/googleanalytics.js?rgf2aa"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","https://legacy.peanutbase.org/sites/default/files/googleanalytics/analytics.js?rgf2aa","ga");ga("create", "UA-49655154-2", {"cookieDomain":"auto"});ga("send", "pageview");
//--><!]]>
</script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/modules/extlink/js/extlink.js?rgf2aa"></script>
<script type="text/javascript" src="https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/scripts.js?rgf2aa"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","setHasJsCookie":0,"ajaxPageState":{"theme":"peanutbase_theme","theme_token":"AECW3hjmaA-PqrZ7RLlBdfoG8iS89DjPhphKmev5FuI","js":{"misc\/jquery.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery-html-prefilter-3.5.0-backport.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/nice_menus\/js\/jquery.bgiframe.js":1,"sites\/all\/modules\/nice_menus\/js\/jquery.hoverIntent.js":1,"sites\/all\/modules\/nice_menus\/js\/superfish.js":1,"sites\/all\/modules\/nice_menus\/js\/nice_menus.js":1,"sites\/all\/modules\/legume\/legume_qtl\/theme\/js\/legume_qtl.js":1,"sites\/all\/modules\/ontology_search\/theme\/js\/ontology_search_scripts.js":1,"sites\/all\/modules\/tripal\/tripal_core\/theme\/js\/tripal.js":1,"sites\/all\/modules\/tripal\/tripal_feature\/theme\/js\/tripal_feature.js":1,"sites\/all\/modules\/lis_tours\/build\/bundle.js":1,"sites\/all\/modules\/lightbox2\/js\/lightbox.js":1,"0":1,"sites\/all\/modules\/google_analytics\/googleanalytics.js":1,"1":1,"sites\/all\/modules\/extlink\/js\/extlink.js":1,"sites\/all\/themes\/peanutbase_theme\/scripts.js":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"modules\/comment\/comment.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/filebrowser\/css\/filebrowser_style.css":1,"sites\/all\/modules\/legume\/legume_featuremap\/theme\/css\/legume_featuremap.css":1,"sites\/all\/modules\/legume\/legume_marker\/theme\/css\/legume_marker.css":1,"sites\/all\/modules\/legume\/legume_qtl\/theme\/css\/legume_qtl.css":1,"sites\/all\/modules\/legume\/legume_stock\/theme\/css\/legume_stock.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/ontology_search\/theme\/css\/ontology_search_css.css":1,"modules\/search\/search.css":1,"sites\/all\/modules\/tripal\/tripal_core\/theme\/css\/tripal.css":1,"sites\/all\/modules\/tripal\/tripal_feature\/theme\/css\/tripal_feature.css":1,"sites\/all\/modules\/tripal_gene\/theme\/css\/tripal_gene.css":1,"sites\/all\/modules\/tripal\/tripal_organism\/theme\/css\/tripal_organism.css":1,"sites\/all\/modules\/tripal\/tripal_pub\/theme\/css\/tripal_pub.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/extlink\/css\/extlink.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/lightbox2\/css\/lightbox.css":1,"sites\/all\/modules\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/themes\/peanutbase_theme\/css\/layout.css":1,"sites\/all\/themes\/peanutbase_theme\/css\/style.css":1,"sites\/all\/themes\/peanutbase_theme\/css\/colors.css":1,"sites\/all\/themes\/peanutbase_theme\/css\/nicemenus_override.css":1,"sites\/all\/themes\/peanutbase_theme\/css\/print.css":1,"themes\/bartik\/css\/ie.css":1,"themes\/bartik\/css\/ie6.css":1}},"lightbox2":{"rtl":0,"file_path":"\/(\\w\\w\/)public:\/","default_image":"\/sites\/all\/modules\/lightbox2\/images\/brokenimage.jpg","border_size":10,"font_color":"000","box_color":"fff","top_position":"","overlay_opacity":"0.8","overlay_color":"000","disable_close_click":true,"resize_sequence":0,"resize_speed":400,"fade_in_speed":400,"slide_down_speed":600,"use_alt_layout":false,"disable_resize":false,"disable_zoom":false,"force_show_nav":false,"show_caption":true,"loop_items":false,"node_link_text":"View Image Details","node_link_target":false,"image_count":"Image !current of !total","video_count":"Video !current of !total","page_count":"Page !current of !total","lite_press_x_close":"press \u003Ca href=\u0022#\u0022 onclick=\u0022hideLightbox(); return FALSE;\u0022\u003E\u003Ckbd\u003Ex\u003C\/kbd\u003E\u003C\/a\u003E to close","download_link_text":"","enable_login":false,"enable_contact":false,"keys_close":"c x 27","keys_previous":"p 37","keys_next":"n 39","keys_zoom":"z","keys_play_pause":"32","display_image_size":"original","image_node_sizes":"()","trigger_lightbox_classes":"","trigger_lightbox_group_classes":"","trigger_slideshow_classes":"","trigger_lightframe_classes":"","trigger_lightframe_group_classes":"","custom_class_handler":0,"custom_trigger_classes":"","disable_for_gallery_lists":true,"disable_for_acidfree_gallery_lists":true,"enable_acidfree_videos":true,"slideshow_interval":5000,"slideshow_automatic_start":true,"slideshow_automatic_exit":true,"show_play_pause":true,"pause_on_next_click":false,"pause_on_previous_click":true,"loop_slides":false,"iframe_width":600,"iframe_height":400,"iframe_border":1,"enable_video":false,"useragent":"Mozilla\/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit\/605.1.15 (KHTML, like Gecko) Version\/18.1 Safari\/605.1.15"},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|(b|s)am|bin|bz2|csv|doc|exe|fa|flv|gif|gff(3)?|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|xz|z|zip"},"nice_menus_options":{"delay":"800","speed":"fast"},"extlink":{"extTarget":0,"extClass":"ext","extLabel":"(link is external)","extImgClass":0,"extIconPlacement":"append","extSubdomains":0,"extExclude":"","extInclude":"","extCssExclude":"","extCssExplicit":"","extAlert":0,"extAlertText":"This link will take you to an external web site.","mailtoClass":"mailto","mailtoLabel":"(link sends e-mail)","extUseFontAwesome":false}});
//--><!]]>
</script>
</head>
<body class="html not-front not-logged-in one-sidebar sidebar-second page-news-page-all" >
  <div id="skip-link">
    <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
    <div id="page-wrapper"><div id="page">

  <div id="header" class="without-secondary-menu"><div class="section clearfix">

          <a href="/" title="Home" rel="home" id="logo">
        <img src="https://legacy.peanutbase.org/sites/all/themes/peanutbase_theme/images/peanutbase_logo.png" alt="Home" />
      </a>
    
          <div id="top-banner">
        <div id="name-and-slogan">

                                    <div id="site-name">
                <strong>
                  <a href="/" title="Home" rel="home"><span>PeanutBase(Legacy)</span></a>
                </strong>
              </div>
                            
                      <div id="site-slogan">
              Genetic and genomic data to enable more rapid crop improvement in peanut.            </div>
                  </div> <!-- /#name-and-slogan -->
        

               
	<div id="contact-and-search">
	  <br>
        <a href="/contact" style="float:right;color:white;text-decoration:underline;">Contact us</a>
          <br>
        <a href="http://eepurl.com/b6zeIL" style="float:right;color:white;text-decoration:underline;">Newsletter signup</a>

        </div>
      </div> <!-- /#top-banner -->
    
    
          <div id="main-menu" class="navigation">
        <ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-0"><li class="menu-267 menu-path-node-4 first odd "><a href="/home">Home</a></li>
<li class="menu-273 menuparent  menu-path-node-7  even "><a href="/species">Species</a><ul><li class="menu-269 menu-path-node-1 first odd "><a href="/organism/Arachis/hypogaea">A. hypogaea (cult. peanut)</a></li>
<li class="menu-4363 menu-path-node-317427  even "><a href="/organism/Arachis/batizocoi">A. batizocoi</a></li>
<li class="menu-4365 menu-path-node-317428  odd "><a href="/organism/Arachis/cardenasii">A. cardenasii</a></li>
<li class="menu-4366 menu-path-node-2613713  even "><a href="/organism/Arachis/correntina">A. correntina</a></li>
<li class="menu-4364 menu-path-node-317429  odd "><a href="/organism/Arachis/diogoi">A. diogoi</a></li>
<li class="menu-270 menu-path-node-2  even "><a href="/organism/Arachis/duranensis">A. duranensis</a></li>
<li class="menu-271 menu-path-node-3  odd "><a href="/organism/Arachis/ipaensis">A. ipaensis</a></li>
<li class="menu-2404 menu-path-node-316839  even "><a href="/organism/Arachis/magna">A. magna</a></li>
<li class="menu-727 menu-path-node-613  odd "><a href="/organism/Arachis/monticola">A. monticola</a></li>
<li class="menu-726 menu-path-node-612  even last"><a href="/organism/Arachis/stenosperma">A. stenosperma</a></li>
</ul></li>
<li class="menu-400 menuparent  menu-path-node-9  odd "><a href="/browse_search">Browse &amp; Search</a><ul><li class="menu-3262 menu-path-arachis-bioproj first odd "><a href="/arachis_bioproj" title="Arachis BioProjects listed from NCBI">BioProjects at NCBI</a></li>
<li class="menu-705 menu-path-node-2617905  even "><a href="/pb_sequenceserver" title="">BLAST sequence search (SequenceServer)</a></li>
<li class="menu-1602 menu-path-node-626  odd "><a href="/blat" title="">BLAT sequence search</a></li>
<li class="menu-4473 menu-path-node-587308  even "><a href="/peanut_genome" title="Peanut genome Page">Genome page</a></li>
<li class="menu-3701 menu-path-node-587307  odd "><a href="/gbrowse_peanut1.0">Browse (GBrowse) - cultivated peanut</a></li>
<li class="menu-2782 menu-path-node-624  even "><a href="/gbrowse_aradu1.0" title="">Genome (GBrowse) - A. duranensis</a></li>
<li class="menu-2783 menu-path-node-625  odd "><a href="/gbrowse_araip1.0" title="">Genome (GBrowse) - A. ipaensis</a></li>
<li class="menu-3712 menu-path-node-319881-  even "><a href="/node/319881/?data=arahy.Tifrunner.gnm1" title="">Genome (JBrowse) - cultivated peanut</a></li>
<li class="menu-2787 menu-path-node-319881-  odd "><a href="/node/319881/?data=Aradu1.0" title="">Genome (JBrowse) - A. duranensis</a></li>
<li class="menu-2788 menu-path-node-319881  even "><a href="/genomes/jbrowse?data=Araip1.0" title="">Genome (JBrowse) - A. ipaensis</a></li>
<li class="menu-2072 menu-path-node-936  odd "><a href="/keyword_search" title="">Genomic feature search by keyword</a></li>
<li class="menu-4321 menu-path-search-gene  even "><a href="/search/gene" title="">Gene and gene family search</a></li>
<li class="menu-424 menu-path-node-13  odd "><a href="/gene_expression">Gene expression atlases</a></li>
<li class="menu-4403 menu-path-node-2616359  even "><a href="/germplasm/gigwa" title="">Genotype data (Gigwa)</a></li>
<li class="menu-3277 menu-path-search-stock  odd "><a href="/search/stock" title="Search for Arachis germplasm">Germplasm search</a></li>
<li class="menu-4404 menu-path-node-936  even "><a href="/keyword_search" title="">Keyword search</a></li>
<li class="menu-1889 menu-path-chado-featuremap  odd "><a href="/chado/featuremap" title="">Map search</a></li>
<li class="menu-4322 menu-path-search-marker  even "><a href="/search/marker" title="">Marker search</a></li>
<li class="menu-3063 menu-path-publications-arachis-recent-pubs  odd "><a href="/publications/arachis_recent_pubs" title="Recent publications on Arachis from Pubmed">Publications (recent at PubMed)</a></li>
<li class="menu-1726 menu-path-chado-publication  even "><a href="/chado/publication" title="Link to publication search form">Publications (with data)</a></li>
<li class="menu-4323 menu-path-search-qtl  odd last"><a href="/search/qtl" title="">QTL search</a></li>
</ul></li>
<li class="menu-1803 menuparent  menu-path-node-715  even "><a href="/traits_maps">Traits &amp; Maps</a><ul><li class="menu-1722 menu-path-node-674 first odd "><a href="/mas">Marker-Assisted Selection</a></li>
<li class="menu-1888 menu-path-chado-featuremap  even "><a href="/chado/featuremap" title="">Map search</a></li>
<li class="menu-1886 menu-path-node-16  odd last"><a href="/qtl">QTL overview</a></li>
</ul></li>
<li class="menu-2462 menuparent  menu-path-node-317377  odd "><a href="/germplasm">Germplasm</a><ul><li class="menu-3108 menu-path-peanutbaseorg-germplasm-map first odd "><a href="http://peanutbase.org/germplasm/map" title="Geographic distribution of Arachis germplasm">Geographic distribution</a></li>
<li class="menu-4470 menu-path-arachisphenopeanutbaseorg-  even "><a href="http://arachispheno.peanutbase.org/" title="ArachisPheno (phenotype server at PeanutBase)">ArachisPheno(phenotype data)</a></li>
<li class="menu-3109 menu-path-node-317377  odd "><a href="/germplasm#germplasmImages" title="Images of Archis germplasm">Germplasm images</a></li>
<li class="menu-3278 menu-path-search-stock  even "><a href="/search/stock" title="Search for Arachis germplasm">Germplasm search</a></li>
<li class="menu-4469 menu-path-node-2617907  odd "><a href="/germplasm/catalogs">Germplasm catalogs</a></li>
<li class="menu-4440 menu-path-node-2616358  even "><a href="/germplasm/gcvit">GenotypeComparisonView:GCViT</a></li>
<li class="menu-4443  menu-path-node-2616361  odd last"><a href="/germplasm/nifa_pccgp" title="NIFA Genotyping Phenotyping Proj.">NIFA PCCGP Project</a></li>
</ul></li>
<li class="menu-2071 menuparent  menu-path-node-10  even "><a href="/download">Download</a><ul><li class="menu-3713 menu-path-speanutbaseorg-data-v2-arachis- first odd "><a href="https://peanutbase.org/data/v2/Arachis/" title="Legume Information System data store">Data store</a></li>
<li class="menu-3705 menu-path-node-587308  even "><a href="/peanut_genome">Download cultivated peanut genome</a></li>
<li class="menu-3068 menu-path-speanutbaseorg-data-v2-arachis-duranensis-  odd "><a href="https://peanutbase.org/data/v2/Arachis/duranensis/" title="A. duranensis Data Store">Download A. duranensis genome &amp; annotations</a></li>
<li class="menu-3066 menu-path-speanutbaseorg-data-v2-arachis-ipaensis-  even last"><a href="https://peanutbase.org/data/v2/Arachis/ipaensis/" title="A. ipaensis Data Store">Download A. ipaensis genome &amp; annotations</a></li>
</ul></li>
<li class="menu-1804 menu-path-node-716  odd "><a href="/submit_data" title="Data submission guide and templates for QTL and MAS data">Submit Data</a></li>
<li class="menu-399 menuparent  menu-path-node-8  even "><a href="/community">Community</a><ul><li class="menu-3544 menu-path-node-8 first odd "><a href="/community#about" title="">About PeanutBase</a></li>
<li class="menu-3541 menu-path-node-587264  even "><a href="/AAGB">AAGB</a></li>
<li class="menu-3540 menu-path-node-587263  odd "><a href="/IPGI">IPGI/PGC</a></li>
<li class="menu-3542 menu-path-node-8  even "><a href="/community#community_resources" title="">Peanut community resources</a></li>
<li class="menu-3543 menu-path-node-8  odd "><a href="/community#outreach" title="">PeanutBase outreach</a></li>
<li class="menu-4369 menu-path-node-2613714  even "><a href="/tutorials" title="">Peanut tutorials</a></li>
<li class="menu-4245 menu-path-node-1294551  odd last"><a href="/workshops">Workshops</a></li>
</ul></li>
<li class="menu-423 menu-path-node-12  odd "><a href="/external">External</a></li>
<li class="menu-1805 menuparent  menu-path-node-717  even last"><a href="/help">Help</a><ul><li class="menu-4370 menu-path-node-717 first odd "><a href="/help" title="">PeanutBase help</a></li>
<li class="menu-4368 menu-path-node-2613714  even last"><a href="/tutorials">Peanut tutorials</a></li>
</ul></li>
</ul>
      </div>  <!--/#main-menu -->

    
    
  </div></div> <!-- /.section, /#header -->

  
  
  <div id="main-wrapper" class="clearfix"><div id="main" class="clearfix">

    
    
    <div id="content" class="column"><div class="section">
      <div id="highlighted">  <div class="region region-highlighted">
    <div id="block-block-9" class="block block-block">

    <h2>MESSAGE</h2>
  
  <div class="content">
    <p><span style="color:red">PeanutBase has transitioned to a new technology format. All updates are now at the <a href="https://peanutbase.org" target="_blank"><b>new site</b></a> still under development. Please visit the new site and give us your <a href="https://peanutbase.org/contact" target="_blank">feedback</a>. The legacy site is still available at <a href="https://legacy.peanutbase.org" target="_blank">legacy.peanutbase.org</a>.</span></p>
  </div>
</div>
  </div>
</div>      <a id="main-content"></a>

                    <!--.<h1 class="title" id="page-title">
                  </h1>-->
            
              <div class="tabs">
                  </div>
                          <div class="region region-content">
    <div id="block-system-main" class="block block-system">

    
  <div class="content">
    <div class="view view-News-list view-id-News_list view-display-id-page_1 view-dom-id-4fdf91dd6343b16fc06328b8bf183ee9">
        
  
  
      <div class="view-content">
      <div class="item-list">    <ul>          <li class="views-row views-row-1 views-row-odd views-row-first">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617919">(June 2023) PeanutBase Migrated to new technology platform.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>New Technology Platform: We have now moved to a new technology platform.</p>
<ul>
<li>
REQUEST: Please provide us your <a href="/contact">feedback</a> when you find something is missing/fails/incomplete. We will try to provide that on a priority basis.</li>
<li>The legacy site is still available at legacy.peanutbase.org but it won’t be updated anymore.</li>
<li>All further developments will now be at this new site.</li>
<li>The migration is not yet complete but some content still at the legacy site will be migrated over time.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-2 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617918">(April 2023) New genomes at PeanutBase DataStore(Bailey, Tifrunner, Pangenes).</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>We now have new genomic data available for Arachis pangene set, Bailey II and Tifrunner genome2.</p>
<ul>
<li>
<strong>Arachis Pangene set:</strong> Pangene set for Arachis species, based on A. hypogaea, A. duranensis, and A. ipaensis. This pangene set includes five annotation sets from A. hypogaea and one each from A. duranensis and A. ipaensis. More details and data at Data Store <a href="https://data.legumeinfo.org/Arachis/GENUS/pangenes/" target="_blank">Arachis.pan1.4LN9</a>.</li>
<li>
<strong>Bailey II genome:</strong> Arachis hypogaea accession Bailey II, genome assembly 1. More details for <a href="https://data.legumeinfo.org/Arachis/hypogaea/genomes/BaileyII.gnm1.1JTF/" target="_blank">BaileyII.gnm1.1JTF</a>.</li>
<li>
<strong>Tifrunner genome2:</strong> Genome assembly 2 for Arachis hypogaea, cultivar Tifrunner(<a href="https://data.legumeinfo.org/Arachis/hypogaea/genomes/Tifrunner.gnm2.J5K5/" target="_blank">Tifrunner.gnm2.J5K5</a>). This version has structural changes relative to assembly 1.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-3 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617917">(April 2023) New genomes at PeanutBase DataStore(Bailey, Tifrunner, Pangenes).</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>We now have new genomic data available for Arachis pangene set, Bailey II and Tifrunner genome2.</p>
<ul>
<li>
<strong>Arachis Pangene set:</strong> Pangene set for Arachis species, based on A. hypogaea, A. duranensis, and A. ipaensis. This pangene set includes five annotation sets from A. hypogaea and one each from A. duranensis and A. ipaensis. More details and data at Data Store <a href="https://data.legumeinfo.org/Arachis/GENUS/pangenes/" target="_blank">Arachis.pan1.4LN9</a>.</li>
<li>
<strong>Bailey II genome:</strong> Arachis hypogaea accession Bailey II, genome assembly 1. More details for <a href="https://data.legumeinfo.org/Arachis/hypogaea/genomes/BaileyII.gnm1.1JTF/" target="_blank">BaileyII.gnm1.1JTF</a>.</li>
<li>
<strong>Tifrunner genome2:</strong> Genome assembly 2 for Arachis hypogaea, cultivar Tifrunner(<a href="https://data.legumeinfo.org/Arachis/hypogaea/genomes/Tifrunner.gnm2.J5K5/" target="_blank">Tifrunner.gnm2.J5K5</a>). This version has structural changes relative to assembly 1.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-4 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617916">(Januar 2023) Traits to associated markers using ArachisMine.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong>Traits To Associated Markers Using ArachisMine</strong><br />
Now it is possible to select a trait from the dropdown and and generate a list of associated markers from QTL/GWAS studies. The markers can be further explored for details using the links in the page.<br />
<a href="https://mines.legumeinfo.org/arachismine/begin.do">Try it at ArachisMine</a>(Click the GENETICS tab for the preset query, 'Trait --&gt; GeneticMarkers') OR<br />
use the link directly to this <a href="https://mines.legumeinfo.org/arachismine/template.do?name=trait_markers&amp;scope=global">query page</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-5 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617915">(November 2022) Search Genes And Traits By Keywords</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong>Search Genes And Traits By Keywords</strong></p>
<p>The ArachisMine <a href="https://dev.peanutbase.org/tools/gene-keyword-search.html">Gene Keyword Search</a> and <a href="https://dev.peanutbase.org/tools/trait-keyword-search.html">Trait Keyword Search</a> utilities are available from the Tools pannel. The search result lists the relevant features and are linked to the corresponding detailed display at ArachisMine.<br />
<a href="https://dev.peanutbase.org/tools/gene-keyword-search.html?page=1&amp;query=photosystem%20II">Example: gene search with keyword</a> 'photosystem II'.<br />
<a href="https://dev.peanutbase.org/tools/trait-keyword-search.html?page=1&amp;query=pod">Example: trait search with keyword</a> 'pod'.</p>
</div>  </div></li>
          <li class="views-row views-row-6 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617914">(October 2022) Jbrowse2, New Genome Browser With Enhanced Functionalities</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong><a href="https://dev.peanutbase.org/tools/jbrowse2/?session=share-spkE_3L8Sy&amp;password=fG6Qf">JBrowse2</a>: A New Genome Browser with Enhanced Functionalities Implented at Peanutbase.</strong><br />
Examples:<br />
1. Aradu structural change, <a href="https://dev.peanutbase.org/tools/jbrowse2/?session=share-oe5Xeyrvyt&amp;password=XRGEo">example</a>.<br />
2. Multiple genome display, <a href="https://dev.peanutbase.org/tools/jbrowse2/?session=share-whX0v_rLpQ&amp;password=5Pjg5">example</a>.<br />
3. Details of features on the right panel, <a href="https://dev.peanutbase.org/tools/jbrowse2/?session=share-mNMZTSz6t2&amp;password=V5eJs">example</a>.<br />
4. Display Variant data from VCF, <a href="https://dev.peanutbase.org/tools/jbrowse2/?session=share-yYlJnsg1bU&amp;password=25Etb">example</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-7 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617913">(September 2022) DSCensor: Genome assembly stats; GIGWA2.5: Visualize variants in multiple lines</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong>DSCensor for summary of genome assembly data &amp; Gigwa2.5 multi-genotype comparison.</strong></p>
<ul>
<li>
<a href="https://dscensor.legumeinfo.org/multiqc-demo/genome_main:Arachis  "><strong>DSCensor</strong></a>:<br />
Shows the summary statistics of assembly data.
</li>
<li>
<a href="https://dev.peanutbase.org/gigwa/"><strong>Gigwa2.5: Genotype comparison</strong></a>:<br />
Now it is possible to visualize variants between multiple lines within Gigwa(v2.5). This is achieved by selecting multiple individuals for display and then download and export as flapjack file format. The ‘keep files in server’ should be selected before export. Finaly, a click at ‘View in Flapjack-Bytes’ draws the visualization showing similarity and differences between the genetic lines at each of the SNPs.
</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-8 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617912">(May 2022) The peanut image catalog of the core collection is now available at PeanutBase.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong>Peanut core catalog:</strong><br />
These images were taken to document unique traits of each accession during a large-scale project to comprehensively catalog the peanut core and mini core collection. Now the images of the accessions in the core collection are available as a <a href="/germplasm/catalogs">catalog</a>. (Contribution from Noelle Anglin, USDA-ARS and others)</p>
</div>  </div></li>
          <li class="views-row views-row-9 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617909">(February 2022) A new and improved Arachis ipaensis genome; peanut expression atlas mapped to genome2 assembly</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
<strong>New and improved assembly of <i>Arachis ipaensis</i></strong><br />
A completely new PacBio and Hi-C sequence and assembly of the genome of <i>Arachis ipaensis</i> is now available (with some usage restrictions) at the PeanutBase <a target="_blank" href="https://data.legumeinfo.org/Arachis/ipaensis/genomes/K30076.gnm2.1GWY/">Data Store</a> as a result of a collaboration between USDA, University of Georgia and Hudson Alpha. <i>Arachis ipaensis</i> is the B subgenome donor of cultivated peanut (<i>A. hypogaea</i>) and its wild counterpart <i>A. monticola</i>. This remarkable accession, K 30076, is the only collection of this species ever made. Its location, far from that of other B genome <i>Arachis</i> species, is incompatible with natural dispersal, indicating past transport by humans. Its extraordinarily high DNA identity with peanut’s B subgenome and location, close to Villa Montes in Bolivia where the landraces of peanut that combine the most primitive, or wild-like, characteristics are grown indicate that it is descended from the very same population that gave rise to the tetraploid species.
</li>
<li>
<strong>Peanut comparative expression data with other legumes</strong><br />
The peanut expression atlas data (<a target="_blank" href="http://journal.frontiersin.org/article/10.3389/fpls.2016.01446/full">Clevenger, et al. 2016</a>) after remapping the reads to gnm2.ann1 is now in the <a target="_blank" href="https://conekt.legumeinfo.org/species/">CoNekT</a> tool where the expression patterns can be compared with their orthologs in other legumes at LIS.<br />
You can 
<ul>
<li><a target="_blank" href="https://conekt.legumeinfo.org/species/view/5">Browse</a> the list of peanut genes with expression profiles in this dataset; click on the gene to visualize the expression profile with links to other features. </li>
<li>Visualization expression <a target="_blank" href="https://conekt.legumeinfo.org/sequence/view/150167">profile of a single peanut gene</a>.</li>
<li><a target="_blank" href="https://conekt.legumeinfo.org/profile_comparison/">Compare the profiles of a group of genes</a> of interest. (Paste this list as an example after choosing peanut for species: arahy.Tifrunner.gnm2.ann1.0FWC6J arahy.Tifrunner.gnm2.ann1.5ZI49M arahy.Tifrunner.gnm2.ann1.61UWXT arahy.Tifrunner.gnm2.ann1.6T97B arahy.Tifrunner.gnm2.ann1.72B7PK)</li>
<li>Look at the available legume orthologs (<a target="_blank" href="https://conekt.legumeinfo.org/family/find_forLIS/legfed_v1_0.L_BQRRPW">gene family members</a>)  and from there the comparative expression pattern (<a target="_blank" href="https://conekt.legumeinfo.org/heatmap/comparative/family/80617/row">tabular view</a>, <a target="_blank" href="https://conekt.legumeinfo.org/tree/find_forLIS/legfed_v1_0.L_BQRRPW_tree">tree view</a>) in those genes.</li>
<li>Search for a <a target="_blank" href="https://conekt.legumeinfo.org/search/specific/profiles">specific expression profile</a> at Conekt. </li>
<li>Coexpressed profile neighbors of a gene presented as a <a target="_blank" href="https://conekt.legumeinfo.org/network/graph/190540">limited network</a> (accessible from the <a target="_blank" href="https://conekt.legumeinfo.org/sequence/view/154464">profile page </a>).</li>
<li>An example of a peanut gene <a target="_blank" href="https://conekt.legumeinfo.org/sequence/view/194988">arahy.Tifrunner.gnm2.ann1.T0173K</a> specifically expressed in roots like its orthologs in <a target="_blank" href="https://conekt.legumeinfo.org/tree/find_forLIS/legfed_v1_0.L_B3L2R6_tree"><i>Glycine</i> but showing divergence in function</a> (expression pattern in non-root tissues) as a result of speciation in <i>Phaseolus</i> and <i>Vigna</i>. </li>
</ul>
</li>
</ul></div>  </div></li>
          <li class="views-row views-row-10 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617908">(August 2021) The peanut mini core image catalog is now at PeanutBase.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><strong>Peanut mini core image catalog:</strong><br />
These images were taken to document unique traits of each accession during a large-scale project to comprehensively catalog the peanut core and mini core collection. Now the images of the accessions in the mini core collection are available as a <a href="/germplasm/catalogs">catalog</a>. (Contributor: Noelle Anglin, USDA-ARS)</p>
</div>  </div></li>
          <li class="views-row views-row-11 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617906">(April 2021) Arachis Phenotype server; PeanutMine update.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>(April 2021) Arachis Phenotype server; PeanutMine update.</p>
<ul>
<li><strong>Phenotype data server</strong> for PeanutBase is now operational at <strong><a target="_blank" href="http://arachispheno.peanutbase.org/">ArachisPheno</a></strong>. We now have the phenotype data for the peanut mini core collection: <a href="https://arachispheno.peanutbase.org/study/1/">agronomic traits</a> and <a href="https://arachispheno.peanutbase.org/study/2/">seed biochemical traits</a>.  If you have phenotype data for peanut lines to share with the community, please <a href="/contact/">contact us</a> to house it at PeanutBase, or you may try a <a href="https://arachispheno.peanutbase.org/submission/">self submission</a> too.
</li>
<li><strong><a target="_blank" href="https://mines.legumeinfo.org/peanutmine/begin.do">PeanutMine</a> updates: <strong></strong></strong></li>
<p>The datasets are listed at the <a target="_blank" href="https://mines.legumeinfo.org/peanutmine/dataCategories.do">data sources</a> page.</p>
<ul>
<li>Several genetic maps along with their derived QTLs are now loaded into the PeanutMIne. </li>
<li>GWAS datasets in PeanutMine: </li>
<p>Gangurde_Wang_2020-NAMFlor7<br />
Gangurde_Wang_2020-NAMTifr<br />
Otyama_Kulkarni_2020
</p></ul>
</ul>
</div>  </div></li>
          <li class="views-row views-row-12 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617904">(February 2021) New genome assembly for A. cardenasii, a diploid relative of peanut.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    The genome of Arachis cardenasii, a relative of cultivated peanut, has been sequenced and is available here  (with some usage restrictions) at the <a href="/data/v2/Arachis/cardenasii/">PeanutBase Data Store</a>. This diploid wild species has been used as an important source of disease resistance in cultivated tetraploid peanut. Genome sequencing was funded by the USDA (Dr. Brian Scheffler leading), under the scientific guidance of Drs. David and Soraya Bertioli and colleagues.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-13 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617903">(November 2020) New trait association viewer (ZZBrowse); phenotype data from NAM project; data store visualizer.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
New tool (<strong>ZZBrowse</strong>) for exploring marker-trait association from GWAS experiments – allowing comparisons with data from other species. <a target="_blank" href="https://zzbrowse.legumeinfo.org/?tab=Chrom&amp;datasets=Peanut%20GWAS&amp;chr=Arahy.16&amp;selected=10803170&amp;window=250000&amp;datasets2=Soybean%20GWAS&amp;chr2=Gm19&amp;selected2=46260000&amp;window2=250000&amp;traits2=Pod%20number;Pod%20shattering&amp;genomicLinkage=true&amp;neighbors=20&amp;matched=4&amp;intermediate=5&amp;selectedGene=arahy.NJI399&amp;relatedRegion=Gm19%2046.13-46.39%20Mbp">Example</a> showing pod/seed weight SNP in peanut that lands in one region of soybean associated with pod number.
</li>
<li>
The GWAS data from the two <strong>Nested Association Mapping (NAM) populations</strong> reported in <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7206994/">Gangurde SS, et al. (2020)</a> is now at PeanutBase. Visualize using ZZBrowse (<a target="_blank" href="https://zzbrowse.legumeinfo.org/?tab=WhGen&amp;datasets=Peanut%20GWAS&amp;chr=Arahy.01&amp;selected=100000&amp;window=250000&amp;datasets2=Arabidopsis%20thaliana%20GWAS&amp;chr2=1&amp;selected2=100000&amp;window2=250000">example</a>), or download at the <a href="https://legumeinfo.org/data/public/Arachis_hypogaea/mixed.gwas.V77H/">DataStore</a>.
</li>
<li>
New tool <strong>(MultiQC / DSCensor)</strong> for visualizing peanut genomic data in our Data Store. Example: <a target="_blank" href="https://dscensor.legumeinfo.org/multiqc-demo/gene_models_main:arachis">Arachis annotation comparisons</a>.
</li>
<li>
New BLAST sequence-search interface (<a target="_blank" href="https://legumeinfo.org/sequenceserver/"><strong>SequenceServer</strong></a>). Please try it and <a href="/contact">let us know</a> your experience.
</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-14 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617902">(October 2020) Core genotype paper published, software update, minicore phenotype data.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li><strong>Genotypic Characterization of the U.S. Peanut Core Collection</strong>: This Publication is now out.<br />
Paul I. Otyama, Roshan Kulkarni, Kelly Chamberlin, Peggy Ozias-Akins, Ye Chu, Lori M. Lincoln, Gregory E. MacDonald, Noelle L. Anglin, Sudhansu Dash, David J. Bertioli, David Fernández-Baca, Michelle A. Graham, Steven B. Cannon and Ethalinda K. S. Cannon. Genotypic Characterization of the U.S. Peanut Core Collection<br />
<a target="_blank" href="https://doi.org/10.1534/g3.120.401306">G3: GENES, GENOMES, GENETICS November 1, 2020 vol. 10 no. 11 4013-4026</a>.
</li>
<li><strong>Software update</strong>: A number of software packages have been updated to their latest versions to keep PeanutBase up to date and functional.</li>
<li><strong>Initial instance of a new tool, ArachisPheno, for exploring peanut phenotype data.</strong> This includes <a href="http://dev.lis.ncgr.org:50007/study/1/" target="_blank">data from the U.S. mini core collection</a>. More data sets will be loaded as they become available.</li>
</ul></div>  </div></li>
          <li class="views-row views-row-15 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617901">(September 2020) Arachis pathway data, PeanutMine update.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li><strong>Pathway data in gene pages at PeanutMine</strong>: Now <i>Arachis hypogaea/duranensis/ipaensis</i> reactome data is available in the gene pages at PeanutMine. <a href="https://mines.legumeinfo.org/peanutmine/report.do?id=33200367">Link</a>: please scroll towards the bottom of the page to the section '<strong>Pathways</strong>' to see if the gene function is related to a metabolic pathway.  Not all genes have this feature, only genes that are listed in the reactome database have this special attribute.
 </li>
<li><strong>PeanutMine update</strong>: It has been updated to v2 with data coming from LIS Data Store.
</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-16 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617900">(August 2020) Nematode resistance QTL dataset from Ballen-Taborda C, Chu, et al. 2019.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>A new source of root-knot nematode resistance from Arachis stenosperma: The QTL dataset from the publication <a href="/pub/130"> Ballen-Taborda C, Chu, et al. 2019</a> is now available at PeanutBase. <a href="/search/qtl?citation=Ballen-Taborda,%20Chu%20et%20al.,%202019a">QTLs</a>, <a href="/featuremap/TT_RunnerIAC886_x_A.batizocoi-A.stenosperma_a">maps</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-17 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2617899">(May 2020) Ongoing work to provide access to phenotype data.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The PeanutBase team is working on new tools for interactively exploring phenotype data.</li>
<li><a href="/data/v2/Arachis/hypogaea/diversity/aradu1_araip1.gnm1.div.2B6N/">Genotype data</a> and <a href="/data/v2/Arachis/hypogaea/traits/minicore.trt.JWYM/">phenotype data</a> for the mini core are also available at the data store.<br />
</li></ul></div>  </div></li>
          <li class="views-row views-row-18 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616362">(Feb, 2020) Instructions for GCViT, genotyping project and site improvement.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Instructions for GCViT, genotyping project and site improvement.</p>
<ul>
<li>The genotype display tool <a href="/germplasm/gigwa">Gigwa</a> and comparison tool <a href="/germplasm/gcvit">GCViT</a> pages have been given a native PeanutBase look.</li>
<li>Instructions have been added to the <a href="/germplasm/gcvit">GCViT</a> page (Please click the 'Instructions ...  ...' bar).</li>
<li>There is a now a new <a href="/germplasm/nifa_pccgp">information page for the genotyping projects</a>.</li>
<li>Links have been corrected in the germplasm page.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-19 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616357">(Jan 2020) African Lines genotyping data and new genotype comparison tool, GCViT for PeanutBase</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
<strong>African Breeding Lines data in PeanutBase gigwa instance: </strong> The PIL African Breeding Lines genotyping data are now in the <a href="/germplasm/gigwa/" target="_blank">PeanutBase GIGWA instance</a>.  The data files can be downloaded from our <a href="/data/public/Arachis_hypogaea/arahy.gnm1.div.LZ50/" target="_blank">DataStore</a> too.
</li>
<li>
<strong>(New) Genotype Comparison Visualization Tool, <a href="/germplasm/gcvit" target="_blank">GCViT</a>: </strong> It is a tool for whole genome visualization of resequencing or SNP array data. GCViT allows a user to compare two or more accessions and visually identify regions of similarity and difference across the genome.<br />
It is capable of multiple chromosome visualisation for the comparison of multiple accessions from a single dataset.  It extracts information from VCF file and does a parallel comparison between two or more lines within that VCF file plotting the location of these differences on the chromosome for visualization.  One can, for example, plot the differences between progeny and its parents to identify which regions are inherited from which parents.  (More information and a <a href="https://youtu.be/B2gPVUipWo0" target="_blank">video tutorial</a> about GCViT is at <a href="https://github.com/LegumeFederation/gcvit" target="_blank">GitHub</a> and  <a href="https://soybase.org/gcvit/" target="_blank">SoyBase</a>)
<p>     We need your feedback on the usage of this tool and will appreciate it if you could let us know about your experience via <a href="/contact">Contact us</a>.
</p></li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-20 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616356">(Dec, 2019) AAGB Conference Abstracts now available in PeanutBase, 2018 &amp; 2019.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The AAGB conference abstracts for <strong>October 21-25, 2019  (Jinan, CHINA)</strong> and <strong>November 12-16, 2018 (Saly, SENEGAL)</strong> along with their programs are now available at the <a href="/AAGB">AAGB page</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-21 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616355">(Oct, 2019) Map and QTL dataset from Kolekar,Sujay, et al. 2016a </a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The <a href="/featuremap/TT_TAG24_x_GPBD4_b">genetic map</a> and <a href="/search/qtl?citation=Kolekar, Sujay et al., 2016a">QTL data</a> from <a href="/pub/129">Kolekar,Sujay, et al. 2016a</a> has been loaded.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-22 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616251">(Aug, 2019) Improvements to the A. hypogaea Tifrunner assembly released.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Corrections to the A. hypogaea var. Tifrunner genome assembly were made by the <a href="https://www.dnazoo.org">DNA Zoo</a>, on the basis of Hi-C data produced by the <a href="https://www.aidenlab.org/">Aiden lab</a>. A summary of the changes is <a href="/peanut_genome_v1_v2">here</a>, the genome data can be downloaded <a href="/data/public/Arachis_hypogaea/Tifrunner.gnm2.J5K5">here</a>, and the gene model annotations are available <a href="/data/public/Arachis_hypogaea/Tifrunner.gnm2.ann1.4K0L">here</a>. See also this <a href="https://www.dnazoo.org/post/tetraploid-peanut-genome-assembly-published-in-nature-genetics">DNA Zoo announcement</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-23 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616233">(June, 2019) US mini core genotype data available</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    <a href="/gigwa">Genotype data</a> for the US peanut mini core (Otyama, et al., in press) is now available at PeanutBase via Gigwa (<a href="http://southgreen.fr/content/gigwa">http://southgreen.fr/content/gigwa</a>), a web-based tool for exploring genotype database. See instructions for <a href="https://southgreen.fr/sites/southgreen.fr/files/uploads/gigwa_releases/2-gigwaV2-searching-commented.mp4">filtering data</a> and <a href="https://southgreen.fr/sites/southgreen.fr/files/uploads/gigwa_releases/3-gigwaV2-outputs-commented.mp4">managing outputs</a>.
  </li>
<li>
    More information added regarding the <a href="/organism/Arachis/hypogaea?pane=genome">Tifrunner genome assembly</a>
  </li>
<li>
    Some small bugs in the website and data have been fixed, particularly regarding germplasm records.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-24 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2616193">(April, 2019) New high-density genetic map from Hu, et al., 2018, CViTjs 1.0</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
  The <a href="/featuremap/TT_Huayu28_x_P76_a">high-density genetic map</a> and <a href="/search/qtl">QTL data</a> from <a href="/pub/128">Hu, et al. 2018</a> has been loaded.
  </li>
<li>
    The <a href="https://github.com/LegumeFederation/cvitjs">1.0 release of CViTjs</a> has been integrated into the <a href="/blast">BLAST search results</a>.
  </li>
<li>Bug fixes to <a href="/search/stock">germplasm search</a>.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-25 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2613841">(Feb, 2019) Map and QTL dataset from Leal-Bertioli, Moretzsohn, et al. 2016</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The <a href="/featuremap/AA_A.duranensis_x_A.stenosperma_d">genetic map</a> and <a href="/search/qtl">QTL data</a> from <a href="/pub/83">Leal-Bertioli, Moretzsohn, et al. 2016</a> has been loaded.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-26 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/2613712">(November, 2018) New comparative map viewer; map downloads in Legume Federation Data Store; new JBrowse tracks</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>A new comparative map viewer, <a href="https://github.com/LegumeFederation/cmap-js">cmap-js</a>, has been released at PeanutBase. For example, see the <a href="/cmap-js/launchCmap-js.php?display=TT_Tifrunner_x_GT-C20_c">Agarwal et al. high-density map</a>.</li>
<li>Genetic map downloads now available via the <a href="/data/public/Arachis_hypogaea">Legume Federation Data Store.</a></li>
<li> Tracks for the 58k and 48k SNP chips added to A. duranensis, A. ipaensis, and A. hypogaea cult. Tifrunner JBrowse instances.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-27 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/1310803">(September, 2018) Data from Leal-Bertioli et al., 2015 available; new root transcriptome expression atlas; more SNPs aligned to A. hypogaea.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    <a href="/featuremap/BB_A.ipaensis_x_A.magna_b">Map</a> and <a href="/search/qtl?citation=Leal-Bertioli,%20Cavalcante%20et%20al.,%202015a">QTL</a> for Rust and several morphological traits from <a href="/pub/81">Leal-Bertioli et al. 2018, "Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection."</a> have been loaded.
  </li>
<li>
    <a href="/gene_expression/atlas_symbiosis">Root transcriptome</a> for A. hypogaea infected with Bradyrhizobium sp. SEMIA 6144 from<a href="https://www.ncbi.nlm.nih.gov/pubmed/30109978"> Karmakar K., et al. (2018) "Transcriptomic analysis with the progress of symbiosis in 'crack-entry' legume Arachis hypogaea highlights its contrast with 'infected thread' adapted legumes. Mol Plant Microbe Interact."</a>
  </li>
<li>
    SNPs for the 2nd generation 48k Affymetrix SNP chip (Axiom_Arachis2) have been aligned on the <a href="/gbrowse_peanut1.0">A. hypogaea genome browser</a>.
</li></ul>
</div>  </div></li>
          <li class="views-row views-row-28 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/1294556">(August, 2018) SNP datasets aligned to genomes, new leaf spot and TSWV QTL, new high density SNP map, NCBI A. hypogaea annotation.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    SNPs for the 2nd generation 48k Affymetrix SNP chip (Axiom_Arachis2) have been aligned on the <a href="/gbrowse_aradu1.0">A. duranensis</a> and <a href="/gbrowse_araip1.0">A. ipaensis</a> genome browsers.
  </li>
<li>
    SNPs from <a href="/pub/124">Agarwal et al. 2018</a> aligned to the A. hypogaea Tifrunner <a href="/gbrowse_peanut1.0">browser</a>.
  </li>
<li>
    <a href="/search/marker?citation=Agarwal,%20Clevenger%20et%20al.,%202018a">SNPs</a> and <a href="/featuremap/TT_Tifrunner_x_GT-C20_c">high density map</a> from <a href="/pub/124">Agarwal et al. 2018</a> loaded into database.
  </li>
<li>
    The <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_hypogaea/100/">gene model annotation from NCBI</a> is now on the A. hypogaea <a href="/gbrowse_peanut1.0">browser</a>.
</li></ul>
</div>  </div></li>
          <li class="views-row views-row-29 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/1294554">(June, 2018) Two sets of germplasm traits (Simpson and Holbrook), tissue expression atlas on A. hypogaea, new gene family BLAST target.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
      Traits assessed between 1982 and 1988 for the 1977-1986 for the accessions collected in South America by Charles Simpson, Antonio Krapovickas, Walton Gregory, José Valls, Don Banks, and colleagues loaded and displayed on germplasm record pages. See an <a href="/stock/Arachis/hypogaea/Accession/PI-468195">example</a>.
  </li>
<li>
      Above and below ground traits collected by Corley Holbrook in the late 1990s for the US Mini Core are loaded and displayed on germplasm record pages. See an <a href="/stock/Arachis/hypogaea/Accession/PI-259658">example</a>.
  </li>
<li>
     The tissue expression atlas from <a href="http://dx.doi.org/10.3389/fpls.2016.01446">Clevenger et al. 2016</a> is now available as a <a href="/gene_expression/atlas">display of differential expression between tissues</a> and showing expression levels on the <a href="/gbrowse_peanut1.0">peanut genome browser</a><a>.
  </a></li>
<li>
     The new <a href="https://legumeinfo.org/node/7138225">Legume Federation gene model families</a> are now available for <a href="/blast">BLASTing</a> against a protein database (blastx or blastp). The older Phytozome gene families are also still available as a BLAST target, but deprecated. These are named "Gene Family Consensus - Legume Federation 1.0" and "Gene Family Consensus - Phytozome 10.2 (deprecated)" respectively.
</li></ul>
</div>  </div></li>
          <li class="views-row views-row-30 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/1294552">(May, 2018) Detailed data records and gene families for cultivated peanut (A. hypogaea) gene models are now available.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Gene models for cultivated peanut (<i>A. hypogaea</i>) have been added to the new Legume Federation gene families, which are searchable at <a href="https://legumeinfo.org/search/phylotree">LegumeInfo.org</a>. More information about the gene families is <a href="https://legumeinfo.org/node/7138225">here</a></li>
<li>Detailed information about individual gene models can be found on the <a href="/search/gene">gene model pages</a>.</li>
<li>Gene models are available for <a href="/data/public/Arachis_hypogaea/">download</a> and <a href="/blast">BLASTing</a>, and can be browsed on the genome using <a href="/gbrowse_peanut1.0">GBrowse</a> and <a href="/node/319881/?data=arahy.Tifrunner.gnm1">JBrowse</a>. </li>
<li>To learn more about the <i>A. hypogaea</i> Tifrunner genome assembly and gene models, see the <a href="/peanut_genome">peanut genome information page</a>.
</li></ul>
</div>  </div></li>
          <li class="views-row views-row-31 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587328">(April, 2018) Sequence search (BLAST) for cultivated peanut genes is available.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Genes (coding sequence / CDS) can now be searched for the cultivated peanut genome, using <a href="/blast">BLAST</a>. This extends the genome BLAST search which was enabled in December.</p>
</div>  </div></li>
          <li class="views-row views-row-32 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587325">Announcement: Breeder Workshop before APRES meeting</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><a href="/workshops#2018Williamsburg">A breeders workshop</a> will be held July 9th, 2018 8:00AM-12:00PM in Williamsburg, VA, the day before the <a href="https://apresinc.com/meetings/future-meeting-dates/">APRES meeting</a>. The focus will be on Marker Assisted Selection. More information coming soon.</p>
</div>  </div></li>
          <li class="views-row views-row-33 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587313">(March, 2018) A. hypogaea gene models, PeanutBase advisory committee report available.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li> The A. hypogaea gene models are now displayed on the genome browsers, both <a href="/gbrowse_peanut1.0">GBrowse</a> and <a href="/node/319881/?data=arahy.Tifrunner.gnm1">JBrowse</a> </li>
<li> The new gene models are also available for <a href="https://peanutbase-stage.usda.iastate.edu/data/public/Arachis_hypogaea/">download</a>. </li>
<li> The 2017 report from the <a href="/community#about">PeanutBase advisory committee</a> is now <a ref="/files/misc/PeanutBase_AdvisoryBoard_FinalReport2017.docx">online</a>. </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-34 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587312">(February, 2018) Gene atlas added to peanut browser</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The tissue atlas from <a href="http://dx.doi.org/10.3389/fpls.2016.01446">Clevenger et al. 2016</a> as been added to both the cultivated peanut (Arachis hypogaea) genome browsers. The new tracks can be viewed in <a href="/gbrowse_peanut1.0">GBrowse</a> or <a href="/node/319881/?data=arahy.Tifrunner.gnm1">JBrowse</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-35 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587309">Cultivated peanut genome assembly available</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The cultivated peanut genome assembly is now available under the <a href="https://www.genome.gov/pages/research/wellcomereport0303.pdf">Ft. Lauderdale agreement</a>. No group may publish whole-genome research until the IPGI has first published a formal description of the genome assembly. The genome assembly can be <a href="/peanut_genome">downloaded</a>, <a href="/gbrowse_peanut1.0">browsed with GBrowse</a> and searched with <a href="/blast">BLAST</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-36 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587303">(December, 2017) Software updates, all downloads served from Legume Federation data store, page updates</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li> PHP has been updated to the current version (7.1)</li>
<li> Database upgraded to to current version of <a href="https://github.com/GMOD/Chado">Chado: 1.31</a>. This change should be invisible, but please let us know if you find problems.
</li><li> <a href="https://jbrowse.org/">JBrowse</a> has been updated to the current release (1.12.3)
</li><li> All downloads have been moved to the <a href="https://peanutbase.org/data/public/">Legume Federation data store</a>, with a consistent naming scheme and directory structure, and mirrored at <a href="https://de.cyverse.org/de/">CyVerse</a> (in the LegumeFederation folder inside "Community Data").</li>
<li> Page updates: updated assembly information on several pages, improved gene model pages and added links to the LIS synteny browser, the <a ref="https://legumeinfo.org/lis_context_viewer/#/search/lis/aradu.Aradu.00IE4">Context Viewer</a>, other bug fixes</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-37 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587302">(November, 2017) Tripal updated to version 1.2, repairs and improvements to gene model pages</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Tripal, which provides most of the PeanutBase functionality, has been updated to version 1.2 so that PeanutBase can take advantage of the latest Tripal improvements. The update should be "invisible": all pages should look the same. Please <a href="/contact">let us know</a> if anything stops working.</li>
<li>Links to term definitions and protein domains have been added to gene model search results and record pages.</li>
<li>Minor bug repairs to the gene model search and record pages.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-38 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587301">(October, 2017) NCBI annotations added to the A. duranensis and A. ipaensis genome browsers.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/">NCBI annotations</a> are now available in the <a href="https://peanutbase-stage.usda.iastate.edu/jbrowse/?data=Aradu1.0&amp;loc=Aradu.A04%3A117039019..117120178&amp;tracks=arduGeneNCBI&amp;highlight=">A. duranensis</a> and <a href="https://peanutbase-stage.usda.iastate.edu/jbrowse/?data=Araip1.0&amp;loc=Araip.B01%3A68030501..68650500&amp;tracks=araipGeneNCBI">A. ipaensis</a> genome browsers. These annotations are calculated by NCBI for the representative genomes of all eukaryotes. See more information about NCBI's <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_duranensis/101/">A. duranensis</a> and <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Arachis_ipaensis/101/">A. ipaensis</a> annotations.</p>
</div>  </div></li>
          <li class="views-row views-row-39 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587266">(August, 2017) Improvements to marker search and new marker data</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Now possible to search for markers by publication.</li>
<li>Complete marker data for <a href="/node/573">Liang et al.,2009</a> has been loaded.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-40 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587265">(June, 2017) PeanutBase now hosts Peanut Bioscience content</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>PeanutBase now hosts the contents of the Peanut Biosciences website in new pages for the <a href="/IPGI">IGPI/PGC meetings</a> and the <a href="/AAGB">AAGB conference</a>.</li>
<li>Added sequences for the 476 markers described in <a href="/pub/113">Nagy and Cho 2010</a></li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-41 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587262">(May, 2017) Peanut downloads in the data store, more marker data</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The <a href="/data/public">Data Store</a> is a new way to access bulk data at PeanutBase. Data sets (assemblies, annotation, repeats, synteny etc.) are now available in the <a href="/data/public">Data Store</a>, and via the mirrored CyVerse <a href="http://datacommons.cyverse.org/browse/iplant/home/shared/Legume_Federation/">Legume Federation Data Commons</a>. The Data Store is also available from a new button on the PeanutBase home page, and the data sets are still available via the older Download tab at PeanutBase.
</li><li><a href="/search/marker">Marker data improvements</a>: sequence and primers added for 7400 markers; an example <a href="/feature/Arachis/spp./genetic_marker/Lec-1">marker page</a>.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-42 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587261">(Mar 2017) Interactive tour for QTL data and new marker track</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>An <a class="link" onclick="lisTours.go('pb-qtl-search-to-gbrowse');">interactive tour</a> of the <a href="/search/qtl">QTL data</a> is now available. All interactive tours can be found on the <a href="/help#tours">help page</a>.</li>
<li>504 markers from the publication <a href="/pub/117">Shirasawa, Hirakawa et al., 2012</a> have been added as tracks to the <a href="/gbrowse_aradu1.0"><i>A. duranensis</i></a> and <a href="/gbrowse_araip1.0"><i>A. ipaensis</i></a> genome browsers.</li>
<li>We are continuing to improve <a href="/search/marker">marker data</a>, adding sequence, position, and other data for each marker.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-43 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/587259">(Feb 2017) SNPs from the Arachis SNP chip now available</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The 15,746 SNPs from the new Arachis SNP chip (<a href="http://dx.doi.org/10.1038/srep40577">Pandey et al., 2017</a>) which were determined to by polymorphic in A. hypogaea (<a href="http://dx.doi.org/10.1016/j.molp.2016.11.015">Clevenger et al., 2017</a>) have been loaded into the <a href="/gbrowse_aradu1.0">A. duranensis</a> and <a href="/gbrowse_araip1.0">A. ipaensis</a> genome browsers. Sequence and position details for <a href="/feature/Arachis/spp./genetic_marker/AX-147221298">each SNP</a> can be found in the <a href="/search/marker">marker pages</a> - search for markers starting with "AX-".</p>
<p>Genomic positions for an additional 3644 markers have been added to their marker records, for example <a href="/feature/Arachis/spp./genetic_marker/Ah1TC06H03">Ah1TC06H03</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-44 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569772">(Jan 2017) Improvements to whole genome view of BLAST results</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>CViTjs is now integrated into the <a href="/blast">BLAST utility</a> to display BLAST hits locations on an entire genome. This view is only available when BLASTing against genome targets (with blastn or tblastn).</p>
<p>CViTjs provides an interactive hit viewer with these features:</p>
<ul>
<li>Zoom in and out using the + and - buttons</li>
<li>Click the wrench button to get a tool palette for: panning, zooming to a selected region, drawing, erasing, and choosing colors</li>
<li>The reset button between the two zoom buttons will return the image to its original state
</li></ul>
</div>  </div></li>
          <li class="views-row views-row-45 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569771">(Dec 2016) Gene expression with eFP browser, whole genome visualization of BLAST hits</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Gene expression for 22 tissues (Clevenger et al., 2016) using the <a href="http://bar.utoronto.ca/efp_arachis/cgi-bin/efpWeb.cgi?dataSource=Arachis_Atlas&amp;modeInput=Absolute&amp;primaryGene=Adur114_comp0_c1_seq8">BAR eFP browser</a> is available on the <a href="/feature/Arachis/duranensis/gene/Aradu.J3GMZ?pane=Expression">gene model pages</a>.
  </li>
<li>Whole genome visualization of hits added to the BLAST results page, for searches against the <i>A. duranensis</i> or <i>A. ipaensis</i> genomes (<a href="/blast/nucleotide/nucleotide">nucleotide-nucleotide</a> search or <a href="/blast/protein/nucleotide">protein-nucleotide</a> search).</li>

</ul>
</div>  </div></li>
          <li class="views-row views-row-46 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569770">(Nov 2016) Newsletter signup; transcript mappings, bug fixes.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Sign up for the PeanutBase <a href="http://eepurl.com/b6zeIL">newsletter</a>. The October issue can be seen <a href="http://us13.campaign-archive1.com/?u=c442fb86dbc3ba6cce030d6d3&amp;id=1b3ff6bf8d&amp;e=cdb826a51a">here</a>, and the June issue <a href="http://us13.campaign-archive1.com/?u=c442fb86dbc3ba6cce030d6d3&amp;id=f938c32e05&amp;e">here</a>.</li>
<li>Transcripts from Clevenger et al., 2016 have been mapped to the <a href="/files/genomes/Arachis_hypogaea/transcriptomes/Arahy.Tifrunner.sSQ6/A.transcripts.mapped.to.genemodels.txt">A. duranensis</a> and <a>A. ipaensis</a> genomes by Josh Clevenger. Available on the <a href="/gene_expression">gene expression atlas page</a>.</li>
<li>Bug fixes in QTL, BLAST, map, and germplasm pages.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-47 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569769">(Oct 2016) Trait data attached to germplasm pages</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Trait data from GRIN has been imported into PeanutBase, permitting germplasm searching by trait.</p>
</div>  </div></li>
          <li class="views-row views-row-48 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569768">(Sept, 2016) New pedigree chart, transcriptome, genome summaries, and germplasm improvements</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    An interactive <a href="/pedigree2000">pedigree chart</a> from 'Use of Plant Introductions in Peanut Cultivar Development.' 2001. T G Isleib, C C Holbrook, D W Gorbet.
  </li>
<li>
    Arachis hypogaea tissue transcriptome for <i>A hypogaea</i> mapped to <a href="/gbrowse_aradu1.0"><i>A. duranensis</i></a> and <a href="/gbrowse_araip1.0"><i>A. ipaensis</i></a>; also, atlas expression data has been imported into PeanutMine, where heatmaps of expression can be generated for arbitrary lists of transcripts (as in the templated query example <a href="http://mines.legumeinfo.org/peanutmine/templates.do">Genes =&gt; GOTerms =&gt; mRNAs</a>).
  </li>
<li>
    Pages describing the <a href="/organism/Arachis/duranensis?pane=genome"><i>A. duranensis</i></a> and <a href="/organism/Arachis/ipaensis?pane=genome"><i>A. ipaensis</i></a> genome assemblies
  </li>
<li>
    Synonyms have been added to <a href="/chado/stock">germplasm data</a><a></a> and can be used to find stock records.
  </li>
<li>
    Instructions for finding promoters added the the <a href="/help#faq">FAQs</a> and an interactive <a class="link" onclick="lisTours.go('germplasm-map');">tour</a> for the germplasm map.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-49 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/569714">(August, 2016) New markers loaded and germplasm can be searched</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>About 15,000 <a href="/search/marker">new markers</a> have been loaded. PeanutBase now holds <a href="/chado/stock">germplasm data</a>, which can be searched and viewed in detail.</p>
</div>  </div></li>
          <li class="views-row views-row-50 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/542066">(July, 2016) New search page for peanut projects recorded at NCBI</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>A new search feature for peanut <a href="/arachis_bioproj">BioProjects at NCBI</a> is now available.</p>
</div>  </div></li>
          <li class="views-row views-row-51 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/542064">(June, 2016) New PeanutMine query tool, more intuitive trait search, improved marker display, new &quot;A&quot; genome report</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li><a href="http://mines.legumeinfo.org/peanutmine/begin.do">PeanutMine</a>: A powerful new interface for querying peanut genetic and genomic data, and operating on lists of features. Developed in collaboration with the Legume Information System and Legume Federation projects.</li>
<li>Improved <a href="/search/qtl">search for peanut traits and QTL</a>, to allow "broad" and "narrow" searches. For example, broad searches will expand a trait search for "oil" to return "seed linoleic" and other oil-related traits.</li>
<li>Improved <a href="/gbrowse_aradu1.0">display of genetic markers in the genome browsers</a> (now colored by by marker type).</li>
<li>Report of <a href="http://www.pnas.org/content/early/2016/05/25/1600899113.full">a new "A" genome (<i>Arachis duranensis</i>)</a> sequence report (for accession PI475845) adds information to complement the <a href="/data_usage_agree_aradu">reference A genome at PeanutBase (accession V14167)</a>.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-52 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/542063">(May, 2016) Upload data to geographic map, improved QTL search</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li> It is now possible to upload datasets to the <a href="/germplasm/map">Arachis interactive geographic map.</a> See an example <a href="/germplasm/map#?zoom=2&amp;maxRecs=200&amp;taxonQuery=Arachis%20hypogaea&amp;traitOverlay=Population&amp;traitScale=global&amp;country=&amp;geocodedOnly=false&amp;traitExcludeUnchar=false&amp;lat=-1.41&amp;lng=17.75&amp;mapHeight=350&amp;baseMap=ESRI%20-%20NatGeo%20(default,%20reference%20map)&amp;ne_lat=51.39920565355378&amp;ne_lng=233.43749999999997&amp;sw_lat=-53.12040528310657&amp;sw_lng=-197.9296875&amp;userDataURL=%2Fgermplasm%2Fmap%2Fstatic%2Fgrin_app%2Fexample-user-data.txt&amp;userDataName=example.csv&amp;limitToMapExtent">here</a> (US mini core population structure, data taken from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21822942">Wang et. al, 2011</a>).</li>
<li><a href="/search/qtl">QTL search</a> has been improved</li>
<li>Bug fixes</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-53 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/542062">(Apr, 2016) A utility for finding peanut/Arachis publications</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    A utility to <a href="/publications/arachis_recent_pubs">scan the recent Arachis literature</a> at Pubmed.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-54 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541660">(Mar, 2016) Arachis diploid genome paper published; interactive map of Arachis germplasm; new markers</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
The paper describing the <i>A. ipaensis</i> and <i>A. duranensis</i> genome, '<b>The genome sequences of Arachis duranensis and Arachis ipaensis, the diploid ancestors of cultivated peanut</b>', has been published in Nature Genetics. It is available <a href="http://www.nature.com/ng/journal/v48/n4/full/ng.3517.html">here</a>.
  </li>
<li>
<a href="/germplasm/map#?zoom=5&amp;maxRecs=200&amp;taxonQuery=&amp;traitOverlay=&amp;traitScale=global&amp;country=&amp;geocodedOnly=false&amp;traitExcludeUnchar=false&amp;limitToMapExtent&amp;lat=-11.70&amp;lng=-59.08&amp;mapHeight=350&amp;baseMap=ESRI%20-%20NatGeo%20(default,%20reference%20map)&amp;ne_lat=-4.083452772038619&amp;ne_lng=-37.1337890625&amp;sw_lat=-19.103648251663632&amp;sw_lng=-81.03515625">Interactive geographic map</a> of <i>Arachis</i> <a href="/germplasm">germplasm</a>, colored by species, and linking to GRIN records, with new functionality:<br />
            - Display Options for <a href="/germplasm/map">Overlay Characterization and Evaluation</a>, to allow coloring and display by trait.<br />
             - Display Options for <a href="/germplasm/map">Search Accession IDs</a>, allowing lists of accession IDs, and coloring by accessions.
  </li>
<li>
    New microsatellite markers from Moretzsohn, Leoi et al. (2005), e.g. <a href="/feature/Arachis/spp./genetic_marker/pPGSSeq17H10"> pPGSSeq17H10</a>.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-55 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541659">(Feb 2016) New gene expression atlases - for nematode and drought responses</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p><a href="/gene_expression">Expression views and atlases</a> for nematode and drought responses, courtesy Patricia Guimaraes and Ana Cristina Miranda Brasileiro et al. </p>
<p>Access these via the Browse &amp; Search menu, or directly:</p>
<ul>
<li>
      <a href="/gene_expression/atlas">Tissue expression atlas</a> (22 tissues from Arachis hypogaea)
  </li>
<li>
      <a href="/gene_expression/atlas_nematode">Nematode response experiment</a> (treatment and control in <i>A. stenosperma</i>)
  </li>
<li>
      <a href="/gene_expression/atlas_nematode">Drought response experiment</a> (treatment and control in <i>A. duranensis</i>)
  </li>
</ul>
<!--
  <li><p>    New <a href="/search/marker">marker sets</a> loaded for identifying diversity in Arachis (XX, XX et al., 2005), e.g. <a href="/feature/Arachis/spp./genetic_marker/Ap158?pane=base">Ap158</a>.
  </li>
<p>-->
</div>  </div></li>
          <li class="views-row views-row-56 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541658">(Dec 2015) New metabolic pathway viewer; improved gene pages; markers loaded for germplasm diversity; improved gene pages</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
    Metabolic pathways now available via collaboration with Reactome for <i>Arachis duranensis</i> and <i>Arachis ipaensis</i>, e.g. <a target="_blank" href="http://plantreactome.gramene.org/PathwayBrowser/#SPECIES=6848275&amp;DIAGRAM=6851900&amp;PATH=6851897,6851896,6851901">glycolipid desaturation pathways for <i>Arachis duranensis</i></a>.
  </li>
<li>
    Improved gene report pages, with more easily accessible sequences, e.g. <a href="/feature/Arachis/ipaensis/gene/Araip.5TB63">Araip.5TB63</a>.
  </li>
<li>
    New <a href="/search/marker">marker sets</a> loaded for identifying botanical varieties (He, Meng et al. 2005), e.g. <a href="/feature/Arachis/spp./genetic_marker/PM350?pane=base">PM350</a>.
  </li>
<li>
    New <a href="/search/marker">marker sets</a> loaded for identifying diversity in Arachis (Palmieri, Bechara et al., 2005), e.g. <a href="/feature/Arachis/spp./genetic_marker/Ap158?pane=base">Ap158</a>.
  </li>
</ul>
<!--
  <li><p>    Expression views and atlases for <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4619257/">nematode</a> and drought responses, courtesy Patricia Guimaraes and Ana Cristina Miranda Brasileiro et al.
  </li>
<p>-->
</div>  </div></li>
          <li class="views-row views-row-57 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541508">(Nov 2015) Genetic markers from three more map publications, and improvements to gene family and synteny views</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
Nearly 500 additional markers (see the <b><a href="/search/marker">marker search page</a></b>) have been added to the database from three publications: Cuc, Mace,  et al. 2008; Ferguson, Burow, et al. 2004; and Moretzsohn, Hopkins, et al. 2004. Some examples: <a href="/feature/Arachis/spp./genetic_marker/IPAHM718">IPAHM023</a>, <a href="/feature/Arachis/spp./genetic_marker/pPGSSeq9H08">pPGSseq9H8</a>, <a href="/feature/Arachis/spp./genetic_marker/Ah-736">Ah-736</a>
  </li>
<li>
Improved <b>gene family and synteny views</b> (at partner site LegumeInfo): clearer legends, help, gene highlighting, and synteny features. An example <a href="http://legumeinfo.org/chado_phylotree/phytozome_10_2.59053124?hilite_node=araip.Araip.47948.1">gene tree with Arachis gene highlighted</a>, and a corresponding <a href="http://legumeinfo.org/lis_context_viewer/index.html#/search/araip.Araip.47948?numNeighbors=8&amp;numMatchedFamilies=6&amp;numNonFamily=5&amp;algorithm=repeat&amp;match=5&amp;mismatch=-1&amp;gap=-1&amp;threshold=25&amp;order=chromosome">synteny view</a>.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-58 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541267">(Oct 2015) New gene expression atlas; new marker search &amp; reports; trait evaluation data; pollination &amp; crossing technique video</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
     <b><a href="/gene_expression/atlas">New gene expression atlas.</a></b> Comparison of RNA-Seq read counts from 22 <i>Arachis hypogaea</i> tissues against <i>Arachis ipaensis</i> and <i>A. duranensis</i> genomes combined - from Dr. Peggy Ozias-Akins and colleagues.
  </li>
<li>
    <b><a href="/search/marker">New marker search page</a></b> and reports, e.g. <a href="/feature/Arachis/spp./genetic_marker/Lec-1">Lec-1</a>.
  </li>
<li><b>Trait evaluation data for peanut germplasm</b>:  PeanutBase has just started attempting serve the <a href="/germplasm">germplasm evaluation data</a> (for traits or descriptors) for peanut.  The evaluation data has been downloaded from the USDA GRIN website and are served <a href="/germplasm/grin/peanut/query?grin_acc_no=289620">here</a> at PeanuBase. This utility is work in progress and would improve in future with better search capability and with more associated data. We request our user community to <a href="/contact">suggest</a> improvements to make this utility more useful.
  </li>
<li><b>Pollination/Crossing technique</b> in peanut: A <a href="https://drive.google.com/file/d/0B83uedJc4U40eXhIUHplMEEyNk0/view?usp=sharing" target="_blank">video tutorial</a> by <a href="http://www.clemson.edu/public/rec/peedee/faculty_staff/tallury.html" target="_blank">Shyam Tallury</a>, Ph D (Peanut Breeder, Advanced Plant Technology, Clemson Pee Dee Research &amp; Education Center)</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-59 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541215">(Sept 2015) New feature to search germplasm; 41 QTL added</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
     New germplasm search feature. The new feature, <a href="/germplasm">GRIN Connect</a>, permits searching the GRIN global germplasm database directly from PeanutBase. (Or the direct link to the query page <a href="/grinconnect/query">here</a>.)
  </li>
<li>
    41 QTL reported by Sarvamangala C, Gowda MVC, Varshney RK. (2011) are now available: <a href="/pub/52">publication</a>, <a href="/featuremap/TT_TG26_x_GPBD4_b">map</a>, and <a href="/search/qtl">QTL features</a>.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-60 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/541028">(Jul 2015) Peanut gene expression atlas; sequence viewer in JBrowse; new repeat/transposable sequences.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>A <strong>gene expression atlas</strong> with 22 tissues from peanut is now available in the <strong>G</strong>Browse genome browsers for <a href="/gbrowse_aradu1.0">A. duranensis</a> and <a href="/gbrowse_araip1.0">A. ipaensis</a>. This pre-publication data is courtesy of Peggy Ozia-Akins lab in collaboration with Brian Scheffler. Transcript densities of these Arachis hypogaea tissues have been mapped onto the combined A. duranensis and A. ipaensis genomes here at PeanutBase by Wei Huang.  Please contact the authors before using or citing this data.</li>
<li>The <strong>J</strong>Browse genome viewers (e.g. <a href="/genomes/jbrowse/?data=Araip1.0&amp;loc=Araip.B01">Aradu.A01</a>, <a href="/genomes/jbrowse/?data=Araip1.0&amp;loc=Araip.B01">Araip.B01</a>) have a new <strong>sequence viewer</strong> plug-in, with the ability to grab up- and down-stream sequence retrieval or reverse-complement sequence around features. Right-click or control-click on any feature (except for synteny), then choose "Sequence Viewer".  Also check the new <strong>fullscreen viewing</strong> capability of this browser ("Fullscreen" button at the top right corner of the JBrowse menu bar).</li>
<li>New repeat and transposable element sequences from the group of Drs. David and Soraya Bertioli: <a href="/download#repeats">download repeats</a></li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-61 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/325033">(Jun 2015) InterPro domain descriptions for all proteins; new JBrowse bookmark feature</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li><a href="/search/protein_domains">InterPro domains</a>: The Arachis gene models have now been annotated with protein domains searchable via InterPro domain IDs or a description word.</li>
<li>New JBrowse feature: save sets of bookmarks (of tracksets and views) under Tools:"Quick Bookmarks"</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-62 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/321556">(May 2015) Phytozome 10.2 gene models, new and improved map and marker data experience; three new QTL datasets loaded.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li><a href="http://legumeinfo.org/search/phylotree">Gene family memberships and trees</a> (at legumeinfo.org) have been rebuilt using Phytozome 10.2 family models, and now include seven legume species, including <i>Arachis ipaensis</i> and <i>Arachis duranensis</i>.</li>
<li>Improvements were made regarding the handling and presentation of genetic maps and marker​​ data.</li>
<li>QTL data from <a href="/pub/80"> Buro​w, Starr et al. (2014a)</a>, <a href="/pub/86">Mondal, Hadapad et al. (2014a)</a>, and <a href="/pub/87">Selvaraj, Narayana et al. (2009a)</a> are now available.</li>
<li>Numerous bug fixes.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-63 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/319882">(Apr 2015) View genomes via the faster JBowse; BLAST hit visualization; Interactive CMap; Enhanced Map Overview.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Now you can browse the genomes via the faster JBrowse:<a href="/genomes/jbrowse?data=Aradu1.0">Aradu</a> and <a href="/genomes/jbrowse?data=Araip1.0">Araip</a></li>
<li><a href="/blast">BLAST</a> hit visualization: Now comes with a graphic presentation to help visualize the location of hits on the target</li>
<li>Access the Arachis genetic maps via the <a href="/traits_maps/cmap">Interactive CMap</a> tool</li>
<li>The Map Overview page has a new design that simplifies access to map sets and individual linkage groups.  Also see the new RFLP-based tetraploid map published in Burow, Starr et al., 2014a.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-64 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/317413">(Mar 2015) Many new peanut images (seeds, pods, plants; various species); New &quot;gene basket&quot; tool. </a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>Peanut (<i>Arachis</i>) <a href="/germplasm">germplasm images</a> can now be viewed at PeanutBase.  Close to 2,000 images have been generously contributed by  Noelle Barkley (USDA ARS, Griffin GA), to make them available to the peanut research community.
    </li>
<li>New "shopping cart" feature (or "basket") for managing sets of genes (e.g. collect protein or CDS sequences and gene descriptions) - under <a href="/search/gene">the gene search page
    </a></li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-65 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/317017">(Feb 2015) Diploid assemblies now in GenBank; Peanut consensus genetic map available.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000817695.1/" target="_blank"><i>A. duranensis</i></a> and <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000816755.1/" target="_blank"><i>A. ipaensis</i></a> genome assemblies were submitted to GenBank and now they have been published.</li>
<li>The <a href="/filebrowser/download/463">consensus genetic map</a> for tetraploid Arachis published in Shirasawa, Bertioli et al., 2013a is now available for download.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-66 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/80860">(Jan 2015) New BLAST targets (genes); New gene pages and descriptors; peanut at PAG</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>
At the <a href="/blast">blast</a> page, we have added A. duranensis and A. ipaensis gene targets - as peptides (blastp and blastx) and nucleotides (blastn and tblasn). These link to both alignment views and to the respective genome browsers.
  </li>
<li>
New functional descriptions available at <a href="/search/gene">gene search page</a>
  </li>
<li>
<a href="https://pag.confex.com/pag/xxiii/webprogram/#srch=words|peanut%20groundnut%20Arachis|method|or|pge|1">Peanut talks at 2015 PAG</a>
  </li>
<li>
Come visit us at PAG, for the <a href="https://pag.confex.com/pag/xxiii/webprogram/Paper15999.html">LIS computer demo</a> or stop by the Plant Databases Booth (#407).
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-67 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/80849">(Dec 2014)  PeanutBase was recognized at AAGB 2014, practice tutorial, BLAST against both diploid genomes ...   ...</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><ul>
<li>The PeanutBase team was recognized for its contribution towards facilitating the assembly of the diploid peanut genomes and the practical utility at the Seventh International Conference of the Peanut Research Community (Advances in Arachis through Genomics &amp; Biotechnology), November, 2014, at Savannah GA USA.</li>
<li>Our activity at AAGB included a tutorial on using PeanutBase, poster and powerpoint presentation, survey to get inputs from the user community and contacting the community for content contribution and further development.</li>
<li>A <a href="/tutorial_aagb2014">practice tutorial</a>  for using PeanutBase is now available too.</li>
<li>Now you can <a href="/blast">BLAST</a> against both diploid genomes in one go:  just select both species from the dropdown for 'Nucleotide BLAST Databases'.</li>
<li><a href="/outreach">Outreach page</a>: Please have a look at this page, our collection of outreach materials so far.</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-68 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/80839">(Nov 2014)  Diploid gene models in genome browser, new sequence search utility, reorganization of site and  more QTL data ...  ...  ...</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>(Nov 2014)  Diploid gene models and markers in genome browser, New sequence search utility, reorganization of menu &amp; site and  more QTL data ...  ...  ...</p>
<ul>
<li>Gene models are now available - both as genome browser tracks for the sequenced wild peanut species <a href="/gbrowse_aradu1.0"><i>A. duranensis</i></a> and <a href="/gbrowse_araip1.0"><i>A. ipaenensis</i></a> - and for <a href="/genomes">download</a>.  Two gene model sets are available: predicted using the MAKER and GLEAN software, respectively. The Peanut Genome Consortium will be using the MAKER models as the primary set for analysis and description of the genes in these species.</li>
<li>Molecular markers from several mapping studies are available as genome browser tracks for <a href="/gbrowse_aradu1.0"><i>A. duranensis</i></a> and <a href="/gbrowse_araip1.0"><i>A. ipaenensis</i></a>.</li>
<li>A new <a href="/keyword_search">keyword search</a> utility has been added to facilitate searching for sequence features, such as gene IDs/names, words in annotation, etc.</li>
<li>Many changes in the navigation and in the home page layout. Please <a href="/contact">let us know</a> if you have questions or suggestions.</li>
<li>New <a href="/search/qtl">QTL data sets</a> have been added, covering a range of important traits: seed characteristics, and fungal and nematode resistance.</li>
<li>New <a href="/submit_data">data submission templates</a> for QTL studies. We would love to work with you if you have such data - published or in process - that you would like to incorporate into PeanutBase!</li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-69 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/682">(Oct 2014)  Oil quality, nematode and rust resistance traits have been added to Marker Assisted Selection list.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>We have added the following three traits to our list in the <a href="/mas">Marker Assisted Selection page</a>.  </p>
<ul>
<li><a href="/nematode_root_knot_xa">Peanut root-knot nematode</a></li>
<li><a href="/seed_oleic_to_linol_ratio_xa">Seed oleic acid to linoleic acid ratio</a></li>
<li><a href="/rust_puccinia_b03a">Rust, Puccinia</a></li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-70 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/677">(Sep 2014) The BLAST sequence search tool has been refined at PeanutBase.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>(Sep 2014) The <a href="/blast">BLAST</a> sequence search tool has been refined at PeanutBase. </p>
<p>The refined utility now includes both protein and nucleotide search against the A. duranensis and A. ipaensis genomes.  We also have modified the user interface to look and feel like the most commonly used  NCBI BLAST interface.</p>
</div>  </div></li>
          <li class="views-row views-row-71 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/676">(Sep 2014) A prototype Marker Assisted Selection (MAS) page.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>(Sep 2014) A prototype Marker Assisted Selection (MAS) page.</p>
<p>Here is a prototype MAS page for the trait <a href="/late_leaf_spot_xa">Late leaf spot (Phaeoisariopsis personata infection)</a> in peanut.  We plan to add more Marker Assisted Selection (MAS) pages for various other traits relevant to peanut breeding to include, primarily information on the nearby markers used for MAS and lines carrying the various alleles of the trait.  We welcome assistance from our user community having expertise on such traits to contribute information to these trait specific pages. Please <a href="/contact">contact</a> us for this purpose if you would like to contribute or maintain a MAS page for a specific trait.</p>
</div>  </div></li>
          <li class="views-row views-row-72 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/645">(August 2014) New sequence-search tool, linking genomic hits to the genome browsers</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>The <a href="/blat">BLAT sequence-search tool</a> has been configured to search the A. duranensis and A. ipaensis genome sequences and link genomic hits to the genome browsers.</p>
</div>  </div></li>
          <li class="views-row views-row-73 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/644">(August 2014) Updated web framework and graphical themes</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>We have moved from the <a href="http://www.tripal.info">Tripal 1</a> / Drupal 6 web framework to <a href="http://www.tripal.info/news/tripal_2.0a_released">Tripal 2.0a</a> / Drupal 7. This shouldn't affect the content of the site, but will make it easier to add a number of important features - such as an integrated BLAST sequence search module - and to keep up with ongoing Tripal development. Please <a href="/contact">let us know</a> if you like (or dislike) any aspects of the updated site. We will be adding a lot of features through the coming year, but are eager to hear about features that you would like to have.</p>
</div>  </div></li>
          <li class="views-row views-row-74 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/643">(August 2014) Redesigned QTL/trait search system and feature report pages</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Redesigned <a href="/search/qtl">QTL/trait search</a> system and <a href="/feature/Arachis/hypogaea/QTL/arahy.Late%20and%20Early%20leaf%20spot%201-4">feature report pages</a></p>
</div>  </div></li>
          <li class="views-row views-row-75 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/642">(Aug 2014) Browser synteny between Arachis diploids and other sequenced legumes</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Corresponding synteny regions with soybean, common bean, and the model species Medicago truncatela for both <a href="/gb2/gbrowse/Aradu1.0">A. duranensis</a> and <a href="/gb2/gbrowse/Araip1.0">A. ipaensis</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-76 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/623">(Jun 2014) Common bean and soybean gene models have been added to the PeanutBase Genome Browser</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>We have now enriched the PeanutBase Genome Browser with <i>Glycine max</i> (ver Glyma2.0) and  <i>Phaseolus vulgaris</i> (build 1.0) gene models.  Some of our users have found this comparison very useful along with the  <i>Arachis hypogaea </i> transcripts (Guo et al.) track.  Here is a <a href="http://peanutbase.agron.iastate.edu/gb2/gbrowse/Aradu1.0/?start=95781580;stop=95791579;ref=Aradu.A01;width=1024;version=100;flip=0;grid=1;l=assembly_supercontigs%1Earachis_hypogaea_guo%1Eglycine_max_gene_models%1Ephaseolus_vulgaris_gene_models">view of the Genome Browser showing such a comparison</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-77 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/622">(May 2014) Pseudomolecule assemblies as 80 column multifasta files are now available for download</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>May 2014<br />
We have made available the genome assemblies in a single 80 column multi-FASTA file, one for each diploid genome.  These files contain the 10 pseudomolecule sequences in a 80 column format with the scaffold sequences concatenated (after ordering and orienting) and separated by spacer sequences of 10,000 "n"s.  This format is compatible with some programs like IGV which requires the sequence lines be of equal width.  They are availble via our <a href="/genomes">Genomes</a> (assembly) page and labeled in the corresponding dipoid pages as follows: </p>
<p>Aradu_v1.0.fa.gz         01-May-2014 14:26         251M         Pseudomolecules, one file</p>
<p>Araip_v1.0.fa.gz         01-May-2014 14:36         338M         Pseudomolecules, one file</p>
</div>  </div></li>
          <li class="views-row views-row-78 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/621">(May 2014) BLAST and BLAT: Sequence search features are now available at PeanutBase</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>May 2014<br />
Sequence search feature is now available at PeanutBase via BLAST and BLAT utilities.  You can search your sequence against Arachis duranensis (assembly ver 1.0) or Arachis ipaensis (assembly ver 1.0)  in the <a href="/arachis_blast">ARACHIS BLAST</a> page (the general purpose NCBI BLAST sequence search utility appropriate for genomic sequences).<br />
OR,<br />
you can use the BLAT alignment tool via the GBrowse pages for <a href="/gb2/gbrowse/Aradu1.0">Arachis duranensis</a> and <a href="/gb2/gbrowse/Araip1.0">Arachis ipaenensis</a>.  The BLAT alignment is recommended for aligning cDNA sequences against the corresponding assembly sequence.</p>
<p>The steps for BLAT alignment are (at the GBrowse page):</p>
<p>1. Select "Find Blat Alignment" in the drop-down menu for Configure.<br />
2. Click "Configure"<br />
3. In the "Find Blat Alignment" section that opens, enter your<br />
(single) sequence in the "Input Sequence to Align" box<br />
4. Click "Find"</p>
<p>If a single hit is found, a new track called "BLAT" will be created in the genome browser with your aligned sequence. If multiple hits are found, you'll have to select one from a list of locations.</p>
<p>If you have questions or need further assistance please <a href="/contact">contact us</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-79 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/619">(Apr 2014) Genome Browser is now accessible for the two Arachis diploid progenitors.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>April 2014<br />
Genome Browser tracks are now available for the two diploid progenitor species, <a href="/gb2/gbrowse/Aradu1.0/"><i>Arachis duranensis</i></a> and <a href="/gb2/gbrowse/Araip1.0/"><i>Arachis ipaensis</i></a>.  In addition to the scaffold assembly, we have also provided a track with alignment of the <i>Arachis hypogaea</i> transcriptome data (<a href="http://www.biomedcentral.com/1471-2164/13/608">Guo et al., BMC Genomics 2012, 13:608</a>) on to each of the diploid genomes.</p>
</div>  </div></li>
          <li class="views-row views-row-80 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/618">(Apr 2014) Draft genome assemblies for the diploid progenitors of peanut are now available.</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>April 2014<br />
Peanut Genome Consortium (PGC) has completed <strong>draft genome assemblies for the diploid progenitors of peanut, Arachis ipaensis and Arachis duranensis</strong>. These are available via our <a href="/download" target="_blank"> <strong>Download</strong></a> page.</p>
</div>  </div></li>
          <li class="views-row views-row-81 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/560">All available QTLs now accessible via a Summary Table</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Nov-Dec, 2013<br />
We have compiled and  uploaded QTL data sets from 4 publications  and now they are available at the '<a href="/qtl">Summary Table of All QTLs</a>'.   More details are available by clicking the QTL and wherever available we have also provided a cmap link to our collaborative site Legume Information System.  You can also <a href="/chado/features?type_id[]=1104">search the QTLs</a> using this search page.  Please explore this data and we encourage the community to give us <a href="/contact">feedback</a> to make this resource more useful as we go along.  As we keep compiling more QTL data sets, they will be available here through this summary page.  All QTL related resources are accessible via our <a href="/qtl">Qtls page</a>.</p>
</div>  </div></li>
          <li class="views-row views-row-82 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/21">QTL terms with ontology compiled and available for download</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Nov-Dec, 2013<br />
Ontologies will be an integral part at PeanutBase and we are trying  to make them consistent with those for other legume species, with specialized terms for peanut where necessary, and with synonyms for existing terms in the literature and in use. This will help towards enabling searches that will integrate trait information for multiple legume species provided by LegumeInfo.org, and will allow integration with other plant genetic databases.  As of November, 2013, peanut QTL terms have been collected from published resources and are being aligned/integrated with the trait ontology developed at SoyBase.  We have made this evolving resource, <a href="/files/qtls/Arachis_traits.xls">QTL terms with ontology</a>, available for download as an excel spreadsheet.  We also welcome and encourage <a href="/contact">comments and suggestions</a> from the community about this resource.</p>
</div>  </div></li>
          <li class="views-row views-row-83 views-row-odd">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/20">A New Look for PeanutBase</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Oct-Nov, 2013</p>
<ul>
<li>
    We have adopted a <strong>new theme</strong> for the site based on colors from a peanut.  If you have suggestions for improvement or find errors, please don't hesitate to send us some <a href="/contact" target="_blank">feedback</a>.
  </li>
<li>
    The <em>sidebar querying features</em> have been temporarily removed while the database is being loaded.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-84 views-row-even">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/19">Transcriptome assembly projects compiled and a few more QTL reports</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>Sep, 2013</p>
<ul>
<li>
    A large number of <a href="/transcriptomes" target="_blank">peanut transcriptome sequencing projects</a> are underway in 2013.  We have compiled here these projects (as of September, 2013)  and it is downloadable as an Excel file.
  </li>
<li>
    Seven new <a href="/qtl">QTL reports</a> are being reviewed and revised before before making them ready for uploading.
  </li>
</ul>
</div>  </div></li>
          <li class="views-row views-row-85 views-row-odd views-row-last">  
  <span class="views-field views-field-title">        <span class="field-content"><a href="/node/18">QTL and map resources downloadable from one place</a></span>  </span>  
  <div class="views-field views-field-body">        <span class="field-content"></span>  </div>  
  <div class="views-field views-field-body-1">        <div class="field-content"><p>July-Aug, 2013<br />
A few <a href="/qtl">QTL</a> and <a href="/maps">map</a> resources were compiled with the goal to make them available for download from a single location.  These resources will be enhanced through the Tripal's Chado database schema for convenient data sharing with other legume projects using the same schema.</p>
</div>  </div></li>
      </ul></div>    </div>
  
  
  
  
  
  
</div>  </div>
</div>
  </div>
      
    </div></div> <!-- /.section, /#content -->

          <div id="sidebar-second" class="column sidebar"><div class="section">
          <div class="region region-sidebar-second">
    <div id="block-views-news-list-block-1" class="block block-views">

    <h2>NEWS</h2>
  
  <div class="content">
    <div class="view view-News-list view-id-News_list view-display-id-block_1 view-dom-id-9a6e650181f690c8309e7e06dfac8cfa">
        
  
  
      <div class="view-content">
      <div class="item-list">    <ul>          <li class="views-row views-row-1 views-row-odd views-row-first">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617919">(June 2023) PeanutBase Migrated to new technology platform.</a></span>  </div></li>
          <li class="views-row views-row-2 views-row-even">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617917">(April 2023) New genomes at PeanutBase DataStore(Bailey, Tifrunner, Pangenes).</a></span>  </div></li>
          <li class="views-row views-row-3 views-row-odd">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617918">(April 2023) New genomes at PeanutBase DataStore(Bailey, Tifrunner, Pangenes).</a></span>  </div></li>
          <li class="views-row views-row-4 views-row-even">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617916">(Januar 2023) Traits to associated markers using ArachisMine.</a></span>  </div></li>
          <li class="views-row views-row-5 views-row-odd">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617915">(November 2022) Search Genes And Traits By Keywords</a></span>  </div></li>
          <li class="views-row views-row-6 views-row-even views-row-last">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="/node/2617914">(October 2022) Jbrowse2, New Genome Browser With Enhanced Functionalities</a></span>  </div></li>
      </ul></div>    </div>
  
  
  
  
      <div class="view-footer">
      <p><strong><a href="/news_page_all" target="_blank">More News</a></strong></p>
    </div>
  
  
</div>  </div>
</div>
  </div>
      </div></div> <!-- /.section, /#sidebar-second -->
    
  </div></div> <!-- /#main, /#main-wrapper -->

  
  <div id="footer-wrapper"><div class="section">

    
          <div id="footer" class="clearfix">
          <div class="region region-footer">
    <div id="block-block-6" class="block block-block">

    
  <div class="content">
    <p><a style="float:left" href="/contact">Feedback</a><br />
<a style="float:right" href="https://twitter.com/peanutbaseorg" target="_blank"><img src="/sites/default/files/images/twitter_transparent.png" height="40" width="40" /></a></p>
  </div>
</div>
  </div>
      </div> <!-- /#footer -->
    
  </div></div> <!-- /.section, /#footer-wrapper -->

</div></div> <!-- /#page, /#page-wrapper -->
  </body>
</html>
